<!DOCTYPE html> 
<html>

<head>
   <meta charset="UTF-8">
   <title> Oncoscape </title>
   <script src="http://s3.amazonaws.com/oncoscape/js/jquery-2.1.0.min.js"></script>
   <script src="//code.jquery.com/ui/1.10.4/jquery-ui.js"></script>
   <link   rel="stylesheet" href="//code.jquery.com/ui/1.10.4/themes/smoothness/jquery-ui.css">

   <script src="http://s3.amazonaws.com/oncoscape/js/cytoscape-2.2.9.min.js"></script>

   <script src="http://s3.amazonaws.com/oncoscape/js/jquery.cytoscape.js-panzoom.js"></script>
   <link   href="http://s3.amazonaws.com/oncoscape/fonts/font-awesome-4.0.3/css/font-awesome.css" rel="stylesheet" type="text/css" />
   <link   href="http://s3.amazonaws.com/oncoscape/js/jquery.cytoscape.js-panzoom.css" rel="stylesheet" type="text/css" />
   <script src="http://s3.amazonaws.com/oncoscape/js/jquery.cytoscape.js-panzoom.js"></script>

   <script src="http://s3.amazonaws.com/oncoscape/js/d3.min.js"></script>

   <script src="http://ajax.aspnetcdn.com/ajax/jquery.dataTables/1.9.4/jquery.dataTables.min.js"></script>
   <link   rel="stylesheet" type="text/css"
           href="http://ajax.aspnetcdn.com/ajax/jquery.dataTables/1.9.4/css/jquery.dataTables.css">

   <link   rel="stylesheet" type="text/css"
           href="http://cdn.datatables.net/colreorder/1.1.0/css/dataTables.colReorder.min.css"></script>

   <!-- img src="//cdn.datatables.net/colreorder/1.1.0/images/insert.png" -->
   <script src="//cdn.datatables.net/colreorder/1.1.0/js/dataTables.colReorder.min.js"></script>
   <script src="//cdn.datatables.net/colvis/1.1.0/js/dataTables.colVis.min.js"></script>

   <link   rel="stylesheet" type="text/css"
           href="http://cdn.datatables.net/colvis/1.1.0/css/dataTables.colVis.css"></script>

   <script src="http://s3.amazonaws.com/oncoscape/js/jquery.multi-select.js" type="text/javascript"></script>
   <link href="http://s3.amazonaws.com/oncoscape/js/multi-select.css" media="screen" rel="stylesheet" type="text/css">

	<script type="text/javascript" src="http://mbostock.github.io/d3/talk/20111018/d3/d3.layout.js"></script>
	 

<script>

</script>
</head>


<script>
var socket;
var dispatchOptions = {};
var socketConnectedFunctions = [];
var onReadyFunctions = [];

//
// Do not modify the following line.  If Oncoscape is launched from LabKey then a new labkey javascript
// object will be created with members {.mode, .reportSession, .filteredPatients}
//
//var labkey = {labkeyOncoscape};

var filteredPatients = [];
//----------------------------------------------------------------------------------------------------
addJavascriptMessageHandler = function(cmd, func)
{
   if(cmd in dispatchOptions){
      alert("javascript message handler for '" +  cmd + " already set");
      }
   else{
      dispatchOptions[cmd] = func
      }
}
//----------------------------------------------------------------------------------------------------
function getRandomFloat (min, max)
{
    return Math.random() * (max - min) + min;
}
//----------------------------------------------------------------------------------------------------
function getRandomInt (min, max) 
{
    return Math.floor(Math.random() * (max - min + 1)) + min;
}
//----------------------------------------------------------------------------------------------------
String.prototype.beginsWith = function (string) 
{
    return(this.toLowerCase().indexOf(string.toLowerCase()) === 0);
};
//----------------------------------------------------------------------------------------------------
// from http://stackoverflow.com/questions/4068373/center-a-popup-window-on-screen
function openCenteredBrowserWindow(url, title, w, h) {
    // Fixes dual-screen position                         Most browsers      Firefox
    var dualScreenLeft = window.screenLeft != undefined ? window.screenLeft : screen.left;
    var dualScreenTop = window.screenTop != undefined ? window.screenTop : screen.top;

    width = window.innerWidth ? window.innerWidth : document.documentElement.clientWidth ? document.documentElement.clientWidth : screen.width;
    height = window.innerHeight ? window.innerHeight : document.documentElement.clientHeight ? document.documentElement.clientHeight : screen.height;

    var left = ((width / 2) - (w / 2)) + dualScreenLeft;
    var top = ((height / 2) - (h / 2)) + dualScreenTop;
    var newWindow = window.open(url, title, 'scrollbars=yes, width=' + w + ', height=' + h + ', top=' + top + ', left=' + left);

    if (window.focus) {
       newWindow.focus();
       }

} // openCenteredBrowserWindow
//----------------------------------------------------------------------------------------------------
dispatchMessage = function(msg)
{
   console.log("--- webapp, index.common, dispatchMessage: " + msg.cmd);

   if (dispatchOptions[msg.cmd])
       dispatchOptions[msg.cmd](msg)
   else
      console.log("unrecognized socket request: " + msg.cmd);
} 
//--------------------------------------------------------------------------------------------------
setupSocket = function (socket)
{
  try {
     socket.onopen = function() {
        console.log("websocket connection now open");
        for(var f=0; f < socketConnectedFunctions.length; f++){
           console.log("calling the next sockectConnectedFunction");
           socketConnectedFunctions[f]();
           } // for f
        } 
     socket.onmessage = function got_packet(msg) {
        msg = JSON.parse(msg.data)
        console.log("index.common onmessage sees " + msg.cmd);
        dispatchMessage(msg)
        } // socket.onmessage, got_packet
     socket.onclose = function(){
        //$("#status").text(msg.cmd)
        console.log("socket closing");
        } // socket.onclose
    } // try
  catch(exception) {
    $("#status").text("Error: " + exception);
    }

} // setupSocket
//----------------------------------------------------------------------------------------------------
function invokeSuccess(r)
{
    socket.rserveExecuting = false;

    if (r.errors.length > 0)
    {
        for (var i=0; i < r.errors.length; i++)
        {
        $("#status").text("Error: " + r.errors[i]);
        }
    }
    else
    if (r.outputParams.length > 0)
    {
        // note that LabKey has handled the JSON parsing already
        var msg = r.outputParams[0].value;
        console.log("index.common onmessage sees " + msg.cmd);
        dispatchMessage(msg);
    }

    //
    // call the next pending command if available
    //
    if (socket.rservePendingCommands.length > 0)
    {
        // note that M4 macro language uses 'shift' so quote it below
        executeRserveCommand(socket.rservePendingCommands.shift());
    }
}
//----------------------------------------------------------------------------------------------------
function invokeFailure(error)
{
    $("#status").text("Error: " + error.exception);
}
//----------------------------------------------------------------------------------------------------
function executeRserveCommand(data)
{
    if (socket.rserveExecuting)
    {
        //
        // put this command on our pending list and execute when the server has responded.
        //

        //console.log("in executeRserveCommand - pending:" + data);
        socket.rservePendingCommands.push(data);
    }
    else
    {
        //
        // execute immediately
        //
        socket.rserveExecuting = true;

        //console.log("in executeRserveCommand - executing:" + data);
        LABKEY.Report.execute( {
            success: invokeSuccess,
            failure: invokeFailure,
            script : 'invokeCommand',
            reportSessionId : socket.rserveSession,
            inputParams : { DATA : data }
        });
    }
}
//----------------------------------------------------------------------------------------------------
// todo: investigate why json returned by LabKey is creating array[1] in some cases.
// both JSON.parse and the LabKey util decode function do this
//----------------------------------------------------------------------------------------------------
function flattenArrays(d)
{
    for (var key in d)
    {
        if (d.hasOwnProperty(key))
        {
            var val = d[key];
            if ((val instanceof Array) && val.length == 1)
            {
                d[key] = val[0];
            }
        }
    }
}
//----------------------------------------------------------------------------------------------------
setupLabKey = function(socket)
{
    // replace socket send for LabKey
    socket.send = executeRserveCommand;
    socket.rservePendingCommands = [];
    socket.rserveExecuting = false;
    socket.rserveSession = labkey.reportSession;
    console.log("rserveSession:  " + labkey.reportSession);

    // kick off init functions
    for(var f=0; f < socketConnectedFunctions.length; f++)
    {
        console.log("calling the next sockectConnectedFunction");
        socketConnectedFunctions[f]();
    }
}

function setupFilteredPatients()
{
    if (typeof labkey != "undefined" && labkey.filteredPatients)
    {
        filteredPatients = labkey.filteredPatients.split(';');
        console.log("==== filteredPatients: " + filteredPatients.length)
    }
}
//--------------------------------------------------------------------------------------------------
$(document).ready(function()
{
    console.log("==== index.common document.ready #1");

    for (var f = 0; f < onReadyFunctions.length; f++)
    {
        console.log("calling on ready function");
        onReadyFunctions[f]();
    }

    //
    // labkeyMode has three states:
    // undefined - labkey not involved
    // labkeyWS - labkey launched Oncoscape; local R must be run with WS
    // labkeyRS - labkey launched oncoscape; Rserve is used
    //
    if (typeof labkey == "undefined")
    {
        socket = new WebSocket("ws://" + window.location.host);
        setupSocket(socket);
    }
    else
    if (labkey.mode == "WS")
    {
        socket = new WebSocket("ws://localhost:7777/");
        setupSocket(socket);
    }
    else
    if (labkey.mode == "RS")
    {
        socket = {};
        setupLabKey(socket);
    }
    else
    {
        console.log("unrecognized labkey.mode was provided: " + labkey.mode);
    }

    // todo: probably a better place to put this
    setupFilteredPatients();
})
//--------------------------------------------------------------------------------------------------
</script>



<script>
onReadyFunctions.push(function() {
    console.log("====== tabapps document ready");
    window.tabsAppRunning = true
    $("#tabs").tabs({
         // todo: distinguish between tabs, only do needed resets
       activate: function(event, ui) {
            console.log("tabs.activate");
            console.log(" ==== tab.activate, tableRef.fnAdjustColumnSizing");
            var tableRef = $("#clinicalTable").dataTable();
            if (tableRef.length > 0) {
               tableRef.fnAdjustColumnSizing();
              } // if
            console.log(" ==== tab.activate, possible cyjs resize and fit");
            if(typeof(cwMarkers) != "undefined") {
               cwMarkers.resize(); 
               cwMarkers.fit(50);
               }
            if(typeof(cwGBM) != "undefined") {
                cwGBM.resize();
                cwGBM.fit(50);
                }
            if(typeof(cwAngio) != "undefined") {
                cwAngio.resize();
                cwAngio.fit(50);
                }
            } // activate
        }); // tabs
    });  // ready

</script>


<body>
<style>
.ui-tabs .ui-tabs-nav li a {font-size:10pt !important;}
</style>

<script>
</script>

<script>
//----------------------------------------------------------------------------------------------------
var DashboardModule = (function () {

     function DashboardInitializeUI(){

      };

     
return{

   init: function(){
      onReadyFunctions.push(DashboardInitializeUI);
//      addJavascriptMessageHandler("String", displayTime);
      }
   };

}); // DateAndTimeModule
//----------------------------------------------------------------------------------------------------
Dashboard = DashboardModule();
Dashboard.init();

</script>
<script>
//----------------------------------------------------------------------------------------------------
var ClinicalTableModule = (function () {

var currentIDs;   // assign this to the full content in the tbl on startup
var tableRef;
var pcaButton;
var timeLinesButton;
var displayDiv;
var ClinicalTableTabNum=1;

   //--------------------------------------------------------------------------------------------
   function initializeUI(){
      pcaButton = $("#toPCAButton");
      pcaButton.click(function(){sendCurrentIDsToModule("PCA")});
      timeLinesButton = $("#toTimeLinesButton")
      timeLinesButton.click(function(){sendCurrentIDsToModule("timeLines")});
      displayDiv = $("#clinicalDataTableDiv");
      $(window).resize(handleWindowResize);
      handleWindowResize();
      $("#ageAtDxMinSlider").slider({
         change: function(event, ui) {$("#ageAtDxMinSliderReadout").text (ui.value)},
         min: 10,
         max: 89,
         value: 10
         });
    $("#ageAtDxMinSliderReadout").text(10);

    $("#ageAtDxMaxSlider").slider({
       change: function(event, ui) {$("#ageAtDxMaxSliderReadout").text (ui.value)},
       min: 10,
       max: 89,
       value: 89
       });
    $("#ageAtDxMaxSliderReadout").text(89);

    $("#overallSurvivalMinSlider").slider({
       change: function(event, ui) {$("#overallSurvivalMinSliderReadout").text (ui.value)},
       min: 0,
       max: 11,
       value: 0
       });
    $("#overallSurvivalMinSliderReadout").text(0);

    $("#overallSurvivalMaxSlider").slider({
       change: function(event, ui) {$("#overallSurvivalMaxSliderReadout").text (ui.value)},
       min: 0,
       max: 11,
       value: 11
       });
    $("#overallSurvivalMaxSliderReadout").text(11);
    //$("#applyClinicalTablesFiltersSlidersButton").button();
    //$("#showAllClinicalTablesRowsButton").button()
    $("#showAllClinicalTablesRowsButton").click(showAllRows)
    $("#applyClinicalTablesFiltersSlidersButton").click(readAndApplyClinicalTableFilters);
    $("#toMarkersAndTissuesButton").prop("disabled",true);
    $("#toGBMPathwaysButton").prop("disabled",true);
    $("#toSurvivalStatsButton").prop("disabled",true);
    $("#toTimeLinesButton").prop("disabled",false);
    };

   //--------------------------------------------------------------------------------------------
   function sendCurrentIDsToModule (moduleName) {
      console.log("entering sendCurrentIDsToModule");
      var rows = tableRef._('tr', {"filter":"applied"});   // cryptic, no?
      var currentIDs = []
      for(var i=0; i < rows.length; i++) 
          currentIDs.push(rows[i][0]);
      console.log(currentIDs.length + " patientIDs going to " + moduleName)
      callback = moduleName + "HandlePatientIDs";
      msg = {cmd:"sendPatientIDsToModule",
             callback: callback,
             status:"request",
             payload:{targetModule: moduleName,
                      ids: currentIDs}
             };
      msg.json = JSON.stringify(msg);
      //console.log(msg.json);
      socket.send(msg.json);
      
      msg = {cmd:"sendPatientIDsToSelectionTree",
              callback: "addSelectionToTree",
              status:"request",
              payload:{  name: "table_filter",
         			PatientIDs : currentIDs,
         			Tab: "ClinicalTable",
         			Settings: {ageAtDxMin: $("#ageAtDxMinSliderReadout").val(),
                      ageAtDxMax: $("#ageAtDxMaxSliderReadout").val(),
                      overallSurvivalMin: $("#overallSurvivalMinSliderReadout").val(),
                      overallSurvivalMax: $("#overallSurvivalMaxSliderReadout").val()}
         		}
             };
      msg.json = JSON.stringify(msg);
           console.log(msg.json);
      socket.send(msg.json);
      } // sendTissueIDsToModule

//----------------------------------------------------------------------------------------------------
   function showAllRows() {
      tableRef.fnFilter("", 0);
      }

   //--------------------------------------------------------------------------------------------
   function readAndApplyClinicalTableFilters() {
      console.log("readAndApplyClinicalTableFilters")
      var ageAtDxMin = $("#ageAtDxMinSliderReadout").val()
      var ageAtDxMax = $("#ageAtDxMaxSliderReadout").val()
      var overallSurvivalMin = $("#overallSurvivalMinSliderReadout").val()
      var overallSurvivalMax = $("#overallSurvivalMaxSliderReadout").val()

      msg = {cmd:"filterPatientHistory", 
             callback: "handleFilterPatientHistory",
             status:"request",
             payload:{ageAtDxMin: ageAtDxMin,
                      ageAtDxMax: ageAtDxMax,
                      overallSurvivalMin: overallSurvivalMin,
                      overallSurvivalMax: overallSurvivalMax}};
      msg.json = JSON.stringify(msg);
      console.log(msg.json);
      socket.send(msg.json);
      } // readAndApplyClinicalTableFilters 


   //--------------------------------------------------------------------------------------------
   function handleFilterPatientHistory(msg) {
      console.log("=== handleFilterPatientHistory");
      console.log(msg);
      if(msg.status != "success"){
         alert("handleFilterClinicalDataTable error: " + msg.payload);
         return;
         }

      var count = msg.payload.count;
      var ids = msg.payload.ids;
      console.log("--- filtered ids returned by Oncoscape: " + ids);

        // having trouble getting visible tissueIDs from DataTable for now
        // an imperfect workaround:  save them to a global variable
        // TODO:  this does not work with direct in-browser filtering of the table

       // if tissueIDs is a single string, then 'length' returns the number
       // of characters in the string, not the number of elements in the array
       // protect against this.

     if(count == 1){
        currentIDs = [ids]
        filterString = ids
        }
     else{
        currentIDs =  ids
        filterString = ids[0];
        for(var i=1; i < ids.length; i++){
           filterString += "|" + ids[i]
           }
        } // if more than one id

      console.log("---- clinicalDataTable2.handleFilterClinicalDataTable");
      console.log(filterString)
      console.log("about to call fnFilter");
      tableRef.fnFilter(filterString, 0, true);
      $("#tabs").tabs( "option", "active", ClinicalTableTabNum);
      } // handleFilterClinicalDataTable 


   //--------------------------------------------------------------------------------------------
  function handleWindowResize(){
      displayDiv.width($(window).width() * 0.95);
      displayDiv.height($(window).height() * 0.95);
     }; // handleWindowResize

   //--------------------------------------------------------------------------------------------
  function requestData (){
     console.log("cdt requests data");
     payload = ""; // demo/clinicalTable320.RData";
     msg = {cmd: "getTabularPatientHistory", callback: "handlePatientHistory", status: "request", 
            payload: payload};
     msg.json = JSON.stringify(msg);
     socket.send(msg.json);
     };

   //--------------------------------------------------------------------------------------------
  function displayTable(msg){
     console.log("entering ctbl displayTable");
     tblColumnNames = msg.payload.colnames;
     columnTitles = [];
     for(var i=0; i < tblColumnNames.length; i++){
        columnTitles.push({sTitle: tblColumnNames[i]});
        }
     
     console.log(columnTitles);

     displayDiv.html('<table cellpadding="0" cellspacing="0" margin-left="10" border="1" class="display" id="clinicalTable"></table>');
     $("#clinicalTable").dataTable({
        "sDom": "Rlfrtip",
         sDom: 'C<"clear">lfrtip',
        "aoColumns": columnTitles,
	"sScrollX": "100px",
        "iDisplayLength": 25,
         bPaginate: true,
        "scrollX": true,
        "fnInitComplete": function(){
            $(".display_results").show();
            }
         }); // dataTable

     console.log("displayTable adding data to table");
     tableRef = $("#clinicalTable").dataTable();
     tableRef.fnAddData(msg.payload.mtx);
     }; // displayTable


  return{
    requestData: requestData,
    init: function(){
      onReadyFunctions.push(initializeUI);
      addJavascriptMessageHandler("handlePatientHistory", displayTable);
      addJavascriptMessageHandler("handleFilterPatientHistory", handleFilterPatientHistory);
      addJavascriptMessageHandler("PatientHistoryHandlePatientIDs", handleFilterPatientHistory);
      socketConnectedFunctions.push(requestData);
      }
    }; // returned object

  }); // DateAndTimeModule

//----------------------------------------------------------------------------------------------------
ctbl = ClinicalTableModule();
ctbl.init();

</script>
<script>
//----------------------------------------------------------------------------------------------------
var PCAModule = (function () {

  var broadcastButton;
  var pcaDisplay;
  var pcaResults;
  var patientClassification;
  var firstTime = true;
  var pcaSelectedRegion;    // from brushing
  var d3PlotBrush;
  var pcaTabNumber = 2;
  var d3pcaDisplay;
  
  //--------------------------------------------------------------------------------------------
  function initializeUI () {
      pcaDisplay = $("#pcaDisplay");
      d3pcaDisplay = d3.select("#pcaDisplay");
      pcaHandleWindowResize();
      broadcastButton = $("#pcaBroadcastSelectionToClinicalTable");
      //broadcastButton.button();
      broadcastButton.click(pcaBroadcastSelection);
      $(window).resize(pcaHandleWindowResize);
      broadcastButton.prop("disabled",true);
      };

  //--------------------------------------------------------------------------------------------
  function runDemo (){
     payload = "";
     msg = {cmd: "calculate_mRNA_PCA", callback: "pcaPlot", status: "request", 
            payload: payload};
     socket.send(JSON.stringify(msg));
     };

  //--------------------------------------------------------------------------------------------
  function getPatientClassification (){
     payload = "";
     msg = {cmd: "getPatientClassification", callback: "handlePatientClassification", 
            status: "request", payload: payload};
     socket.send(JSON.stringify(msg));
     };

  //--------------------------------------------------------------------------------------------
  function handlePatientClassification (msg){
     console.log("=== handlePatientClassification");
     //console.log(msg)
     if(msg.status == "success"){
        patientClassification = JSON.parse(msg.payload)
        console.log("got classification, length " + patientClassification.length);
        }
     else{
       alert("error!" + msg.payload)
       }
      }; // handlePatientIDs

  //--------------------------------------------------------------------------------------------
  function pcaHandleWindowResize () {
     pcaDisplay.width($(window).width() * 0.95);
     pcaDisplay.height($(window).height() * 0.80);
     if(!firstTime) {d3PcaScatterPlot(pcaResults);}
     };

   //--------------------------------------------------------------------------------------------
  function pcaBroadcastSelection (){
      console.log("pcaBroadcastSelection: " + pcaSelectedRegion);
      x1=pcaSelectedRegion[0][0];
      y1=pcaSelectedRegion[0][1];
      x2=pcaSelectedRegion[1][0];
      y2=pcaSelectedRegion[1][1];
      ids = [];
      for(var i=0; i < pcaResults.length; i++){
         p = pcaResults[i];
         if(p.PC1 >= x1 & p.PC1 <= x2 & p.PC2 >= y1 & p.PC2 <= y2)
            ids.push(p.id[0]);
         } // for i
      if(ids.length > 0)
         sendIDsToModule(ids, "PatientHistory", "HandlePatientIDs");
      };

  //--------------------------------------------------------------------------------------------
  function sendIDsToModule (ids, moduleName, title){
       callback = moduleName + title;
       msg = {cmd:"sendIDsToModule",
              callback: callback,
              status:"request",
              payload:{targetModule: moduleName,
                       ids: ids}
             };
      socket.send(JSON.stringify(msg));
      } // sendTissueIDsToModule


  //--------------------------------------------------------------------------------------------
  function pcaPlot (msg){
      console.log("==== pcaPlot");
      //console.log(msg);
      if(msg.status == "success"){
         pcaResults = JSON.parse(msg.payload);
         d3PcaScatterPlot(pcaResults);
         if(!firstTime)  // first call comes at startup.  do not want to raise tab then.
             $("#tabs").tabs( "option", "active", pcaTabNumber);
         } // success
    else{
      console.log("pcaPlot about to call alert: " + msg)
      alert(msg.payload)
      }
     firstTime = false;
     };

  //--------------------------------------------------------------------------------------------
  function handlePatientIDs(msg){
      console.log("Module.pca: handlePatientIDs");
      //console.log(msg)
      if(msg.status == "success"){
         patientIDs = msg.payload
         //console.log("pca handlePatientIds: " + patientIDs);
         payload = patientIDs
         msg = {cmd: "calculate_mRNA_PCA", callback: "pcaPlot", status: "request", 
                payload: payload};
         socket.send(JSON.stringify(msg));
         }
    else{
      console.log("handlePatientIDs about to call alert: " + msg)
      alert(msg.payload)
      }
     }; // handlePatientIDs

  //--------------------------------------------------------------------------------------------
  function d3PlotBrushReader () {
     console.log("plotBrushReader 1037a 22jul2014");
     pcaSelectedRegion = d3PlotBrush.extent();
     //console.log("region: " + pcaSelectedRegion);
     x0 = pcaSelectedRegion[0][0];
     x1 = pcaSelectedRegion[1][0];
     width = Math.abs(x0-x1);
     //console.log("width: " + width);
     if(width > 1){
        broadcastButton.prop("disabled", false);
        console.log("width > 1, new button state, disabled?: " + broadcastButton.prop("disabled"));
        }
     else{
        broadcastButton.prop("disabled", true);
        console.log("width !> 1, new button state, disabled?: " + broadcastButton.prop("disabled"));
        }
     }; // d3PlotBrushReader

  //-------------------------------------------------------------------------------------------
  function chooseColor (d){
     id = d.id[0];
     for(var i=0; i<patientClassification.length; i++){
        if (id == patientClassification[i].rowname[0]){
          result = patientClassification[i].color[0]
          return(result)
          } // if match
        } // for i
     //console.log("chooseColor, no match for id " + id);
     return("black");
     }
  //-------------------------------------------------------------------------------------------
  function d3PcaScatterPlot(dataset) {

     broadcastButton.prop("disabled",true);
     var padding = 50;
     var width = $("#pcaDisplay").width();
     var height = $("#pcaDisplay").height();

     var xMax = d3.max(dataset, function(d) { return +d.PC1;} );
     var xMin = d3.min(dataset, function(d) { return +d.PC1;} );
     var yMax = d3.max(dataset, function(d) { return +d.PC2;} );
     var yMin = d3.min(dataset, function(d) { return +d.PC2;} );
 
       // todo:  after finding min and max, determine largest of each axis in abs value
       // todo:  then find next larger even number, use that throughout
     
     xMax = xMax * 1.1
     xMin = xMin * 1.1
     xMax = 40
     xMin = -40
     yMax = 30
     yMin = -30

     //console.log("xMax: " + xMax);   console.log("xMin: " + xMin);
     //console.log("yMax: " + yMax);   console.log("yMin: " + yMin);


     d3pcaDisplay.select("#pcaSVG").remove();  // so that append("svg") is not cumulative
 
     var xScale = d3.scale.linear()
                    .domain([xMin,xMax])
                    .range([padding, width - padding]);

     var yScale = d3.scale.linear()
                    .domain([yMin, yMax])
                    .range([height - padding, padding]); // note inversion 

     var xTranslationForYAxis = xScale(0);
     var yTranslationForXAxis = yScale(0);

     var xAxis = d3.svg.axis()
                   .scale(xScale)
                   .orient("bottom")
                   .ticks(5);

     var yAxis = d3.svg.axis()
                   .scale(yScale)
                   .orient("left")
                   .ticks(5);

     var tooltip = d3.select("body")
                     .attr("class", "tooltip")
                     .append("div")
                     .style("position", "absolute")
                     .style("z-index", "10")
                     .style("visibility", "hidden")
                     .text("a simple tooltip");

     d3PlotBrush = d3.svg.brush()
        .x(xScale)
        .y(yScale)
        .on("brushend", d3PlotBrushReader);

     var svg = d3pcaDisplay.append("svg")
                 .attr("id", "pcaSVG")
                 .attr("width", width)
                 .attr("height", height)
                 .call(d3PlotBrush);

     svg.append("g")
        .attr("class", "x axis")
        .attr("transform", "translate(0, " + yTranslationForXAxis + ")")
        .call(xAxis);

     svg.append("g")
        .attr("class", "y axis")
        .attr("transform", "translate(" + xTranslationForYAxis + ", 0)")
        .call(yAxis);

     var circle = svg.append("g").selectAll("circle")
                     .data(dataset)
                     .enter()
                     .append("circle")
                     .attr("cx", function(d,i) {return xScale(d.PC1);})
                     .attr("cy", function(d,i) {return yScale(d.PC2);})
                     .attr("r", function(d) { return 3;})
                     .style("fill", function(d) {return(chooseColor(d))})
                     //.style("fill", function(d) { return "blue"})
                     .on("mouseover", function(d,i){
                         tooltip.text(d.id);
                         return tooltip.style("visibility", "visible");
                         })
                    .on("mousemove", function(){return tooltip.style("top",
                           (d3.event.pageY-10)+"px").style("left",(d3.event.pageX+10)+"px");})
                    .on("mouseout", function(){return tooltip.style("visibility", "hidden");});
      
 
     } // d3PcaScatterPlot


  //--------------------------------------------------------------------------------------------
  return{
   init: function(){
      onReadyFunctions.push(initializeUI);
      addJavascriptMessageHandler("pcaPlot", pcaPlot);
      addJavascriptMessageHandler("PCAHandlePatientIDs", handlePatientIDs);
      addJavascriptMessageHandler("handlePatientClassification", handlePatientClassification)
      socketConnectedFunctions.push(getPatientClassification);
      socketConnectedFunctions.push(runDemo);
      }
   };

}); // PCAModule
//----------------------------------------------------------------------------------------------------
pca = PCAModule();
pca.init();

</script>
<script>
//----------------------------------------------------------------------------------------------------
var TimeLineModule = (function () {

     

     //--------------------------------------------------------------------------------------------------
     var TimelineTabNumber = 3;
     var TimeLineMargin = {top: 10, right: 15, bottom: 30, left: 20};
     var AlignBy = "--";
     var OrderBy = "--";
     var SidePlotEvent = "--";
     var TimeLineDisplay;
     var MainEvents = ["DOB","Encounter", "Diagnosis", "OR",  "MRI","Radiation", "Chemo","Progression",  "Status"]
     var MainEventColors = ["#17becf", "#d62728", "#8c564b","#ff7f0e", "#7f7f7f","#9467bd","#1f77b4","#2ca02c", "#bcbd22"]
     var MainEventTextSpacing = [0, 70, 82, 87, 75, 80, 78, 75, 80];
     var TimeLineSelectedRegion;    // from brushing
     var TimeLined3PlotBrush;
     var dialog;
     var TimeLinebroadcastButton;
     var CalculatedEvents =[{Name:"Survival", Event1: "Diagnosis", Event2: "Status", TimeScale: "Months"},
                            {Name:"AgeDx",Event1: "DOB", Event2: "Diagnosis", TimeScale: "Years"},
                            {Name:"TimeToProgression",Event1: "Diagnosis", Event2: "Progression", TimeScale: "Days"},
                            {Name:"FirstProgressionToDeath",Event1: "Progression", Event2: "Status", TimeScale: "Days"} ];
         CalculatedEvents = d3.nest()
              .key(function(d) { return d.Name; })
              .map(CalculatedEvents, d3.map);
              
     var OneDay = 1000 *60 * 60*24;
     var TimeLineColor = d3.scale.ordinal().range(MainEventColors).domain(MainEvents);
     var PatientHeight = 3;

     var Events,EventsByID, FormatDate, EventTypes, ShowEvents;
     var dispatch = d3.dispatch("load","LoadOptions", "DisplayPatients", "Update", "UpdateMenuOptions");
     var TimeLineInitialLoad=true;

       
        //--------------------------------------------------------------------------------------------
       function initializeUI(){
             console.log("========== initializing Timeline UI")
           TimeLineDisplay = $("#TimeLineDisplay");
           TimeLineHandleWindowResize();
           TimeLinebroadcastButton = $("#SendSelectionToClinicalTable");
              TimeLinebroadcastButton.click(broadcastSelection);
           $(window).resize(TimeLineHandleWindowResize);
           TimeLinebroadcastButton.prop("disabled",true);
          
           dispatch.load();
      };

     //--------------------------------------------------------------------------------------------------     
     function getDateDiff(Name, Event1, Event2, TimeScale){
          console.log("Date Difference of " +Event2+ " - " + Event1 + " in " + TimeScale)          
          var DateDiff = []; var TimeScaleValue=1
          if(TimeScale==="Days"){TimeScaleValue=OneDay;}
          else if(TimeScale ==="Months"){TimeScaleValue=OneDay*30.425}
          else if(TimeScale==="Years"){ TimeScaleValue = OneDay*365.25}
               
          EventsByID.forEach(function(ID, Patient){
               var dateDiff = 0; var date1, date2;
                if( Name=== "Survival" & (!Patient.has("Status") | Patient.get("Status")[0].Type !== "Dead")) {
                     dateDiff = null;
                } else{
                  if(Patient.has(Event1) && Patient.has(Event2) ){
                     if(Patient.get(Event1)[0].date.length >1){
                         date1 = Patient.get(Event1).sort(AscendingStartDate)[0].date[0] }
                     else{ date1 = Patient.get(Event1).sort(AscendingDate)[0].date         }
                     if(Patient.get(Event2)[0].date.length >1){
                        date2 = Patient.get(Event2).sort(AscendingStartDate)[0].date[0] }
                     else{ date2 = Patient.get(Event2).sort(AscendingDate)[0].date         }
                    
                     dateDiff = (date2 - date1 )/TimeScaleValue
//                    DateDiff.push( {ID: ID,PtNum: Patient.get(Event1)[0].PtNum, value: dateDiff, Scale: TimeScale})
                   }else{ dateDiff=null;}
               }
            DateDiff.push( {ID: ID,PtNum: Patient.get(Event1)[0].PtNum, value: dateDiff, Scale: TimeScale})
          })
          return DateDiff;
     }
     
     //--------------------------------------------------------------------------------------------------     
     function getHorizontalBarSize(Patient){
     
          console.log("Creating Horizontal BarPlot: ", Patient)
          var BarSizes = []
          Patient.forEach(function(d){
               xBar = 0; barWidth = ~~d.value
               if(d.value < 0){ xBar = ~~d.value; barWidth = Math.abs(d.value);  }
               BarSizes.push( {ID: d.ID, info: ~~d.value, Scale: d.Scale, xBar: xBar, yBar: d.PtNum,  width: barWidth})
          })     
          return BarSizes;
     }
     
      //--------------------------------------------------------------------------------------------
     function TimeLineHandleWindowResize(){
       	console.log("===== resizing Timeline window")
          TimeLineDisplay.width($(window).width() * 0.95);
           TimeLineDisplay.height($(window).height() * 0.80);
          if(!TimeLineInitialLoad) {dispatch.DisplayPatients();}
     };

   //--------------------------------------------------------------------------------------------
   function broadcastSelection(){
      console.log("broadcastSelection: " + TimeLineSelectedRegion);
      x1=TimeLineSelectedRegion[0][0];
      y1=TimeLineSelectedRegion[0][1];
      x2=TimeLineSelectedRegion[1][0];
      y2=TimeLineSelectedRegion[1][1];
      ids = [];
      
      console.log(TimeLineSelectedRegion)
      function LogTime(t){
                     if(AlignBy === "--"){ 
                               return t;
                     } else{ var Dir = (t<0 ? -1 : 1); 
                         return Dir * Math.log(Math.abs(t/OneDay)+1)/Math.log(2)
                    }
               }     
  
      for(var i=0; i < Events.length; i++){
         event = Events[i];
         if(event.PtNum >= y1/PatientHeight & event.PtNum <= y2/PatientHeight){
			// Patient within range
            
            if(event.date.length>1 ){
                 if( (LogTime(event.date[0]-event.offset) >=x1 & LogTime(event.date[0]-event.offset) <= x2) ||
	                 (LogTime(event.date[1]-event.offset) >=x1 & LogTime(event.date[1]-event.offset) <= x2) ){
	                  // date endpoints within range
	                
                      if(ids.indexOf(event.PatientID) === -1)
                        	ids.push(event.PatientID);
                }
            } else{
                 if (LogTime(event.date-event.offset) >=x1 & LogTime(event.date-event.offset) <= x2) {
	                  // date within range
                      if(ids.indexOf(event.PatientID) === -1)
                        	ids.push(event.PatientID);
                }
            }
         }
      } // for i
    
    if(ids.length > 0)
         sendIDsToModule(ids, "PatientHistory", "HandlePatientIDs");
    };

//--------------------------------------------------------------------------------------------
  function d3PlotBrushReader(){
     console.log("plotBrushReader");
     TimeLineSelectedRegion = TimeLined3PlotBrush.extent();
     //console.log("region: " + pcaSelectedRegion);
     y0 = TimeLineSelectedRegion[0][1];
     y1 = TimeLineSelectedRegion[1][1];
     selectHeight = Math.abs(y0-y1);
     if(selectHeight > 1)
        TimeLinebroadcastButton.prop("disabled", false);
     else
        TimeLinebroadcastButton.prop("disabled", true);
     }; // d3PlotBrushReader

   //--------------------------------------------------------------------------------------------
  function sendIDsToModule(ids, moduleName, title){
       callback = moduleName + title;
       msg = {cmd:"sendIDsToModule",
              callback: callback,
              status:"request",
              payload:{targetModule: moduleName,
                       ids: ids}
             };
     socket.send(JSON.stringify(msg));
      } // sendTissueIDsToModule

   //--------------------------------------------------------------------------------------------------     
   function handlePatientIDs(msg){
      console.log("Module.TimeLine: handlePatientIDs");
      console.log(msg)
    $("#tabs").tabs( "option", "active", TimelineTabNumber);
 //       TimeLineInitialLoad=true;
     if(msg.status == "success"){
         patientIDs = msg.payload
         console.log("TimeLine handlePatientIds: " + patientIDs);
         payload = patientIDs
         msg = {cmd: "getCaisisPatientHistory", callback: "DisplayPatientTimeLine", status: "request", 
                payload: payload};
         socket.send(JSON.stringify(msg));
         }
      else{
         console.log("handlePatientIDs about to call alert: " + msg)
         alert(msg.payload)
         }
      }; // handlePatientIDs


    //----------------------------------------------------------------------------------------------------
    function loadPatientDemoData(){

       console.log("==== patientTimeLines  get Events from File");
//       TimeLineInitialLoad=true;
       cmd = "getCaisisPatientHistory";
       status = "request"
       callback = "DisplayPatientTimeLine"
          filename = "" // was 'BTC_clinicaldata_6-18-14.RData', now learned from manifest file
          msg = {cmd: cmd, callback: callback, status: "request", payload: filename};
        // console.log(JSON.stringify(msg))
       socket.send(JSON.stringify(msg));
       } // loadPatientDemoData

     //--------------------------------------------------------------------------------------------------
     function DisplayPatientTimeLine(msg) {
         console.log("==== DisplayPatientTimeLine  Module.js document.ready");
          console.log(msg);

         console.log("==== DisplayPatientTimeLine  Events");     
          Events = msg.payload;
          console.log("Event count: " + Events.length);
         
          parseDate = d3.time.format ("%m/%d/%Y").parse;
          FormatDate = d3.time.format ("%x");
     
            Events.forEach(function(d) {
               d.Keep=true;
               if(d.date instanceof Array){
                    for(var i=0;i<d.date.length;i++){ 
                         if(parseDate(d.date[i])===null || !isValidDate(d.date[i])){ 
                              console.log("Flag "+ d.date[i]); d.Keep = false;
                         }else{  d.date[i] = parseDate(d.date[i]); }
               }}else{
                    if(parseDate(d.date)===null || !isValidDate(d.date)){ console.log("Flag "+ d.date); d.Keep=false;
                    } else{d.date = parseDate(d.date); }
               }
               if(d.Name === "Death") d.Name="Status"
               console.log(ShowEvents);
              d.disabled = false;
              if(ShowEvents){
               if(ShowEvents.indexOf(d.Name) === -1){  d.disabled = true;} }

             d.showPatient = true;
              d.offset = 0    });
     
          console.log("Remove Invalid Dates")
          Events = Events.filter(function(d) {if(!d.Keep){ console.log(d.date)}
               return d.Keep })

           EventsByID = d3.nest()
              .key(function(d) { return d.PatientID; })
               .key(function(d) { return d.Name; })
              .map(Events, d3.map);
 
          EventTypes = d3.map()
          Events.forEach(function(d){
               if(EventTypes.has(d.Name)){
                    if(EventTypes.get(d.Name).indexOf(d.Type) === -1){
                         EventTypes.get(d.Name).push(d.Type)            }
               } else {
                    EventTypes.set(d.Name, [d.Type])
               }
          });          
     
 
          CalculatedEvents.forEach(function(key, entry){
               var value = entry[0];
               if(!EventTypes.has(value.Event1) ||  !EventTypes.has(value.Event2) ){
                    console.log("Removing Calculated Event: " + key)
                    CalculatedEvents.remove(key)
               }
          })

          console.log("CaculatedEvents stored:", CalculatedEvents)
          console.log("Events stored:", Events)
          console.log("EventsByID stored:", EventsByID)

          if(TimeLineInitialLoad){
               ShowEvents = EventTypes.keys();
               dispatch.LoadOptions();
               TimeLineInitialLoad=false;
          }
          dispatch.DisplayPatients();
     }

     //--------------------------------------------------------------------------------------------------
     dispatch.on("LoadOptions.AllDisplays", function(){

          console.log("======== LoadOptions.AllDisplays")
          
          var width = $("#TimeLineDisplay").width();
          var height = $("#TimeLineDisplay").height();
          
          var   TimeLineSize = {width: (0.8*width - TimeLineMargin.left - TimeLineMargin.right), height: (0.95*height - TimeLineMargin.top - TimeLineMargin.bottom)},
                SideBarSize = {width: (0.2*width - TimeLineMargin.left - TimeLineMargin.right),  height: (0.95*height - TimeLineMargin.top - TimeLineMargin.bottom)},
                legendSize = {height: 0.05*height, width: TimeLineSize.width};

          var svg = d3.select("#TimeLineDisplay").append("svg")
                   .attr("width", TimeLineSize.width + SideBarSize.width + 2*TimeLineMargin.left + 2*TimeLineMargin.right )
                   .attr("height", SideBarSize.height + TimeLineMargin.top + TimeLineMargin.bottom + legendSize.height)
                       ;

          var SidePlot = svg.append("g").attr("class", "SidePlotSVG")
                  .attr("transform", "translate(" + TimeLineMargin.left + "," + TimeLineMargin.top + ")");     
             
          var TimeLine = svg.append("g").attr("class", "TimeLineSVG")
                  .attr("transform", "translate(" + (SideBarSize.width+2*TimeLineMargin.left + TimeLineMargin.right) + "," + TimeLineMargin.top + ")");
          
          
          var TextOffSet = d3.scale.ordinal()
               .range(MainEventTextSpacing)
               .domain(MainEvents);

          console.log("==== Event Types")
          var legend = svg
                     .append("g")
                   .attr("class", "legend")
                   .attr("transform", "translate(" + (SideBarSize.width+2* TimeLineMargin.left + TimeLineMargin.right) + "," + (TimeLineSize.height+ TimeLineMargin.top + TimeLineMargin.bottom) + ")")
                    .selectAll(".legend")
                     .data(TimeLineColor.domain().filter(function(d){
                          return EventTypes.keys().indexOf(d) !== -1 })  )
              .enter().append("g")
            .attr("transform", function(d, i) { 
                      return "translate(" + i*TextOffSet(d) + ",0)" })
               ;
            legend.append("rect")
           .attr("width", 10)
           .attr("height", 10)
           .style("fill", function(d) { return TimeLineColor(d)})
           .on("click", ToggleVisibleEvent);

            legend.append("text")
            .attr("y", 9)
            .attr("x", 12)
            .style("font-size", 12)
           .text(function(d) { return d; });

          //--------------------------------------------------------------------------------------------------
          dispatch.on("DisplayPatients.SidePlotDisplay",  function(){

               console.log("======== DisplayPatients.SidePlotDisplay")
          
               SidePlot.selectAll("g").remove();

                   width = $("#TimeLineDisplay").width();
                    height = $("#TimeLineDisplay").height();
          
                 TimeLineSize = {width: (0.8*width - TimeLineMargin.left - TimeLineMargin.right), height: (0.95*height - TimeLineMargin.top - TimeLineMargin.bottom)}
                SideBarSize = {width: (0.2*width - TimeLineMargin.left - TimeLineMargin.right),  height: (0.95*height - TimeLineMargin.top - TimeLineMargin.bottom)}
                legendSize = {height: 0.05*height, width: TimeLineSize.width};
          
               svg.select(".legend")
                   .attr("transform", "translate(" + (SideBarSize.width+2* TimeLineMargin.left+TimeLineMargin.right) + "," + (TimeLineSize.height+ TimeLineMargin.top + TimeLineMargin.bottom) + ")")
          
               svg.attr("width", TimeLineSize.width + SideBarSize.width + 2*TimeLineMargin.left + 2*TimeLineMargin.right )
                   .attr("height", SideBarSize.height + TimeLineMargin.top + TimeLineMargin.bottom + legendSize.height)
                       ;
               TimeLine.attr("transform", "translate(" + (SideBarSize.width+2*TimeLineMargin.left + TimeLineMargin.right) + "," + TimeLineMargin.top + ")");
  
          
               var y = d3.scale.linear().range([SideBarSize.height, 0]), 
                    yAxis = d3.svg.axis().scale(y).orient("left").ticks(0),          
                    x = d3.scale.linear().range([0, SideBarSize.width]),
                    xAxis = d3.svg.axis().scale(x).orient("bottom")
                    y.domain([d3.min(Events,function(d) { return PatientHeight*d.PtNum; })-PatientHeight,d3.max(Events,function(d) { return PatientHeight*d.PtNum; })+PatientHeight]);
//                  y.domain(d3.extent(Events, function(d) { return PatientHeight*d.PtNum; }));
//                   y.domain([d3.min(Events,function(d) { return d.PtNum; }),d3.max(Events,function(d) { return PatientHeight*d.PtNum; })]);
                             
               var PatientOrderBy = []
               var Categories = []

               
               console.log("Display Current Side Plot Event: " + SidePlotEvent)

               if(SidePlotEvent === "--"){     return;     
               } else if (CalculatedEvents.has(SidePlotEvent) ){
                    var event = CalculatedEvents.get(SidePlotEvent)[0]
                     console.log("Using event: ", event)
                     PatientOrderBy =  getHorizontalBarSize(getDateDiff(SidePlotEvent, event.Event1,event.Event2,event.TimeScale)); 
                    
               } else if(["Chemo", "Radiation", "Diagnosis", "OR", "Status", "MRI"].indexOf(SidePlotEvent) !== -1){
                    //get number of categories
                    EventsByID.forEach(function(ID, Patient){
                         if(Patient.has(SidePlotEvent) && Patient.get(SidePlotEvent)[0].showPatient){
                              Patient.get(SidePlotEvent).forEach(function(k){
                              if(Categories.indexOf(k.Type) == -1){ Categories.push(k.Type)} })
                         }
                    })
                    Categories.sort();
                    console.log(Categories)
                    console.log(Categories.length)
                    var xWidth = 1/Categories.length;
                    EventsByID.forEach(function(ID, Patient){
                         if(!Patient.has(SidePlotEvent)){
                         } else if(Patient.get(SidePlotEvent)[0].showPatient){               
                              var xPos = Categories.indexOf(Patient.get(SidePlotEvent)[0].Type)
                              PatientOrderBy.push( {ID: ID,info:Patient.get(SidePlotEvent)[0].Type, yBar: Patient.get(SidePlotEvent)[0].PtNum,
                                              xBar: xPos * xWidth , width: xWidth})
                         }
                    })
                    xAxis.ticks(Categories.length)
                         .tickValues(makeArray(Categories.length,  function(i) { return i/Categories.length; }))
                         .tickFormat(function (d) {console.log(Categories[d*Categories.length]); return Categories[d*Categories.length]     })
               } 

               console.log("PatientOrderBy")
               console.log(PatientOrderBy)

                x.domain([d3.min([d3.min(PatientOrderBy, function(d){return d.width}),d3.min(PatientOrderBy, function(d){return d.xBar})]),
                               d3.max(PatientOrderBy, function(d){ return d.xBar + d.width})]).nice();

               console.log("SidePlotDomain: ", d3.min([d3.min(PatientOrderBy, function(d){return d.width}),d3.min(PatientOrderBy, function(d){return d.xBar})]),
               d3.max(PatientOrderBy, function(d){ return d.xBar + d.width}) )

 //            d3PlotBrush = d3.svg.brush()
   //            .x(x)
     //          .y(y)
       //        .on("brushend", d3PlotBrushReader);

          //     SidePlot.call(d3PlotBrush);

//			var HorizLine = svg.append("g").attr("class", "rect")

             var tooltip = d3.select("body")
                .attr("class", "tooltip")
                .append("div")
                .style("position", "absolute")
                .style("z-index", "10")
                .style("visibility", "hidden")
                .text("a simple tooltip");
     
                 SidePlot.append("g")
                .attr("class", "x axis")
                .attr("transform", "translate(0," + (SideBarSize.height+TimeLineMargin.top) + ")")
                .call(xAxis)
                .selectAll("text")  
                    .style("text-anchor", "end")
                    .style("font-size", 12)
                      .attr("dy", ".55em")
                      .attr("dx", "-.45em")
                     .attr("transform", function(d) {
                         return "rotate(-75)" 
                        });
 
                 SidePlot.append("g")
                .attr("class", "y axis")
                .call(yAxis)
              .append("text")
                .on("mouseover", function(d){
                        tooltip.text("click to reorder by SidePlot value");
                     return tooltip.style("visibility", "visible"); })
                 .on("mousemove", function(){return tooltip.style("top",
                         (d3.event.pageY-10)+"px").style("left",(d3.event.pageX+10)+"px");})
               .on("mouseout", function(){return tooltip.style("visibility", "hidden");})
                 .on("click", function(){
                           OrderBySidePlot(); 
                           dispatch.Update();
                           dispatch.DisplayPatients();})
                .attr("transform", "rotate(-90)")
                .attr("y", 2)
                .attr("dy", "-.71em")
                .style("font-size", 12)
                .style("text-anchor", "end")
                .text(SidePlotEvent)
                ;
            
               var BarPlot_Horiz = SidePlot.append("g").selectAll("rect")
                    .data(PatientOrderBy)
                    .enter()
                         .append("rect")
                         .attr("x", function(d) { return x(d.xBar);  })
                         .attr("y", function(d) { return y(PatientHeight*d.yBar); })
                         .attr("width", function(d) { return Math.abs(x(d.width) - x(0));  })
                         .attr("height", function(d) { return PatientHeight; })
                         .attr("fill", function(d){ 
                              var ColorShade =  d3.rgb(TimeLineColor(SidePlotEvent)); //d3.rgb("grey"); //
                              if(Categories.length>0){ 
                                   return ColorShade.brighter((Categories.indexOf(d.info) % 5)/2) };
                              return ColorShade;
                              })
                         .on("mouseover", function(d,i){
                          tooltip.text(d.ID + ": " + d.info);
                     return tooltip.style("visibility", "visible");
                     })
                          .on("mousemove", function(){return tooltip.style("top",
                         (d3.event.pageY-10)+"px").style("left",(d3.event.pageX+10)+"px");})
                          .on("mouseout", function(){return tooltip.style("visibility", "hidden");})
                         ;
          })

          //--------------------------------------------------------------------------------------------------
          dispatch.on("DisplayPatients.TimeLineDisplay",  function(){

               console.log("======== DisplayPatients.TimeLineDisplay")
               TimeLine.selectAll("g").remove();


                var tooltip = d3.select("body")
                .attr("class", "tooltip")
                .append("div")
                .style("position", "absolute")
                .style("z-index", "10")
                .style("visibility", "hidden")
                .text("a simple tooltip");

               var EventOffset = d3.map({"Radiation": 1.5, "Chemo": -1.5})
               var x,TimeScale, xTitle, xAxis, 
                    y = d3.scale.linear().range([TimeLineSize.height, 0]), 
                    yAxis = d3.svg.axis().scale(y).orient("left").ticks(0)
                    ;
               function LogTime(t){
                     if(AlignBy === "--"){ 
                               return t;
                     } else{ var Dir = (t<0 ? -1 : 1); 
                         return Dir * Math.log(Math.abs(t)+1)/Math.log(2) 
                    }
               }     
          
               if(OrderBy !== "--"){ OrderEvents();}
               if(AlignBy === "--"){ 
                     x = d3.time.scale().range([0, TimeLineSize.width]); TimeScale =1;
                     Xtitle="Year";
                     xAxis = d3.svg.axis().scale(x).orient("bottom")
               } else{
                    AlignEvents();
                     x =  d3.scale.linear().range([0, TimeLineSize.width])
                     TimeScale = OneDay
                     Xtitle = "Days"
                     xAxis = d3.svg.axis().scale(x).orient("bottom")
                         .ticks(10)
                         .tickFormat(function (d) { 
                              var Dir = (d<0 ? -1 : 1); 
                              return Math.round(Dir * (Math.pow(2, (Math.abs(d)))-1) *100)/100})
                    }

               var EventMin      = d3.min(Events.filter(function(d){ return d.showPatient && !d.disabled}), function(d){ 
                                        var date = (d.date instanceof Array ? d.date.sort(DescendingDate)[0] : d.date);  return LogTime((date - d.offset)/TimeScale);})
               var EventMax      = d3.max(Events.filter(function(d){ return d.showPatient && !d.disabled}), function(d){ 
                                        var date = (d.date instanceof Array ? d.date.sort(DescendingDate)[d.date.length-1] : d.date);  return LogTime((date - d.offset)/TimeScale);})

               console.log("Min: " + EventMin + ", Max: "+ EventMax)
               x.domain([EventMin, EventMax]);
               y.domain([d3.min(Events,function(d) { return PatientHeight*d.PtNum; })-PatientHeight,d3.max(Events,function(d) { return PatientHeight*d.PtNum; })+PatientHeight]);


             TimeLined3PlotBrush = d3.svg.brush()
               .x(x)
               .y(y)
               .on("brushend", d3PlotBrushReader);

               TimeLine.call(TimeLined3PlotBrush);


            TimeLine.append("g")
                .attr("class", "x axis")
                .attr("transform", "translate(0," + TimeLineSize.height + ")")
                .call(xAxis)
               .append("text")
               .style("font-size", 12)
                .text(Xtitle);
 
            TimeLine.append("g")
                .attr("class", "y axis")
                .call(yAxis)
              .append("text")
                     .attr("transform", "rotate(-90)")
                .attr("y", 2)
                .attr("dy", ".71em")
                .style("text-anchor", "end")
                .style("font-size", 12)
                .text("Patients");
      
               var TimeSeries = TimeLine.append("g").selectAll("path")
                    .data(Events.filter(function(d){ return (d.date instanceof Array) && !d.disabled && d.showPatient}))
               ;

               console.log("TimeSeries")
               console.log(TimeSeries)

               TimeSeries.enter()
                    .append("line")
                    .attr("class", "path")
                    .attr("x1", function(d) { return x(LogTime((d.date[0] - d.offset)/TimeScale));  })
                    .attr("y1", function(d) { return y(PatientHeight*d.PtNum + EventOffset.get(d.Name)); })
                    .attr("x2", function(d) { return x(LogTime((d.date[1] - d.offset)/TimeScale));  })
                    .attr("y2", function(d) { return y(PatientHeight*d.PtNum+ EventOffset.get(d.Name)); })
                    .attr("stroke", function(d){ 
                         var ColorShade = d3.rgb(TimeLineColor(d.Name)); 
                         if(d3.keys(d).indexOf("Type") !== -1){ 
                              return ColorShade.brighter((EventTypes.get(d.Name).indexOf(d.Type) % 5)/2) };
                         return ColorShade;
                         })
                    .attr("stroke-width", PatientHeight*0.75)
                    .attr("data-legend",function(d) { return d.Name})
                    .on("mouseover", function(d,i){
                         var Type = ""; if(d3.keys(d).indexOf("Type") !== -1){ Type = d.Type}
                     tooltip.text(d.PatientID + ": " + d.Name + " (" + FormatDate(d.date[0]) + ", "+ FormatDate(d.date[1]) + ") " + Type);
                     return tooltip.style("visibility", "visible");
                     })
                     .on("mousemove", function(){return tooltip.style("top",
                    (d3.event.pageY-10)+"px").style("left",(d3.event.pageX+10)+"px");})
                     .on("mouseout", function(){return tooltip.style("visibility", "hidden");})
                    ;

               TimeSeries.exit().remove();

               var TimePoint = TimeLine.append("g").selectAll("circle")
                    .data(Events.filter(function(d){ return !(d.date instanceof Array) && !d.disabled && d.showPatient}));

               console.log("TimePoint")
               console.log(TimePoint)
                    
               TimePoint.enter()
                    .append("circle")
                    .attr("class", "circle")
                    .style("fill", function(d){ return TimeLineColor(d.Name) })
                    .attr("cx", function(d) {return x(LogTime((d.date -d.offset)/TimeScale)); })
                    .attr("cy", function(d) { return y(PatientHeight*d.PtNum); })
                    .attr("r", function(d) { return PatientHeight;})
                    .attr("data-legend",function(d) { return d.Name})
                    .on("mouseover", function(d,i){
                         var Type = ""; if(d3.keys(d).indexOf("Type") !== -1){ Type = d.Type}
                      tooltip.text(d.PatientID + ": " + d.Name + " (" + FormatDate(d.date) +") " + Type);
                      return tooltip.style("visibility", "visible");
                     })
                     .on("mousemove", function(){return tooltip.style("top",
                    (d3.event.pageY-10)+"px").style("left",(d3.event.pageX+10)+"px");})
                     .on("mouseout", function(){return tooltip.style("visibility", "hidden");})
               ;

               TimePoint.exit().remove();                    

               })
          })

          //--------------------------------------------------------------------------------------------------
          dispatch.on("load.Menu", function(){
         
              var width = $("#TimeLineDisplay").width();
               var height = $("#TimeLineDisplay").height();
          
               var   TimeLineSize = {width: (0.8*width - TimeLineMargin.left - TimeLineMargin.right), height: (0.95*height - TimeLineMargin.top - TimeLineMargin.bottom)},
                     SideBarSize = {width: (0.2*width - TimeLineMargin.left - TimeLineMargin.right),  height: (0.95*height - TimeLineMargin.top - TimeLineMargin.bottom)},
                     legendSize = {height: 0.05*height, width: TimeLineSize.width};

               console.log("======== load.Menu")
                 var  AlignByMenu = d3.select("#AlignByDiv")
                   .attr("transform", "translate(" + (3*TimeLineMargin.left+SideBarSize.width+TimeLineMargin.right) + ",0)")
                    .append("g")
                     .append("select")
                    .on("change", function() {
                         AlignBy = this.value; 
                         AlignEvents(); 
                         dispatch.DisplayPatients()});
                 ;
               AlignByMenu.selectAll("option")
                    .data(["--"])
                    .enter()
                         .append("option")
                         .attr("value", function(d){return d})
                         .text(function(d) { return d})
                ;

                 var  OrderByMenu = d3.select("#OrderByDiv")
                   .attr("transform", "translate(" + (TimeLineMargin.left+SideBarSize.width+TimeLineMargin.left+TimeLineMargin.right) + ",0)")
                     .append("g")
                   .append("select")
                    .on("change", function() {
                         OrderBy = this.value; 
                         if(OrderBy === "+Add"){
                              console.log("== changing OrderBy with ", OrderBy)
                              OpenDialogForAddedEvents("OrderBy");
                         } else{
                              console.log("== changing OrderBy ", OrderBy)
                         OrderEvents();      
                              dispatch.DisplayPatients();
//                              dialog.dialog("destroy");
                         }
                    })
                 ;
               OrderByMenu.selectAll("option")
                    .data(["--", "+Add"])
                    .enter()
                         .append("option")
                         .attr("value", function(d){return d})
                         .text(function(d) { return d})
                ;
               var  AddSideBarMenu = d3.select("#AddSideBar")
                    .style("display", "inline-block") 
                    .style("width", (SideBarSize.width + 2*TimeLineMargin.left + TimeLineMargin.right) +"px" )
                    .append("g")
                    .append("select")
                    .on("change", function() {SidePlotEvent = this.value; 
                         if(SidePlotEvent === "+Add"){
                              console.log("== changing SideBar with ", SidePlotEvent)
                              OpenDialogForAddedEvents("SidePlot");
                         } else{ 
                              console.log("== changing SideBar", SidePlotEvent)
                                                  dispatch.DisplayPatients(); 
//                         dialog.dialog("destroy");
                         }
                                                  });
               AddSideBarMenu.selectAll("option")
                    .data(["--", "+Add"])
                    .enter()
                         .append("option")
                         .attr("value", function(d){return d})
                         .text(function(d) { return d})
                ;
                var  SaveImage = d3.select("#SaveImageDiv")
                    .on("click", function() {
                         console.log("========Save Image: ")
          
                         var html = d3.select("svg")
                                      .attr("version", 1.1)
                                  .attr("xmlns", "http://www.w3.org/2000/svg")
                                      .node().parentNode.innerHTML;
                        var imgsrc = 'data:image/svg+xml;base64,'+ btoa(html);
                         var img = '<img src="'+imgsrc+'">';
                    
                         var canvas = document.querySelector("canvas"),
                             context = canvas.getContext("2d");

                         var image = new Image;
                         image.src = imgsrc;
                         image.onload = function() {
                                context.drawImage(image, 0, 0);
          
                              var canvasdata = canvas.toDataURL("image/png");
                              var pngimg = '<img src="'+canvasdata+'">'; 

                                var a = document.createElement("a");
                                   a.download = "test.png"
                                     a.href = canvasdata;
                                a.click();
                         }
                    })
     
         //--------------------------------------------------------------------------------------------------
         dispatch.on("UpdateMenuOptions.Menu", function(){
     
            console.log("======== UpdateMenuOptions")
            console.log("CalculatedEvents keys: ", CalculatedEvents.keys())
            console.log("EventTypes keys: ",EventTypes.keys())
            console.log("OrderByMenu: ", OrderByMenu)
     
          OrderByMenu.selectAll("option").remove()     
          AlignByMenu.selectAll("option").remove()
          AddSideBarMenu.selectAll("option").remove()
              
           OrderByMenu.selectAll("option")
                    .data(d3.merge([["--", "+Add"], CalculatedEvents.keys() ,EventTypes.keys()]))
                    .enter()
                         .append("option")
                         .attr("value", function(d){return d})
                         .text(function(d) { return d})
                ;
       
                 AlignByMenu.selectAll("option")
                    .data(d3.merge([["--"],EventTypes.keys()]))
                    .enter()
                         .append("option")
                         .attr("value", function(d){return d})
                         .text(function(d) { return d})
           ;
           
           AddSideBarMenu.selectAll("option")
                    .data(d3.merge([["--", "+Add"], CalculatedEvents.keys(), ["Radiation", "Chemo", "Diagnosis", "OR", "MRI", "Status"].filter(function(d){console.log(d, EventTypes.has(d)); return EventTypes.has(d) })]))
                    .enter()
                         .append("option")
                         .attr("value", function(d){return d})
                         .text(function(d) { return d})
           ;
                     
        })
        //--------------------------------------------------------------------------------------------------
          dispatch.on("LoadOptions.Menu",  function(){
               console.log("======== LoadOptions.Menu")     
               dispatch.UpdateMenuOptions()
          })
     
          
          //--------------------------------------------------------------------------------------------------
          dispatch.on("Update.Menu", function(){
               console.log("======== Update.Menu")
                
               OrderByMenu.selectAll("option")
                         .each(function(d){console.log(d); if(d === OrderBy) return d3.select(this).attr("selected", "selected")})
               AddSideBarMenu.selectAll("option")
                         .each(function(d){console.log(d); if(d === SidePlotEvent) return d3.select(this).attr("selected", "selected")})				
               AlignByMenu.selectAll("option")
                         .each(function(d){console.log(d); if(d === AlignBy) return d3.select(this).attr("selected", "selected")})
          })
     })
     //--------------------------------------------------------------------------------------------
     function UpdateCalculatedEvent(value){
               
          }
        //--------------------------------------------------------------------------------------------
       function CreateCalculatedEvent() {
                          var Name = $( "#Name" ).val();
                          var Event1 = $( "#Event1" ).val();
                          var Event2 = $( "#Event2" ).val();
                          var TimeScale = $( "#TimeScale").val();
                         console.log("Calculated Event: ")
                         console.log(Name); console.log(Event1); console.log(Event2); console.log(TimeScale);

                          var valid = true;
      
                          valid = valid && EventTypes.has(Event1);
                          valid = valid && EventTypes.has(Event2);
                           valid = valid && ["Days", "Months", "Years"].indexOf(TimeScale) !== -1;
  
                          if ( valid ) {
                            CalculatedEvents.set(Name, [{Name: Name, Event1: Event1, Event2: Event2,  TimeScale: TimeScale}])
                            console.log("Calculated Events Added: " + Name)
                            console.log(CalculatedEvents);
                             dialog.dialog( "close" );
                             return(Name);
                          } else {
                                 console.log("Invalid Calculated Event");
                               return("--");
                          }
                   }

      
      //--------------------------------------------------------------------------------------------
       function OpenDialogForAddedEvents(MenuType) {
          console.log("======== Dialog for Adding Events")
          dialog = $( "#AddCalculatedEvent" ).dialog({
                autoOpen: false,
                title: "Calculate Event",
                height: 300,
                width: 400,
                buttons: {
                  "Create": function(){
                            var value = CreateCalculatedEvent()
                            
                            if(MenuType == "OrderBy"){
                                   OrderBy = value;
                                   console.log("OrderBy is now: " + OrderBy);
                                   OrderEvents();
                              } else if (MenuType == "SidePlot"){
                                   SidePlotEvent = value;
                                   console.log("SidePlotEvent is now: " + SidePlotEvent);
                              }
                              dialog.dialog("close")
                               dispatch.UpdateMenuOptions();
                               dispatch.Update(); 
                               dispatch.DisplayPatients();
                       },
                  Cancel: function() { dialog.dialog( "close" ); UpdateCalculatedEvent("--");}
                },
                close: function() {}
                
         });
           dialog.dialog( "open" );
          console.log("open dialog")

     }

     //--------------------------------------------------------------------------------------------------
     function AlignEvents(){
     
          console.log("========Align Event: "+ AlignBy);
          Events.forEach(function(d){ d.offset = 0; d.showPatient=true;})
          EventsByID.forEach(function(ID, Patient){
                         Patient.forEach(function(id, event){event.showPatient = true; event.offset=0;})})

          if (AlignBy === "AgeDx"){ 
          } else if(EventTypes.keys().indexOf(AlignBy) !== -1){
               EventsByID.forEach(function(ID, Patient){
                    if( Patient.has(AlignBy)){
                         var MinPatientAlignBy = d3.min(Patient.get(AlignBy), function(d) {var date = (d.date instanceof Array ? d3.min(d.date) : d.date);  return date})
                         Events.filter(function(d){return d.PatientID === ID})
                              .forEach(function(d){ d.offset = MinPatientAlignBy;})
                    }
                    else{          // hide Patients that can't be aligned
                         Events.filter(function(d){return d.PatientID === ID})
                                   .forEach(function(d){ d.showPatient = false; d.offset=0;})
                         Patient.forEach(function(id, event){event.showPatient = false; event.offset=0;})
                    }
               })
          }
     }     

     //--------------------------------------------------------------------------------------------------
     function OrderEvents(){
     
          console.log("========Order Event: "+ OrderBy);
          var PatientOrderBy = []


          if(CalculatedEvents.has(OrderBy) ){
                var event = CalculatedEvents.get(OrderBy)[0]
                console.log(event)
                PatientOrderBy = getDateDiff(OrderBy, event.Event1,event.Event2,""); 
          } else if(EventTypes.keys().indexOf(OrderBy) !== -1){
               EventsByID.forEach(function(ID, Patient){
                    if( Patient.has(OrderBy)){
                         PatientOrderBy.push(d3.min(Patient.get(OrderBy), function(d) 
                              {var date = (d.date instanceof Array ? d3.min(d.date) : d.date);   
                                   return {ID: ID, value: date} }))
                    } else{
                         PatientOrderBy.push({ID: ID, value: 0} )
                    }   })
          } 
     
          PatientOrderBy.sort(DescendingValues).forEach(function(Ordered, i){
                         Events.filter(function(d){return d.PatientID === Ordered.ID})
                                   .forEach(function(d){ d.PtNum = i})
                         if(EventsByID.has(Ordered.ID)){EventsByID.get(Ordered.ID)
                                   .forEach(function(d){ d.PtNum = i})}
               })
          
          
          
          console.log("Reordered")
          console.log(PatientOrderBy)
     }     
     //--------------------------------------------------------------------------------------------------
     function OrderBySidePlot(){
     
          console.log("========Order by SidePlot: "+ SidePlotEvent);
          var PatientOrderBy = [], Categories = [];

          OrderBy = "--";
          if(CalculatedEvents.has(SidePlotEvent) ){
                var event = CalculatedEvents.get(SidePlotEvent)[0]
                console.log(event)
                PatientOrderBy = getDateDiff(SidePlotEvent, event.Event1,event.Event2,""); OrderBy=SidePlotEvent;
          
          } else if(EventTypes.keys().indexOf(SidePlotEvent) !== -1){

               EventsByID.forEach(function(ID, Patient){
                    if(Patient.has(SidePlotEvent) && Patient.get(SidePlotEvent)[0].showPatient){
                         Patient.get(SidePlotEvent).forEach(function(k){
                              if(Categories.indexOf(k.Type) == -1){ Categories.push(k.Type)}
                         })
                         PatientOrderBy.push({ID:ID, value: Patient.get(SidePlotEvent)[0].Type})
                    }
               })
               
               Categories.sort();
               console.log(Categories)
               console.log(Categories.length)
          
               PatientOrderBy.forEach(function(d,i){
                    d.value = Categories.indexOf(d.value)
               })
               
          }
          
          PatientOrderBy.sort(DescendingValues).forEach(function(Ordered, i){
                         Events.filter(function(d){return d.PatientID === Ordered.ID})
                                   .forEach(function(d){ d.PtNum = i})
                         if(EventsByID.has(Ordered.ID)){EventsByID.get(Ordered.ID)
                                   .forEach(function(d){ d.PtNum = i})}
               })
          
          
          console.log("Reordered")
          console.log(PatientOrderBy)
     }     

     //--------------------------------------------------------------------------------------------------
     function DescendingDate(a,b) {
       if (a.date > b.date)
          return -1;
       if (a.date < b.date)
         return 1;
       return 0;
     }
    //--------------------------------------------------------------------------------------------------
     function DescendingStartDate(a,b) {
       if (a.date[0] > b.date[0])
          return -1;
       if (a.date[0] < b.date[0])
         return 1;
       return 0;
     }
     //--------------------------------------------------------------------------------------------------
     function AscendingDate(a,b) {
       if (a.date < b.date)
          return -1;
       if (a.date > b.date)
         return 1;
       return 0;
     }
    //--------------------------------------------------------------------------------------------------
     function AscendingStartDate(a,b) {
       if (a.date[0] < b.date[0])
          return -1;
       if (a.date[0] > b.date[0])
         return 1;
       return 0;
     }
     //--------------------------------------------------------------------------------------------------
     function DescendingValues(a,b) {
       if (a.value > b.value)
          return -1;
       if (a.value < b.value)
         return 1;
       return 0;
     }
     //--------------------------------------------------------------------------------------------------
     function makeArray(count, content) {
        var result = [];
        if(typeof(content) == "function") {
           for(var i=0; i<count; i++) {
              result.push(content(i));
           }
        } else {
           for(var i=0; i<count; i++) {
              result.push(content);
           }
        }
        return result;
     }
     //--------------------------------------------------------------------------------------------------
     function isValidDate(text) {
          //eg text = '2/30/2011';
     
          var comp = text.split('/');
          var m = parseInt(comp[0], 10);
          var d = parseInt(comp[1], 10);
          var y = parseInt(comp[2], 10);
          var date = new Date(y,m-1,d);
     
          if (date.getFullYear() !== y || date.getMonth() + 1 !== m || date.getDate() !== d){
               console.log("Invalid Format Month/Day/Year " + text + ": " + m + "/" + d+ "/" + y)
             return false;
          }
               
          // check month and year
          if(y < 1000 || y > 3000 || m == 0 || m > 12){
               console.log("Invalid Month/Year: " + m + "/" + y)
             return false;
          }
         var monthLength = [ 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31 ];
     
         // Adjust Days for leap years
         if(y % 400 == 0 || (y % 100 != 0 && y % 4 == 0))
             monthLength[1] = 29;
     
         // Check the range of the day
         if( d <= 0 || d > monthLength[m - 1]){
               console.log("Invalid Day/Month: " + d+ "/" + m)    
               return false;
          }
          
          return true;
     }
     //--------------------------------------------------------------------------------------------------     
     function ToggleVisibleEvent(d){
         
         var hide = false;
         var newFilters = [];
      
         ShowEvents.forEach(function (f) {
           if (d === f) {  
                hide = true;
                console.log("Hiding " + d);
                Events.forEach(function(D){ 
                     if(D.Name === d) {D.disabled=true}});
           } else { newFilters.push(f);}
         });
     
          // Hide the shape or show it
          if (hide) {  d3.select(this).style("opacity", 0.2);
         } else {
           d3.select(this).style("opacity", 1);
           newFilters.push(d);
           Events.forEach(function(D){ 
                     if(D.Name === d) {D.disabled=false}});
          }
          ShowEvents = newFilters;
          dispatch.DisplayPatients();
     }
     //--------------------------------------------------------------------------------------------------
     
     //--------------------------------------------------------------------------------------------------

  return{
   init: function(){
             console.log("========== initializing PatientModule")
          onReadyFunctions.push(initializeUI);
          
          addJavascriptMessageHandler("DisplayPatientTimeLine", DisplayPatientTimeLine);
         addJavascriptMessageHandler("timeLinesHandlePatientIDs", handlePatientIDs);
      
         socketConnectedFunctions.push(loadPatientDemoData);
   }
  };

}); // TimeLineModule
//----------------------------------------------------------------------------------------------------
PatientTimeLine = TimeLineModule();
PatientTimeLine.init();
</script>


<script>
//----------------------------------------------------------------------------------------------------
var cwGBM;

var gbmPathwaysModule = (function () {

  var cyDiv;
  var viewAbstractsButton, zoomSelectedButton;
  var searchBox;
  var mouseOverReadout;
  var edgeSelectionOn = false;

  //--------------------------------------------------------------------------------------------
  function initializeUI () {
      cyDiv = $("#cyGbmPathways");
      viewAbstractsButton = $("#cwGBMViewAbstractsButton");
      zoomSelectedButton  = $("#cwGBMZoomSelectedButton");
      searchBox = $("#gbmPathwaysSearchBox");
      mouseOverReadout = $("#gbmPathwaysMouseOverReadoutDiv")
      loadNetwork();
      $(window).resize(handleWindowResize);
      };

  //--------------------------------------------------------------------------------------------
  function loadNetwork () {

       // the pathways graph is included explicitly by widget.html, so the
       // curatedGBMpathways is already defined
    console.log("loadGBMPathwaysNetwork, node count: " + curatedGBMpathways.elements.nodes.length);
    cwGBM = $("#cyGbmPathways");
    cwGBM.cytoscape({
       elements: curatedGBMpathways.elements,
       style: curatedGBMVizmap[0].style,
       showOverlay: false,
       minZoom: 0.01,
       maxZoom: 8.0,
       layout: {
         name: "preset",
         fit: true
         },
     ready: function() {
        console.log("cwGBM ready");
        cwGBM = this;
        cwGBM.on('mouseover', 'node', function(evt){
           var node = evt.cyTarget;
           mouseOverReadout.text(node.data().label);
           })
        cwGBM.on('mouseover', 'edge', function(evt){
           var edge = evt.cyTarget;
           mouseOverReadout.text(edge.data().canonicalName);
           })
        cwGBM.on('select', 'edge', function(evt){
           var edge = evt.cyTarget;
           console.log("selected edge");
           var pmid = edge.data().pmid;
           console.log("pmid: " + pmid);
           openCenteredBrowserWindow("http://www.ncbi.nlm.nih.gov/pubmed/?term=" + pmid, "pubmed abstract", 800, 600)
           });
        $("#cwGBMMovieButton").button()
        zoomSelectedButton.button();
        zoomSelectedButton.click(zoomSelection);
        viewAbstractsButton.button();
        viewAbstractsButton.click(toggleEdgeSelection);
        searchBox.keydown(doSearch);
        //$("#cwGBMMovieButton").click(cwGBMtogglePlayMovie);
        //$("#gbmPathwaysSearchBox").keydown(readGbmPathwaysSearchBox);

        cwGBM.edges().unselectify();
        console.log("cwGBM.reset");
        cwGBM.reset();
        handleWindowResize();
        //requestNanoStringExpressionData();
        } // cy.ready
       })
    .cytoscapePanzoom({ });   // need to learn about options

    } // loadGBMPathwaysNetwork

   //----------------------------------------------------------------------------------------------------
   function handleWindowResize () {
      console.log("gbmPathways window resize: " + $(window).width() + ", " + $(window).height());
      cyDiv.width(0.95 * $(window).width());
      cyDiv.height(0.8 * $(window).height());
      cwGBM.resize();
      cwGBM.fit(50);
      } // handleWindowResize


   //----------------------------------------------------------------------------------------------------
   function zoomSelection() {
      cwGBM.fit(cwGBM.$(':selected'), 50)
      }

   //----------------------------------------------------------------------------------------------------
   function toggleEdgeSelection () {
     if(edgeSelectionOn){
        cwGBM.edges().unselectify();
        edgeSelectionOn = false;
        viewAbstractsButton.button("option", "label", "Enable Abstracts");
        }
      else{
        cwGBM.edges().selectify();
        edgeSelectionOn = true;
        viewAbstractsButton.button("option", "label", "Disable Abstracts");
        }
      } // toggleEdgeSelection


   //----------------------------------------------------------------------------------------------------
   nodeNames = function(){
     nodes = cwGBM.filter("node:visible");
     result = [];
     for(var i=0; i < nodes.length; i++){
       result.push(nodes[i].data().label)
       } // for i
     return(result)
     } // nodeNames

   //----------------------------------------------------------------------------------------------------
   function doSearch(e) {
      var keyCode = e.keyCode || e.which;
      if (keyCode == 13) {
         searchString = searchBox.val();
         console.log("searchString: " + searchString);
         names = nodeNames()
         matches = []
         for(var i=0; i < names.length; i++){
            if(names[i].beginsWith(searchString)) {
               console.log(searchString + " matched " + names[i]);
               s = "cwGBM.filter('node[name=\"" + names[i] + "\"]').select()";
               eval (s);
               } // if searchString matched beginning of node
            } // for i
         } // if 13 (return key)
      } // doSearch

   //----------------------------------------------------------------------------------------------------
   return{
     init: function(){
       onReadyFunctions.push(initializeUI);
       //addJavascriptMessageHandler("pairedDistributionsPlot", pairedDistributionsPlot);
       //socketConnectedFunctions.push(runDemo);
       }
     };

   }); // gbmPathwaysModule
//----------------------------------------------------------------------------------------------------
gbmPathway = gbmPathwaysModule()
gbmPathway.init();

</script>


<script>
//----------------------------------------------------------------------------------------------------
var SavedSelectionModule = (function (){
     
     var nodecount_i;
     var height;
     var root;
    
    var	tree, diagonal;
	var SaveSelectedDisplay;
    var InitialLoad;
    var SaveSelectedsvg;
    var CurrentSelection;
    var SelectionNames= [];
  
//--------------------------------------------------------------------------------------------
  function handleWindowResize(){
      SaveSelectedDisplay.width($(window).width() * 0.95);
      SaveSelectedDisplay.height($(window).height() * 0.95);
     }; // handleWindowResize

   
//----------------------------------------------------------------------------------------------------
	function initializeSelectionUI(){			     
	 console.log("====== Initializing Selection UI")

        nodecount_i=0;
         var margin = {top: 10, right: 15, bottom: 30, left: 20};
        InitialLoad = true;
	    SaveSelectedDisplay = $("#SavedSelectionTreeDiv");
        handleWindowResize();
        
        height = SaveSelectedDisplay.height(); //200;      
        var width = SaveSelectedDisplay.width(); //200;       
         
        tree = d3.layout.tree()
         .size([(height-margin.top-margin.bottom), (width-margin.left-margin.right)]);
     
        diagonal = d3.svg.diagonal()
         .projection(function(d) { return [d.y, d.x]; });
          
        SaveSelectedsvg = d3.select("#SavedSelectionTreeDiv").append("svg")
         .attr("width", width)
         .attr("height", height)
         .append("g")
             .attr("transform", "translate(" + margin.left + "," + margin.top + ")");

         $(window).resize(handleWindowResize);

		// if date modified needs updated
		//http://www.dynamicdrive.com/forums/archive/index.php/t-63637.html

       };
       
//----------------------------------------------------------------------------------------------------
     function testingAddSavedSelection() {
       msg = {cmd:"test",
              callback: "",
              status:"request",
              payload:{  name: "test set",
         			PatientIDs : "random set of IDs here",
         			Tab: "None",
         			Settings: "gibberish"
         		}
             };
        addSavedSelection(msg);

}
//----------------------------------------------------------------------------------------------------
     function addSavedSelection(msg) {
       
       console.log("=== Saving Selection", msg)
 
 		var i=0;
 		var nameSuffix = ""
 		var nodeName =msg.payload.name 
        while(SelectionNames.indexOf(nodeName.concat(nameSuffix)) !== -1){
		    i++;
		    nameSuffix = "_".concat(i)	
		}
       newNode = msg.payload
       newNode.name = nodeName.concat(nameSuffix)
       SelectionNames.push(newNode.name)
       
       if(CurrentSelection.children){ 
            CurrentSelection.children.push(newNode);
       } else { CurrentSelection.children = [newNode] }
       
       var currentNode = CurrentSelection
       var update_id = CurrentSelection.id
       while(currentNode.parent){ 
          currentNode.parent.children[ currentNode.parent.children.indexOf(function(d) {  d.id ===update_id})] = currentNode;
          currentNode = currentNode.parent
        }
        root = currentNode;
       updateSavedSelection(root);
       
       CurrentSelection = CurrentSelection.children.filter(function(d){ return d.id === nodecount_i })[0]
    console.log("new tree: ", root)
    console.log("CurrentSelection: ", CurrentSelection)
	}       


//----------------------------------------------------------------------------------------------------
function make_editable(d, field)
{
// code from https://gist.github.com/GerHobbelt/2653660

    console.log("make_editable", arguments);
 
    this
      .on("mouseover", function() {
        d3.select(this).style("fill", "red");
      })
      .on("mouseout", function() {
        d3.select(this).style("fill", null);
      })
      .on("click", function(d) {
        var p = this.parentNode;
        console.log(this, arguments);
 
        // inject a HTML form to edit the content here...
 
        // bug in the getBBox logic here, but don't know what I've done wrong here;
        // anyhow, the coordinates are completely off & wrong. :-((
        var xy = this.getBBox();
        var p_xy = p.getBBox();
 
        xy.x -= p_xy.x;
        xy.y -= p_xy.y;
 
        var el = d3.select(this);
        var p_el = d3.select(p);
 
        var frm = p_el.append("foreignObject");
 
        var inp = frm
            .attr("x", xy.x)
            .attr("y", xy.y)
            .attr("width", 300)
            .attr("height", 25)
            .append("xhtml:form")
                    .append("input")
                        .attr("value", function() {
                            // nasty spot to place this call, but here we are sure that the <input> tag is available
                            // and is handily pointed at by 'this':
                            this.focus();
 
                            return d[field];
                        })
                        .attr("style", "width: 294px;")
                        // make the form go away when you jump out (form looses focus) or hit ENTER:
                        .on("blur", function() {
                            console.log("blur", this, arguments);
 
                            var txt = inp.node().value;
 
                            d[field] = txt;
                            el
                                .text(function(d) { return d[field]; });
 
                            // Note to self: frm.remove() will remove the entire <g> group! Remember the D3 selection logic!
                            p_el.selectAll(function() { return this.getElementsByTagName("foreignObject"); }).remove();
                        })
                        .on("keydown", function() {
                            console.log("keypress", this, arguments);
 
                            // IE fix
                            if (!d3.event)
                                d3.event = window.event;
 
                            var e = d3.event;
                            if (e.keyCode == 13)
                            {
                                if (typeof(e.cancelBubble) !== 'undefined') // IE
                                  e.cancelBubble = true;
                                if (e.stopPropagation)
                                  e.stopPropagation();
                                e.preventDefault();
 
                                var txt = inp.node().value;
 
                                d[field] = txt;
                                el
                                    .text(function(d) { return d[field]; });
 
                                // odd. Should work in Safari, but the debugger crashes on this instead.
                                // Anyway, it SHOULD be here and it doesn't hurt otherwise.
                                p_el.selectAll(function() { return this.getElementsByTagName("foreignObject"); }).remove();
                            }
                        });
      });
}
//----------------------------------------------------------------------------------------------------
     function updateSavedSelection(source) {
       var duration = d3.event && d3.event.altKey ? 5000 : 500;
     
     var tooltip = d3.select("body")
                .attr("class", "tooltip")
                .append("div")
                .style("position", "absolute")
                .style("z-index", "20")
                .style("visibility", "hidden")
                ;
     
       // Compute the new tree layout.
       var nodes = tree.nodes(root).reverse();
       console.log("Nodes", nodes);
       
       // Normalize for fixed-depth.
       nodes.forEach(function(d) { d.y = d.depth * 180; });
     
       // Update the nodes
       var node =  SaveSelectedsvg.selectAll("g.node")
           .data(nodes, function(d) { return d.id || (d.id = ++nodecount_i); });
     
       // Enter any new nodes at the parent's previous position.
       var nodeEnter = node.enter().append("g")
           .attr("class", "node")
           .attr("transform", function(d) { return "translate(" + source.y0 + "," + source.x0 + ")"; })
            .on("click", function(d) { toggle(d); updateSavedSelection(d); });
     
       nodeEnter.append("circle")
           .attr("r", 1e-6)
           .style("fill", function(d) { return d._children ? "lightsteelblue" : "grey"; })
           .on("mouseover", function(d){
                  var settings = getSettingsString(d);
                    tooltip.html(d.PatientIDs.length + " Patients from " + d.Tab + "<br></br>Settings: " + settings)
                     return tooltip.style("visibility", "visible"); })
           .on("mousemove", function(){return tooltip.style("top",
                         (d3.event.pageY-10)+"px").style("left",(d3.event.pageX+10)+"px");})
           .on("mouseout", function(){return tooltip.style("visibility", "hidden");})
           ;
     
       nodeEnter.append("text")
           .attr("class", "NodeName")
           .attr("x", function(d) { return d.children || d._children ? -10 : 10; })
           .attr("dy", "1em")
           .attr("text-anchor", function(d) { return d.children || d._children ? "end" : "start"; })
           .text(function(d) { return d.name; })
           .style("fill-opacity", 1e-6)
           .call(make_editable, "NodeName");
     
       // Transition nodes to their new position.
       var nodeUpdate = node.transition()
           .duration(duration)
           .attr("transform", function(d) { return "translate(" + d.y + "," + d.x + ")"; });
     
       nodeUpdate.select("circle")
           .attr("r", 4.5)
           .style("fill", function(d) { return d._children ? "lightsteelblue" : "grey"; });
     
       nodeUpdate.select("text")
           .style("fill-opacity", 1);
     
       // Transition exiting nodes to the parent's new position.
       var nodeExit = node.exit().transition()
           .duration(duration)
           .attr("transform", function(d) { return "translate(" + source.y + "," + source.x + ")"; })
           .remove();
     
       nodeExit.select("circle")
           .attr("r", 1e-6);
     
       nodeExit.select("text")
           .style("fill-opacity", 1e-6);
     
       // Update the links
       var link = SaveSelectedsvg.selectAll("path.link")
           .data(tree.links(nodes), function(d) { return d.target.id; });
     
       // Enter any new links at the parent's previous position.
       link.enter().insert("path", "g")
           .attr("class", "link")
           .attr("d", function(d) {
             var o = {x: source.x0, y: source.y0};
             return diagonal({source: o, target: o});
           })
         .transition()
           .duration(duration)
           .attr("d", diagonal);
     
       // Transition links to their new position.
       link.transition()
           .duration(duration)
           .attr("d", diagonal);
     
       // Transition exiting nodes to the parent's new position.
       link.exit().transition()
           .duration(duration)
           .attr("d", function(d) {
             var o = {x: source.x, y: source.y};
             return diagonal({source: o, target: o});
           })
           .remove();
     
       // Stash the old positions for transition.
       nodes.forEach(function(d) {
         d.x0 = d.x;
         d.y0 = d.y;
       });
     }
     
//----------------------------------------------------------------------------------------------------
     // Toggle children.
     function toggle(d) {
       if (d.children) {
         d._children = d.children;
         d.children = null;
       } else {
         d.children = d._children;
         d._children = null;
       }
     }
//----------------------------------------------------------------------------------------------------
    function getSettingsString(d){
		console.log("change setting string: ",  d)
		var SettingsString = d.Settings;
		if(d.Tab === "ClinicalTable"){
		   if(d.Settings.ageAtDxMin){
		       SettingsString = "AgeDx [" + d.Settings.ageAtDxMin + ", " + d.Settings.ageAtDxMax + "] <br>"
		                  + "Survival [" + d.Settings.overallSurvivalMin + ", " + d.Settings.overallSurvivalMax + "]";
		    }
		}
		return SettingsString;
		
}
//----------------------------------------------------------------------------------------------------
    function loadPatientData(){

       console.log("==== SavedSelection  get all PatientIDs from ClinicalTable");
       cmd = "getCaisisPatientHistory"; //sendCurrentIDsToModule
       status = "request"
       callback = "SetupSavedSelectionTree"
          filename = "" // was 'BTC_clinicaldata_6-18-14.RData', now learned from manifest file
          msg = {cmd: cmd, callback: callback, status: "request", payload: filename};
        // console.log(JSON.stringify(msg))
       socket.send(JSON.stringify(msg));
       } // loadPatientDemoData

//----------------------------------------------------------------------------------------------------
     function SetupSavedSelectionTree(msg){			     

		console.log("===== Setup SavedSelection Tree")
         console.log(msg)

         var AllData = msg.payload
         var PtIDs = []; 
         for(var i=0;i<AllData.length; i++){
         	if(PtIDs.indexOf(AllData[i].PatientID) === -1)
         		PtIDs.push(AllData[i].PatientID)
         }
         console.log("All Patients: ", PtIDs)
         root = {   "name": "All Patients",
         			"PatientIDs" : PtIDs,
         			"Tab": "ClinicalTable",
         			"Settings": "None"
         		}
         
         root.x0 = height / 2;
         root.y0 = 0;
             	
         function toggleAll(d) {
           if (d.children) {
             d.children.forEach(toggleAll);
             toggle(d);
           }
         }
         
       InitialLoad = false;
       updateSavedSelection(root); 
       CurrentSelection = root;
       SelectionNames.push(root.name);
  
       console.log("root: ", root)
      
 //      testingAddSavedSelection()
       
     }     
  
//--------------------------------------------------------------------------------------------
 //    function HandleWindowResize(){
 //         Display.width($(window).width() * 0.95);
 //         Display.height($(window).height() * 0.80);
 //         if(!InitialLoad) {updateSavedSelection(root);}
 //    };
  
//----------------------------------------------------------------------------------------------------
     
       return{
     
        init: function(){
           onReadyFunctions.push(initializeSelectionUI);
           addJavascriptMessageHandler("SetupSavedSelectionTree", SetupSavedSelectionTree);
 		   addJavascriptMessageHandler("addSelectionToTree", addSavedSelection);
          socketConnectedFunctions.push(loadPatientData);
           }
        };
     
}); // SavedSelectionModule
 

//----------------------------------------------------------------------------------------------------
SavedSelection = SavedSelectionModule();
SavedSelection.init();

</script>
<script>
//----------------------------------------------------------------------------------------------------
var PairedDistributionsModule = (function () {

  var generateRandomPairedDistributionsDataButton
  var pairedDistributionsDisplay;
  var pairedDistributionsResults;
  var patientClassification;
  var firstTime = true;
  var pairedDistributionsSelectedRegion;    // from brushing
  var d3PlotBrush;

  //--------------------------------------------------------------------------------------------
  function initializeUI (){
      generateRandomPairedDistributionsDataButton =  $("#generateRandomPairedDistributionsDataButton");
      generateRandomPairedDistributionsDataButton.click(requestRandomPairedDistributions);

      pairedDistributionsDisplay = $("#pairedDistributionsDisplay");
      pairedDistributionsHandleWindowResize();
      pairedDistributionsBroadcastButton = $("#pairedDistributionsBroadcastSelectionToClinicalTable");
      //pairedDistributionsBroadcastButton.button();
      $(window).resize(pairedDistributionsHandleWindowResize);
      pairedDistributionsBroadcastButton.prop("disabled",true);
      };

  //--------------------------------------------------------------------------------------------
  requestRandomPairedDistributions = function(){
     msg = {cmd: "calculatePairedDistributionsOfPatientHistoryData",
            callback:  "pairedDistributionsPlot",
            status: "request", 
            payload: {mode:"test", attribute:"FirstProgression"}};

     socket.send(JSON.stringify(msg));
     };

  //--------------------------------------------------------------------------------------------
  runDemo = function(){
     requestRandomPairedDistributions();
     };

  //--------------------------------------------------------------------------------------------
  getPatientClassification = function(){
     payload = "";
     msg = {cmd: "getPatientClassification", callback: "handlePatientClassification", 
            status: "request", payload: payload};
     socket.send(JSON.stringify(msg));
     };

  //--------------------------------------------------------------------------------------------
  handlePatientClassification = function(msg){
     console.log("=== handlePatientClassification");
     //console.log(msg)
     if(msg.status == "success"){
        patientClassification = JSON.parse(msg.payload)
        console.log("got classification, length " + patientClassification.length);
        }
     else{
       alert("error!" + msg.payload)
       }
      }; // handlePatientIDs

  //--------------------------------------------------------------------------------------------
  pairedDistributionsHandleWindowResize = function(){
     pairedDistributionsDisplay.width($(window).width() * 0.95);
     pairedDistributionsDisplay.height($(window).height() * 0.80);
     if(!firstTime) {d3PairedDistributionsScatterPlot(pairedDistributionsResults);}
     };

   //--------------------------------------------------------------------------------------------
   pairedDistributionsBroadcastSelection = function(){
      console.log("pairedDistributionsBroadcastSelection: " + pairedDistributionsSelectedRegion);
      x1=pairedDistributionsSelectedRegion[0][0];
      y1=pairedDistributionsSelectedRegion[0][1];
      x2=pairedDistributionsSelectedRegion[1][0];
      y2=pairedDistributionsSelectedRegion[1][1];
      ids = [];
      for(var i=0; i < pairedDistributionsResults.length; i++){
         p = pairedDistributionsResults[i];
         if(p.PC1 >= x1 & p.PC1 <= x2 & p.PC2 >= y1 & p.PC2 <= y2)
            ids.push(p.id[0]);
         } // for i
      if(ids.length > 0)
         sendIDsToModule(ids, "PatientHistory", "HandlePatientIDs");
      };

  //--------------------------------------------------------------------------------------------
  sendIDsToModule = function(ids, moduleName, title){
       callback = moduleName + title;
       msg = {cmd:"sendIDsToModule",
              callback: callback,
              status:"request",
              payload:{targetModule: moduleName,
                       ids: ids}
             };
      socket.send(JSON.stringify(msg));
      }; // sendTissueIDsToModule


  //--------------------------------------------------------------------------------------------
  pairedDistributionsPlot = function(msg){
      console.log("==== pairedDistributionsPlot");
      // console.log(msg);
      if(msg.status == "success"){
         pairedDistributionsResults = msg.payload;
         console.log(msg.payload);
         d3PairedDistributionsScatterPlot(pairedDistributionsResults);
         // tab activation not needed here - pshannon (7 aug 2014)
         //tabIndex = $('#tabs a[href="#pairedDistributionsDiv"]').parent().index();
         //if(!firstTime)  // first call comes at startup.  do not want to raise tab then.
         //    $("#tabs").tabs( "option", "active", tabIndex);
         } // success
    else{
      console.log("pairedDistributionsPlot about to call alert: " + msg)
      alert(msg.payload)
      }
     firstTime = false;
     };

  //--------------------------------------------------------------------------------------------
  handlePatientIDs = function(msg){
      console.log("Module.pairedDistributions: handlePatientIDs");
      //console.log(msg)
      if(msg.status == "success"){
         patientIDs = msg.payload
         //console.log("pairedDistributions handlePatientIds: " + patientIDs);
         payload = patientIDs
         msg = {cmd: "calculate_mRNA_PairedDistributions", callback: "pairedDistributionsPlot", status: "request", 
                payload: payload};
         socket.send(JSON.stringify(msg));
         }
    else{
      console.log("handlePatientIDs about to call alert: " + msg)
      alert(msg.payload)
      }
     }; // handlePatientIDs

  //--------------------------------------------------------------------------------------------
  d3PlotBrushReader = function(){
     console.log("plotBrushReader");
     pairedDistributionsSelectedRegion = d3PlotBrush.extent();
     //console.log("region: " + pairedDistributionsSelectedRegion);
     x0 = pairedDistributionsSelectedRegion[0][0];
     x1 = pairedDistributionsSelectedRegion[1][0];
     width = Math.abs(x0-x1);
     //console.log("width: " + width);
     if(width > 1)
        pairedDistributionsBroadcastButton.prop("disabled", false);
     else
        pairedDistributionsBroadcastButton.prop("disabled", true);
     }; // d3PlotBrushReader

  //-------------------------------------------------------------------------------------------
  chooseColor = function(d){
     id = d;
     for(var i=0; i<patientClassification.length; i++){
        if (id == patientClassification[i].rowname[0]){
          result = patientClassification[i].color[0]
          return(result)
          } // if match
        } // for i
     console.log("chooseColor, no match for id " + id);
     return("black");
     };
  //-------------------------------------------------------------------------------------------   
  getX = function(pop){
  	pop = pop + "";
  	if (pop == "pop1"){
  		return ((Math.random() * xMax/3)) + xMax/4;
  		}else{
  		return (Math.floor(Math.random() * -xMax/3)) - xMax/4;
  		}
  };
  //-------------------------------------------------------------------------------------------   
  getColor = function(pop){
  	pop = pop + "";
  	if (pop == "pop1"){
  		return 'red';
  		}else{
  		return 'blue';
  		}
  };
  //-------------------------------------------------------------------------------------------
  d3PairedDistributionsScatterPlot = function(data) {
  
  	 //console.log(d3.values(dataset.pop2));
  	 
  	 dataset = [];
  	 
  	 console.log(d3.values(data.pop2).length);
  	 
  	 for(i = 0; i < d3.values(data.pop1).length; i++){
  	 	if (d3.values(data.pop1)[i]!=null){
  	 		dataset.push({patient: d3.keys(data.pop1)[i], value: d3.values(data.pop1)[i], pop: "pop1"});
  	 		}
  	 }
  	 for(i = 0; i < d3.values(data.pop2).length; i++){
  	 	if (d3.values(data.pop2)[i]!=null){
  	 		dataset.push({patient: d3.keys(data.pop2)[i], value: d3.values(data.pop2)[i], pop: "pop2"});
  	 		}
  	 }
  
     pairedDistributionsBroadcastButton.prop("disabled",true);
     var padding = 50;
     var width = $("#pairedDistributionsDisplay").width();
     var height = $("#pairedDistributionsDisplay").height();
     
     var max = d3.max(dataset, function(d) { return +d.value;} );
     
     xMax = 40
     xMin = -40
     yMax = max * 1.1
     yMin = 0

        // select our svg by identifier, remove it
     d3.select("#pairedDistributionsSVG").remove()

        //flip axis
     var xScale = d3.scale.linear()
                    .domain([xMin,xMax])
                    .range([padding, width - padding]);

     var yScale = d3.scale.linear()
                    .domain([yMin, yMax])
                    .range([height - padding, padding]); // note inversion 

     var xAxis = d3.svg.axis()
                   .scale(xScale)
                   .orient("bottom")
                   .ticks(0);

     var yAxis = d3.svg.axis()
                   .scale(yScale)
                   .orient("left")
                   .ticks(5);


     d3PlotBrush = d3.svg.brush()
        .x(xScale)
        .y(yScale)
        .on("brushend", d3PlotBrushReader);

     var svg = d3.select("#pairedDistributionsDisplay")
                 .append("svg")
                 .attr("id", "pairedDistributionsSVG")
                 .attr("width", width)
                 .attr("height", height)
                 .call(d3PlotBrush);

     var tooltip = d3.select("body")
                     .attr("class", "tooltip")
                     .append("div")
                     .style("position", "absolute")
                     .style("z-index", "10")
                     .style("visibility", "hidden")
                     .text("a simple tooltip");


    var circle = svg.selectAll("circle")
        			.data(dataset)
   					.enter()
   					.append("circle")
   					.attr("cx", function(d) {return xScale(getX(d.pop));})
   					.attr("cy", function(d) {return yScale(d.value);})
   					.attr("r", 3)
                    .style("fill", function(d){return getColor(d.pop);})
                 	.on("mouseover", function(d,i){tooltip.text(d.patient); return tooltip.style("visibility", "visible");})
                	.on("mousemove", function(){return tooltip.style("top",
                           (d3.event.pageY-10)+"px").style("left",(d3.event.pageX+10)+"px");})
                	.on("mouseout", function(){return tooltip.style("visibility", "hidden");});
      
     var xTranslationForYAxis = xScale(0);
     var yTranslationForXAxis = yScale(0);

     svg.append("g")
        .attr("class", "x axis")
        .attr("transform", "translate(0, " + yTranslationForXAxis + ")")
        .call(xAxis);

     svg.append("g")
        .attr("class", "y axis")
        .attr("transform", "translate(" + xTranslationForYAxis + ", 0)")
        .call(yAxis);

     }; // d3PairedDistributionsScatterPlot


  //--------------------------------------------------------------------------------------------
  return{
   init: function(){
      onReadyFunctions.push(initializeUI);
      addJavascriptMessageHandler("pairedDistributionsPlot", pairedDistributionsPlot);
      //addJavascriptMessageHandler("PairedDistributionsHandlePatientIDs", handlePatientIDs);
      //addJavascriptMessageHandler("handlePatientClassification", handlePatientClassification)
      //socketConnectedFunctions.push(getPatientClassification);
      socketConnectedFunctions.push(runDemo);
      }
   };

}); // PairedDistributionsModule
//----------------------------------------------------------------------------------------------------
pairedDistributions = PairedDistributionsModule();
pairedDistributions.init();

</script>
<script>
onReadyFunctions.push(function() {
    console.log("====== tabapps document ready");
    window.tabsAppRunning = true
    $("#tabs").tabs({
         // todo: distinguish between tabs, only do needed resets
       activate: function(event, ui) {
            console.log("tabs.activate");
            console.log(" ==== tab.activate, tableRef.fnAdjustColumnSizing");
            var tableRef = $("#clinicalTable").dataTable();
            if (tableRef.length > 0) {
               tableRef.fnAdjustColumnSizing();
              } // if
            console.log(" ==== tab.activate, possible cyjs resize and fit");
            if(typeof(cwMarkers) != "undefined") {
               cwMarkers.resize(); 
               cwMarkers.fit(50);
               }
            if(typeof(cwGBM) != "undefined") {
                cwGBM.resize();
                cwGBM.fit(50);
                }
            if(typeof(cwAngio) != "undefined") {
                cwAngio.resize();
                cwAngio.fit(50);
                }
            } // activate
        }); // tabs
    });  // ready

</script>


<script>
//----------------------------------------------------------------------------------------------------
var todoModule = (function () {

var ToDoURL = "https://docs.google.com/spreadsheets/d/1Rqqpma1M8aF5bX4BM2cYYgDd5hazUysqzCxLhGUwldo/edit?usp=sharing"
var ToDobutton;

initializeUI = function(){
    ToDobutton = $("#ToDoLink");
    ToDobutton.on("click",function(d){window.open(ToDoURL) }   )
    };


return{

   init: function(){
      onReadyFunctions.push(initializeUI);
      }
   };

}); // DateAndTimeModule
//----------------------------------------------------------------------------------------------------
todo = todoModule();
todo.init();

</script>

<div id="tabs">
   <ul>
     <li><a href="#DashboardDiv">Dashboard</a></li>
     <li><a href="#clinicalDataModuleDiv">Clinical Data</a></li>
     <li><a href="#pcaDiv">PCA</a></li>
     <li><a href="#patientTimeLinesDiv">Timelines</a></li>
     <li><a href="#gbmPathwaysDiv">GBM Pathways</a></li>
     <li><a href="#SavedSelectionDiv">Saved Selection</a></li>
     <li><a href="#pairedDistributionsDiv">Paired Distributions</a></li>
     <li><a href="#todoDiv">Todo</a></li>
   </ul>


<style>
ul.UserDocs {
    list-style-type: circle;
}
</style>


<div id="DashboardDiv">
<h3> Oncoscape </h3>
   <p> Welcome to Oncoscape! </p>
   <p> This page will include:</p>
   <ul class="UserDocs">
     <li> a welcome message</li>
     <li> links to download vignettes </li>
     <li> Explanation of available data </li>
     <li> Tooltips/Tutorial </li>
     <li> link to download/install software </li>
   </ul>
</div>

<style>


#clinicalDataTableDiv {
   min-width: 900px;
   margin: 0 auto;
   background-color: #E8E8E8;
   font-family: sans-serif;
   font-size: 14px;
   }

#oldclinicalDataTableDiv {
   border: 1px solid #aaa;
   height: 500px;
   font-family: sans-serif;
   background-color: #E8E8E8;
   font-size: 12px;
   margin-top: 1px;
   margin-left: 10px;
   margin-right: 10px;
   margin-bottom: 1px;
   padding: 1px;
   padding-bottom: 20px;
   }

th, td { white-space: nowrap; } 



#compactViewButton{
   margin-left: 40px;
   }


#clinicalDataTableControlsDiv {
   border: 1px solid #ccc;
   font-family: sans-serif;
   font-size: 12px;
   margin-top: 2px;
   margin-left: 10px;
   margin-right: 10px;
   margin-bottom: 5px;
   background-color: #E8E8E8;
   }

#clinicalTablesButtonsDiv{
   padding-left: 20px;
   padding-bottom: 5px;
   }

#clinicalDataTableFilterControlsDiv{
   font-family: sans-serif;
   font-size: 14px;
   margin-top: 40px;
   margin-left: 100px;
   margin-right: 100px;
   margin-bottom: 20px;
   }

#clinicalDataTableOutboundButtonDiv{
   margin-left: 240px;
   }

.clinicalDataFilterSlider { 
   margin-left:20px; 
   margin-right:5px; 
   margin-top:0px;
   border: 1px solid #FFF;
   height: 20px;
   max: 0;  min: 10; 
   orientation: 'horizontal';
   background-color: #FFF;
   color: #00F;
   }

.clinicalDataFilterSliderReadout {
  font-family:"Courier";
  color: #0000FF;
  background-color: #FFF;
  font-size: 14px;
  border: 1px solid #DDD;
  width: 20px;
  height: 16px;
  resize: none;
  margin-top: 4px;
  margin-left: 2px;
  }

.clinicalDataFilterSliderTitleDiv {
  font-family:"Arial";
  color: #000000;
  font-size: 14px;
  padding-top: 10px;
  padding-left: 40px;
  }

.tblClinicalControlButton {
   margin-bottom: 2px;
   }




</style>


<div id="clinicalDataModuleDiv">
   <div id="clinicalDataTableControlsDiv">
      <div id=clinicalDataTableFilterControlsDiv" style="border: 1px;">

       <div id="slider1Div" style="display: inline-block">
           <div id="sliderTitleDiv" class="clinicalDataFilterSliderTitleDiv" style="margin-left: 10px;">ageAtDx min</div>
           <div style="width: 200px; overflow: hidden;">
               <div style="width: 120px; height: 10px; float: left; margin-top: 10px;" id="ageAtDxMinSlider" class="clinicalDataFilterSlider"></div>
               <div style="margin-left: 20px;" id='msgBox'>  
                  <textarea id="ageAtDxMinSliderReadout" readonly class="clinicalDataFilterSliderReadout"'></textarea> 
               </div>
           </div>
        </div>

       <div id="slider2Div" style="display: inline-block">
           <div id="sliderTitleDiv"  class="clinicalDataFilterSliderTitleDiv" style="margin-left: 10px;">ageAtDx max</div>
           <div style="width: 300px; overflow: hidden;">
               <div style="width: 120px; height: 10px; float: left; margin-top: 10px;" id="ageAtDxMaxSlider" class="clinicalDataFilterSlider"></div>
               <div style="margin-left: 20px;" id='msgBox'>  
                  <textarea id="ageAtDxMaxSliderReadout" readonly class="clinicalDataFilterSliderReadout"'></textarea> 
               </div>
           </div>
        </div>

       <div id="slider3Div" style="display: inline-block">
           <div id="sliderTitle3Div" class="clinicalDataFilterSliderTitleDiv" style="margin-left: 10px;">survival min</div>
           <div style="width: 200px; overflow: hidden;">
               <div style="width: 120px; height: 10px; float: left; margin-top: 10px;" id="overallSurvivalMinSlider" class="clinicalDataFilterSlider"></div>
               <div style="margin-left: 20px;" id='msgBox'>  
                  <textarea id="overallSurvivalMinSliderReadout" readonly class="clinicalDataFilterSliderReadout"'></textarea> 
               </div>
           </div>
        </div>

       <div id="slider4Div" style="display: inline-block">
           <div id="sliderTitle4Div" class="clinicalDataFilterSliderTitleDiv" style="margin-left: 10px;">survival max</div>
           <div style="width: 200px; overflow: hidden;">
               <div style="width: 120px; height: 10px; float: left; margin-top: 10px;" id="overallSurvivalMaxSlider" class="clinicalDataFilterSlider"></div>
               <div style="margin-left: 20px;" id='msgBox'>  
                  <textarea id="overallSurvivalMaxSliderReadout" readonly class="clinicalDataFilterSliderReadout"'></textarea> 
               </div>
           </div>
        </div>

       <div id="clinicalTablesButtonsDiv" style="display: block">
           <div id="sliderApplyDiv" style="display: inline-block">
              <button id="applyClinicalTablesFiltersSlidersButton" class="tblClinicalControlButton" type="button">Apply Filters</button>
           </div>
           <div id="viewButtonDiv" style="display: inline-block">
              <button id="showAllClinicalTablesRowsButton" class="tblClinicalControlButton" type="button">Remove Filters</button>
              <!-- button id="toggleMoreFewerColumnsButton" class="tblClinicalControlButton" type="button">More Columns</button -->
           </div>

          <div id="clinicalDataTableOutboundButtonDiv" style="display: inline-block">
             <button id="toMarkersAndTissuesButton" class="tblClinicalControlButton" type="button">Markers &amp; Tissues</button>
             <button id="toGBMPathwaysButton" class="tblClinicalControlButton" type="button">GBM Pathways</button>
             <button id="toPCAButton" class="tblClinicalControlButton" type="button">PCA</button>
             <button id="toSurvivalStatsButton" class="tblClinicalControlButton" type="button">Survival</button>
             <button id="toTimeLinesButton" class="tblClinicalControlButton" type="button">Timelines</button>
          </div>
       </div>

    </div>
   </div>
   <div id="clinicalDataTableDiv"></div>
</div>
<style>

.domain { 
  fill: none; 
  } 


.extent {
  fill-opacity: .1;
  stroke: #f00;
}


.axis path, .axis line {
    stroke: black;
    stroke-width: 1px;
    }

#pcaDiv {
   margin: 0px;
   padding: 0px;
   }

#pcaDisplay {
   border: 1px solid #aaa;
   width: 50px;
   height: 50px;
   background-color: #FAFAFA;
   margin-top: 1px;
   margin-left: 3px;
   margin-right: 0;
   margin-bottom: 1px;
   padding: 1px;
   padding-bottom: 0px;
   }

#pcaBroadcastSelectionToClinicalTable{
   font-size: 10pt;
   }

</style>

<div id="pcaDiv">
   <button id="pcaBroadcastSelectionToClinicalTable">To Clinical Data... </button>
   <div id="pcaDisplay"></div>
</div>

<style>

.axis path,
.axis line {
  fill: none;
  stroke: #000;
  shape-rendering: crispEdges;
}

.x.axis path {
  display: none;
}

.line {
  fill: none;
  stroke: steelblue;
  stroke-width: 1.5px;
}

.select{
	background-color: white;
}
.button{
	background-color: white;
	border-radius: 0 0 5px 5px; 
	border-style: none solid solid; 
	border-width: 0 1px 1px;

}
.resizable {
    resize: both;   /* Options: horizontal, vertical, both */
    overflow: auto; /* fix for Safari */
}

.domain { 
  fill: none; 
  } 



</style>

<canvas width="960" height="500" style="display:none"></canvas>

<div id="patientTimeLinesDiv">
	<div id="ArrangeDataDiv">
	     <span id="AddSideBar">Features: </span>
	     <span id="AlignByDiv">Align By: </span>
	     <span id="OrderByDiv">Order By: </span>
<!--	     <button id="SaveImageDiv">Save Image </button>  -->
       	 <button id="SendSelectionToClinicalTable">To Clinical Data... </button>
     </div>
     <div id="TimeLineDisplay">
      </div>
     
     <div id="AddCalculatedEvent"  style="display:none">
     <form>
          <label for="Name">Name</label>
          <input type="text" id="Name" value="e.g. Survival"><br>
          <label for="Event1">Event 1</label>
          <input type="text" id="Event1" value="(legend)"><br>
          <label for="Event2">Event 2</label>
          <input type="text" id="Event2" value="(legend)"><br>
          <label for="TimeScale"> Time Scale</label>
          <input type="text" id="TimeScale" value="Days, Months, Years"><br>
	</form>
     </div>
</div>

<script>
curatedGBMpathways = {
  "format_version" : "1.0",
  "generated_by" : "cytoscape-3.1.0",
  "target_cytoscapejs_version" : "~2.1",
  "data" : {
    "selected" : true,
    "__Annotations" : [ ],
    "shared_name" : "GBM (mskcc & hbolouri) + Holland Lab Data",
    "SUID" : 7631,
    "name" : "GBM (mskcc & hbolouri) + Holland Lab Data"
  },
  "elements" : {
    "nodes" : [ {
      "data" : {
        "id" : "7797",
        "mut" : 0.0,
        "SUID" : 7797,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "ABCA1",
        "selected" : false,
        "canonicalName" : "ABCA1",
        "name" : "ABCA1",
        "nodeType" : "gene",
        "shared_name" : "ABCA1"
      },
      "position" : {
        "x" : -22.0,
        "y" : 3068.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7796",
        "mut" : 0.0,
        "SUID" : 7796,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "AKT family",
        "selected" : false,
        "canonicalName" : "AKT family",
        "name" : "AKT family",
        "nodeType" : "family",
        "shared_name" : "AKT family"
      },
      "position" : {
        "x" : -347.0,
        "y" : 361.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7795",
        "mut" : 0.0,
        "SUID" : 7795,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "AKT1",
        "selected" : false,
        "canonicalName" : "AKT1",
        "name" : "AKT1",
        "nodeType" : "protein",
        "shared_name" : "AKT1"
      },
      "position" : {
        "x" : -1030.0,
        "y" : 779.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7794",
        "mut" : 0.0,
        "SUID" : 7794,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "ATM",
        "selected" : false,
        "canonicalName" : "ATM",
        "name" : "ATM",
        "nodeType" : "gene",
        "shared_name" : "ATM"
      },
      "position" : {
        "x" : -2522.0,
        "y" : 1030.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7793",
        "mut" : 0.0,
        "SUID" : 7793,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "BASC complex",
        "selected" : false,
        "canonicalName" : "BASC complex",
        "name" : "BASC complex",
        "nodeType" : "complex",
        "shared_name" : "BASC complex"
      },
      "position" : {
        "x" : -2661.0,
        "y" : 2365.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7792",
        "mut" : 0.0,
        "SUID" : 7792,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "BCL2L1",
        "selected" : false,
        "canonicalName" : "BCL2L1",
        "name" : "BCL2L1",
        "nodeType" : "gene",
        "shared_name" : "BCL2L1"
      },
      "position" : {
        "x" : -2070.0,
        "y" : 2358.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7791",
        "mut" : 0.0,
        "SUID" : 7791,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "BRCA complex",
        "selected" : false,
        "canonicalName" : "BRCA complex",
        "name" : "BRCA complex",
        "nodeType" : "complex",
        "shared_name" : "BRCA complex"
      },
      "position" : {
        "x" : -2385.0,
        "y" : 2365.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7790",
        "mut" : 0.0,
        "SUID" : 7790,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "CBL",
        "selected" : false,
        "canonicalName" : "CBL",
        "name" : "CBL",
        "nodeType" : "gene",
        "shared_name" : "CBL"
      },
      "position" : {
        "x" : 1284.0,
        "y" : -265.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7789",
        "mut" : 0.0,
        "SUID" : 7789,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "CCND2",
        "selected" : false,
        "canonicalName" : "CCND2",
        "name" : "CCND2",
        "nodeType" : "gene",
        "shared_name" : "CCND2"
      },
      "position" : {
        "x" : 1284.0,
        "y" : 2049.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7788",
        "mut" : 0.0,
        "SUID" : 7788,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "CCND-CDK4 complex",
        "selected" : false,
        "canonicalName" : "CCND-CDK4 complex",
        "name" : "CCND-CDK4 complex",
        "nodeType" : "complex",
        "shared_name" : "CCND-CDK4 complex"
      },
      "position" : {
        "x" : 745.0,
        "y" : 2056.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7787",
        "mut" : 0.0,
        "SUID" : 7787,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "CCND-CDK6 complex",
        "selected" : false,
        "canonicalName" : "CCND-CDK6 complex",
        "name" : "CCND-CDK6 complex",
        "nodeType" : "complex",
        "shared_name" : "CCND-CDK6 complex"
      },
      "position" : {
        "x" : 1895.0,
        "y" : 2063.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7786",
        "mut" : 0.0,
        "SUID" : 7786,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "CCNE-CDK2 complex",
        "selected" : false,
        "canonicalName" : "CCNE-CDK2 complex",
        "name" : "CCNE-CDK2 complex",
        "nodeType" : "complex",
        "shared_name" : "CCNE-CDK2 complex"
      },
      "position" : {
        "x" : 95.0,
        "y" : 2208.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7785",
        "mut" : 0.0,
        "SUID" : 7785,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "CDKN1A",
        "selected" : false,
        "canonicalName" : "CDKN1A",
        "name" : "CDKN1A",
        "nodeType" : "gene",
        "shared_name" : "CDKN1A"
      },
      "position" : {
        "x" : 1030.0,
        "y" : 914.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7784",
        "mut" : 0.0,
        "SUID" : 7784,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "CDKN1B",
        "selected" : false,
        "canonicalName" : "CDKN1B",
        "name" : "CDKN1B",
        "nodeType" : "gene",
        "shared_name" : "CDKN1B"
      },
      "position" : {
        "x" : 95.0,
        "y" : 1806.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7783",
        "mut" : 0.0,
        "SUID" : 7783,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "CDKN2A:ARF",
        "selected" : false,
        "canonicalName" : "CDKN2A:ARF",
        "name" : "CDKN2A:ARF",
        "nodeType" : "gene",
        "shared_name" : "CDKN2A:ARF"
      },
      "position" : {
        "x" : -1735.0,
        "y" : 367.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7782",
        "mut" : 0.0,
        "SUID" : 7782,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "CENTG1",
        "selected" : false,
        "canonicalName" : "CENTG1",
        "name" : "CENTG1",
        "nodeType" : "gene",
        "shared_name" : "CENTG1"
      },
      "position" : {
        "x" : -1115.0,
        "y" : -889.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7781",
        "mut" : 0.0,
        "SUID" : 7781,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "CIC",
        "selected" : false,
        "canonicalName" : "CIC",
        "name" : "CIC",
        "nodeType" : "gene",
        "shared_name" : "CIC"
      },
      "position" : {
        "x" : -3290.0,
        "y" : 1692.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7780",
        "mut" : 0.0,
        "SUID" : 7780,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "CMYB",
        "selected" : false,
        "canonicalName" : "CMYB",
        "name" : "CMYB",
        "nodeType" : "protein",
        "shared_name" : "CMYB"
      },
      "position" : {
        "x" : -1295.0,
        "y" : 2465.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7779",
        "mut" : 0.0,
        "SUID" : 7779,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "CTNNB1",
        "selected" : false,
        "canonicalName" : "CTNNB1",
        "name" : "CTNNB1",
        "nodeType" : "gene",
        "shared_name" : "CTNNB1"
      },
      "position" : {
        "x" : 3523.0,
        "y" : 2512.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7778",
        "mut" : 0.0,
        "SUID" : 7778,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "E2F1",
        "selected" : false,
        "canonicalName" : "E2F1",
        "name" : "E2F1",
        "nodeType" : "gene",
        "shared_name" : "E2F1"
      },
      "position" : {
        "x" : 1284.0,
        "y" : 2911.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7777",
        "mut" : 0.0,
        "SUID" : 7777,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "EGFR family",
        "selected" : false,
        "canonicalName" : "EGFR family",
        "name" : "EGFR family",
        "nodeType" : "family",
        "shared_name" : "EGFR family"
      },
      "position" : {
        "x" : 1993.0,
        "y" : -1441.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7776",
        "mut" : 0.0,
        "SUID" : 7776,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "EP300",
        "selected" : false,
        "canonicalName" : "EP300",
        "name" : "EP300",
        "nodeType" : "gene",
        "shared_name" : "EP300"
      },
      "position" : {
        "x" : -1946.0,
        "y" : 1209.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7775",
        "mut" : 0.0,
        "SUID" : 7775,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "ERK1/2",
        "selected" : false,
        "canonicalName" : "ERK1/2",
        "name" : "ERK1/2",
        "nodeType" : "gene",
        "shared_name" : "ERK1/2"
      },
      "position" : {
        "x" : 2295.0,
        "y" : 1692.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7774",
        "mut" : 0.0,
        "SUID" : 7774,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "ERRFI1",
        "selected" : false,
        "canonicalName" : "ERRFI1",
        "name" : "ERRFI1",
        "nodeType" : "gene",
        "shared_name" : "ERRFI1"
      },
      "position" : {
        "x" : 1984.0,
        "y" : -1706.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7773",
        "mut" : 0.0,
        "SUID" : 7773,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "FGFR family",
        "selected" : false,
        "canonicalName" : "FGFR family",
        "name" : "FGFR family",
        "nodeType" : "family",
        "shared_name" : "FGFR family"
      },
      "position" : {
        "x" : 2155.0,
        "y" : -720.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7772",
        "mut" : 0.0,
        "SUID" : 7772,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "FOXO family",
        "selected" : false,
        "canonicalName" : "FOXO family",
        "name" : "FOXO family",
        "nodeType" : "family",
        "shared_name" : "FOXO family"
      },
      "position" : {
        "x" : 324.0,
        "y" : 395.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7771",
        "mut" : 0.0,
        "SUID" : 7771,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "FUBP1",
        "selected" : false,
        "canonicalName" : "FUBP1",
        "name" : "FUBP1",
        "nodeType" : "gene",
        "shared_name" : "FUBP1"
      },
      "position" : {
        "x" : 3196.0,
        "y" : 2789.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7770",
        "mut" : 0.0,
        "SUID" : 7770,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "GAB1",
        "selected" : false,
        "canonicalName" : "GAB1",
        "name" : "GAB1",
        "nodeType" : "gene",
        "shared_name" : "GAB1"
      },
      "position" : {
        "x" : 707.0,
        "y" : -921.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7769",
        "mut" : 0.0,
        "SUID" : 7769,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "GLUT1",
        "selected" : false,
        "canonicalName" : "GLUT1",
        "name" : "GLUT1",
        "nodeType" : "gene",
        "shared_name" : "GLUT1"
      },
      "position" : {
        "x" : 3639.0,
        "y" : 3018.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7768",
        "mut" : 0.0,
        "SUID" : 7768,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "GRB2",
        "selected" : false,
        "canonicalName" : "GRB2",
        "name" : "GRB2",
        "nodeType" : "gene",
        "shared_name" : "GRB2"
      },
      "position" : {
        "x" : 1829.0,
        "y" : -486.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7767",
        "mut" : 0.0,
        "SUID" : 7767,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "HIF1A",
        "selected" : false,
        "canonicalName" : "HIF1A",
        "name" : "HIF1A",
        "nodeType" : "protein",
        "shared_name" : "HIF1A"
      },
      "position" : {
        "x" : -1030.0,
        "y" : 2600.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7766",
        "mut" : 0.0,
        "SUID" : 7766,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "HGF",
        "selected" : false,
        "canonicalName" : "HGF",
        "name" : "HGF",
        "nodeType" : "protein",
        "shared_name" : "HGF"
      },
      "position" : {
        "x" : 973.0,
        "y" : -1706.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7765",
        "mut" : 0.0,
        "SUID" : 7765,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "HOXA9",
        "selected" : false,
        "canonicalName" : "HOXA9",
        "name" : "HOXA9",
        "nodeType" : "protein",
        "shared_name" : "HOXA9"
      },
      "position" : {
        "x" : -3096.0,
        "y" : 2728.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7764",
        "mut" : 0.0,
        "SUID" : 7764,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "IDH1",
        "selected" : false,
        "canonicalName" : "IDH1",
        "name" : "IDH1",
        "nodeType" : "gene",
        "shared_name" : "IDH1"
      },
      "position" : {
        "x" : -698.0,
        "y" : 2429.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7763",
        "mut" : 0.0,
        "SUID" : 7763,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "IDH2",
        "selected" : false,
        "canonicalName" : "IDH2",
        "name" : "IDH2",
        "nodeType" : "gene",
        "shared_name" : "IDH2"
      },
      "position" : {
        "x" : -840.0,
        "y" : 2307.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7762",
        "mut" : 0.0,
        "SUID" : 7762,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "IGF1R",
        "selected" : false,
        "canonicalName" : "IGF1R",
        "name" : "IGF1R",
        "nodeType" : "gene",
        "shared_name" : "IGF1R"
      },
      "position" : {
        "x" : 1284.0,
        "y" : -1087.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7761",
        "mut" : 0.0,
        "SUID" : 7761,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "INK4 family",
        "selected" : false,
        "canonicalName" : "INK4 family",
        "name" : "INK4 family",
        "nodeType" : "family",
        "shared_name" : "INK4 family"
      },
      "position" : {
        "x" : 1284.0,
        "y" : 1809.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7760",
        "mut" : 0.0,
        "SUID" : 7760,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "IRS1",
        "selected" : false,
        "canonicalName" : "IRS1",
        "name" : "IRS1",
        "nodeType" : "gene",
        "shared_name" : "IRS1"
      },
      "position" : {
        "x" : 540.0,
        "y" : -1087.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7759",
        "mut" : 0.0,
        "SUID" : 7759,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "LDHA",
        "selected" : false,
        "canonicalName" : "LDHA",
        "name" : "LDHA",
        "nodeType" : "gene",
        "shared_name" : "LDHA"
      },
      "position" : {
        "x" : 3426.0,
        "y" : 3018.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7758",
        "mut" : 0.0,
        "SUID" : 7758,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "LDLR",
        "selected" : false,
        "canonicalName" : "LDLR",
        "name" : "LDLR",
        "nodeType" : "gene",
        "shared_name" : "LDLR"
      },
      "position" : {
        "x" : 608.0,
        "y" : 3061.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7757",
        "mut" : 0.0,
        "SUID" : 7757,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "LPA",
        "selected" : false,
        "canonicalName" : "LPA",
        "name" : "LPA",
        "nodeType" : "smallMolecule",
        "shared_name" : "LPA"
      },
      "position" : {
        "x" : -3259.0,
        "y" : -2.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7756",
        "mut" : 0.0,
        "SUID" : 7756,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "LXR family",
        "selected" : false,
        "canonicalName" : "LXR family",
        "name" : "LXR family",
        "nodeType" : "family",
        "shared_name" : "LXR family"
      },
      "position" : {
        "x" : 119.0,
        "y" : 2850.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7755",
        "mut" : 0.0,
        "SUID" : 7755,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "MDM2",
        "selected" : false,
        "canonicalName" : "MDM2",
        "name" : "MDM2",
        "nodeType" : "gene",
        "shared_name" : "MDM2"
      },
      "position" : {
        "x" : -1734.0,
        "y" : 655.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7754",
        "mut" : 0.0,
        "SUID" : 7754,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "MDM4",
        "selected" : false,
        "canonicalName" : "MDM4",
        "name" : "MDM4",
        "nodeType" : "gene",
        "shared_name" : "MDM4"
      },
      "position" : {
        "x" : -1489.0,
        "y" : 1187.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7753",
        "mut" : 0.0,
        "SUID" : 7753,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "MEK1/2",
        "selected" : false,
        "canonicalName" : "MEK1/2",
        "name" : "MEK1/2",
        "nodeType" : "gene",
        "shared_name" : "MEK1/2"
      },
      "position" : {
        "x" : 2289.0,
        "y" : 1344.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7752",
        "mut" : 0.0,
        "SUID" : 7752,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "MET:EGFR",
        "selected" : false,
        "canonicalName" : "MET:EGFR",
        "name" : "MET:EGFR",
        "nodeType" : "complex",
        "shared_name" : "MET:EGFR"
      },
      "position" : {
        "x" : 1677.0,
        "y" : -1599.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7751",
        "mut" : 0.0,
        "SUID" : 7751,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "MGMT",
        "selected" : false,
        "canonicalName" : "MGMT",
        "name" : "MGMT",
        "nodeType" : "protein",
        "shared_name" : "MGMT"
      },
      "position" : {
        "x" : -1030.0,
        "y" : 2895.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7750",
        "mut" : 0.0,
        "SUID" : 7750,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "MIR200C",
        "selected" : false,
        "canonicalName" : "MIR200C",
        "name" : "MIR200C",
        "nodeType" : "miRNA",
        "shared_name" : "MIR200C"
      },
      "position" : {
        "x" : -1244.0,
        "y" : 2173.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7749",
        "mut" : 0.0,
        "SUID" : 7749,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "MIR26A1",
        "selected" : false,
        "canonicalName" : "MIR26A1",
        "name" : "MIR26A1",
        "nodeType" : "miRNA",
        "shared_name" : "MIR26A1"
      },
      "position" : {
        "x" : 434.0,
        "y" : -495.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7748",
        "mut" : 0.0,
        "SUID" : 7748,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "MSK1/2",
        "selected" : false,
        "canonicalName" : "MSK1/2",
        "name" : "MSK1/2",
        "nodeType" : "gene",
        "shared_name" : "MSK1/2"
      },
      "position" : {
        "x" : 2731.0,
        "y" : 1767.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7747",
        "mut" : 0.0,
        "SUID" : 7747,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "mTORC1",
        "selected" : false,
        "canonicalName" : "mTORC1",
        "name" : "mTORC1",
        "nodeType" : "complex",
        "shared_name" : "mTORC1"
      },
      "position" : {
        "x" : -647.0,
        "y" : 1560.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7746",
        "mut" : 0.0,
        "SUID" : 7746,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "MYC",
        "selected" : false,
        "canonicalName" : "MYC",
        "name" : "MYC",
        "nodeType" : "gene",
        "shared_name" : "MYC"
      },
      "position" : {
        "x" : 3523.0,
        "y" : 2776.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7745",
        "mut" : 0.0,
        "SUID" : 7745,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "MYLIP",
        "selected" : false,
        "canonicalName" : "MYLIP",
        "name" : "MYLIP",
        "nodeType" : "gene",
        "shared_name" : "MYLIP"
      },
      "position" : {
        "x" : 390.0,
        "y" : 2918.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7744",
        "mut" : 0.0,
        "SUID" : 7744,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "NF1",
        "selected" : false,
        "canonicalName" : "NF1",
        "name" : "NF1",
        "nodeType" : "gene",
        "shared_name" : "NF1"
      },
      "position" : {
        "x" : 1593.0,
        "y" : -14.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7743",
        "mut" : 0.0,
        "SUID" : 7743,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "NF-kB:HIF1A",
        "selected" : false,
        "canonicalName" : "NF-kB:HIF1A",
        "name" : "NF-kB:HIF1A",
        "nodeType" : "complex",
        "shared_name" : "NF-kB:HIF1A"
      },
      "position" : {
        "x" : 3523.0,
        "y" : 2025.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7742",
        "mut" : 0.0,
        "SUID" : 7742,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "PDGFR family",
        "selected" : false,
        "canonicalName" : "PDGFR family",
        "name" : "PDGFR family",
        "nodeType" : "family",
        "shared_name" : "PDGFR family"
      },
      "position" : {
        "x" : 2081.0,
        "y" : -986.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7741",
        "mut" : 0.0,
        "SUID" : 7741,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "PDPK1",
        "selected" : false,
        "canonicalName" : "PDPK1",
        "name" : "PDPK1",
        "nodeType" : "gene",
        "shared_name" : "PDPK1"
      },
      "position" : {
        "x" : -347.0,
        "y" : 38.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7740",
        "mut" : 0.0,
        "SUID" : 7740,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "PI3K class1a",
        "selected" : false,
        "canonicalName" : "PI3K class1a",
        "name" : "PI3K class1a",
        "nodeType" : "class",
        "shared_name" : "PI3K class1a"
      },
      "position" : {
        "x" : -347.0,
        "y" : -889.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7739",
        "mut" : 0.0,
        "SUID" : 7739,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "PI3K class1b",
        "selected" : false,
        "canonicalName" : "PI3K class1b",
        "name" : "PI3K class1b",
        "nodeType" : "class",
        "shared_name" : "PI3K class1b"
      },
      "position" : {
        "x" : 579.0,
        "y" : -213.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7738",
        "mut" : 0.0,
        "SUID" : 7738,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "PI3K class2",
        "selected" : false,
        "canonicalName" : "PI3K class2",
        "name" : "PI3K class2",
        "nodeType" : "class",
        "shared_name" : "PI3K class2"
      },
      "position" : {
        "x" : -3434.0,
        "y" : -190.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7737",
        "mut" : 0.0,
        "SUID" : 7737,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "PI-3-P",
        "selected" : false,
        "canonicalName" : "PI-3-P",
        "name" : "PI-3-P",
        "nodeType" : "gene",
        "shared_name" : "PI-3-P"
      },
      "position" : {
        "x" : -3434.0,
        "y" : 632.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7736",
        "mut" : 0.0,
        "SUID" : 7736,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "PIK3C2B",
        "selected" : false,
        "canonicalName" : "PIK3C2B",
        "name" : "PIK3C2B",
        "nodeType" : "gene",
        "shared_name" : "PIK3C2B"
      },
      "position" : {
        "x" : -3434.0,
        "y" : 273.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7735",
        "mut" : 0.0,
        "SUID" : 7735,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "PIP3",
        "selected" : false,
        "canonicalName" : "PIP3",
        "name" : "PIP3",
        "nodeType" : "gene",
        "shared_name" : "PIP3"
      },
      "position" : {
        "x" : -347.0,
        "y" : -287.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7734",
        "mut" : 0.0,
        "SUID" : 7734,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "PKC class",
        "selected" : false,
        "canonicalName" : "PKC class",
        "name" : "PKC class",
        "nodeType" : "class",
        "shared_name" : "PKC class"
      },
      "position" : {
        "x" : 2678.0,
        "y" : 1030.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7733",
        "mut" : 0.0,
        "SUID" : 7733,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "PKM2",
        "selected" : false,
        "canonicalName" : "PKM2",
        "name" : "PKM2",
        "nodeType" : "gene",
        "shared_name" : "PKM2"
      },
      "position" : {
        "x" : 3523.0,
        "y" : 2259.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7732",
        "mut" : 0.0,
        "SUID" : 7732,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "PLCG",
        "selected" : false,
        "canonicalName" : "PLCG",
        "name" : "PLCG",
        "nodeType" : "gene",
        "shared_name" : "PLCG"
      },
      "position" : {
        "x" : 2678.0,
        "y" : 14.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7731",
        "mut" : 0.0,
        "SUID" : 7731,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "polycomb repressive complex",
        "selected" : false,
        "canonicalName" : "polycomb repressive complex",
        "name" : "polycomb repressive complex",
        "nodeType" : "complex",
        "shared_name" : "polycomb repressive complex"
      },
      "position" : {
        "x" : -3096.0,
        "y" : 646.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7730",
        "mut" : 0.0,
        "SUID" : 7730,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "PRKCE",
        "selected" : false,
        "canonicalName" : "PRKCE",
        "name" : "PRKCE",
        "nodeType" : "gene",
        "shared_name" : "PRKCE"
      },
      "position" : {
        "x" : 3523.0,
        "y" : -363.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7729",
        "mut" : 0.0,
        "SUID" : 7729,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "pS6K",
        "selected" : false,
        "canonicalName" : "pS6K",
        "name" : "pS6K",
        "nodeType" : "protein",
        "shared_name" : "pS6K"
      },
      "position" : {
        "x" : -188.0,
        "y" : 1367.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7728",
        "mut" : 0.0,
        "SUID" : 7728,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "PTEN",
        "selected" : false,
        "canonicalName" : "PTEN",
        "name" : "PTEN",
        "nodeType" : "gene",
        "shared_name" : "PTEN"
      },
      "position" : {
        "x" : 151.0,
        "y" : -405.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7727",
        "mut" : 0.0,
        "SUID" : 7727,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "RAF family",
        "selected" : false,
        "canonicalName" : "RAF family",
        "name" : "RAF family",
        "nodeType" : "family",
        "shared_name" : "RAF family"
      },
      "position" : {
        "x" : 1829.0,
        "y" : 1030.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7726",
        "mut" : 0.0,
        "SUID" : 7726,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "RAS family",
        "selected" : false,
        "canonicalName" : "RAS family",
        "name" : "RAS family",
        "nodeType" : "family",
        "shared_name" : "RAS family"
      },
      "position" : {
        "x" : 1829.0,
        "y" : 188.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7725",
        "mut" : 0.0,
        "SUID" : 7725,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "RB1",
        "selected" : false,
        "canonicalName" : "RB1",
        "name" : "RB1",
        "nodeType" : "gene",
        "shared_name" : "RB1"
      },
      "position" : {
        "x" : 1284.0,
        "y" : 2518.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7724",
        "mut" : 0.0,
        "SUID" : 7724,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "RTK class",
        "selected" : false,
        "canonicalName" : "RTK class",
        "name" : "RTK class",
        "nodeType" : "class",
        "shared_name" : "RTK class"
      },
      "position" : {
        "x" : 1829.0,
        "y" : -771.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7723",
        "mut" : 0.0,
        "SUID" : 7723,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "SPRY2",
        "selected" : false,
        "canonicalName" : "SPRY2",
        "name" : "SPRY2",
        "nodeType" : "gene",
        "shared_name" : "SPRY2"
      },
      "position" : {
        "x" : 1284.0,
        "y" : 361.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7722",
        "mut" : 0.0,
        "SUID" : 7722,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "SRC",
        "selected" : false,
        "canonicalName" : "SRC",
        "name" : "SRC",
        "nodeType" : "gene",
        "shared_name" : "SRC"
      },
      "position" : {
        "x" : 528.0,
        "y" : -771.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7721",
        "mut" : 0.0,
        "SUID" : 7721,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "SREBP",
        "selected" : false,
        "canonicalName" : "SREBP",
        "name" : "SREBP",
        "nodeType" : "family",
        "shared_name" : "SREBP"
      },
      "position" : {
        "x" : 320.0,
        "y" : 1246.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7720",
        "mut" : 0.0,
        "SUID" : 7720,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "SREBP1a",
        "selected" : false,
        "canonicalName" : "SREBP1a",
        "name" : "SREBP1a",
        "nodeType" : "gene",
        "shared_name" : "SREBP1a"
      },
      "position" : {
        "x" : 586.0,
        "y" : 1198.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7719",
        "mut" : 0.0,
        "SUID" : 7719,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "SREBP2",
        "selected" : false,
        "canonicalName" : "SREBP2",
        "name" : "SREBP2",
        "nodeType" : "protein",
        "shared_name" : "SREBP2"
      },
      "position" : {
        "x" : 768.0,
        "y" : 1430.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7718",
        "mut" : 0.0,
        "SUID" : 7718,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "STAT3",
        "selected" : false,
        "canonicalName" : "STAT3",
        "name" : "STAT3",
        "nodeType" : "gene",
        "shared_name" : "STAT3"
      },
      "position" : {
        "x" : -3678.0,
        "y" : -1592.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7717",
        "mut" : 0.0,
        "SUID" : 7717,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "STAT5",
        "selected" : false,
        "canonicalName" : "STAT5",
        "name" : "STAT5",
        "nodeType" : "gene",
        "shared_name" : "STAT5"
      },
      "position" : {
        "x" : -2070.0,
        "y" : -1441.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7716",
        "mut" : 0.0,
        "SUID" : 7716,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "TET2",
        "selected" : false,
        "canonicalName" : "TET2",
        "name" : "TET2",
        "nodeType" : "gene",
        "shared_name" : "TET2"
      },
      "position" : {
        "x" : 2260.0,
        "y" : 2766.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7715",
        "mut" : 0.0,
        "SUID" : 7715,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "TP53",
        "selected" : false,
        "canonicalName" : "TP53",
        "name" : "TP53",
        "nodeType" : "gene",
        "shared_name" : "TP53"
      },
      "position" : {
        "x" : -1754.0,
        "y" : 928.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7714",
        "mut" : 0.0,
        "SUID" : 7714,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "TSC complex",
        "selected" : false,
        "canonicalName" : "TSC complex",
        "name" : "TSC complex",
        "nodeType" : "complex",
        "shared_name" : "TSC complex"
      },
      "position" : {
        "x" : -575.0,
        "y" : 1328.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7713",
        "mut" : 0.0,
        "SUID" : 7713,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "ZEB1",
        "selected" : false,
        "canonicalName" : "ZEB1",
        "name" : "ZEB1",
        "nodeType" : "protein",
        "shared_name" : "ZEB1"
      },
      "position" : {
        "x" : -1244.0,
        "y" : 1868.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7712",
        "mut" : 0.0,
        "SUID" : 7712,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "Lipid metabolism",
        "selected" : false,
        "canonicalName" : "Lipid metabolism",
        "name" : "Lipid metabolism",
        "nodeType" : "process",
        "shared_name" : "Lipid metabolism"
      },
      "position" : {
        "x" : -12.0,
        "y" : 3309.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7711",
        "mut" : 0.0,
        "SUID" : 7711,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "CELL SURVIVAL",
        "selected" : false,
        "canonicalName" : "CELL SURVIVAL",
        "name" : "CELL SURVIVAL",
        "nodeType" : "process",
        "shared_name" : "CELL SURVIVAL"
      },
      "position" : {
        "x" : -347.0,
        "y" : 3309.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7710",
        "mut" : 0.0,
        "SUID" : 7710,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "AKT2",
        "selected" : false,
        "canonicalName" : "AKT2",
        "name" : "AKT2",
        "nodeType" : "gene",
        "shared_name" : "AKT2"
      },
      "position" : {
        "x" : -797.0,
        "y" : 361.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7709",
        "mut" : 0.0,
        "SUID" : 7709,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "AKT3",
        "selected" : false,
        "canonicalName" : "AKT3",
        "name" : "AKT3",
        "nodeType" : "gene",
        "shared_name" : "AKT3"
      },
      "position" : {
        "x" : -651.0,
        "y" : 171.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7708",
        "mut" : 0.0,
        "SUID" : 7708,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "BRCA1",
        "selected" : false,
        "canonicalName" : "BRCA1",
        "name" : "BRCA1",
        "nodeType" : "gene",
        "shared_name" : "BRCA1"
      },
      "position" : {
        "x" : -2522.0,
        "y" : 2156.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7707",
        "mut" : 0.0,
        "SUID" : 7707,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "DNA Damage Repair",
        "selected" : false,
        "canonicalName" : "DNA Damage Repair",
        "name" : "DNA Damage Repair",
        "nodeType" : "process",
        "shared_name" : "DNA Damage Repair"
      },
      "position" : {
        "x" : -2504.0,
        "y" : 3309.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7706",
        "mut" : 0.0,
        "SUID" : 7706,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "MSH6",
        "selected" : false,
        "canonicalName" : "MSH6",
        "name" : "MSH6",
        "nodeType" : "gene",
        "shared_name" : "MSH6"
      },
      "position" : {
        "x" : -2840.0,
        "y" : 2156.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7705",
        "mut" : 0.0,
        "SUID" : 7705,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "CELL MIGRATION",
        "selected" : false,
        "canonicalName" : "CELL MIGRATION",
        "name" : "CELL MIGRATION",
        "nodeType" : "process",
        "shared_name" : "CELL MIGRATION"
      },
      "position" : {
        "x" : -3434.0,
        "y" : 3309.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7704",
        "mut" : 0.0,
        "SUID" : 7704,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "BRCA2",
        "selected" : false,
        "canonicalName" : "BRCA2",
        "name" : "BRCA2",
        "nodeType" : "gene",
        "shared_name" : "BRCA2"
      },
      "position" : {
        "x" : -2221.0,
        "y" : 2156.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7703",
        "mut" : 0.0,
        "SUID" : 7703,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "CCND1",
        "selected" : false,
        "canonicalName" : "CCND1",
        "name" : "CCND1",
        "nodeType" : "gene",
        "shared_name" : "CCND1"
      },
      "position" : {
        "x" : 1284.0,
        "y" : 2191.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7702",
        "mut" : 0.0,
        "SUID" : 7702,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "CDK4",
        "selected" : false,
        "canonicalName" : "CDK4",
        "name" : "CDK4",
        "nodeType" : "gene",
        "shared_name" : "CDK4"
      },
      "position" : {
        "x" : 739.0,
        "y" : 1830.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7701",
        "mut" : 0.0,
        "SUID" : 7701,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "CDK6",
        "selected" : false,
        "canonicalName" : "CDK6",
        "name" : "CDK6",
        "nodeType" : "gene",
        "shared_name" : "CDK6"
      },
      "position" : {
        "x" : 2181.0,
        "y" : 2063.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7700",
        "mut" : 0.0,
        "SUID" : 7700,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "CDK2",
        "selected" : false,
        "canonicalName" : "CDK2",
        "name" : "CDK2",
        "nodeType" : "gene",
        "shared_name" : "CDK2"
      },
      "position" : {
        "x" : 269.0,
        "y" : 1998.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7699",
        "mut" : 0.0,
        "SUID" : 7699,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "CCNE1",
        "selected" : false,
        "canonicalName" : "CCNE1",
        "name" : "CCNE1",
        "nodeType" : "gene",
        "shared_name" : "CCNE1"
      },
      "position" : {
        "x" : -60.0,
        "y" : 1991.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7698",
        "mut" : 0.0,
        "SUID" : 7698,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "G1/S PROGRESSION",
        "selected" : false,
        "canonicalName" : "G1/S PROGRESSION",
        "name" : "G1/S PROGRESSION",
        "nodeType" : "process",
        "shared_name" : "G1/S PROGRESSION"
      },
      "position" : {
        "x" : 1284.0,
        "y" : 3309.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7697",
        "mut" : 0.0,
        "SUID" : 7697,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "ETV1/4/5",
        "selected" : false,
        "canonicalName" : "ETV1/4/5",
        "name" : "ETV1/4/5",
        "nodeType" : "gene",
        "shared_name" : "ETV1/4/5"
      },
      "position" : {
        "x" : -3218.0,
        "y" : 2234.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7696",
        "mut" : 0.0,
        "SUID" : 7696,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "EGFR",
        "selected" : false,
        "canonicalName" : "EGFR",
        "name" : "EGFR",
        "nodeType" : "gene",
        "shared_name" : "EGFR"
      },
      "position" : {
        "x" : 2337.0,
        "y" : -1624.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7695",
        "mut" : 0.0,
        "SUID" : 7695,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "ERBB2",
        "selected" : false,
        "canonicalName" : "ERBB2",
        "name" : "ERBB2",
        "nodeType" : "gene",
        "shared_name" : "ERBB2"
      },
      "position" : {
        "x" : 2396.0,
        "y" : -1488.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7694",
        "mut" : 0.0,
        "SUID" : 7694,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "ERBB3",
        "selected" : false,
        "canonicalName" : "ERBB3",
        "name" : "ERBB3",
        "nodeType" : "gene",
        "shared_name" : "ERBB3"
      },
      "position" : {
        "x" : 2387.0,
        "y" : -1359.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7693",
        "mut" : 0.0,
        "SUID" : 7693,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "FGFR2",
        "selected" : false,
        "canonicalName" : "FGFR2",
        "name" : "FGFR2",
        "nodeType" : "gene",
        "shared_name" : "FGFR2"
      },
      "position" : {
        "x" : 2399.0,
        "y" : -779.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7692",
        "mut" : 0.0,
        "SUID" : 7692,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "FGFR1",
        "selected" : false,
        "canonicalName" : "FGFR1",
        "name" : "FGFR1",
        "nodeType" : "gene",
        "shared_name" : "FGFR1"
      },
      "position" : {
        "x" : 2376.0,
        "y" : -565.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7691",
        "mut" : 0.0,
        "SUID" : 7691,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "FOXO1",
        "selected" : false,
        "canonicalName" : "FOXO1",
        "name" : "FOXO1",
        "nodeType" : "gene",
        "shared_name" : "FOXO1"
      },
      "position" : {
        "x" : 216.0,
        "y" : 122.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7690",
        "mut" : 0.0,
        "SUID" : 7690,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "FOXO3",
        "selected" : false,
        "canonicalName" : "FOXO3",
        "name" : "FOXO3",
        "nodeType" : "gene",
        "shared_name" : "FOXO3"
      },
      "position" : {
        "x" : 439.0,
        "y" : 122.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7689",
        "mut" : 0.0,
        "SUID" : 7689,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "FOXO4",
        "selected" : false,
        "canonicalName" : "FOXO4",
        "name" : "FOXO4",
        "nodeType" : "gene",
        "shared_name" : "FOXO4"
      },
      "position" : {
        "x" : 671.0,
        "y" : 122.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7688",
        "mut" : 0.0,
        "SUID" : 7688,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "Glycolysis",
        "selected" : false,
        "canonicalName" : "Glycolysis",
        "name" : "Glycolysis",
        "nodeType" : "process",
        "shared_name" : "Glycolysis"
      },
      "position" : {
        "x" : 3528.0,
        "y" : 3309.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7687",
        "mut" : 0.0,
        "SUID" : 7687,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "MET",
        "selected" : false,
        "canonicalName" : "MET",
        "name" : "MET",
        "nodeType" : "gene",
        "shared_name" : "MET"
      },
      "position" : {
        "x" : 1393.0,
        "y" : -1832.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7686",
        "mut" : 0.0,
        "SUID" : 7686,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "APOPTOSIS",
        "selected" : false,
        "canonicalName" : "APOPTOSIS",
        "name" : "APOPTOSIS",
        "nodeType" : "process",
        "shared_name" : "APOPTOSIS"
      },
      "position" : {
        "x" : -1921.0,
        "y" : 3309.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7685",
        "mut" : 0.0,
        "SUID" : 7685,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "Histone methylation",
        "selected" : false,
        "canonicalName" : "Histone methylation",
        "name" : "Histone methylation",
        "nodeType" : "process",
        "shared_name" : "Histone methylation"
      },
      "position" : {
        "x" : -684.0,
        "y" : 3309.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7684",
        "mut" : 0.0,
        "SUID" : 7684,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "CDKN2A:p16",
        "selected" : false,
        "canonicalName" : "CDKN2A:p16",
        "name" : "CDKN2A:p16",
        "nodeType" : "gene",
        "shared_name" : "CDKN2A:p16"
      },
      "position" : {
        "x" : 1284.0,
        "y" : 1427.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7683",
        "mut" : 0.0,
        "SUID" : 7683,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "CDKN2C",
        "selected" : false,
        "canonicalName" : "CDKN2C",
        "name" : "CDKN2C",
        "nodeType" : "gene",
        "shared_name" : "CDKN2C"
      },
      "position" : {
        "x" : 1141.0,
        "y" : 1575.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7682",
        "mut" : 0.0,
        "SUID" : 7682,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "CDKN2B",
        "selected" : false,
        "canonicalName" : "CDKN2B",
        "name" : "CDKN2B",
        "nodeType" : "gene",
        "shared_name" : "CDKN2B"
      },
      "position" : {
        "x" : 1463.0,
        "y" : 1570.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7681",
        "mut" : 0.0,
        "SUID" : 7681,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "NR1H1",
        "selected" : false,
        "canonicalName" : "NR1H1",
        "name" : "NR1H1",
        "nodeType" : "gene",
        "shared_name" : "NR1H1"
      },
      "position" : {
        "x" : -85.0,
        "y" : 2698.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7680",
        "mut" : 0.0,
        "SUID" : 7680,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "NR1H2",
        "selected" : false,
        "canonicalName" : "NR1H2",
        "name" : "NR1H2",
        "nodeType" : "gene",
        "shared_name" : "NR1H2"
      },
      "position" : {
        "x" : 101.0,
        "y" : 2629.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7679",
        "mut" : 0.0,
        "SUID" : 7679,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "NR1H3",
        "selected" : false,
        "canonicalName" : "NR1H3",
        "name" : "NR1H3",
        "nodeType" : "gene",
        "shared_name" : "NR1H3"
      },
      "position" : {
        "x" : 297.0,
        "y" : 2673.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7678",
        "mut" : 0.0,
        "SUID" : 7678,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "Therapy Resistance",
        "selected" : false,
        "canonicalName" : "Therapy Resistance",
        "name" : "Therapy Resistance",
        "nodeType" : "process",
        "shared_name" : "Therapy Resistance"
      },
      "position" : {
        "x" : -1024.0,
        "y" : 3309.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7677",
        "mut" : 0.0,
        "SUID" : 7677,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "OLIG2",
        "selected" : false,
        "canonicalName" : "OLIG2",
        "name" : "OLIG2",
        "nodeType" : "gene",
        "shared_name" : "OLIG2"
      },
      "position" : {
        "x" : -1520.0,
        "y" : 2006.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7676",
        "mut" : 0.0,
        "SUID" : 7676,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "Chromatin remodeling",
        "selected" : false,
        "canonicalName" : "Chromatin remodeling",
        "name" : "Chromatin remodeling",
        "nodeType" : "process",
        "shared_name" : "Chromatin remodeling"
      },
      "position" : {
        "x" : 2748.0,
        "y" : 3309.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7675",
        "mut" : 0.0,
        "SUID" : 7675,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "mTOR",
        "selected" : false,
        "canonicalName" : "mTOR",
        "name" : "mTOR",
        "nodeType" : "protein",
        "shared_name" : "mTOR"
      },
      "position" : {
        "x" : -859.0,
        "y" : 1795.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7674",
        "mut" : 0.0,
        "SUID" : 7674,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "DEPTOR",
        "selected" : false,
        "canonicalName" : "DEPTOR",
        "name" : "DEPTOR",
        "nodeType" : "protein",
        "shared_name" : "DEPTOR"
      },
      "position" : {
        "x" : -441.0,
        "y" : 1786.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7673",
        "mut" : 0.0,
        "SUID" : 7673,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "PRAS40",
        "selected" : false,
        "canonicalName" : "PRAS40",
        "name" : "PRAS40",
        "nodeType" : "protein",
        "shared_name" : "PRAS40"
      },
      "position" : {
        "x" : -647.0,
        "y" : 1845.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7672",
        "mut" : 0.0,
        "SUID" : 7672,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "MLST8",
        "selected" : false,
        "canonicalName" : "MLST8",
        "name" : "MLST8",
        "nodeType" : "protein",
        "shared_name" : "MLST8"
      },
      "position" : {
        "x" : -916.0,
        "y" : 1601.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7671",
        "mut" : 0.0,
        "SUID" : 7671,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "NFKB1",
        "selected" : false,
        "canonicalName" : "NFKB1",
        "name" : "NFKB1",
        "nodeType" : "gene",
        "shared_name" : "NFKB1"
      },
      "position" : {
        "x" : 3636.0,
        "y" : 1823.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7670",
        "mut" : 0.0,
        "SUID" : 7670,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "RELA",
        "selected" : false,
        "canonicalName" : "RELA",
        "name" : "RELA",
        "nodeType" : "gene",
        "shared_name" : "RELA"
      },
      "position" : {
        "x" : 3417.0,
        "y" : 1823.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7669",
        "mut" : 0.0,
        "SUID" : 7669,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "PDGFRB",
        "selected" : false,
        "canonicalName" : "PDGFRB",
        "name" : "PDGFRB",
        "nodeType" : "gene",
        "shared_name" : "PDGFRB"
      },
      "position" : {
        "x" : 2154.0,
        "y" : -1201.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7668",
        "mut" : 0.0,
        "SUID" : 7668,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "PDGFRA",
        "selected" : false,
        "canonicalName" : "PDGFRA",
        "name" : "PDGFRA",
        "nodeType" : "gene",
        "shared_name" : "PDGFRA"
      },
      "position" : {
        "x" : 2343.0,
        "y" : -1026.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7667",
        "mut" : 0.0,
        "SUID" : 7667,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "PIK3R1",
        "selected" : false,
        "canonicalName" : "PIK3R1",
        "name" : "PIK3R1",
        "nodeType" : "gene",
        "shared_name" : "PIK3R1"
      },
      "position" : {
        "x" : -165.0,
        "y" : -1219.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7666",
        "mut" : 0.0,
        "SUID" : 7666,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "PIK3R2",
        "selected" : false,
        "canonicalName" : "PIK3R2",
        "name" : "PIK3R2",
        "nodeType" : "gene",
        "shared_name" : "PIK3R2"
      },
      "position" : {
        "x" : -13.0,
        "y" : -1104.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7665",
        "mut" : 0.0,
        "SUID" : 7665,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "PIK3CA",
        "selected" : false,
        "canonicalName" : "PIK3CA",
        "name" : "PIK3CA",
        "nodeType" : "gene",
        "shared_name" : "PIK3CA"
      },
      "position" : {
        "x" : -347.0,
        "y" : -1289.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7664",
        "mut" : 0.0,
        "SUID" : 7664,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "PIK3CB",
        "selected" : false,
        "canonicalName" : "PIK3CB",
        "name" : "PIK3CB",
        "nodeType" : "gene",
        "shared_name" : "PIK3CB"
      },
      "position" : {
        "x" : -703.0,
        "y" : -1085.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7663",
        "mut" : 0.0,
        "SUID" : 7663,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "PIK3CD",
        "selected" : false,
        "canonicalName" : "PIK3CD",
        "name" : "PIK3CD",
        "nodeType" : "gene",
        "shared_name" : "PIK3CD"
      },
      "position" : {
        "x" : -545.0,
        "y" : -1206.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7662",
        "mut" : 0.0,
        "SUID" : 7662,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "PIK3CG",
        "selected" : false,
        "canonicalName" : "PIK3CG",
        "name" : "PIK3CG",
        "nodeType" : "gene",
        "shared_name" : "PIK3CG"
      },
      "position" : {
        "x" : 723.0,
        "y" : -409.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7661",
        "mut" : 0.0,
        "SUID" : 7661,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "PIK3C2A",
        "selected" : false,
        "canonicalName" : "PIK3C2A",
        "name" : "PIK3C2A",
        "nodeType" : "gene",
        "shared_name" : "PIK3C2A"
      },
      "position" : {
        "x" : -3356.0,
        "y" : -515.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7660",
        "mut" : 0.0,
        "SUID" : 7660,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "PIK3C2G",
        "selected" : false,
        "canonicalName" : "PIK3C2G",
        "name" : "PIK3C2G",
        "nodeType" : "gene",
        "shared_name" : "PIK3C2G"
      },
      "position" : {
        "x" : -3247.0,
        "y" : -346.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7659",
        "mut" : 0.0,
        "SUID" : 7659,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "PRKCA",
        "selected" : false,
        "canonicalName" : "PRKCA",
        "name" : "PRKCA",
        "nodeType" : "gene",
        "shared_name" : "PRKCA"
      },
      "position" : {
        "x" : 3122.0,
        "y" : 1229.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7658",
        "mut" : 0.0,
        "SUID" : 7658,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "PRKCB1",
        "selected" : false,
        "canonicalName" : "PRKCB1",
        "name" : "PRKCB1",
        "nodeType" : "gene",
        "shared_name" : "PRKCB1"
      },
      "position" : {
        "x" : 2914.0,
        "y" : 286.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7657",
        "mut" : 0.0,
        "SUID" : 7657,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "PRKCD",
        "selected" : false,
        "canonicalName" : "PRKCD",
        "name" : "PRKCD",
        "nodeType" : "gene",
        "shared_name" : "PRKCD"
      },
      "position" : {
        "x" : 3056.0,
        "y" : 1399.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7656",
        "mut" : 0.0,
        "SUID" : 7656,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "PRKCG",
        "selected" : false,
        "canonicalName" : "PRKCG",
        "name" : "PRKCG",
        "nodeType" : "gene",
        "shared_name" : "PRKCG"
      },
      "position" : {
        "x" : 3100.0,
        "y" : 495.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7655",
        "mut" : 0.0,
        "SUID" : 7655,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "PRKCH",
        "selected" : false,
        "canonicalName" : "PRKCH",
        "name" : "PRKCH",
        "nodeType" : "gene",
        "shared_name" : "PRKCH"
      },
      "position" : {
        "x" : 2907.0,
        "y" : 1515.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7654",
        "mut" : 0.0,
        "SUID" : 7654,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "PRKCI",
        "selected" : false,
        "canonicalName" : "PRKCI",
        "name" : "PRKCI",
        "nodeType" : "gene",
        "shared_name" : "PRKCI"
      },
      "position" : {
        "x" : 3128.0,
        "y" : 670.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7653",
        "mut" : 0.0,
        "SUID" : 7653,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "PRKCQ",
        "selected" : false,
        "canonicalName" : "PRKCQ",
        "name" : "PRKCQ",
        "nodeType" : "gene",
        "shared_name" : "PRKCQ"
      },
      "position" : {
        "x" : 3128.0,
        "y" : 1064.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7652",
        "mut" : 0.0,
        "SUID" : 7652,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "PRKCZ",
        "selected" : false,
        "canonicalName" : "PRKCZ",
        "name" : "PRKCZ",
        "nodeType" : "gene",
        "shared_name" : "PRKCZ"
      },
      "position" : {
        "x" : 3128.0,
        "y" : 872.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7651",
        "mut" : 0.0,
        "SUID" : 7651,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "RAF1",
        "selected" : false,
        "canonicalName" : "RAF1",
        "name" : "RAF1",
        "nodeType" : "gene",
        "shared_name" : "RAF1"
      },
      "position" : {
        "x" : 1968.0,
        "y" : 1314.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7650",
        "mut" : 0.0,
        "SUID" : 7650,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "ARAF",
        "selected" : false,
        "canonicalName" : "ARAF",
        "name" : "ARAF",
        "nodeType" : "gene",
        "shared_name" : "ARAF"
      },
      "position" : {
        "x" : 1580.0,
        "y" : 1314.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7649",
        "mut" : 0.0,
        "SUID" : 7649,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "BRAF",
        "selected" : false,
        "canonicalName" : "BRAF",
        "name" : "BRAF",
        "nodeType" : "gene",
        "shared_name" : "BRAF"
      },
      "position" : {
        "x" : 1776.0,
        "y" : 1314.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7648",
        "mut" : 0.0,
        "SUID" : 7648,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "HRAS",
        "selected" : false,
        "canonicalName" : "HRAS",
        "name" : "HRAS",
        "nodeType" : "gene",
        "shared_name" : "HRAS"
      },
      "position" : {
        "x" : 2062.0,
        "y" : 365.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7647",
        "mut" : 0.0,
        "SUID" : 7647,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "KRAS",
        "selected" : false,
        "canonicalName" : "KRAS",
        "name" : "KRAS",
        "nodeType" : "gene",
        "shared_name" : "KRAS"
      },
      "position" : {
        "x" : 2102.0,
        "y" : 193.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7646",
        "mut" : 0.0,
        "SUID" : 7646,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "NRAS",
        "selected" : false,
        "canonicalName" : "NRAS",
        "name" : "NRAS",
        "nodeType" : "gene",
        "shared_name" : "NRAS"
      },
      "position" : {
        "x" : 2061.0,
        "y" : 8.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7645",
        "mut" : 0.0,
        "SUID" : 7645,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "DNA methylation",
        "selected" : false,
        "canonicalName" : "DNA methylation",
        "name" : "DNA methylation",
        "nodeType" : "process",
        "shared_name" : "DNA methylation"
      },
      "position" : {
        "x" : 2265.0,
        "y" : 3309.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7644",
        "mut" : 0.0,
        "SUID" : 7644,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "G2/M ARREST",
        "selected" : false,
        "canonicalName" : "G2/M ARREST",
        "name" : "G2/M ARREST",
        "nodeType" : "process",
        "shared_name" : "G2/M ARREST"
      },
      "position" : {
        "x" : -1576.0,
        "y" : 3309.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7643",
        "mut" : 0.0,
        "SUID" : 7643,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "TSC1",
        "selected" : false,
        "canonicalName" : "TSC1",
        "name" : "TSC1",
        "nodeType" : "gene",
        "shared_name" : "TSC1"
      },
      "position" : {
        "x" : -435.0,
        "y" : 1126.0
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7642",
        "mut" : 0.0,
        "SUID" : 7642,
        "cnv" : "0",
        "score" : 0.0,
        "label" : "TSC2",
        "selected" : false,
        "canonicalName" : "TSC2",
        "name" : "TSC2",
        "nodeType" : "gene",
        "shared_name" : "TSC2"
      },
      "position" : {
        "x" : -688.0,
        "y" : 1126.0
      },
      "selected" : false
    } ],
    "edges" : [ {
      "data" : {
        "id" : "7998",
        "source" : "7797",
        "target" : "7712",
        "pmid" : "24217114",
        "selected" : false,
        "canonicalName" : "ABCA1 (inhibits.process) Lipid metabolism",
        "edgeType" : "inhibits.process",
        "SUID" : 7998,
        "name" : "ABCA1 (inhibits.process) Lipid metabolism",
        "interaction" : "inhibits.process",
        "shared_interaction" : "inhibits.process",
        "shared_name" : "ABCA1 (inhibits.process) Lipid metabolism",
        "edgeDetail" : "ABCA1 (inhibits.process) Lipid metabolism\n\n1. Cancers (Basel). 2013 Nov 8;5(4):1469-84. doi: 10.3390/cancers5041469.\n\nTumor metabolism of malignant gliomas.\n\nRu P(1), Williams TM, Chakravarti A, Guo D.\n\nAuthor information: \n(1)Department of Radiation Oncology, Ohio State University Comprehensive Cancer\nCenter & Arthur G James Cancer Hospital, Columbus, OH 43012, USA.\ndeliang.guo@osumc.edu.\n\nConstitutively activated oncogenic signaling via genetic mutations such as in the\nEGFR/PI3K/Akt and Ras/RAF/MEK pathways has been recognized as a major driver for \ntumorigenesis in most cancers. Recent insights into tumor metabolism have further\nrevealed that oncogenic signaling pathways directly promote metabolic\nreprogramming to upregulate biosynthesis of lipids, carbohydrates, protein, DNA\nand RNA, leading to enhanced growth of human tumors. Therefore, targeting cell\nmetabolism has become a novel direction for drug development in oncology. In\nmalignant gliomas, metabolism pathways of glucose, glutamine and lipid are\nsignificantly reprogrammed. Moreover, molecular mechanisms causing these\nmetabolic changes are just starting to be unraveled. In this review, we will\nsummarize recent studies revealing critical gene alterations that lead to\nmetabolic changes in malignant gliomas, and also discuss promising therapeutic\nstrategies via targeting the key players in metabolic regulation.\n\nPMCID: PMC3875949\nPMID: 24217114  [PubMed]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7997",
        "source" : "7796",
        "target" : "7711",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "AKT family (activates.process) CELL SURVIVAL",
        "edgeType" : "activates.process",
        "SUID" : 7997,
        "name" : "AKT family (activates.process) CELL SURVIVAL",
        "interaction" : "activates.process",
        "shared_interaction" : "activates.process",
        "shared_name" : "AKT family (activates.process) CELL SURVIVAL",
        "edgeDetail" : "AKT family (activates.process) CELL SURVIVAL\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7996",
        "source" : "7796",
        "target" : "7714",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "AKT family (inhibits) TSC complex",
        "edgeType" : "inhibits",
        "SUID" : 7996,
        "name" : "AKT family (inhibits) TSC complex",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "AKT family (inhibits) TSC complex",
        "edgeDetail" : "AKT family (inhibits) TSC complex\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7995",
        "source" : "7796",
        "target" : "7772",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "AKT family (inhibits) FOXO family",
        "edgeType" : "inhibits",
        "SUID" : 7995,
        "name" : "AKT family (inhibits) FOXO family",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "AKT family (inhibits) FOXO family",
        "edgeDetail" : "AKT family (inhibits) FOXO family\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7994",
        "source" : "7796",
        "target" : "7784",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "AKT family (inhibits) CDKN1B",
        "edgeType" : "inhibits",
        "SUID" : 7994,
        "name" : "AKT family (inhibits) CDKN1B",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "AKT family (inhibits) CDKN1B",
        "edgeDetail" : "AKT family (inhibits) CDKN1B\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7993",
        "source" : "7796",
        "target" : "7755",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "AKT family (activates) MDM2",
        "edgeType" : "activates",
        "SUID" : 7993,
        "name" : "AKT family (activates) MDM2",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "AKT family (activates) MDM2",
        "edgeDetail" : "AKT family (activates) MDM2\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7992",
        "source" : "7796",
        "target" : "7710",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "AKT family (family.includes) AKT2",
        "edgeType" : "family.includes",
        "SUID" : 7992,
        "name" : "AKT family (family.includes) AKT2",
        "interaction" : "family.includes",
        "shared_interaction" : "family.includes",
        "shared_name" : "AKT family (family.includes) AKT2",
        "edgeDetail" : "AKT family (family.includes) AKT2\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7991",
        "source" : "7796",
        "target" : "7709",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "AKT family (family.includes) AKT3",
        "edgeType" : "family.includes",
        "SUID" : 7991,
        "name" : "AKT family (family.includes) AKT3",
        "interaction" : "family.includes",
        "shared_interaction" : "family.includes",
        "shared_name" : "AKT family (family.includes) AKT3",
        "edgeDetail" : "AKT family (family.includes) AKT3\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7990",
        "source" : "7796",
        "target" : "7795",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "AKT family (family.includes) AKT1",
        "edgeType" : "family.includes",
        "SUID" : 7990,
        "name" : "AKT family (family.includes) AKT1",
        "interaction" : "family.includes",
        "shared_interaction" : "family.includes",
        "shared_name" : "AKT family (family.includes) AKT1",
        "edgeDetail" : "AKT family (family.includes) AKT1\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7989",
        "source" : "7796",
        "target" : "7720",
        "pmid" : "24217114",
        "selected" : false,
        "canonicalName" : "AKT family (activates) SREBP1a",
        "edgeType" : "activates",
        "SUID" : 7989,
        "name" : "AKT family (activates) SREBP1a",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "AKT family (activates) SREBP1a",
        "edgeDetail" : "AKT family (activates) SREBP1a\n\n1. Cancers (Basel). 2013 Nov 8;5(4):1469-84. doi: 10.3390/cancers5041469.\n\nTumor metabolism of malignant gliomas.\n\nRu P(1), Williams TM, Chakravarti A, Guo D.\n\nAuthor information: \n(1)Department of Radiation Oncology, Ohio State University Comprehensive Cancer\nCenter & Arthur G James Cancer Hospital, Columbus, OH 43012, USA.\ndeliang.guo@osumc.edu.\n\nConstitutively activated oncogenic signaling via genetic mutations such as in the\nEGFR/PI3K/Akt and Ras/RAF/MEK pathways has been recognized as a major driver for \ntumorigenesis in most cancers. Recent insights into tumor metabolism have further\nrevealed that oncogenic signaling pathways directly promote metabolic\nreprogramming to upregulate biosynthesis of lipids, carbohydrates, protein, DNA\nand RNA, leading to enhanced growth of human tumors. Therefore, targeting cell\nmetabolism has become a novel direction for drug development in oncology. In\nmalignant gliomas, metabolism pathways of glucose, glutamine and lipid are\nsignificantly reprogrammed. Moreover, molecular mechanisms causing these\nmetabolic changes are just starting to be unraveled. In this review, we will\nsummarize recent studies revealing critical gene alterations that lead to\nmetabolic changes in malignant gliomas, and also discuss promising therapeutic\nstrategies via targeting the key players in metabolic regulation.\n\nPMCID: PMC3875949\nPMID: 24217114  [PubMed]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7988",
        "source" : "7795",
        "target" : "7767",
        "pmid" : "10691731",
        "selected" : false,
        "canonicalName" : "AKT1 (activates) HIF1A",
        "edgeType" : "activates",
        "SUID" : 7988,
        "name" : "AKT1 (activates) HIF1A",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "AKT1 (activates) HIF1A",
        "edgeDetail" : "AKT1 (activates) HIF1A\n\n1. Genes Dev. 2000 Feb 15;14(4):391-6.\n\nLoss of PTEN facilitates HIF-1-mediated gene expression.\n\nZundel W, Schindler C, Haas-Kogan D, Koong A, Kaper F, Chen E, Gottschalk AR,\nRyan HE, Johnson RS, Jefferson AB, Stokoe D, Giaccia AJ.\n\nMayer Cancer Biology Research Laboratory, Department of Radiation Oncology,\nStanford University, Stanford, California 94305-5468 USA.\n\nIn glioblastoma-derived cell lines, PTEN does not significantly alter apoptotic\nsensitivity or cause complete inhibition of DNA synthesis. However, in these cell\nlines PTEN regulates hypoxia- and IGF-1-induced angiogenic gene expression by\nregulating Akt activation of HIF-1 activity. Restoration of wild-type PTEN to\nglioblastoma cell lines lacking functional PTEN ablates hypoxia and IGF-1\ninduction of HIF-1-regulated genes. In addition, Akt activation leads to\nHIF-1alpha stabilization, whereas PTEN attenuates hypoxia-mediated HIF-1alpha\nstabilization. We propose that loss of PTEN during malignant progression\ncontributes to tumor expansion through the deregulation of Akt activity and\nHIF-1-regulated gene expression.\n\nPMCID: PMC316386\nPMID: 10691731  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7987",
        "source" : "7795",
        "target" : "7747",
        "pmid" : "23625692",
        "selected" : false,
        "canonicalName" : "AKT1 (activates) mTORC1",
        "edgeType" : "activates",
        "SUID" : 7987,
        "name" : "AKT1 (activates) mTORC1",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "AKT1 (activates) mTORC1",
        "edgeDetail" : "AKT1 (activates) mTORC1\n\n1. Tumour Biol. 2013 Aug;34(4):1991-2002. doi: 10.1007/s13277-013-0800-5. Epub 2013 \nApr 30.\n\nTargeting the PI3K/AKT/mTOR signaling pathway in glioblastoma: novel therapeutic \nagents and advances in understanding.\n\nSami A, Karsy M.\n\nDepartment of Radiology and Imaging Sciences, Emory University School of\nMedicine, Atlanta, GA 30329, USA.\n\nGlioblastoma multiforme (GBM) is a grade IV astrocytoma with a median survival of\n12 months despite current multi-modal treatment options. GBM is distinguished\nclinicopathologically into primary and secondary subtypes. Mutations of\nphosphatase and tensin homolog, and subsequent upregulation of the downstream\nprotein kinase B/mammalian target of rapamycin (mTOR) signaling pathway, are\ncommonly seen in primary GBM and less predominantly in secondary GBM. While\ninvestigations into targeted treatments of mTOR have been attempted, feedback\nregulation within the mTOR signaling pathway may account for therapeutic\nresistance. Currently, rapamycin analogs, dual-targeted mTOR complex 1 and 2\nagents as well as dual mTOR and phosphatidylinositol-3 kinase-targeted agents are\nbeing investigated experimentally and in clinical trials. This review will\ndiscuss the experimental potential of these agents in the treatment of GBM and\ntheir current stage in the GBM drug pipeline. Knowledge obtained from the\napplication of these agents can help in understanding the pathogenesis of GBM as \nwell as delineating subsequent treatment strategies.\n\nPMID: 23625692  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7986",
        "source" : "7794",
        "target" : "7715",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "ATM (activates) TP53",
        "edgeType" : "activates",
        "SUID" : 7986,
        "name" : "ATM (activates) TP53",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "ATM (activates) TP53",
        "edgeDetail" : "ATM (activates) TP53\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7985",
        "source" : "7794",
        "target" : "7708",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "ATM (activates) BRCA1",
        "edgeType" : "activates",
        "SUID" : 7985,
        "name" : "ATM (activates) BRCA1",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "ATM (activates) BRCA1",
        "edgeDetail" : "ATM (activates) BRCA1\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7984",
        "source" : "7793",
        "target" : "7707",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "BASC complex (activates.process) DNA Damage Repair",
        "edgeType" : "activates.process",
        "SUID" : 7984,
        "name" : "BASC complex (activates.process) DNA Damage Repair",
        "interaction" : "activates.process",
        "shared_interaction" : "activates.process",
        "shared_name" : "BASC complex (activates.process) DNA Damage Repair",
        "edgeDetail" : "BASC complex (activates.process) DNA Damage Repair\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7983",
        "source" : "7793",
        "target" : "7708",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "BASC complex (contains) BRCA1",
        "edgeType" : "contains",
        "SUID" : 7983,
        "name" : "BASC complex (contains) BRCA1",
        "interaction" : "contains",
        "shared_interaction" : "contains",
        "shared_name" : "BASC complex (contains) BRCA1",
        "edgeDetail" : "BASC complex (contains) BRCA1\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7982",
        "source" : "7793",
        "target" : "7706",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "BASC complex (contains) MSH6",
        "edgeType" : "contains",
        "SUID" : 7982,
        "name" : "BASC complex (contains) MSH6",
        "interaction" : "contains",
        "shared_interaction" : "contains",
        "shared_name" : "BASC complex (contains) MSH6",
        "edgeDetail" : "BASC complex (contains) MSH6\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7981",
        "source" : "7792",
        "target" : "7711",
        "pmid" : "2279867",
        "selected" : false,
        "canonicalName" : "BCL2L1 (activates.process) CELL SURVIVAL",
        "edgeType" : "activates.process",
        "SUID" : 7981,
        "name" : "BCL2L1 (activates.process) CELL SURVIVAL",
        "interaction" : "activates.process",
        "shared_interaction" : "activates.process",
        "shared_name" : "BCL2L1 (activates.process) CELL SURVIVAL",
        "edgeDetail" : "BCL2L1 (activates.process) CELL SURVIVAL\n\n1. Int J Neurosci. 1990 Apr;51(3-4):205-6.\n\nModulatory effects of vasoactive intestinal peptide on immune functions of spleen\nand lymphoid cells.\n\nNock IH(1), Sharma JD.\n\nAuthor information: \n(1)School of Environmental Sciences, Jawaharlal Nehru University, New Delhi, India.\n\nPMID: 2279867  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7980",
        "source" : "7792",
        "target" : "7705",
        "pmid" : "2279867",
        "selected" : false,
        "canonicalName" : "BCL2L1 (activates.process) CELL MIGRATION",
        "edgeType" : "activates.process",
        "SUID" : 7980,
        "name" : "BCL2L1 (activates.process) CELL MIGRATION",
        "interaction" : "activates.process",
        "shared_interaction" : "activates.process",
        "shared_name" : "BCL2L1 (activates.process) CELL MIGRATION",
        "edgeDetail" : "BCL2L1 (activates.process) CELL MIGRATION\n\n1. Int J Neurosci. 1990 Apr;51(3-4):205-6.\n\nModulatory effects of vasoactive intestinal peptide on immune functions of spleen\nand lymphoid cells.\n\nNock IH(1), Sharma JD.\n\nAuthor information: \n(1)School of Environmental Sciences, Jawaharlal Nehru University, New Delhi, India.\n\nPMID: 2279867  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7979",
        "source" : "7791",
        "target" : "7707",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "BRCA complex (activates.process) DNA Damage Repair",
        "edgeType" : "activates.process",
        "SUID" : 7979,
        "name" : "BRCA complex (activates.process) DNA Damage Repair",
        "interaction" : "activates.process",
        "shared_interaction" : "activates.process",
        "shared_name" : "BRCA complex (activates.process) DNA Damage Repair",
        "edgeDetail" : "BRCA complex (activates.process) DNA Damage Repair\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7978",
        "source" : "7791",
        "target" : "7704",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "BRCA complex (contains) BRCA2",
        "edgeType" : "contains",
        "SUID" : 7978,
        "name" : "BRCA complex (contains) BRCA2",
        "interaction" : "contains",
        "shared_interaction" : "contains",
        "shared_name" : "BRCA complex (contains) BRCA2",
        "edgeDetail" : "BRCA complex (contains) BRCA2\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7977",
        "source" : "7791",
        "target" : "7708",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "BRCA complex (contains) BRCA1",
        "edgeType" : "contains",
        "SUID" : 7977,
        "name" : "BRCA complex (contains) BRCA1",
        "interaction" : "contains",
        "shared_interaction" : "contains",
        "shared_name" : "BRCA complex (contains) BRCA1",
        "edgeDetail" : "BRCA complex (contains) BRCA1\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7976",
        "source" : "7790",
        "target" : "7724",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "CBL (inhibits) RTK class",
        "edgeType" : "inhibits",
        "SUID" : 7976,
        "name" : "CBL (inhibits) RTK class",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "CBL (inhibits) RTK class",
        "edgeDetail" : "CBL (inhibits) RTK class\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7975",
        "source" : "7789",
        "target" : "7788",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "CCND2 (contains) CCND-CDK4 complex",
        "edgeType" : "contains",
        "SUID" : 7975,
        "name" : "CCND2 (contains) CCND-CDK4 complex",
        "interaction" : "contains",
        "shared_interaction" : "contains",
        "shared_name" : "CCND2 (contains) CCND-CDK4 complex",
        "edgeDetail" : "CCND2 (contains) CCND-CDK4 complex\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7974",
        "source" : "7789",
        "target" : "7787",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "CCND2 (contains) CCND-CDK6 complex",
        "edgeType" : "contains",
        "SUID" : 7974,
        "name" : "CCND2 (contains) CCND-CDK6 complex",
        "interaction" : "contains",
        "shared_interaction" : "contains",
        "shared_name" : "CCND2 (contains) CCND-CDK6 complex",
        "edgeDetail" : "CCND2 (contains) CCND-CDK6 complex\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7973",
        "source" : "7788",
        "target" : "7725",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "CCND-CDK4 complex (inhibits) RB1",
        "edgeType" : "inhibits",
        "SUID" : 7973,
        "name" : "CCND-CDK4 complex (inhibits) RB1",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "CCND-CDK4 complex (inhibits) RB1",
        "edgeDetail" : "CCND-CDK4 complex (inhibits) RB1\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7972",
        "source" : "7788",
        "target" : "7703",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "CCND-CDK4 complex (contains) CCND1",
        "edgeType" : "contains",
        "SUID" : 7972,
        "name" : "CCND-CDK4 complex (contains) CCND1",
        "interaction" : "contains",
        "shared_interaction" : "contains",
        "shared_name" : "CCND-CDK4 complex (contains) CCND1",
        "edgeDetail" : "CCND-CDK4 complex (contains) CCND1\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7971",
        "source" : "7788",
        "target" : "7702",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "CCND-CDK4 complex (contains) CDK4",
        "edgeType" : "contains",
        "SUID" : 7971,
        "name" : "CCND-CDK4 complex (contains) CDK4",
        "interaction" : "contains",
        "shared_interaction" : "contains",
        "shared_name" : "CCND-CDK4 complex (contains) CDK4",
        "edgeDetail" : "CCND-CDK4 complex (contains) CDK4\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7970",
        "source" : "7787",
        "target" : "7725",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "CCND-CDK6 complex (inhibits) RB1",
        "edgeType" : "inhibits",
        "SUID" : 7970,
        "name" : "CCND-CDK6 complex (inhibits) RB1",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "CCND-CDK6 complex (inhibits) RB1",
        "edgeDetail" : "CCND-CDK6 complex (inhibits) RB1\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7969",
        "source" : "7787",
        "target" : "7703",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "CCND-CDK6 complex (contains) CCND1",
        "edgeType" : "contains",
        "SUID" : 7969,
        "name" : "CCND-CDK6 complex (contains) CCND1",
        "interaction" : "contains",
        "shared_interaction" : "contains",
        "shared_name" : "CCND-CDK6 complex (contains) CCND1",
        "edgeDetail" : "CCND-CDK6 complex (contains) CCND1\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7968",
        "source" : "7787",
        "target" : "7701",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "CCND-CDK6 complex (contains) CDK6",
        "edgeType" : "contains",
        "SUID" : 7968,
        "name" : "CCND-CDK6 complex (contains) CDK6",
        "interaction" : "contains",
        "shared_interaction" : "contains",
        "shared_name" : "CCND-CDK6 complex (contains) CDK6",
        "edgeDetail" : "CCND-CDK6 complex (contains) CDK6\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7967",
        "source" : "7786",
        "target" : "7725",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "CCNE-CDK2 complex (inhibits) RB1",
        "edgeType" : "inhibits",
        "SUID" : 7967,
        "name" : "CCNE-CDK2 complex (inhibits) RB1",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "CCNE-CDK2 complex (inhibits) RB1",
        "edgeDetail" : "CCNE-CDK2 complex (inhibits) RB1\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7966",
        "source" : "7786",
        "target" : "7700",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "CCNE-CDK2 complex (contains) CDK2",
        "edgeType" : "contains",
        "SUID" : 7966,
        "name" : "CCNE-CDK2 complex (contains) CDK2",
        "interaction" : "contains",
        "shared_interaction" : "contains",
        "shared_name" : "CCNE-CDK2 complex (contains) CDK2",
        "edgeDetail" : "CCNE-CDK2 complex (contains) CDK2\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7965",
        "source" : "7786",
        "target" : "7699",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "CCNE-CDK2 complex (contains) CCNE1",
        "edgeType" : "contains",
        "SUID" : 7965,
        "name" : "CCNE-CDK2 complex (contains) CCNE1",
        "interaction" : "contains",
        "shared_interaction" : "contains",
        "shared_name" : "CCNE-CDK2 complex (contains) CCNE1",
        "edgeDetail" : "CCNE-CDK2 complex (contains) CCNE1\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7964",
        "source" : "7785",
        "target" : "7698",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "CDKN1A (activates.process) G1/S PROGRESSION",
        "edgeType" : "activates.process",
        "SUID" : 7964,
        "name" : "CDKN1A (activates.process) G1/S PROGRESSION",
        "interaction" : "activates.process",
        "shared_interaction" : "activates.process",
        "shared_name" : "CDKN1A (activates.process) G1/S PROGRESSION",
        "edgeDetail" : "CDKN1A (activates.process) G1/S PROGRESSION\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7963",
        "source" : "7785",
        "target" : "7788",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "CDKN1A (inhibits) CCND-CDK4 complex",
        "edgeType" : "inhibits",
        "SUID" : 7963,
        "name" : "CDKN1A (inhibits) CCND-CDK4 complex",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "CDKN1A (inhibits) CCND-CDK4 complex",
        "edgeDetail" : "CDKN1A (inhibits) CCND-CDK4 complex\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7962",
        "source" : "7785",
        "target" : "7787",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "CDKN1A (inhibits) CCND-CDK6 complex",
        "edgeType" : "inhibits",
        "SUID" : 7962,
        "name" : "CDKN1A (inhibits) CCND-CDK6 complex",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "CDKN1A (inhibits) CCND-CDK6 complex",
        "edgeDetail" : "CDKN1A (inhibits) CCND-CDK6 complex\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7961",
        "source" : "7784",
        "target" : "7786",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "CDKN1B (inhibits) CCNE-CDK2 complex",
        "edgeType" : "inhibits",
        "SUID" : 7961,
        "name" : "CDKN1B (inhibits) CCNE-CDK2 complex",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "CDKN1B (inhibits) CCNE-CDK2 complex",
        "edgeDetail" : "CDKN1B (inhibits) CCNE-CDK2 complex\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7960",
        "source" : "7783",
        "target" : "7755",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "CDKN2A:ARF (inhibits) MDM2",
        "edgeType" : "inhibits",
        "SUID" : 7960,
        "name" : "CDKN2A:ARF (inhibits) MDM2",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "CDKN2A:ARF (inhibits) MDM2",
        "edgeDetail" : "CDKN2A:ARF (inhibits) MDM2\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7959",
        "source" : "7782",
        "target" : "7740",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "CENTG1 (activates) PI3K class1a",
        "edgeType" : "activates",
        "SUID" : 7959,
        "name" : "CENTG1 (activates) PI3K class1a",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "CENTG1 (activates) PI3K class1a",
        "edgeDetail" : "CENTG1 (activates) PI3K class1a\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7958",
        "source" : "7781",
        "target" : "7697",
        "pmid" : "21087211",
        "selected" : false,
        "canonicalName" : "CIC (inhibits.process) ETV1/4/5",
        "edgeType" : "inhibits.process",
        "SUID" : 7958,
        "name" : "CIC (inhibits.process) ETV1/4/5",
        "interaction" : "inhibits.process",
        "shared_interaction" : "inhibits.process",
        "shared_name" : "CIC (inhibits.process) ETV1/4/5",
        "edgeDetail" : "CIC (inhibits.process) ETV1/4/5\n\n1. Biochem J. 2011 Feb 1;433(3):515-25. doi: 10.1042/BJ20101562.\n\nERK/p90(RSK)/14-3-3 signalling has an impact on expression of PEA3 Ets\ntranscription factors via the transcriptional repressor capica.\n\nDissanayake K(1), Toth R, Blakey J, Olsson O, Campbell DG, Prescott AR,\nMacKintosh C.\n\nAuthor information: \n(1)University of Dundee, Scotland, UK.\n\nCompounds that inhibit signalling upstream of ERK (extracellular-signal-regulated\nkinase) are promising anticancer therapies, motivating research to define how\nthis pathway promotes cancers. In the present study, we show that human capica\nrepresses mRNA expression for PEA3 (polyoma enhancer activator 3) Ets\ntranscription factors ETV1, ETV4 and ETV5 (ETV is Ets translocation variant), and\nthis repression is relieved by multisite controls of capica by ERK, p90(RSK)\n(p90 ribosomal S6 kinase) and 14-3-3 proteins. Specifically, 14-3-3 binds to\np90(RSK)-phosphorylated Ser of capica thereby modulating DNA binding to its\nHMG (high-mobility group) box, whereas ERK phosphorylations prevent binding of a \nC-terminal NLS (nuclear localization sequence) to importin 4 (KPNA3). ETV1, ETV4\nand ETV5 mRNA levels in melanoma cells are elevated by siRNA (small interfering\nRNA) knockdown of capica, and decreased by inhibiting ERK and/or expressing a\nform of capica that cannot bind to 14-3-3 proteins. Capica knockdown also\nenhances cell migration. The findings of the present study give further\nmechanistic insights into why ETV1 is highly expressed in certain cancers,\nindicate that loss of capica can desensitize cells to the effects of ERK pathway\ninhibitors, and highlight interconnections among growth factor signalling,\nspinocerebellar ataxias and cancers.\n\nPMCID: PMC3025492\nPMID: 21087211  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7957",
        "source" : "7781",
        "target" : "7705",
        "pmid" : "21087211",
        "selected" : false,
        "canonicalName" : "CIC (inhibits) CELL MIGRATION",
        "edgeType" : "inhibits",
        "SUID" : 7957,
        "name" : "CIC (inhibits) CELL MIGRATION",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "CIC (inhibits) CELL MIGRATION",
        "edgeDetail" : "CIC (inhibits) CELL MIGRATION\n\n1. Biochem J. 2011 Feb 1;433(3):515-25. doi: 10.1042/BJ20101562.\n\nERK/p90(RSK)/14-3-3 signalling has an impact on expression of PEA3 Ets\ntranscription factors via the transcriptional repressor capica.\n\nDissanayake K(1), Toth R, Blakey J, Olsson O, Campbell DG, Prescott AR,\nMacKintosh C.\n\nAuthor information: \n(1)University of Dundee, Scotland, UK.\n\nCompounds that inhibit signalling upstream of ERK (extracellular-signal-regulated\nkinase) are promising anticancer therapies, motivating research to define how\nthis pathway promotes cancers. In the present study, we show that human capica\nrepresses mRNA expression for PEA3 (polyoma enhancer activator 3) Ets\ntranscription factors ETV1, ETV4 and ETV5 (ETV is Ets translocation variant), and\nthis repression is relieved by multisite controls of capica by ERK, p90(RSK)\n(p90 ribosomal S6 kinase) and 14-3-3 proteins. Specifically, 14-3-3 binds to\np90(RSK)-phosphorylated Ser of capica thereby modulating DNA binding to its\nHMG (high-mobility group) box, whereas ERK phosphorylations prevent binding of a \nC-terminal NLS (nuclear localization sequence) to importin 4 (KPNA3). ETV1, ETV4\nand ETV5 mRNA levels in melanoma cells are elevated by siRNA (small interfering\nRNA) knockdown of capica, and decreased by inhibiting ERK and/or expressing a\nform of capica that cannot bind to 14-3-3 proteins. Capica knockdown also\nenhances cell migration. The findings of the present study give further\nmechanistic insights into why ETV1 is highly expressed in certain cancers,\nindicate that loss of capica can desensitize cells to the effects of ERK pathway\ninhibitors, and highlight interconnections among growth factor signalling,\nspinocerebellar ataxias and cancers.\n\nPMCID: PMC3025492\nPMID: 21087211  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7956",
        "source" : "7780",
        "target" : "7751",
        "pmid" : "23818228",
        "selected" : false,
        "canonicalName" : "CMYB (activates) MGMT",
        "edgeType" : "activates",
        "SUID" : 7956,
        "name" : "CMYB (activates) MGMT",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "CMYB (activates) MGMT",
        "edgeDetail" : "CMYB (activates) MGMT\n\n1. EMBO Mol Med. 2013 Aug;5(8):1196-212. doi: 10.1002/emmm.201302827. Epub 2013 Jul \n1.\n\nThe ZEB1 pathway links glioblastoma initiation, invasion and chemoresistance.\n\nSiebzehnrubl FA, Silver DJ, Tugertimur B, Deleyrolle LP, Siebzehnrubl D,\nSarkisian MR, Devers KG, Yachnis AT, Kupper MD, Neal D, Nabilsi NH, Kladde MP,\nSuslov O, Brabletz S, Brabletz T, Reynolds BA, Steindler DA.\n\nDepartment of Neurosurgery, University of Florida, Gainesville, FL, USA.\nfas@ufl.edu\n\nGlioblastoma remains one of the most lethal types of cancer, and is the most\ncommon brain tumour in adults. In particular, tumour recurrence after surgical\nresection and radiation invariably occurs regardless of aggressive chemotherapy. \nHere, we provide evidence that the transcription factor ZEB1 (zinc finger E-box\nbinding homeobox 1) exerts simultaneous influence over invasion, chemoresistance \nand tumourigenesis in glioblastoma. ZEB1 is preferentially expressed in invasive \nglioblastoma cells, where the ZEB1-miR-200 feedback loop interconnects these\nprocesses through the downstream effectors ROBO1, c-MYB and MGMT. Moreover, ZEB1 \nexpression in glioblastoma patients is predictive of shorter survival and poor\nTemozolomide response. Our findings indicate that this regulator of\nepithelial-mesenchymal transition orchestrates key features of cancer stem cells \nin malignant glioma and identify ROBO1, OLIG2, CD133 and MGMT as novel targets of\nthe ZEB1 pathway. Thus, ZEB1 is an important candidate molecule for glioblastoma \nrecurrence, a marker of invasive tumour cells and a potential therapeutic target,\nalong with its downstream effectors.\n\n 2013 The Authors. Published by John Wiley and Sons, Ltd on behalf of EMBO.\n\nPMID: 23818228  [PubMed - in process]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7955",
        "source" : "7779",
        "target" : "7746",
        "pmid" : "24217114",
        "selected" : false,
        "canonicalName" : "CTNNB1 (activates) MYC",
        "edgeType" : "activates",
        "SUID" : 7955,
        "name" : "CTNNB1 (activates) MYC",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "CTNNB1 (activates) MYC",
        "edgeDetail" : "CTNNB1 (activates) MYC\n\n1. Ann Med Interne (Paris). 1974 Aug-Sep;125(8-9):641-50.\n\n[Obstructive myocardiopathy, Turner-Ullrich syndrome and von Recklinghausen's\ndisease].\n\n[Article in French]\n\nGerbaux A, Belfante M, Hiltgen M.\n\nPMID: 4217114  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7954",
        "source" : "7779",
        "target" : "7703",
        "pmid" : "24217114",
        "selected" : false,
        "canonicalName" : "CTNNB1 (activates) CCND1",
        "edgeType" : "activates",
        "SUID" : 7954,
        "name" : "CTNNB1 (activates) CCND1",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "CTNNB1 (activates) CCND1",
        "edgeDetail" : "CTNNB1 (activates) CCND1\n\n1. Ann Med Interne (Paris). 1974 Aug-Sep;125(8-9):641-50.\n\n[Obstructive myocardiopathy, Turner-Ullrich syndrome and von Recklinghausen's\ndisease].\n\n[Article in French]\n\nGerbaux A, Belfante M, Hiltgen M.\n\nPMID: 4217114  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7953",
        "source" : "7778",
        "target" : "7698",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "E2F1 (activates.process) G1/S PROGRESSION",
        "edgeType" : "activates.process",
        "SUID" : 7953,
        "name" : "E2F1 (activates.process) G1/S PROGRESSION",
        "interaction" : "activates.process",
        "shared_interaction" : "activates.process",
        "shared_name" : "E2F1 (activates.process) G1/S PROGRESSION",
        "edgeDetail" : "E2F1 (activates.process) G1/S PROGRESSION\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7952",
        "source" : "7777",
        "target" : "7730",
        "pmid" : "24217114",
        "selected" : false,
        "canonicalName" : "EGFR family (activates) PRKCE",
        "edgeType" : "activates",
        "SUID" : 7952,
        "name" : "EGFR family (activates) PRKCE",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "EGFR family (activates) PRKCE",
        "edgeDetail" : "EGFR family (activates) PRKCE\n\n1. Cancers (Basel). 2013 Nov 8;5(4):1469-84. doi: 10.3390/cancers5041469.\n\nTumor metabolism of malignant gliomas.\n\nRu P(1), Williams TM, Chakravarti A, Guo D.\n\nAuthor information: \n(1)Department of Radiation Oncology, Ohio State University Comprehensive Cancer\nCenter & Arthur G James Cancer Hospital, Columbus, OH 43012, USA.\ndeliang.guo@osumc.edu.\n\nConstitutively activated oncogenic signaling via genetic mutations such as in the\nEGFR/PI3K/Akt and Ras/RAF/MEK pathways has been recognized as a major driver for \ntumorigenesis in most cancers. Recent insights into tumor metabolism have further\nrevealed that oncogenic signaling pathways directly promote metabolic\nreprogramming to upregulate biosynthesis of lipids, carbohydrates, protein, DNA\nand RNA, leading to enhanced growth of human tumors. Therefore, targeting cell\nmetabolism has become a novel direction for drug development in oncology. In\nmalignant gliomas, metabolism pathways of glucose, glutamine and lipid are\nsignificantly reprogrammed. Moreover, molecular mechanisms causing these\nmetabolic changes are just starting to be unraveled. In this review, we will\nsummarize recent studies revealing critical gene alterations that lead to\nmetabolic changes in malignant gliomas, and also discuss promising therapeutic\nstrategies via targeting the key players in metabolic regulation.\n\nPMCID: PMC3875949\nPMID: 24217114  [PubMed]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7951",
        "source" : "7777",
        "target" : "7696",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "EGFR family (family.includes) EGFR",
        "edgeType" : "family.includes",
        "SUID" : 7951,
        "name" : "EGFR family (family.includes) EGFR",
        "interaction" : "family.includes",
        "shared_interaction" : "family.includes",
        "shared_name" : "EGFR family (family.includes) EGFR",
        "edgeDetail" : "EGFR family (family.includes) EGFR\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7950",
        "source" : "7777",
        "target" : "7695",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "EGFR family (family.includes) ERBB2",
        "edgeType" : "family.includes",
        "SUID" : 7950,
        "name" : "EGFR family (family.includes) ERBB2",
        "interaction" : "family.includes",
        "shared_interaction" : "family.includes",
        "shared_name" : "EGFR family (family.includes) ERBB2",
        "edgeDetail" : "EGFR family (family.includes) ERBB2\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7949",
        "source" : "7777",
        "target" : "7694",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "EGFR family (family.includes) ERBB3",
        "edgeType" : "family.includes",
        "SUID" : 7949,
        "name" : "EGFR family (family.includes) ERBB3",
        "interaction" : "family.includes",
        "shared_interaction" : "family.includes",
        "shared_name" : "EGFR family (family.includes) ERBB3",
        "edgeDetail" : "EGFR family (family.includes) ERBB3\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7948",
        "source" : "7777",
        "target" : "7718",
        "pmid" : "241352280",
        "selected" : false,
        "canonicalName" : "EGFR family (activates) STAT3",
        "edgeType" : "activates",
        "SUID" : 7948,
        "name" : "EGFR family (activates) STAT3",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "EGFR family (activates) STAT3",
        "edgeDetail" : "EGFR family (activates) STAT3\n\n1.  Error occurred: \nPMID: 241352280 \n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7947",
        "source" : "7777",
        "target" : "7717",
        "pmid" : "241352280",
        "selected" : false,
        "canonicalName" : "EGFR family (activates) STAT5",
        "edgeType" : "activates",
        "SUID" : 7947,
        "name" : "EGFR family (activates) STAT5",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "EGFR family (activates) STAT5",
        "edgeDetail" : "EGFR family (activates) STAT5\n\n1.  Error occurred: \nPMID: 241352280 \n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7946",
        "source" : "7776",
        "target" : "7715",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "EP300 (activates) TP53",
        "edgeType" : "activates",
        "SUID" : 7946,
        "name" : "EP300 (activates) TP53",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "EP300 (activates) TP53",
        "edgeDetail" : "EP300 (activates) TP53\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7945",
        "source" : "7775",
        "target" : "7746",
        "pmid" : "17188374",
        "selected" : false,
        "canonicalName" : "ERK1/2 (activates) MYC",
        "edgeType" : "activates",
        "SUID" : 7945,
        "name" : "ERK1/2 (activates) MYC",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "ERK1/2 (activates) MYC",
        "edgeDetail" : "ERK1/2 (activates) MYC\n\n1. Biochim Biophys Acta. 2007 Aug;1773(8):1299-310. Epub 2006 Nov 17.\n\nERK implication in cell cycle regulation.\n\nChambard JC(1), Lefloch R, Pouyssgur J, Lenormand P.\n\nAuthor information: \n(1)Institute of Signaling Developmental Biology and Cancer, CNRS UMR 6543,\nUniversite de Nice-Sofia Antipolis, Centre A. Lacassagne, 33 Avenue de\nValombrose, 06189 Nice, France.\n\nThe Ras/Raf/MEK/ERK signaling cascade that integrates an extreme variety of\nextracellular stimuli into key biological responses controlling cell\nproliferation, differentiation or death is one of the most studied intracellular \npathways. Here we present some evidences that have been accumulated over the last\n15 years proving the requirement of ERK in the control of cell proliferation. In \nthis review we focus (i) on the spatio-temporal control of ERK signaling, (ii) on\nthe key cellular components linking extracellular signals to the induction and\nactivation of cell cycle events controlling G1 to S-phase transition and (iii) on\nthe role of ERK in the growth factor-independent G2/M phase of the cell cycle. As\nERK pathway is often co-activated with the PI3 kinase signaling, we highlight\nsome of the key points of convergence leading to a full activation of mTOR via\nERK and AKT synergies. Finally, ERK and AKT targets being constitutively\nactivated in so many human cancers, we briefly touched the cure issue of using\nmore specific drugs in rationally selected cancer patients.\n\nPMID: 17188374  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7944",
        "source" : "7775",
        "target" : "7748",
        "pmid" : "17188374",
        "selected" : false,
        "canonicalName" : "ERK1/2 (activates) MSK1/2",
        "edgeType" : "activates",
        "SUID" : 7944,
        "name" : "ERK1/2 (activates) MSK1/2",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "ERK1/2 (activates) MSK1/2",
        "edgeDetail" : "ERK1/2 (activates) MSK1/2\n\n1. Biochim Biophys Acta. 2007 Aug;1773(8):1299-310. Epub 2006 Nov 17.\n\nERK implication in cell cycle regulation.\n\nChambard JC(1), Lefloch R, Pouyssgur J, Lenormand P.\n\nAuthor information: \n(1)Institute of Signaling Developmental Biology and Cancer, CNRS UMR 6543,\nUniversite de Nice-Sofia Antipolis, Centre A. Lacassagne, 33 Avenue de\nValombrose, 06189 Nice, France.\n\nThe Ras/Raf/MEK/ERK signaling cascade that integrates an extreme variety of\nextracellular stimuli into key biological responses controlling cell\nproliferation, differentiation or death is one of the most studied intracellular \npathways. Here we present some evidences that have been accumulated over the last\n15 years proving the requirement of ERK in the control of cell proliferation. In \nthis review we focus (i) on the spatio-temporal control of ERK signaling, (ii) on\nthe key cellular components linking extracellular signals to the induction and\nactivation of cell cycle events controlling G1 to S-phase transition and (iii) on\nthe role of ERK in the growth factor-independent G2/M phase of the cell cycle. As\nERK pathway is often co-activated with the PI3 kinase signaling, we highlight\nsome of the key points of convergence leading to a full activation of mTOR via\nERK and AKT synergies. Finally, ERK and AKT targets being constitutively\nactivated in so many human cancers, we briefly touched the cure issue of using\nmore specific drugs in rationally selected cancer patients.\n\nPMID: 17188374  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7943",
        "source" : "7775",
        "target" : "7788",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "ERK1/2 (activates) CCND-CDK4 complex",
        "edgeType" : "activates",
        "SUID" : 7943,
        "name" : "ERK1/2 (activates) CCND-CDK4 complex",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "ERK1/2 (activates) CCND-CDK4 complex",
        "edgeDetail" : "ERK1/2 (activates) CCND-CDK4 complex\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7942",
        "source" : "7775",
        "target" : "7787",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "ERK1/2 (activates) CCND-CDK6 complex",
        "edgeType" : "activates",
        "SUID" : 7942,
        "name" : "ERK1/2 (activates) CCND-CDK6 complex",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "ERK1/2 (activates) CCND-CDK6 complex",
        "edgeDetail" : "ERK1/2 (activates) CCND-CDK6 complex\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7941",
        "source" : "7775",
        "target" : "7781",
        "pmid" : "21817013",
        "selected" : false,
        "canonicalName" : "ERK1/2 (inhibits) CIC",
        "edgeType" : "inhibits",
        "SUID" : 7941,
        "name" : "ERK1/2 (inhibits) CIC",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "ERK1/2 (inhibits) CIC",
        "edgeDetail" : "ERK1/2 (inhibits) CIC\n\n1. Science. 2011 Sep 9;333(6048):1453-5. doi: 10.1126/science.1210557. Epub 2011 Aug\n4.\n\nMutations in CIC and FUBP1 contribute to human oligodendroglioma.\n\nBettegowda C(1), Agrawal N, Jiao Y, Sausen M, Wood LD, Hruban RH, Rodriguez FJ,\nCahill DP, McLendon R, Riggins G, Velculescu VE, Oba-Shinjo SM, Marie SK,\nVogelstein B, Bigner D, Yan H, Papadopoulos N, Kinzler KW.\n\nAuthor information: \n(1)Ludwig Center for Cancer Genetics and Howard Hughes Medical Institutions, Johns\nHopkins Kimmel Cancer Center, Baltimore, MD 21287, USA.\n\nOligodendrogliomas are the second most common malignant brain tumor in adults and\nexhibit characteristic losses of chromosomes 1p and 19q. To identify the\nmolecular genetic basis for this alteration, we performed exomic sequencing of\nseven tumors. Among other changes, we found that the CIC gene (homolog of the\nDrosophila gene capicua) on chromosome 19q was somatically mutated in six cases\nand that the FUBP1 gene [encoding far-upstream element (FUSE) binding protein] on\nchromosome 1p was somatically mutated in two tumors. Examination of 27 additional\noligodendrogliomas revealed 12 and 3 more tumors with mutations of CIC and FUBP1,\nrespectively, 58% of which were predicted to result in truncations of the encoded\nproteins. These results suggest a critical role for these genes in the biology\nand pathology of oligodendrocytes.\n\nPMCID: PMC3170506\nPMID: 21817013  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7940",
        "source" : "7774",
        "target" : "7777",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "ERRFI1 (inhibits) EGFR family",
        "edgeType" : "inhibits",
        "SUID" : 7940,
        "name" : "ERRFI1 (inhibits) EGFR family",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "ERRFI1 (inhibits) EGFR family",
        "edgeDetail" : "ERRFI1 (inhibits) EGFR family\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7939",
        "source" : "7773",
        "target" : "7693",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "FGFR family (family.includes) FGFR2",
        "edgeType" : "family.includes",
        "SUID" : 7939,
        "name" : "FGFR family (family.includes) FGFR2",
        "interaction" : "family.includes",
        "shared_interaction" : "family.includes",
        "shared_name" : "FGFR family (family.includes) FGFR2",
        "edgeDetail" : "FGFR family (family.includes) FGFR2\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7938",
        "source" : "7773",
        "target" : "7692",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "FGFR family (family.includes) FGFR1",
        "edgeType" : "family.includes",
        "SUID" : 7938,
        "name" : "FGFR family (family.includes) FGFR1",
        "interaction" : "family.includes",
        "shared_interaction" : "family.includes",
        "shared_name" : "FGFR family (family.includes) FGFR1",
        "edgeDetail" : "FGFR family (family.includes) FGFR1\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7937",
        "source" : "7772",
        "target" : "7691",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "FOXO family (family.includes) FOXO1",
        "edgeType" : "family.includes",
        "SUID" : 7937,
        "name" : "FOXO family (family.includes) FOXO1",
        "interaction" : "family.includes",
        "shared_interaction" : "family.includes",
        "shared_name" : "FOXO family (family.includes) FOXO1",
        "edgeDetail" : "FOXO family (family.includes) FOXO1\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7936",
        "source" : "7772",
        "target" : "7690",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "FOXO family (family.includes) FOXO3",
        "edgeType" : "family.includes",
        "SUID" : 7936,
        "name" : "FOXO family (family.includes) FOXO3",
        "interaction" : "family.includes",
        "shared_interaction" : "family.includes",
        "shared_name" : "FOXO family (family.includes) FOXO3",
        "edgeDetail" : "FOXO family (family.includes) FOXO3\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7935",
        "source" : "7772",
        "target" : "7689",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "FOXO family (family.includes) FOXO4",
        "edgeType" : "family.includes",
        "SUID" : 7935,
        "name" : "FOXO family (family.includes) FOXO4",
        "interaction" : "family.includes",
        "shared_interaction" : "family.includes",
        "shared_name" : "FOXO family (family.includes) FOXO4",
        "edgeDetail" : "FOXO family (family.includes) FOXO4\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7934",
        "source" : "7772",
        "target" : "7723",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "FOXO family (activates) SPRY2",
        "edgeType" : "activates",
        "SUID" : 7934,
        "name" : "FOXO family (activates) SPRY2",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "FOXO family (activates) SPRY2",
        "edgeDetail" : "FOXO family (activates) SPRY2\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7933",
        "source" : "7772",
        "target" : "7785",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "FOXO family (activates) CDKN1A",
        "edgeType" : "activates",
        "SUID" : 7933,
        "name" : "FOXO family (activates) CDKN1A",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "FOXO family (activates) CDKN1A",
        "edgeDetail" : "FOXO family (activates) CDKN1A\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7932",
        "source" : "7772",
        "target" : "7784",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "FOXO family (activates) CDKN1B",
        "edgeType" : "activates",
        "SUID" : 7932,
        "name" : "FOXO family (activates) CDKN1B",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "FOXO family (activates) CDKN1B",
        "edgeDetail" : "FOXO family (activates) CDKN1B\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7931",
        "source" : "7771",
        "target" : "7746",
        "pmid" : "21817013",
        "selected" : false,
        "canonicalName" : "FUBP1 (inhibits) MYC",
        "edgeType" : "inhibits",
        "SUID" : 7931,
        "name" : "FUBP1 (inhibits) MYC",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "FUBP1 (inhibits) MYC",
        "edgeDetail" : "FUBP1 (inhibits) MYC\n\n1. Science. 2011 Sep 9;333(6048):1453-5. doi: 10.1126/science.1210557. Epub 2011 Aug\n4.\n\nMutations in CIC and FUBP1 contribute to human oligodendroglioma.\n\nBettegowda C(1), Agrawal N, Jiao Y, Sausen M, Wood LD, Hruban RH, Rodriguez FJ,\nCahill DP, McLendon R, Riggins G, Velculescu VE, Oba-Shinjo SM, Marie SK,\nVogelstein B, Bigner D, Yan H, Papadopoulos N, Kinzler KW.\n\nAuthor information: \n(1)Ludwig Center for Cancer Genetics and Howard Hughes Medical Institutions, Johns\nHopkins Kimmel Cancer Center, Baltimore, MD 21287, USA.\n\nOligodendrogliomas are the second most common malignant brain tumor in adults and\nexhibit characteristic losses of chromosomes 1p and 19q. To identify the\nmolecular genetic basis for this alteration, we performed exomic sequencing of\nseven tumors. Among other changes, we found that the CIC gene (homolog of the\nDrosophila gene capicua) on chromosome 19q was somatically mutated in six cases\nand that the FUBP1 gene [encoding far-upstream element (FUSE) binding protein] on\nchromosome 1p was somatically mutated in two tumors. Examination of 27 additional\noligodendrogliomas revealed 12 and 3 more tumors with mutations of CIC and FUBP1,\nrespectively, 58% of which were predicted to result in truncations of the encoded\nproteins. These results suggest a critical role for these genes in the biology\nand pathology of oligodendrocytes.\n\nPMCID: PMC3170506\nPMID: 21817013  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7930",
        "source" : "7770",
        "target" : "7740",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "GAB1 (activates) PI3K class1a",
        "edgeType" : "activates",
        "SUID" : 7930,
        "name" : "GAB1 (activates) PI3K class1a",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "GAB1 (activates) PI3K class1a",
        "edgeDetail" : "GAB1 (activates) PI3K class1a\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7929",
        "source" : "7769",
        "target" : "7688",
        "pmid" : "24217114",
        "selected" : false,
        "canonicalName" : "GLUT1 (activates) Glycolysis",
        "edgeType" : "activates",
        "SUID" : 7929,
        "name" : "GLUT1 (activates) Glycolysis",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "GLUT1 (activates) Glycolysis",
        "edgeDetail" : "GLUT1 (activates) Glycolysis\n\n1. Ann Med Interne (Paris). 1974 Aug-Sep;125(8-9):641-50.\n\n[Obstructive myocardiopathy, Turner-Ullrich syndrome and von Recklinghausen's\ndisease].\n\n[Article in French]\n\nGerbaux A, Belfante M, Hiltgen M.\n\nPMID: 4217114  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7928",
        "source" : "7768",
        "target" : "7770",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "GRB2 (activates) GAB1",
        "edgeType" : "activates",
        "SUID" : 7928,
        "name" : "GRB2 (activates) GAB1",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "GRB2 (activates) GAB1",
        "edgeDetail" : "GRB2 (activates) GAB1\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7927",
        "source" : "7768",
        "target" : "7726",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "GRB2 (activates) RAS family",
        "edgeType" : "activates",
        "SUID" : 7927,
        "name" : "GRB2 (activates) RAS family",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "GRB2 (activates) RAS family",
        "edgeDetail" : "GRB2 (activates) RAS family\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7926",
        "source" : "7767",
        "target" : "7751",
        "pmid" : "23076220",
        "selected" : false,
        "canonicalName" : "HIF1A (activates) MGMT",
        "edgeType" : "activates",
        "SUID" : 7926,
        "name" : "HIF1A (activates) MGMT",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "HIF1A (activates) MGMT",
        "edgeDetail" : "HIF1A (activates) MGMT\n\n1. Cell Death Dis. 2012 Oct 18;3:e412. doi: 10.1038/cddis.2012.153.\n\nBMP2 sensitizes glioblastoma stem-like cells to Temozolomide by affecting HIF-1 \nstability and MGMT expression.\n\nPersano L, Pistollato F, Rampazzo E, Della Puppa A, Abbadi S, Frasson C, Volpin\nF, Indraccolo S, Scienza R, Basso G.\n\nOncohematology Laboratory, Department of Woman and Child Health, University of\nPadova, Padova, Italy. luca.persano@unipd.it\n\nGlioblastoma multiforme (GBM) is the most common brain tumour, characterized by a\ncentral and partially necrotic (i.e., hypoxic) core enriched in cancer stem cells\n(CSCs). We previously showed that the most hypoxic and immature (i.e., CSCs) GBM \ncells were resistant to Temozolomide (TMZ) in vitro, owing to a particularly high\nexpression of O6-methylguanine-DNA-methyltransferase (MGMT), the most important\nfactor associated to therapy resistance in GBM. Bone morphogenetic proteins\n(BMPs), and in particular BMP2, are known to promote differentiation and growth\ninhibition in GBM cells. For this reason, we investigated whether a BMP2-based\ntreatment would increase TMZ response in hypoxic drug-resistant GBM-derived\ncells. Here we show that BMP2 induced strong differentiation of GBM stem-like\ncells and subsequent addition of TMZ caused dramatic increase of apoptosis.\nImportantly, we correlated these effects to a BMP2-induced downregulation of both\nhypoxia-inducible factor-1 (HIF-1) and MGMT. We report here a novel mechanism\ninvolving the HIF-1-dependent regulation of MGMT, highlighting the existence of \na HIF-1/MGMT axis supporting GBM resistance to therapy. As confirmed from this\nevidence, over-stabilization of HIF-1 in TMZ-sensitive GBM cells abolished their\nresponsiveness to it. In conclusion, we describe a HIF-1-dependent regulation of\nMGMT and suggest that BMP2, by down-modulating the HIF-1/MGMT axis, should\nincrease GBM responsiveness to chemotherapy, thus opening the way to the\ndevelopment of future strategies for GBM treatment.\n\nPMCID: PMC3481140\nPMID: 23076220  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7925",
        "source" : "7766",
        "target" : "7687",
        "pmid" : "23359207",
        "selected" : false,
        "canonicalName" : "HGF (activates) MET",
        "edgeType" : "activates",
        "SUID" : 7925,
        "name" : "HGF (activates) MET",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "HGF (activates) MET",
        "edgeDetail" : "HGF (activates) MET\n\n1. Neoplasia. 2013 Jan;15(1):73-84.\n\nRegulation of HGF expression by EGFR-mediated c-Met activation in glioblastoma\ncells.\n\nGarnett J(1), Chumbalkar V, Vaillant B, Gururaj AE, Hill KS, Latha K, Yao J,\nPriebe W, Colman H, Elferink LA, Bogler O.\n\nAuthor information: \n(1)Department of Neurosurgery, University of Texas MD Anderson Cancer Center,\nHouston, TX 77030, USA.\n\nThe hepatocyte growth factor receptor (c-Met) and a constitutively active mutant \nof the epidermal growth factor receptor (EGFR/EGFRvIII) are frequently\noverexpressed in glioblastoma (GBM) and promote tumorigenesis. The mechanisms\nunderlying elevated hepatocyte growth factor (HGF) production in GBM are not\nunderstood. We found higher, coordinated mRNA expression levels of HGF and c-Met \nin mesenchymal (Mes) GBMs, a subtype associated with poor treatment response and \nshorter overall survival. In an HGF/c-Met-dependent GBM cell line, HGF expression\ndeclined upon silencing of c-Met using RNAi or by inhibiting its activity with\nSU11274. Silencing c-Met decreased anchorage-independent colony formation and\nincreased the survival of mice bearing intracranial GBM xenografts. Consistent\nwith these findings, c-Met activation by EGFR also elevated HGF expression, and \nthe inhibition of EGFR with AG1478 reduced HGF levels. Interestingly, c-Met\nexpression was required for EGFR-mediated HGF production, anchorage-independent \ngrowth, and in vivo tumorigenicity, suggesting that these pathways are coupled.\nUsing an unbiased mass spectrometry-based screen, we show that signal transducer \nand activator of transcription 3 (STAT3) Y705 is a downstream target of c-Met\nsignaling. Suppression of STAT3 phosphorylation with WP1193 reduced HGF\nexpression in EGFR-expressing GBM cells, whereas constitutively active STAT3\npartially rescued HGF expression and colony formation in c-Met knockdown cells\nexpressing EGFR. These results suggest that the c-Met/HGF signaling axis is\nenhanced by EGFR through increased STAT3-dependent HGF expression and that\ntargeting c-Met in Mes GBMs may be an important strategy for therapy.\n\nPMCID: PMC3556940\nPMID: 23359207  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7924",
        "source" : "7766",
        "target" : "7752",
        "pmid" : "23359207",
        "selected" : false,
        "canonicalName" : "HGF (activates) MET:EGFR",
        "edgeType" : "activates",
        "SUID" : 7924,
        "name" : "HGF (activates) MET:EGFR",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "HGF (activates) MET:EGFR",
        "edgeDetail" : "HGF (activates) MET:EGFR\n\n1. Neoplasia. 2013 Jan;15(1):73-84.\n\nRegulation of HGF expression by EGFR-mediated c-Met activation in glioblastoma\ncells.\n\nGarnett J(1), Chumbalkar V, Vaillant B, Gururaj AE, Hill KS, Latha K, Yao J,\nPriebe W, Colman H, Elferink LA, Bogler O.\n\nAuthor information: \n(1)Department of Neurosurgery, University of Texas MD Anderson Cancer Center,\nHouston, TX 77030, USA.\n\nThe hepatocyte growth factor receptor (c-Met) and a constitutively active mutant \nof the epidermal growth factor receptor (EGFR/EGFRvIII) are frequently\noverexpressed in glioblastoma (GBM) and promote tumorigenesis. The mechanisms\nunderlying elevated hepatocyte growth factor (HGF) production in GBM are not\nunderstood. We found higher, coordinated mRNA expression levels of HGF and c-Met \nin mesenchymal (Mes) GBMs, a subtype associated with poor treatment response and \nshorter overall survival. In an HGF/c-Met-dependent GBM cell line, HGF expression\ndeclined upon silencing of c-Met using RNAi or by inhibiting its activity with\nSU11274. Silencing c-Met decreased anchorage-independent colony formation and\nincreased the survival of mice bearing intracranial GBM xenografts. Consistent\nwith these findings, c-Met activation by EGFR also elevated HGF expression, and \nthe inhibition of EGFR with AG1478 reduced HGF levels. Interestingly, c-Met\nexpression was required for EGFR-mediated HGF production, anchorage-independent \ngrowth, and in vivo tumorigenicity, suggesting that these pathways are coupled.\nUsing an unbiased mass spectrometry-based screen, we show that signal transducer \nand activator of transcription 3 (STAT3) Y705 is a downstream target of c-Met\nsignaling. Suppression of STAT3 phosphorylation with WP1193 reduced HGF\nexpression in EGFR-expressing GBM cells, whereas constitutively active STAT3\npartially rescued HGF expression and colony formation in c-Met knockdown cells\nexpressing EGFR. These results suggest that the c-Met/HGF signaling axis is\nenhanced by EGFR through increased STAT3-dependent HGF expression and that\ntargeting c-Met in Mes GBMs may be an important strategy for therapy.\n\nPMCID: PMC3556940\nPMID: 23359207  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7923",
        "source" : "7765",
        "target" : "7686",
        "pmid" : "20068170",
        "selected" : false,
        "canonicalName" : "HOXA9 (inhibits.process) APOPTOSIS",
        "edgeType" : "inhibits.process",
        "SUID" : 7923,
        "name" : "HOXA9 (inhibits.process) APOPTOSIS",
        "interaction" : "inhibits.process",
        "shared_interaction" : "inhibits.process",
        "shared_name" : "HOXA9 (inhibits.process) APOPTOSIS",
        "edgeDetail" : "HOXA9 (inhibits.process) APOPTOSIS\n\n1. Cancer Res. 2010 Jan 15;70(2):453-62. doi: 10.1158/0008-5472.CAN-09-2189. Epub\n2010 Jan 12.\n\nReversing HOXA9 oncogene activation by PI3K inhibition: epigenetic mechanism and \nprognostic significance in human glioblastoma.\n\nCosta BM, Smith JS, Chen Y, Chen J, Phillips HS, Aldape KD, Zardo G, Nigro J,\nJames CD, Fridlyand J, Reis RM, Costello JF.\n\nThe Brain Tumor Research Center, Department of Neurological Surgery, University\nof California-San Francisco, San Francisco, California 94158-9001, USA.\n\nHOXA genes encode critical transcriptional regulators of embryonic development\nthat have been implicated in cancer. In this study, we documented functional\nrelevance and mechanism of activation of HOXA9 in glioblastoma (GBM), the most\ncommon malignant brain tumor. Expression of HOXA genes was investigated using\nreverse transcription-PCR in primary gliomas and glioblastoma cell lines and was \nvalidated in two sets of expression array data. In a subset of GBM, HOXA genes\nare aberrently activated within confined chromosomal domains. Transcriptional\nactivation of the HOXA cluster was reversible by a phosphoinostide 3-kinase\n(PI3K) inhibitor through an epigenetic mechanism involving histone H3K27\ntrimethylation. Functional studies of HOXA9 showed its capacity to decrease\napoptosis and increase cellular proliferation along with tumor necrosis\nfactor-related apoptosis-including ligand resistance. Notably, aberrant\nexpression of HOXA9 was independently predictive of shorter overall and\nprogression-free survival in two GBM patient sets and improved survival\nprediction by MGMT promoter methylation. Thus, HOXA9 activation is a novel,\nindependent, and negative prognostic marker in GBM that is reversible through a\nPI3K-associated epigenetic mechanism. Our findings suggest a transcriptional\npathway through which PI3K activates oncogenic HOXA expression with implications \nfor mTOR or PI3K targeted therapies.\n\nPMCID: PMC2849935\nPMID: 20068170  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7922",
        "source" : "7764",
        "target" : "7767",
        "pmid" : "24217114",
        "selected" : false,
        "canonicalName" : "IDH1 (inhibits) HIF1A",
        "edgeType" : "inhibits",
        "SUID" : 7922,
        "name" : "IDH1 (inhibits) HIF1A",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "IDH1 (inhibits) HIF1A",
        "edgeDetail" : "IDH1 (inhibits) HIF1A\n\n1. Ann Med Interne (Paris). 1974 Aug-Sep;125(8-9):641-50.\n\n[Obstructive myocardiopathy, Turner-Ullrich syndrome and von Recklinghausen's\ndisease].\n\n[Article in French]\n\nGerbaux A, Belfante M, Hiltgen M.\n\nPMID: 4217114  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7921",
        "source" : "7764",
        "target" : "7712",
        "pmid" : "24077277",
        "selected" : false,
        "canonicalName" : "IDH1 (activates.process) Lipid metabolism",
        "edgeType" : "activates.process",
        "SUID" : 7921,
        "name" : "IDH1 (activates.process) Lipid metabolism",
        "interaction" : "activates.process",
        "shared_interaction" : "activates.process",
        "shared_name" : "IDH1 (activates.process) Lipid metabolism",
        "edgeDetail" : "IDH1 (activates.process) Lipid metabolism\n\n1. Neurol Med Chir (Tokyo). 2013;53(10):645-54. Epub 2013 Sep 27.\n\nAn R132H mutation in isocitrate dehydrogenase 1 enhances p21 expression and\ninhibits phosphorylation of retinoblastoma protein in glioma cells.\n\nMiyata S(1), Urabe M, Gomi A, Nagai M, Yamaguchi T, Tsukahara T, Mizukami H, Kume\nA, Ozawa K, Watanabe E.\n\nAuthor information: \n(1)Department of Neurosurgery, Center for Molecular Medicine, Jichi Medical\nUniversity.\n\nCytosolic isocitrate dehydrogenase 1 (IDH1) with an R132H mutation in brain\ntumors loses its enzymatic activity for catalyzing isocitrate to -ketoglutarate \n(-KG) and acquires new activity whereby it converts -KG to 2-hydroxyglutarate. \nThe IDH1 mutation induces down-regulation of tricarboxylic acid cycle\nintermediates and up-regulation of lipid metabolism. Sterol regulatory\nelement-binding proteins (SREBPs) regulate not only the synthesis of cholesterol \nand fatty acids but also acyclin-dependent kinase inhibitor p21 that halts the\ncell cycle at G1. Here we show that SREBPs were up-regulated in U87 human\nglioblastoma cells transfected with an IDH1(R132H)-expression plasmid. Small\ninterfering ribonucleic acid (siRNA) for SREBP1 specifically decreased p21\nmessenger RNA (mRNA) levels independent of the p53 pathway. In\nIDH1(R132H)-expressing U87 cells, phosphorylation of Retinoblastoma (Rb) protein \nalso decreased. We propose that metabolic changes induced by the IDH1 mutation\nenhance p21 expression via SREBP1 and inhibit phosphorylation of Rb, which slows \nprogression of the cell cycle and may be associated with non-aggressive features \nof gliomas with an IDH1 mutation.\n\nPMID: 24077277  [PubMed - in process]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7920",
        "source" : "7764",
        "target" : "7720",
        "pmid" : "24077277",
        "selected" : false,
        "canonicalName" : "IDH1 (activates) SREBP1a",
        "edgeType" : "activates",
        "SUID" : 7920,
        "name" : "IDH1 (activates) SREBP1a",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "IDH1 (activates) SREBP1a",
        "edgeDetail" : "IDH1 (activates) SREBP1a\n\n1. Neurol Med Chir (Tokyo). 2013;53(10):645-54. Epub 2013 Sep 27.\n\nAn R132H mutation in isocitrate dehydrogenase 1 enhances p21 expression and\ninhibits phosphorylation of retinoblastoma protein in glioma cells.\n\nMiyata S(1), Urabe M, Gomi A, Nagai M, Yamaguchi T, Tsukahara T, Mizukami H, Kume\nA, Ozawa K, Watanabe E.\n\nAuthor information: \n(1)Department of Neurosurgery, Center for Molecular Medicine, Jichi Medical\nUniversity.\n\nCytosolic isocitrate dehydrogenase 1 (IDH1) with an R132H mutation in brain\ntumors loses its enzymatic activity for catalyzing isocitrate to -ketoglutarate \n(-KG) and acquires new activity whereby it converts -KG to 2-hydroxyglutarate. \nThe IDH1 mutation induces down-regulation of tricarboxylic acid cycle\nintermediates and up-regulation of lipid metabolism. Sterol regulatory\nelement-binding proteins (SREBPs) regulate not only the synthesis of cholesterol \nand fatty acids but also acyclin-dependent kinase inhibitor p21 that halts the\ncell cycle at G1. Here we show that SREBPs were up-regulated in U87 human\nglioblastoma cells transfected with an IDH1(R132H)-expression plasmid. Small\ninterfering ribonucleic acid (siRNA) for SREBP1 specifically decreased p21\nmessenger RNA (mRNA) levels independent of the p53 pathway. In\nIDH1(R132H)-expressing U87 cells, phosphorylation of Retinoblastoma (Rb) protein \nalso decreased. We propose that metabolic changes induced by the IDH1 mutation\nenhance p21 expression via SREBP1 and inhibit phosphorylation of Rb, which slows \nprogression of the cell cycle and may be associated with non-aggressive features \nof gliomas with an IDH1 mutation.\n\nPMID: 24077277  [PubMed - in process]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7919",
        "source" : "7764",
        "target" : "7716",
        "pmid" : "23863747",
        "selected" : false,
        "canonicalName" : "IDH1 (inhibits) TET2",
        "edgeType" : "inhibits",
        "SUID" : 7919,
        "name" : "IDH1 (inhibits) TET2",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "IDH1 (inhibits) TET2",
        "edgeDetail" : "IDH1 (inhibits) TET2\n\n1. Nat Commun. 2013;4:2166. doi: 10.1038/ncomms3166.\n\nMeta-analysis of IDH-mutant cancers identifies EBF1 as an interaction partner for\nTET2.\n\nGuilhamon P(1), Eskandarpour M, Halai D, Wilson GA, Feber A, Teschendorff AE,\nGomez V, Hergovich A, Tirabosco R, Fernanda Amary M, Baumhoer D, Jundt G, Ross\nMT, Flanagan AM, Beck S.\n\nAuthor information: \n(1)Medical Genomics, UCL Cancer Institute, University College London, London, UK.\n\nIsocitrate dehydrogenase (IDH) genes 1 and 2 are frequently mutated in acute\nmyeloid leukaemia (AML), low-grade glioma, cholangiocarcinoma (CC) and\nchondrosarcoma (CS). For AML, low-grade glioma and CC, mutant IDH status is\nassociated with a DNA hypermethylation phenotype, implicating altered epigenome\ndynamics in the aetiology of these cancers. Here we show that the IDH variants in\nCS are also associated with a hypermethylation phenotype and display increased\nproduction of the oncometabolite 2-hydroxyglutarate, supporting the role of\nmutant IDH-produced 2-hydroxyglutarate as an inhibitor of TET-mediated DNA\ndemethylation. Meta-analysis of the acute myeloid leukaemia, low-grade glioma,\ncholangiocarcinoma and CS methylation data identifies cancer-specific effectors\nwithin the retinoic acid receptor activation pathway among the hypermethylated\ntargets. By analysing sequence motifs surrounding hypermethylated sites across\nthe four cancer types, and using chromatin immunoprecipitation and western\nblotting, we identify the transcription factor EBF1 (early B-cell factor 1) as an\ninteraction partner for TET2, suggesting a sequence-specific mechanism for\nregulating DNA methylation.\n\nPMCID: PMC3759038\nPMID: 23863747  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7918",
        "source" : "7764",
        "target" : "7685",
        "pmid" : "22343901",
        "selected" : false,
        "canonicalName" : "IDH1 (activates.process) Histone methylation",
        "edgeType" : "activates.process",
        "SUID" : 7918,
        "name" : "IDH1 (activates.process) Histone methylation",
        "interaction" : "activates.process",
        "shared_interaction" : "activates.process",
        "shared_name" : "IDH1 (activates.process) Histone methylation",
        "edgeDetail" : "IDH1 (activates.process) Histone methylation\n\n1. Nature. 2012 Feb 15;483(7390):474-8. doi: 10.1038/nature10860.\n\nIDH mutation impairs histone demethylation and results in a block to cell\ndifferentiation.\n\nLu C(1), Ward PS, Kapoor GS, Rohle D, Turcan S, Abdel-Wahab O, Edwards CR, Khanin\nR, Figueroa ME, Melnick A, Wellen KE, O'Rourke DM, Berger SL, Chan TA, Levine RL,\nMellinghoff IK, Thompson CB.\n\nAuthor information: \n(1)Cancer Biology and Genetics Program, Memorial Sloan-Kettering Cancer Center, New \nYork, New York 10065, USA.\n\nComment in\n    Nat Rev Cancer. 2012 Apr;12(4):229.\n\nRecurrent mutations in isocitrate dehydrogenase 1 (IDH1) and IDH2 have been\nidentified in gliomas, acute myeloid leukaemias (AML) and chondrosarcomas, and\nshare a novel enzymatic property of producing 2-hydroxyglutarate (2HG) from\n-ketoglutarate. Here we report that 2HG-producing IDH mutants can prevent the\nhistone demethylation that is required for lineage-specific progenitor cells to\ndifferentiate into terminally differentiated cells. In tumour samples from glioma\npatients, IDH mutations were associated with a distinct gene expression profile\nenriched for genes expressed in neural progenitor cells, and this was associated \nwith increased histone methylation. To test whether the ability of IDH mutants to\npromote histone methylation contributes to a block in cell differentiation in\nnon-transformed cells, we tested the effect of neomorphic IDH mutants on\nadipocyte differentiation in vitro. Introduction of either mutant IDH or\ncell-permeable 2HG was associated with repression of the inducible expression of \nlineage-specific differentiation genes and a block to differentiation. This\ncorrelated with a significant increase in repressive histone methylation marks\nwithout observable changes in promoter DNA methylation. Gliomas were found to\nhave elevated levels of similar histone repressive marks. Stable transfection of \na 2HG-producing mutant IDH into immortalized astrocytes resulted in progressive\naccumulation of histone methylation. Of the marks examined, increased H3K9\nmethylation reproducibly preceded a rise in DNA methylation as cells were\npassaged in culture. Furthermore, we found that the 2HG-inhibitable H3K9\ndemethylase KDM4C was induced during adipocyte differentiation, and that\nRNA-interference suppression of KDM4C was sufficient to block differentiation.\nTogether these data demonstrate that 2HG can inhibit histone demethylation and\nthat inhibition of histone demethylation can be sufficient to block the\ndifferentiation of non-transformed cells.\n\nPMCID: PMC3478770\nPMID: 22343901  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7917",
        "source" : "7763",
        "target" : "7767",
        "pmid" : "24217114",
        "selected" : false,
        "canonicalName" : "IDH2 (inhibits) HIF1A",
        "edgeType" : "inhibits",
        "SUID" : 7917,
        "name" : "IDH2 (inhibits) HIF1A",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "IDH2 (inhibits) HIF1A",
        "edgeDetail" : "IDH2 (inhibits) HIF1A\n\n1. Ann Med Interne (Paris). 1974 Aug-Sep;125(8-9):641-50.\n\n[Obstructive myocardiopathy, Turner-Ullrich syndrome and von Recklinghausen's\ndisease].\n\n[Article in French]\n\nGerbaux A, Belfante M, Hiltgen M.\n\nPMID: 4217114  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7916",
        "source" : "7762",
        "target" : "7760",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "IGF1R (activates) IRS1",
        "edgeType" : "activates",
        "SUID" : 7916,
        "name" : "IGF1R (activates) IRS1",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "IGF1R (activates) IRS1",
        "edgeDetail" : "IGF1R (activates) IRS1\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7915",
        "source" : "7761",
        "target" : "7788",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "INK4 family (inhibits) CCND-CDK4 complex",
        "edgeType" : "inhibits",
        "SUID" : 7915,
        "name" : "INK4 family (inhibits) CCND-CDK4 complex",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "INK4 family (inhibits) CCND-CDK4 complex",
        "edgeDetail" : "INK4 family (inhibits) CCND-CDK4 complex\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7914",
        "source" : "7761",
        "target" : "7787",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "INK4 family (inhibits) CCND-CDK6 complex",
        "edgeType" : "inhibits",
        "SUID" : 7914,
        "name" : "INK4 family (inhibits) CCND-CDK6 complex",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "INK4 family (inhibits) CCND-CDK6 complex",
        "edgeDetail" : "INK4 family (inhibits) CCND-CDK6 complex\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7913",
        "source" : "7761",
        "target" : "7684",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "INK4 family (family.includes) CDKN2A:p16",
        "edgeType" : "family.includes",
        "SUID" : 7913,
        "name" : "INK4 family (family.includes) CDKN2A:p16",
        "interaction" : "family.includes",
        "shared_interaction" : "family.includes",
        "shared_name" : "INK4 family (family.includes) CDKN2A:p16",
        "edgeDetail" : "INK4 family (family.includes) CDKN2A:p16\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7912",
        "source" : "7761",
        "target" : "7683",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "INK4 family (family.includes) CDKN2C",
        "edgeType" : "family.includes",
        "SUID" : 7912,
        "name" : "INK4 family (family.includes) CDKN2C",
        "interaction" : "family.includes",
        "shared_interaction" : "family.includes",
        "shared_name" : "INK4 family (family.includes) CDKN2C",
        "edgeDetail" : "INK4 family (family.includes) CDKN2C\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7911",
        "source" : "7761",
        "target" : "7682",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "INK4 family (family.includes) CDKN2B",
        "edgeType" : "family.includes",
        "SUID" : 7911,
        "name" : "INK4 family (family.includes) CDKN2B",
        "interaction" : "family.includes",
        "shared_interaction" : "family.includes",
        "shared_name" : "INK4 family (family.includes) CDKN2B",
        "edgeDetail" : "INK4 family (family.includes) CDKN2B\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7910",
        "source" : "7760",
        "target" : "7740",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "IRS1 (activates) PI3K class1a",
        "edgeType" : "activates",
        "SUID" : 7910,
        "name" : "IRS1 (activates) PI3K class1a",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "IRS1 (activates) PI3K class1a",
        "edgeDetail" : "IRS1 (activates) PI3K class1a\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7909",
        "source" : "7759",
        "target" : "7688",
        "pmid" : "24217114",
        "selected" : false,
        "canonicalName" : "LDHA (activates) Glycolysis",
        "edgeType" : "activates",
        "SUID" : 7909,
        "name" : "LDHA (activates) Glycolysis",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "LDHA (activates) Glycolysis",
        "edgeDetail" : "LDHA (activates) Glycolysis\n\n1. Ann Med Interne (Paris). 1974 Aug-Sep;125(8-9):641-50.\n\n[Obstructive myocardiopathy, Turner-Ullrich syndrome and von Recklinghausen's\ndisease].\n\n[Article in French]\n\nGerbaux A, Belfante M, Hiltgen M.\n\nPMID: 4217114  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7908",
        "source" : "7758",
        "target" : "7712",
        "pmid" : "24217114",
        "selected" : false,
        "canonicalName" : "LDLR (activates.process) Lipid metabolism",
        "edgeType" : "activates.process",
        "SUID" : 7908,
        "name" : "LDLR (activates.process) Lipid metabolism",
        "interaction" : "activates.process",
        "shared_interaction" : "activates.process",
        "shared_name" : "LDLR (activates.process) Lipid metabolism",
        "edgeDetail" : "LDLR (activates.process) Lipid metabolism\n\n1. Cancers (Basel). 2013 Nov 8;5(4):1469-84. doi: 10.3390/cancers5041469.\n\nTumor metabolism of malignant gliomas.\n\nRu P(1), Williams TM, Chakravarti A, Guo D.\n\nAuthor information: \n(1)Department of Radiation Oncology, Ohio State University Comprehensive Cancer\nCenter & Arthur G James Cancer Hospital, Columbus, OH 43012, USA.\ndeliang.guo@osumc.edu.\n\nConstitutively activated oncogenic signaling via genetic mutations such as in the\nEGFR/PI3K/Akt and Ras/RAF/MEK pathways has been recognized as a major driver for \ntumorigenesis in most cancers. Recent insights into tumor metabolism have further\nrevealed that oncogenic signaling pathways directly promote metabolic\nreprogramming to upregulate biosynthesis of lipids, carbohydrates, protein, DNA\nand RNA, leading to enhanced growth of human tumors. Therefore, targeting cell\nmetabolism has become a novel direction for drug development in oncology. In\nmalignant gliomas, metabolism pathways of glucose, glutamine and lipid are\nsignificantly reprogrammed. Moreover, molecular mechanisms causing these\nmetabolic changes are just starting to be unraveled. In this review, we will\nsummarize recent studies revealing critical gene alterations that lead to\nmetabolic changes in malignant gliomas, and also discuss promising therapeutic\nstrategies via targeting the key players in metabolic regulation.\n\nPMCID: PMC3875949\nPMID: 24217114  [PubMed]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7907",
        "source" : "7757",
        "target" : "7736",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "LPA (activates) PIK3C2B",
        "edgeType" : "activates",
        "SUID" : 7907,
        "name" : "LPA (activates) PIK3C2B",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "LPA (activates) PIK3C2B",
        "edgeDetail" : "LPA (activates) PIK3C2B\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7906",
        "source" : "7756",
        "target" : "7745",
        "pmid" : "24217114",
        "selected" : false,
        "canonicalName" : "LXR family (activates) MYLIP",
        "edgeType" : "activates",
        "SUID" : 7906,
        "name" : "LXR family (activates) MYLIP",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "LXR family (activates) MYLIP",
        "edgeDetail" : "LXR family (activates) MYLIP\n\n1. Cancers (Basel). 2013 Nov 8;5(4):1469-84. doi: 10.3390/cancers5041469.\n\nTumor metabolism of malignant gliomas.\n\nRu P(1), Williams TM, Chakravarti A, Guo D.\n\nAuthor information: \n(1)Department of Radiation Oncology, Ohio State University Comprehensive Cancer\nCenter & Arthur G James Cancer Hospital, Columbus, OH 43012, USA.\ndeliang.guo@osumc.edu.\n\nConstitutively activated oncogenic signaling via genetic mutations such as in the\nEGFR/PI3K/Akt and Ras/RAF/MEK pathways has been recognized as a major driver for \ntumorigenesis in most cancers. Recent insights into tumor metabolism have further\nrevealed that oncogenic signaling pathways directly promote metabolic\nreprogramming to upregulate biosynthesis of lipids, carbohydrates, protein, DNA\nand RNA, leading to enhanced growth of human tumors. Therefore, targeting cell\nmetabolism has become a novel direction for drug development in oncology. In\nmalignant gliomas, metabolism pathways of glucose, glutamine and lipid are\nsignificantly reprogrammed. Moreover, molecular mechanisms causing these\nmetabolic changes are just starting to be unraveled. In this review, we will\nsummarize recent studies revealing critical gene alterations that lead to\nmetabolic changes in malignant gliomas, and also discuss promising therapeutic\nstrategies via targeting the key players in metabolic regulation.\n\nPMCID: PMC3875949\nPMID: 24217114  [PubMed]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7905",
        "source" : "7756",
        "target" : "7797",
        "pmid" : "24217114",
        "selected" : false,
        "canonicalName" : "LXR family (activates) ABCA1",
        "edgeType" : "activates",
        "SUID" : 7905,
        "name" : "LXR family (activates) ABCA1",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "LXR family (activates) ABCA1",
        "edgeDetail" : "LXR family (activates) ABCA1\n\n1. Cancers (Basel). 2013 Nov 8;5(4):1469-84. doi: 10.3390/cancers5041469.\n\nTumor metabolism of malignant gliomas.\n\nRu P(1), Williams TM, Chakravarti A, Guo D.\n\nAuthor information: \n(1)Department of Radiation Oncology, Ohio State University Comprehensive Cancer\nCenter & Arthur G James Cancer Hospital, Columbus, OH 43012, USA.\ndeliang.guo@osumc.edu.\n\nConstitutively activated oncogenic signaling via genetic mutations such as in the\nEGFR/PI3K/Akt and Ras/RAF/MEK pathways has been recognized as a major driver for \ntumorigenesis in most cancers. Recent insights into tumor metabolism have further\nrevealed that oncogenic signaling pathways directly promote metabolic\nreprogramming to upregulate biosynthesis of lipids, carbohydrates, protein, DNA\nand RNA, leading to enhanced growth of human tumors. Therefore, targeting cell\nmetabolism has become a novel direction for drug development in oncology. In\nmalignant gliomas, metabolism pathways of glucose, glutamine and lipid are\nsignificantly reprogrammed. Moreover, molecular mechanisms causing these\nmetabolic changes are just starting to be unraveled. In this review, we will\nsummarize recent studies revealing critical gene alterations that lead to\nmetabolic changes in malignant gliomas, and also discuss promising therapeutic\nstrategies via targeting the key players in metabolic regulation.\n\nPMCID: PMC3875949\nPMID: 24217114  [PubMed]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7904",
        "source" : "7756",
        "target" : "7681",
        "pmid" : "24217114",
        "selected" : false,
        "canonicalName" : "LXR family (contains) NR1H1",
        "edgeType" : "contains",
        "SUID" : 7904,
        "name" : "LXR family (contains) NR1H1",
        "interaction" : "contains",
        "shared_interaction" : "contains",
        "shared_name" : "LXR family (contains) NR1H1",
        "edgeDetail" : "LXR family (contains) NR1H1\n\n1. Cancers (Basel). 2013 Nov 8;5(4):1469-84. doi: 10.3390/cancers5041469.\n\nTumor metabolism of malignant gliomas.\n\nRu P(1), Williams TM, Chakravarti A, Guo D.\n\nAuthor information: \n(1)Department of Radiation Oncology, Ohio State University Comprehensive Cancer\nCenter & Arthur G James Cancer Hospital, Columbus, OH 43012, USA.\ndeliang.guo@osumc.edu.\n\nConstitutively activated oncogenic signaling via genetic mutations such as in the\nEGFR/PI3K/Akt and Ras/RAF/MEK pathways has been recognized as a major driver for \ntumorigenesis in most cancers. Recent insights into tumor metabolism have further\nrevealed that oncogenic signaling pathways directly promote metabolic\nreprogramming to upregulate biosynthesis of lipids, carbohydrates, protein, DNA\nand RNA, leading to enhanced growth of human tumors. Therefore, targeting cell\nmetabolism has become a novel direction for drug development in oncology. In\nmalignant gliomas, metabolism pathways of glucose, glutamine and lipid are\nsignificantly reprogrammed. Moreover, molecular mechanisms causing these\nmetabolic changes are just starting to be unraveled. In this review, we will\nsummarize recent studies revealing critical gene alterations that lead to\nmetabolic changes in malignant gliomas, and also discuss promising therapeutic\nstrategies via targeting the key players in metabolic regulation.\n\nPMCID: PMC3875949\nPMID: 24217114  [PubMed]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7903",
        "source" : "7756",
        "target" : "7680",
        "pmid" : "24217114",
        "selected" : false,
        "canonicalName" : "LXR family (contains) NR1H2",
        "edgeType" : "contains",
        "SUID" : 7903,
        "name" : "LXR family (contains) NR1H2",
        "interaction" : "contains",
        "shared_interaction" : "contains",
        "shared_name" : "LXR family (contains) NR1H2",
        "edgeDetail" : "LXR family (contains) NR1H2\n\n1. Cancers (Basel). 2013 Nov 8;5(4):1469-84. doi: 10.3390/cancers5041469.\n\nTumor metabolism of malignant gliomas.\n\nRu P(1), Williams TM, Chakravarti A, Guo D.\n\nAuthor information: \n(1)Department of Radiation Oncology, Ohio State University Comprehensive Cancer\nCenter & Arthur G James Cancer Hospital, Columbus, OH 43012, USA.\ndeliang.guo@osumc.edu.\n\nConstitutively activated oncogenic signaling via genetic mutations such as in the\nEGFR/PI3K/Akt and Ras/RAF/MEK pathways has been recognized as a major driver for \ntumorigenesis in most cancers. Recent insights into tumor metabolism have further\nrevealed that oncogenic signaling pathways directly promote metabolic\nreprogramming to upregulate biosynthesis of lipids, carbohydrates, protein, DNA\nand RNA, leading to enhanced growth of human tumors. Therefore, targeting cell\nmetabolism has become a novel direction for drug development in oncology. In\nmalignant gliomas, metabolism pathways of glucose, glutamine and lipid are\nsignificantly reprogrammed. Moreover, molecular mechanisms causing these\nmetabolic changes are just starting to be unraveled. In this review, we will\nsummarize recent studies revealing critical gene alterations that lead to\nmetabolic changes in malignant gliomas, and also discuss promising therapeutic\nstrategies via targeting the key players in metabolic regulation.\n\nPMCID: PMC3875949\nPMID: 24217114  [PubMed]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7902",
        "source" : "7756",
        "target" : "7679",
        "pmid" : "24217114",
        "selected" : false,
        "canonicalName" : "LXR family (contains) NR1H3",
        "edgeType" : "contains",
        "SUID" : 7902,
        "name" : "LXR family (contains) NR1H3",
        "interaction" : "contains",
        "shared_interaction" : "contains",
        "shared_name" : "LXR family (contains) NR1H3",
        "edgeDetail" : "LXR family (contains) NR1H3\n\n1. Cancers (Basel). 2013 Nov 8;5(4):1469-84. doi: 10.3390/cancers5041469.\n\nTumor metabolism of malignant gliomas.\n\nRu P(1), Williams TM, Chakravarti A, Guo D.\n\nAuthor information: \n(1)Department of Radiation Oncology, Ohio State University Comprehensive Cancer\nCenter & Arthur G James Cancer Hospital, Columbus, OH 43012, USA.\ndeliang.guo@osumc.edu.\n\nConstitutively activated oncogenic signaling via genetic mutations such as in the\nEGFR/PI3K/Akt and Ras/RAF/MEK pathways has been recognized as a major driver for \ntumorigenesis in most cancers. Recent insights into tumor metabolism have further\nrevealed that oncogenic signaling pathways directly promote metabolic\nreprogramming to upregulate biosynthesis of lipids, carbohydrates, protein, DNA\nand RNA, leading to enhanced growth of human tumors. Therefore, targeting cell\nmetabolism has become a novel direction for drug development in oncology. In\nmalignant gliomas, metabolism pathways of glucose, glutamine and lipid are\nsignificantly reprogrammed. Moreover, molecular mechanisms causing these\nmetabolic changes are just starting to be unraveled. In this review, we will\nsummarize recent studies revealing critical gene alterations that lead to\nmetabolic changes in malignant gliomas, and also discuss promising therapeutic\nstrategies via targeting the key players in metabolic regulation.\n\nPMCID: PMC3875949\nPMID: 24217114  [PubMed]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7901",
        "source" : "7755",
        "target" : "7715",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "MDM2 (inhibits) TP53",
        "edgeType" : "inhibits",
        "SUID" : 7901,
        "name" : "MDM2 (inhibits) TP53",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "MDM2 (inhibits) TP53",
        "edgeDetail" : "MDM2 (inhibits) TP53\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7900",
        "source" : "7754",
        "target" : "7715",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "MDM4 (inhibits) TP53",
        "edgeType" : "inhibits",
        "SUID" : 7900,
        "name" : "MDM4 (inhibits) TP53",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "MDM4 (inhibits) TP53",
        "edgeDetail" : "MDM4 (inhibits) TP53\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7899",
        "source" : "7753",
        "target" : "7775",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "MEK1/2 (activates) ERK1/2",
        "edgeType" : "activates",
        "SUID" : 7899,
        "name" : "MEK1/2 (activates) ERK1/2",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "MEK1/2 (activates) ERK1/2",
        "edgeDetail" : "MEK1/2 (activates) ERK1/2\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7898",
        "source" : "7752",
        "target" : "7687",
        "pmid" : "23359207",
        "selected" : false,
        "canonicalName" : "MET:EGFR (contains) MET",
        "edgeType" : "contains",
        "SUID" : 7898,
        "name" : "MET:EGFR (contains) MET",
        "interaction" : "contains",
        "shared_interaction" : "contains",
        "shared_name" : "MET:EGFR (contains) MET",
        "edgeDetail" : "MET:EGFR (contains) MET\n\n1. Neoplasia. 2013 Jan;15(1):73-84.\n\nRegulation of HGF expression by EGFR-mediated c-Met activation in glioblastoma\ncells.\n\nGarnett J(1), Chumbalkar V, Vaillant B, Gururaj AE, Hill KS, Latha K, Yao J,\nPriebe W, Colman H, Elferink LA, Bogler O.\n\nAuthor information: \n(1)Department of Neurosurgery, University of Texas MD Anderson Cancer Center,\nHouston, TX 77030, USA.\n\nThe hepatocyte growth factor receptor (c-Met) and a constitutively active mutant \nof the epidermal growth factor receptor (EGFR/EGFRvIII) are frequently\noverexpressed in glioblastoma (GBM) and promote tumorigenesis. The mechanisms\nunderlying elevated hepatocyte growth factor (HGF) production in GBM are not\nunderstood. We found higher, coordinated mRNA expression levels of HGF and c-Met \nin mesenchymal (Mes) GBMs, a subtype associated with poor treatment response and \nshorter overall survival. In an HGF/c-Met-dependent GBM cell line, HGF expression\ndeclined upon silencing of c-Met using RNAi or by inhibiting its activity with\nSU11274. Silencing c-Met decreased anchorage-independent colony formation and\nincreased the survival of mice bearing intracranial GBM xenografts. Consistent\nwith these findings, c-Met activation by EGFR also elevated HGF expression, and \nthe inhibition of EGFR with AG1478 reduced HGF levels. Interestingly, c-Met\nexpression was required for EGFR-mediated HGF production, anchorage-independent \ngrowth, and in vivo tumorigenicity, suggesting that these pathways are coupled.\nUsing an unbiased mass spectrometry-based screen, we show that signal transducer \nand activator of transcription 3 (STAT3) Y705 is a downstream target of c-Met\nsignaling. Suppression of STAT3 phosphorylation with WP1193 reduced HGF\nexpression in EGFR-expressing GBM cells, whereas constitutively active STAT3\npartially rescued HGF expression and colony formation in c-Met knockdown cells\nexpressing EGFR. These results suggest that the c-Met/HGF signaling axis is\nenhanced by EGFR through increased STAT3-dependent HGF expression and that\ntargeting c-Met in Mes GBMs may be an important strategy for therapy.\n\nPMCID: PMC3556940\nPMID: 23359207  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7897",
        "source" : "7752",
        "target" : "7777",
        "pmid" : "23359207",
        "selected" : false,
        "canonicalName" : "MET:EGFR (contains) EGFR family",
        "edgeType" : "contains",
        "SUID" : 7897,
        "name" : "MET:EGFR (contains) EGFR family",
        "interaction" : "contains",
        "shared_interaction" : "contains",
        "shared_name" : "MET:EGFR (contains) EGFR family",
        "edgeDetail" : "MET:EGFR (contains) EGFR family\n\n1. Neoplasia. 2013 Jan;15(1):73-84.\n\nRegulation of HGF expression by EGFR-mediated c-Met activation in glioblastoma\ncells.\n\nGarnett J(1), Chumbalkar V, Vaillant B, Gururaj AE, Hill KS, Latha K, Yao J,\nPriebe W, Colman H, Elferink LA, Bogler O.\n\nAuthor information: \n(1)Department of Neurosurgery, University of Texas MD Anderson Cancer Center,\nHouston, TX 77030, USA.\n\nThe hepatocyte growth factor receptor (c-Met) and a constitutively active mutant \nof the epidermal growth factor receptor (EGFR/EGFRvIII) are frequently\noverexpressed in glioblastoma (GBM) and promote tumorigenesis. The mechanisms\nunderlying elevated hepatocyte growth factor (HGF) production in GBM are not\nunderstood. We found higher, coordinated mRNA expression levels of HGF and c-Met \nin mesenchymal (Mes) GBMs, a subtype associated with poor treatment response and \nshorter overall survival. In an HGF/c-Met-dependent GBM cell line, HGF expression\ndeclined upon silencing of c-Met using RNAi or by inhibiting its activity with\nSU11274. Silencing c-Met decreased anchorage-independent colony formation and\nincreased the survival of mice bearing intracranial GBM xenografts. Consistent\nwith these findings, c-Met activation by EGFR also elevated HGF expression, and \nthe inhibition of EGFR with AG1478 reduced HGF levels. Interestingly, c-Met\nexpression was required for EGFR-mediated HGF production, anchorage-independent \ngrowth, and in vivo tumorigenicity, suggesting that these pathways are coupled.\nUsing an unbiased mass spectrometry-based screen, we show that signal transducer \nand activator of transcription 3 (STAT3) Y705 is a downstream target of c-Met\nsignaling. Suppression of STAT3 phosphorylation with WP1193 reduced HGF\nexpression in EGFR-expressing GBM cells, whereas constitutively active STAT3\npartially rescued HGF expression and colony formation in c-Met knockdown cells\nexpressing EGFR. These results suggest that the c-Met/HGF signaling axis is\nenhanced by EGFR through increased STAT3-dependent HGF expression and that\ntargeting c-Met in Mes GBMs may be an important strategy for therapy.\n\nPMCID: PMC3556940\nPMID: 23359207  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7896",
        "source" : "7752",
        "target" : "7718",
        "pmid" : "23359207",
        "selected" : false,
        "canonicalName" : "MET:EGFR (activates) STAT3",
        "edgeType" : "activates",
        "SUID" : 7896,
        "name" : "MET:EGFR (activates) STAT3",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "MET:EGFR (activates) STAT3",
        "edgeDetail" : "MET:EGFR (activates) STAT3\n\n1. Neoplasia. 2013 Jan;15(1):73-84.\n\nRegulation of HGF expression by EGFR-mediated c-Met activation in glioblastoma\ncells.\n\nGarnett J(1), Chumbalkar V, Vaillant B, Gururaj AE, Hill KS, Latha K, Yao J,\nPriebe W, Colman H, Elferink LA, Bogler O.\n\nAuthor information: \n(1)Department of Neurosurgery, University of Texas MD Anderson Cancer Center,\nHouston, TX 77030, USA.\n\nThe hepatocyte growth factor receptor (c-Met) and a constitutively active mutant \nof the epidermal growth factor receptor (EGFR/EGFRvIII) are frequently\noverexpressed in glioblastoma (GBM) and promote tumorigenesis. The mechanisms\nunderlying elevated hepatocyte growth factor (HGF) production in GBM are not\nunderstood. We found higher, coordinated mRNA expression levels of HGF and c-Met \nin mesenchymal (Mes) GBMs, a subtype associated with poor treatment response and \nshorter overall survival. In an HGF/c-Met-dependent GBM cell line, HGF expression\ndeclined upon silencing of c-Met using RNAi or by inhibiting its activity with\nSU11274. Silencing c-Met decreased anchorage-independent colony formation and\nincreased the survival of mice bearing intracranial GBM xenografts. Consistent\nwith these findings, c-Met activation by EGFR also elevated HGF expression, and \nthe inhibition of EGFR with AG1478 reduced HGF levels. Interestingly, c-Met\nexpression was required for EGFR-mediated HGF production, anchorage-independent \ngrowth, and in vivo tumorigenicity, suggesting that these pathways are coupled.\nUsing an unbiased mass spectrometry-based screen, we show that signal transducer \nand activator of transcription 3 (STAT3) Y705 is a downstream target of c-Met\nsignaling. Suppression of STAT3 phosphorylation with WP1193 reduced HGF\nexpression in EGFR-expressing GBM cells, whereas constitutively active STAT3\npartially rescued HGF expression and colony formation in c-Met knockdown cells\nexpressing EGFR. These results suggest that the c-Met/HGF signaling axis is\nenhanced by EGFR through increased STAT3-dependent HGF expression and that\ntargeting c-Met in Mes GBMs may be an important strategy for therapy.\n\nPMCID: PMC3556940\nPMID: 23359207  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7895",
        "source" : "7751",
        "target" : "7678",
        "pmid" : "23818228",
        "selected" : false,
        "canonicalName" : "MGMT (activates.process) Therapy Resistance",
        "edgeType" : "activates.process",
        "SUID" : 7895,
        "name" : "MGMT (activates.process) Therapy Resistance",
        "interaction" : "activates.process",
        "shared_interaction" : "activates.process",
        "shared_name" : "MGMT (activates.process) Therapy Resistance",
        "edgeDetail" : "MGMT (activates.process) Therapy Resistance\n\n1. EMBO Mol Med. 2013 Aug;5(8):1196-212. doi: 10.1002/emmm.201302827. Epub 2013 Jul \n1.\n\nThe ZEB1 pathway links glioblastoma initiation, invasion and chemoresistance.\n\nSiebzehnrubl FA, Silver DJ, Tugertimur B, Deleyrolle LP, Siebzehnrubl D,\nSarkisian MR, Devers KG, Yachnis AT, Kupper MD, Neal D, Nabilsi NH, Kladde MP,\nSuslov O, Brabletz S, Brabletz T, Reynolds BA, Steindler DA.\n\nDepartment of Neurosurgery, University of Florida, Gainesville, FL, USA.\nfas@ufl.edu\n\nGlioblastoma remains one of the most lethal types of cancer, and is the most\ncommon brain tumour in adults. In particular, tumour recurrence after surgical\nresection and radiation invariably occurs regardless of aggressive chemotherapy. \nHere, we provide evidence that the transcription factor ZEB1 (zinc finger E-box\nbinding homeobox 1) exerts simultaneous influence over invasion, chemoresistance \nand tumourigenesis in glioblastoma. ZEB1 is preferentially expressed in invasive \nglioblastoma cells, where the ZEB1-miR-200 feedback loop interconnects these\nprocesses through the downstream effectors ROBO1, c-MYB and MGMT. Moreover, ZEB1 \nexpression in glioblastoma patients is predictive of shorter survival and poor\nTemozolomide response. Our findings indicate that this regulator of\nepithelial-mesenchymal transition orchestrates key features of cancer stem cells \nin malignant glioma and identify ROBO1, OLIG2, CD133 and MGMT as novel targets of\nthe ZEB1 pathway. Thus, ZEB1 is an important candidate molecule for glioblastoma \nrecurrence, a marker of invasive tumour cells and a potential therapeutic target,\nalong with its downstream effectors.\n\n 2013 The Authors. Published by John Wiley and Sons, Ltd on behalf of EMBO.\n\nPMID: 23818228  [PubMed - in process]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7894",
        "source" : "7751",
        "target" : "7707",
        "pmid" : "23076220",
        "selected" : false,
        "canonicalName" : "MGMT (activates.process) DNA Damage Repair",
        "edgeType" : "activates.process",
        "SUID" : 7894,
        "name" : "MGMT (activates.process) DNA Damage Repair",
        "interaction" : "activates.process",
        "shared_interaction" : "activates.process",
        "shared_name" : "MGMT (activates.process) DNA Damage Repair",
        "edgeDetail" : "MGMT (activates.process) DNA Damage Repair\n\n1. Cell Death Dis. 2012 Oct 18;3:e412. doi: 10.1038/cddis.2012.153.\n\nBMP2 sensitizes glioblastoma stem-like cells to Temozolomide by affecting HIF-1 \nstability and MGMT expression.\n\nPersano L, Pistollato F, Rampazzo E, Della Puppa A, Abbadi S, Frasson C, Volpin\nF, Indraccolo S, Scienza R, Basso G.\n\nOncohematology Laboratory, Department of Woman and Child Health, University of\nPadova, Padova, Italy. luca.persano@unipd.it\n\nGlioblastoma multiforme (GBM) is the most common brain tumour, characterized by a\ncentral and partially necrotic (i.e., hypoxic) core enriched in cancer stem cells\n(CSCs). We previously showed that the most hypoxic and immature (i.e., CSCs) GBM \ncells were resistant to Temozolomide (TMZ) in vitro, owing to a particularly high\nexpression of O6-methylguanine-DNA-methyltransferase (MGMT), the most important\nfactor associated to therapy resistance in GBM. Bone morphogenetic proteins\n(BMPs), and in particular BMP2, are known to promote differentiation and growth\ninhibition in GBM cells. For this reason, we investigated whether a BMP2-based\ntreatment would increase TMZ response in hypoxic drug-resistant GBM-derived\ncells. Here we show that BMP2 induced strong differentiation of GBM stem-like\ncells and subsequent addition of TMZ caused dramatic increase of apoptosis.\nImportantly, we correlated these effects to a BMP2-induced downregulation of both\nhypoxia-inducible factor-1 (HIF-1) and MGMT. We report here a novel mechanism\ninvolving the HIF-1-dependent regulation of MGMT, highlighting the existence of \na HIF-1/MGMT axis supporting GBM resistance to therapy. As confirmed from this\nevidence, over-stabilization of HIF-1 in TMZ-sensitive GBM cells abolished their\nresponsiveness to it. In conclusion, we describe a HIF-1-dependent regulation of\nMGMT and suggest that BMP2, by down-modulating the HIF-1/MGMT axis, should\nincrease GBM responsiveness to chemotherapy, thus opening the way to the\ndevelopment of future strategies for GBM treatment.\n\nPMCID: PMC3481140\nPMID: 23076220  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7893",
        "source" : "7750",
        "target" : "7713",
        "pmid" : "23818228",
        "selected" : false,
        "canonicalName" : "MIR200C (inhibits) ZEB1",
        "edgeType" : "inhibits",
        "SUID" : 7893,
        "name" : "MIR200C (inhibits) ZEB1",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "MIR200C (inhibits) ZEB1",
        "edgeDetail" : "MIR200C (inhibits) ZEB1\n\n1. EMBO Mol Med. 2013 Aug;5(8):1196-212. doi: 10.1002/emmm.201302827. Epub 2013 Jul \n1.\n\nThe ZEB1 pathway links glioblastoma initiation, invasion and chemoresistance.\n\nSiebzehnrubl FA, Silver DJ, Tugertimur B, Deleyrolle LP, Siebzehnrubl D,\nSarkisian MR, Devers KG, Yachnis AT, Kupper MD, Neal D, Nabilsi NH, Kladde MP,\nSuslov O, Brabletz S, Brabletz T, Reynolds BA, Steindler DA.\n\nDepartment of Neurosurgery, University of Florida, Gainesville, FL, USA.\nfas@ufl.edu\n\nGlioblastoma remains one of the most lethal types of cancer, and is the most\ncommon brain tumour in adults. In particular, tumour recurrence after surgical\nresection and radiation invariably occurs regardless of aggressive chemotherapy. \nHere, we provide evidence that the transcription factor ZEB1 (zinc finger E-box\nbinding homeobox 1) exerts simultaneous influence over invasion, chemoresistance \nand tumourigenesis in glioblastoma. ZEB1 is preferentially expressed in invasive \nglioblastoma cells, where the ZEB1-miR-200 feedback loop interconnects these\nprocesses through the downstream effectors ROBO1, c-MYB and MGMT. Moreover, ZEB1 \nexpression in glioblastoma patients is predictive of shorter survival and poor\nTemozolomide response. Our findings indicate that this regulator of\nepithelial-mesenchymal transition orchestrates key features of cancer stem cells \nin malignant glioma and identify ROBO1, OLIG2, CD133 and MGMT as novel targets of\nthe ZEB1 pathway. Thus, ZEB1 is an important candidate molecule for glioblastoma \nrecurrence, a marker of invasive tumour cells and a potential therapeutic target,\nalong with its downstream effectors.\n\n 2013 The Authors. Published by John Wiley and Sons, Ltd on behalf of EMBO.\n\nPMID: 23818228  [PubMed - in process]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7892",
        "source" : "7750",
        "target" : "7780",
        "pmid" : "23818228",
        "selected" : false,
        "canonicalName" : "MIR200C (inhibits) CMYB",
        "edgeType" : "inhibits",
        "SUID" : 7892,
        "name" : "MIR200C (inhibits) CMYB",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "MIR200C (inhibits) CMYB",
        "edgeDetail" : "MIR200C (inhibits) CMYB\n\n1. EMBO Mol Med. 2013 Aug;5(8):1196-212. doi: 10.1002/emmm.201302827. Epub 2013 Jul \n1.\n\nThe ZEB1 pathway links glioblastoma initiation, invasion and chemoresistance.\n\nSiebzehnrubl FA, Silver DJ, Tugertimur B, Deleyrolle LP, Siebzehnrubl D,\nSarkisian MR, Devers KG, Yachnis AT, Kupper MD, Neal D, Nabilsi NH, Kladde MP,\nSuslov O, Brabletz S, Brabletz T, Reynolds BA, Steindler DA.\n\nDepartment of Neurosurgery, University of Florida, Gainesville, FL, USA.\nfas@ufl.edu\n\nGlioblastoma remains one of the most lethal types of cancer, and is the most\ncommon brain tumour in adults. In particular, tumour recurrence after surgical\nresection and radiation invariably occurs regardless of aggressive chemotherapy. \nHere, we provide evidence that the transcription factor ZEB1 (zinc finger E-box\nbinding homeobox 1) exerts simultaneous influence over invasion, chemoresistance \nand tumourigenesis in glioblastoma. ZEB1 is preferentially expressed in invasive \nglioblastoma cells, where the ZEB1-miR-200 feedback loop interconnects these\nprocesses through the downstream effectors ROBO1, c-MYB and MGMT. Moreover, ZEB1 \nexpression in glioblastoma patients is predictive of shorter survival and poor\nTemozolomide response. Our findings indicate that this regulator of\nepithelial-mesenchymal transition orchestrates key features of cancer stem cells \nin malignant glioma and identify ROBO1, OLIG2, CD133 and MGMT as novel targets of\nthe ZEB1 pathway. Thus, ZEB1 is an important candidate molecule for glioblastoma \nrecurrence, a marker of invasive tumour cells and a potential therapeutic target,\nalong with its downstream effectors.\n\n 2013 The Authors. Published by John Wiley and Sons, Ltd on behalf of EMBO.\n\nPMID: 23818228  [PubMed - in process]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7891",
        "source" : "7750",
        "target" : "7677",
        "pmid" : "23818228",
        "selected" : false,
        "canonicalName" : "MIR200C (inhibits) OLIG2",
        "edgeType" : "inhibits",
        "SUID" : 7891,
        "name" : "MIR200C (inhibits) OLIG2",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "MIR200C (inhibits) OLIG2",
        "edgeDetail" : "MIR200C (inhibits) OLIG2\n\n1. EMBO Mol Med. 2013 Aug;5(8):1196-212. doi: 10.1002/emmm.201302827. Epub 2013 Jul \n1.\n\nThe ZEB1 pathway links glioblastoma initiation, invasion and chemoresistance.\n\nSiebzehnrubl FA, Silver DJ, Tugertimur B, Deleyrolle LP, Siebzehnrubl D,\nSarkisian MR, Devers KG, Yachnis AT, Kupper MD, Neal D, Nabilsi NH, Kladde MP,\nSuslov O, Brabletz S, Brabletz T, Reynolds BA, Steindler DA.\n\nDepartment of Neurosurgery, University of Florida, Gainesville, FL, USA.\nfas@ufl.edu\n\nGlioblastoma remains one of the most lethal types of cancer, and is the most\ncommon brain tumour in adults. In particular, tumour recurrence after surgical\nresection and radiation invariably occurs regardless of aggressive chemotherapy. \nHere, we provide evidence that the transcription factor ZEB1 (zinc finger E-box\nbinding homeobox 1) exerts simultaneous influence over invasion, chemoresistance \nand tumourigenesis in glioblastoma. ZEB1 is preferentially expressed in invasive \nglioblastoma cells, where the ZEB1-miR-200 feedback loop interconnects these\nprocesses through the downstream effectors ROBO1, c-MYB and MGMT. Moreover, ZEB1 \nexpression in glioblastoma patients is predictive of shorter survival and poor\nTemozolomide response. Our findings indicate that this regulator of\nepithelial-mesenchymal transition orchestrates key features of cancer stem cells \nin malignant glioma and identify ROBO1, OLIG2, CD133 and MGMT as novel targets of\nthe ZEB1 pathway. Thus, ZEB1 is an important candidate molecule for glioblastoma \nrecurrence, a marker of invasive tumour cells and a potential therapeutic target,\nalong with its downstream effectors.\n\n 2013 The Authors. Published by John Wiley and Sons, Ltd on behalf of EMBO.\n\nPMID: 23818228  [PubMed - in process]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7890",
        "source" : "7749",
        "target" : "7728",
        "pmid" : "23807265",
        "selected" : false,
        "canonicalName" : "MIR26A1 (inhibits) PTEN",
        "edgeType" : "inhibits",
        "SUID" : 7890,
        "name" : "MIR26A1 (inhibits) PTEN",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "MIR26A1 (inhibits) PTEN",
        "edgeDetail" : "MIR26A1 (inhibits) PTEN\n\n1. Epigenetics. 2013 Aug;8(8):785-95. doi: 10.4161/epi.25440. Epub 2013 Jun 27.\n\nEpigenetic pathways and glioblastoma treatment.\n\nClarke J(1), Penas C, Pastori C, Komotar RJ, Bregy A, Shah AH, Wahlestedt C, Ayad\nNG.\n\nAuthor information: \n(1)Division of Biostatistics; Department of Epidemiology and Public Health;\nUniversity of Miami Miller School of Medicine; Miami, FL USA.\n\nGlioblastoma multiforme (GBM) is the most common malignant adult brain tumor.\nStandard GBM treatment includes maximal safe surgical resection with combination \nradiotherapy and adjuvant temozolomide (TMZ) chemotherapy. Alarmingly, patient\nsurvival at five-years is below 10%. This is in part due to the invasive behavior\nof the tumor and the resulting inability to resect greater than 98% of some\ntumors. In fact, recurrence after such treatment may be inevitable, even in cases\nwhere gross total resection is achieved. The Cancer Genome Atlas (TCGA) research \nnetwork performed whole genome sequencing of GBM tumors and found that GBM\nrecurrence is linked to epigenetic mechanisms and pathways. Central to these\npathways are epigenetic enzymes, which have recently emerged as possible new drug\ntargets for multiple cancers, including GBM. Here we review GBM treatment, and\nprovide a systems approach to identifying epigenetic drivers of GBM tumor\nprogression based on temporal modeling of putative GBM cells of origin. We also\ndiscuss advances in defining epigenetic mechanisms controlling GBM initiation and\nrecurrence and the drug discovery considerations associated with targeting\nepigenetic enzymes for GBM treatment.\n\nPMCID: PMC3883781\nPMID: 23807265  [PubMed - in process]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7889",
        "source" : "7748",
        "target" : "7676",
        "pmid" : "17188374",
        "selected" : false,
        "canonicalName" : "MSK1/2 (activates.process) Chromatin remodeling",
        "edgeType" : "activates.process",
        "SUID" : 7889,
        "name" : "MSK1/2 (activates.process) Chromatin remodeling",
        "interaction" : "activates.process",
        "shared_interaction" : "activates.process",
        "shared_name" : "MSK1/2 (activates.process) Chromatin remodeling",
        "edgeDetail" : "MSK1/2 (activates.process) Chromatin remodeling\n\n1. Biochim Biophys Acta. 2007 Aug;1773(8):1299-310. Epub 2006 Nov 17.\n\nERK implication in cell cycle regulation.\n\nChambard JC(1), Lefloch R, Pouyssgur J, Lenormand P.\n\nAuthor information: \n(1)Institute of Signaling Developmental Biology and Cancer, CNRS UMR 6543,\nUniversite de Nice-Sofia Antipolis, Centre A. Lacassagne, 33 Avenue de\nValombrose, 06189 Nice, France.\n\nThe Ras/Raf/MEK/ERK signaling cascade that integrates an extreme variety of\nextracellular stimuli into key biological responses controlling cell\nproliferation, differentiation or death is one of the most studied intracellular \npathways. Here we present some evidences that have been accumulated over the last\n15 years proving the requirement of ERK in the control of cell proliferation. In \nthis review we focus (i) on the spatio-temporal control of ERK signaling, (ii) on\nthe key cellular components linking extracellular signals to the induction and\nactivation of cell cycle events controlling G1 to S-phase transition and (iii) on\nthe role of ERK in the growth factor-independent G2/M phase of the cell cycle. As\nERK pathway is often co-activated with the PI3 kinase signaling, we highlight\nsome of the key points of convergence leading to a full activation of mTOR via\nERK and AKT synergies. Finally, ERK and AKT targets being constitutively\nactivated in so many human cancers, we briefly touched the cure issue of using\nmore specific drugs in rationally selected cancer patients.\n\nPMID: 17188374  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7888",
        "source" : "7747",
        "target" : "7675",
        "pmid" : "23641065",
        "selected" : false,
        "canonicalName" : "mTORC1 (complex.includes) mTOR",
        "edgeType" : "complex.includes",
        "SUID" : 7888,
        "name" : "mTORC1 (complex.includes) mTOR",
        "interaction" : "complex.includes",
        "shared_interaction" : "complex.includes",
        "shared_name" : "mTORC1 (complex.includes) mTOR",
        "edgeDetail" : "mTORC1 (complex.includes) mTOR\n\n1. J Cell Sci. 2013 Apr 15;126(Pt 8):1713-9. doi: 10.1242/jcs.125773. Epub 2013 May \n2.\n\nRegulation of mTORC1 and its impact on gene expression at a glance.\n\nLaplante M, Sabatini DM.\n\nCentre de Recherche de l'Institut Universitaire de Cardiologie et de Pneumologie \nde Qubec (CRIUCPQ), Facult de Mdecine, Universit Laval, 2725 Chemin Ste-Foy, \nQubec, QC, G1V 4G5, Canada. mathieu.laplante@criucpq.ulaval.ca\n\nThe mechanistic (or mammalian) target of rapamycin (mTOR) is a kinase that\nregulates key cellular functions linked to the promotion of cell growth and\nmetabolism. This kinase, which is part of two protein complexes termed mTOR\ncomplex 1 (mTORC1) and 2 (mTORC2), has a fundamental role in coordinating\nanabolic and catabolic processes in response to growth factors and nutrients. Of \nthe two mTOR complexes, mTORC1 is by far the best characterized. When active,\nmTORC1 triggers cell growth and proliferation by promoting protein synthesis,\nlipid biogenesis, and metabolism, and by reducing autophagy. The fact that mTORC1\nderegulation is associated with several human diseases, such as type 2 diabetes, \ncancer, obesity and neurodegeneration, highlights its importance in the\nmaintenance of cellular homeostasis. Over the last years, several groups observed\nthat mTORC1 inhibition, in addition to reducing protein synthesis, deeply affects\ngene transcription. Here, we review the connections between mTORC1 and gene\ntranscription by focusing on its impact in regulating the activation of specific \ntranscription factors including including STAT3, SREBPs, PPAR, PPAR, HIF1,\nYY1PGC1 and TFEB. We also discuss the importance of these transcription factors\nin mediating the effects of mTORC1 on various cellular processes in physiological\nand pathological contexts.\n\nPMCID: PMC3678406 [Available on 2014/4/15]\nPMID: 23641065  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7887",
        "source" : "7747",
        "target" : "7674",
        "pmid" : "23641065",
        "selected" : false,
        "canonicalName" : "mTORC1 (complex.includes) DEPTOR",
        "edgeType" : "complex.includes",
        "SUID" : 7887,
        "name" : "mTORC1 (complex.includes) DEPTOR",
        "interaction" : "complex.includes",
        "shared_interaction" : "complex.includes",
        "shared_name" : "mTORC1 (complex.includes) DEPTOR",
        "edgeDetail" : "mTORC1 (complex.includes) DEPTOR\n\n1. J Cell Sci. 2013 Apr 15;126(Pt 8):1713-9. doi: 10.1242/jcs.125773. Epub 2013 May \n2.\n\nRegulation of mTORC1 and its impact on gene expression at a glance.\n\nLaplante M, Sabatini DM.\n\nCentre de Recherche de l'Institut Universitaire de Cardiologie et de Pneumologie \nde Qubec (CRIUCPQ), Facult de Mdecine, Universit Laval, 2725 Chemin Ste-Foy, \nQubec, QC, G1V 4G5, Canada. mathieu.laplante@criucpq.ulaval.ca\n\nThe mechanistic (or mammalian) target of rapamycin (mTOR) is a kinase that\nregulates key cellular functions linked to the promotion of cell growth and\nmetabolism. This kinase, which is part of two protein complexes termed mTOR\ncomplex 1 (mTORC1) and 2 (mTORC2), has a fundamental role in coordinating\nanabolic and catabolic processes in response to growth factors and nutrients. Of \nthe two mTOR complexes, mTORC1 is by far the best characterized. When active,\nmTORC1 triggers cell growth and proliferation by promoting protein synthesis,\nlipid biogenesis, and metabolism, and by reducing autophagy. The fact that mTORC1\nderegulation is associated with several human diseases, such as type 2 diabetes, \ncancer, obesity and neurodegeneration, highlights its importance in the\nmaintenance of cellular homeostasis. Over the last years, several groups observed\nthat mTORC1 inhibition, in addition to reducing protein synthesis, deeply affects\ngene transcription. Here, we review the connections between mTORC1 and gene\ntranscription by focusing on its impact in regulating the activation of specific \ntranscription factors including including STAT3, SREBPs, PPAR, PPAR, HIF1,\nYY1PGC1 and TFEB. We also discuss the importance of these transcription factors\nin mediating the effects of mTORC1 on various cellular processes in physiological\nand pathological contexts.\n\nPMCID: PMC3678406 [Available on 2014/4/15]\nPMID: 23641065  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7886",
        "source" : "7747",
        "target" : "7673",
        "pmid" : "23641065",
        "selected" : false,
        "canonicalName" : "mTORC1 (complex.includes) PRAS40",
        "edgeType" : "complex.includes",
        "SUID" : 7886,
        "name" : "mTORC1 (complex.includes) PRAS40",
        "interaction" : "complex.includes",
        "shared_interaction" : "complex.includes",
        "shared_name" : "mTORC1 (complex.includes) PRAS40",
        "edgeDetail" : "mTORC1 (complex.includes) PRAS40\n\n1. J Cell Sci. 2013 Apr 15;126(Pt 8):1713-9. doi: 10.1242/jcs.125773. Epub 2013 May \n2.\n\nRegulation of mTORC1 and its impact on gene expression at a glance.\n\nLaplante M, Sabatini DM.\n\nCentre de Recherche de l'Institut Universitaire de Cardiologie et de Pneumologie \nde Qubec (CRIUCPQ), Facult de Mdecine, Universit Laval, 2725 Chemin Ste-Foy, \nQubec, QC, G1V 4G5, Canada. mathieu.laplante@criucpq.ulaval.ca\n\nThe mechanistic (or mammalian) target of rapamycin (mTOR) is a kinase that\nregulates key cellular functions linked to the promotion of cell growth and\nmetabolism. This kinase, which is part of two protein complexes termed mTOR\ncomplex 1 (mTORC1) and 2 (mTORC2), has a fundamental role in coordinating\nanabolic and catabolic processes in response to growth factors and nutrients. Of \nthe two mTOR complexes, mTORC1 is by far the best characterized. When active,\nmTORC1 triggers cell growth and proliferation by promoting protein synthesis,\nlipid biogenesis, and metabolism, and by reducing autophagy. The fact that mTORC1\nderegulation is associated with several human diseases, such as type 2 diabetes, \ncancer, obesity and neurodegeneration, highlights its importance in the\nmaintenance of cellular homeostasis. Over the last years, several groups observed\nthat mTORC1 inhibition, in addition to reducing protein synthesis, deeply affects\ngene transcription. Here, we review the connections between mTORC1 and gene\ntranscription by focusing on its impact in regulating the activation of specific \ntranscription factors including including STAT3, SREBPs, PPAR, PPAR, HIF1,\nYY1PGC1 and TFEB. We also discuss the importance of these transcription factors\nin mediating the effects of mTORC1 on various cellular processes in physiological\nand pathological contexts.\n\nPMCID: PMC3678406 [Available on 2014/4/15]\nPMID: 23641065  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7885",
        "source" : "7747",
        "target" : "7672",
        "pmid" : "23641065",
        "selected" : false,
        "canonicalName" : "mTORC1 (complex.includes) MLST8",
        "edgeType" : "complex.includes",
        "SUID" : 7885,
        "name" : "mTORC1 (complex.includes) MLST8",
        "interaction" : "complex.includes",
        "shared_interaction" : "complex.includes",
        "shared_name" : "mTORC1 (complex.includes) MLST8",
        "edgeDetail" : "mTORC1 (complex.includes) MLST8\n\n1. J Cell Sci. 2013 Apr 15;126(Pt 8):1713-9. doi: 10.1242/jcs.125773. Epub 2013 May \n2.\n\nRegulation of mTORC1 and its impact on gene expression at a glance.\n\nLaplante M, Sabatini DM.\n\nCentre de Recherche de l'Institut Universitaire de Cardiologie et de Pneumologie \nde Qubec (CRIUCPQ), Facult de Mdecine, Universit Laval, 2725 Chemin Ste-Foy, \nQubec, QC, G1V 4G5, Canada. mathieu.laplante@criucpq.ulaval.ca\n\nThe mechanistic (or mammalian) target of rapamycin (mTOR) is a kinase that\nregulates key cellular functions linked to the promotion of cell growth and\nmetabolism. This kinase, which is part of two protein complexes termed mTOR\ncomplex 1 (mTORC1) and 2 (mTORC2), has a fundamental role in coordinating\nanabolic and catabolic processes in response to growth factors and nutrients. Of \nthe two mTOR complexes, mTORC1 is by far the best characterized. When active,\nmTORC1 triggers cell growth and proliferation by promoting protein synthesis,\nlipid biogenesis, and metabolism, and by reducing autophagy. The fact that mTORC1\nderegulation is associated with several human diseases, such as type 2 diabetes, \ncancer, obesity and neurodegeneration, highlights its importance in the\nmaintenance of cellular homeostasis. Over the last years, several groups observed\nthat mTORC1 inhibition, in addition to reducing protein synthesis, deeply affects\ngene transcription. Here, we review the connections between mTORC1 and gene\ntranscription by focusing on its impact in regulating the activation of specific \ntranscription factors including including STAT3, SREBPs, PPAR, PPAR, HIF1,\nYY1PGC1 and TFEB. We also discuss the importance of these transcription factors\nin mediating the effects of mTORC1 on various cellular processes in physiological\nand pathological contexts.\n\nPMCID: PMC3678406 [Available on 2014/4/15]\nPMID: 23641065  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7884",
        "source" : "7747",
        "target" : "7729",
        "pmid" : "18701474",
        "selected" : false,
        "canonicalName" : "mTORC1 (activates) pS6K",
        "edgeType" : "activates",
        "SUID" : 7884,
        "name" : "mTORC1 (activates) pS6K",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "mTORC1 (activates) pS6K",
        "edgeDetail" : "mTORC1 (activates) pS6K\n\n1. Cancer Res. 2008 Aug 15;68(16):6516-23. doi: 10.1158/0008-5472.CAN-07-6188.\n\nS6K1 plays a key role in glial transformation.\n\nNakamura JL, Garcia E, Pieper RO.\n\nDepartment of Radiation Oncology, University of California, San Francisco,\nCalifornia 94143, USA. jnakamura@radonc.ucsf.edu\n\nThe mammalian target of rapamycin (mTOR) is a nutrient and ATP sensor suggested\nto play an important role in tumorigenesis, particularly in the setting of PTEN\nloss or activated Akt/PKB. Of mTOR's two known effectors, eIF4E has been\nimplicated in tumorigenesis, whereas the role of S6 kinase (S6K1) in\ntransformation is less understood. To assess the contribution of S6K1 to the\ntransformed phenotype, we pharmacologically and genetically manipulated the\nmTOR-S6K pathway in glioma cells and monitored its effects on growth in soft\nagar, a hallmark of cellular transformation, and also assessed in vivo\nintracranial growth. Anchorage-independent growth by HRas(V12)-transformed human \nastrocytes as well as by U251 and U373 human glioma cells was inhibited by\npharmacologic mTOR inhibition. Similarly, short hairpin RNA-mediated suppression \nof mTOR also reduced anchorage-independent growth of glioma cell lines.\nExpression of wild-type eIF4E in rapamycin-treated E6/E7/hTert/HRas(V12) and U373\ncells failed to rescue colony formation, although expression of wild-type S6K1 or\nrapamycin-resistant S6K1 in rapamycin-treated U373 and U251 provided partial\nrescue. Consistent with the latter observation, small interfering RNA-mediated\nsuppression of S6K1 in HRas(V12)-transformed human astrocytes, U251, and U373\ncells resulted in a significant loss of anchorage-independent growth.\nFurthermore, we found that in vivo short hairpin RNA-mediated suppression of S6K1\nin HRas(V12)-transformed human astrocytes reduced intracranial tumor size, in\nassociation with reduced tumor levels of phosphorylated ribosomal protein S6.\nThese findings implicate the mTOR-S6K pathway as a critical mediator of glial\ncell transformation.\n\nPMCID: PMC2538583\nPMID: 18701474  [PubMed - indexed for MEDLINE]\n\n\n\n1. Int J Oncol. 2009 Oct;35(4):731-40.\n\nInvolvement of mTORC1 and mTORC2 in regulation of glioblastoma multiforme growth \nand motility.\n\nGulati N, Karsy M, Albert L, Murali R, Jhanwar-Uniyal M.\n\nDepartment of Neurosurgery, New York Medical College, Valhalla, NY 10595, USA.\n\nThe AKT/mammalian target of rapamycin (mTOR) signaling pathway plays a critical\nrole in glioblastoma multiforme (GBM) oncogenesis due to activation of AKT. We\nstudied two distinct complexes, mTOR complex 1 (mTORC1) and mTOR complex 2\n(mTORC2), through which mTOR controls cell survival, growth and motility.\nInhibition of mTOR by rapamycin (RAPA) resulted in time-dependent suppression of \nS6 ribosomal protein (pS6KSer235/236; mTORC1 substrate) and caused transient\nsuppression of pAKTSer473 (mTORC2 substrate) at 1 to 3 h followed by a consistent\nincrease from 6 to 24 h. Inhibition of mTOR or phosphoinositide 3-kinase (PI3K)\nsuppressed platelet-derived growth factor (PDGF)- or fibronectin (FN)-induced\nactivation of p70S6KThr389. Combined inhibition of mTOR and PI3K abolished PDGF- \nor FN-induced activation of STAT3Ser727. Expression of pAKT was suppressed by\nsiRNA silencing of mTORC2 co-protein rictor, but not by mTORC1 co-protein raptor.\nGBM cell proliferation and motility paralleled the activation of mTORC2. Combined\ninhibition of mTOR and PI3K had an additive effect on suppressing cell growth and\nmotility. PDGF-induced nuclear localization of mTOR was blocked by pre-treatment \nwith RAPA. The results demonstrated that an activation of mTORC2 occurs when\nmTORC1 is inhibited by RAPA. Therefore, simultaneous suppression of mTORC1 and\nmTORC2 may provide novel therapy for GBM.\n\nPMID: 19724909  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7883",
        "source" : "7746",
        "target" : "7769",
        "pmid" : "24217114",
        "selected" : false,
        "canonicalName" : "MYC (activates) GLUT1",
        "edgeType" : "activates",
        "SUID" : 7883,
        "name" : "MYC (activates) GLUT1",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "MYC (activates) GLUT1",
        "edgeDetail" : "MYC (activates) GLUT1\n\n1. Ann Med Interne (Paris). 1974 Aug-Sep;125(8-9):641-50.\n\n[Obstructive myocardiopathy, Turner-Ullrich syndrome and von Recklinghausen's\ndisease].\n\n[Article in French]\n\nGerbaux A, Belfante M, Hiltgen M.\n\nPMID: 4217114  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7882",
        "source" : "7746",
        "target" : "7759",
        "pmid" : "24217114",
        "selected" : false,
        "canonicalName" : "MYC (activates) LDHA",
        "edgeType" : "activates",
        "SUID" : 7882,
        "name" : "MYC (activates) LDHA",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "MYC (activates) LDHA",
        "edgeDetail" : "MYC (activates) LDHA\n\n1. Ann Med Interne (Paris). 1974 Aug-Sep;125(8-9):641-50.\n\n[Obstructive myocardiopathy, Turner-Ullrich syndrome and von Recklinghausen's\ndisease].\n\n[Article in French]\n\nGerbaux A, Belfante M, Hiltgen M.\n\nPMID: 4217114  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7881",
        "source" : "7745",
        "target" : "7758",
        "pmid" : "24217114",
        "selected" : false,
        "canonicalName" : "MYLIP (inhibits) LDLR",
        "edgeType" : "inhibits",
        "SUID" : 7881,
        "name" : "MYLIP (inhibits) LDLR",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "MYLIP (inhibits) LDLR",
        "edgeDetail" : "MYLIP (inhibits) LDLR\n\n1. Cancers (Basel). 2013 Nov 8;5(4):1469-84. doi: 10.3390/cancers5041469.\n\nTumor metabolism of malignant gliomas.\n\nRu P(1), Williams TM, Chakravarti A, Guo D.\n\nAuthor information: \n(1)Department of Radiation Oncology, Ohio State University Comprehensive Cancer\nCenter & Arthur G James Cancer Hospital, Columbus, OH 43012, USA.\ndeliang.guo@osumc.edu.\n\nConstitutively activated oncogenic signaling via genetic mutations such as in the\nEGFR/PI3K/Akt and Ras/RAF/MEK pathways has been recognized as a major driver for \ntumorigenesis in most cancers. Recent insights into tumor metabolism have further\nrevealed that oncogenic signaling pathways directly promote metabolic\nreprogramming to upregulate biosynthesis of lipids, carbohydrates, protein, DNA\nand RNA, leading to enhanced growth of human tumors. Therefore, targeting cell\nmetabolism has become a novel direction for drug development in oncology. In\nmalignant gliomas, metabolism pathways of glucose, glutamine and lipid are\nsignificantly reprogrammed. Moreover, molecular mechanisms causing these\nmetabolic changes are just starting to be unraveled. In this review, we will\nsummarize recent studies revealing critical gene alterations that lead to\nmetabolic changes in malignant gliomas, and also discuss promising therapeutic\nstrategies via targeting the key players in metabolic regulation.\n\nPMCID: PMC3875949\nPMID: 24217114  [PubMed]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7880",
        "source" : "7744",
        "target" : "7726",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "NF1 (inhibits) RAS family",
        "edgeType" : "inhibits",
        "SUID" : 7880,
        "name" : "NF1 (inhibits) RAS family",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "NF1 (inhibits) RAS family",
        "edgeDetail" : "NF1 (inhibits) RAS family\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7879",
        "source" : "7743",
        "target" : "7767",
        "pmid" : "24217114",
        "selected" : false,
        "canonicalName" : "NF-kB:HIF1A (contains) HIF1A",
        "edgeType" : "contains",
        "SUID" : 7879,
        "name" : "NF-kB:HIF1A (contains) HIF1A",
        "interaction" : "contains",
        "shared_interaction" : "contains",
        "shared_name" : "NF-kB:HIF1A (contains) HIF1A",
        "edgeDetail" : "NF-kB:HIF1A (contains) HIF1A\n\n1. Cancers (Basel). 2013 Nov 8;5(4):1469-84. doi: 10.3390/cancers5041469.\n\nTumor metabolism of malignant gliomas.\n\nRu P(1), Williams TM, Chakravarti A, Guo D.\n\nAuthor information: \n(1)Department of Radiation Oncology, Ohio State University Comprehensive Cancer\nCenter & Arthur G James Cancer Hospital, Columbus, OH 43012, USA.\ndeliang.guo@osumc.edu.\n\nConstitutively activated oncogenic signaling via genetic mutations such as in the\nEGFR/PI3K/Akt and Ras/RAF/MEK pathways has been recognized as a major driver for \ntumorigenesis in most cancers. Recent insights into tumor metabolism have further\nrevealed that oncogenic signaling pathways directly promote metabolic\nreprogramming to upregulate biosynthesis of lipids, carbohydrates, protein, DNA\nand RNA, leading to enhanced growth of human tumors. Therefore, targeting cell\nmetabolism has become a novel direction for drug development in oncology. In\nmalignant gliomas, metabolism pathways of glucose, glutamine and lipid are\nsignificantly reprogrammed. Moreover, molecular mechanisms causing these\nmetabolic changes are just starting to be unraveled. In this review, we will\nsummarize recent studies revealing critical gene alterations that lead to\nmetabolic changes in malignant gliomas, and also discuss promising therapeutic\nstrategies via targeting the key players in metabolic regulation.\n\nPMCID: PMC3875949\nPMID: 24217114  [PubMed]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7878",
        "source" : "7743",
        "target" : "7671",
        "pmid" : "24217114",
        "selected" : false,
        "canonicalName" : "NF-kB:HIF1A (contains) NFKB1",
        "edgeType" : "contains",
        "SUID" : 7878,
        "name" : "NF-kB:HIF1A (contains) NFKB1",
        "interaction" : "contains",
        "shared_interaction" : "contains",
        "shared_name" : "NF-kB:HIF1A (contains) NFKB1",
        "edgeDetail" : "NF-kB:HIF1A (contains) NFKB1\n\n1. Cancers (Basel). 2013 Nov 8;5(4):1469-84. doi: 10.3390/cancers5041469.\n\nTumor metabolism of malignant gliomas.\n\nRu P(1), Williams TM, Chakravarti A, Guo D.\n\nAuthor information: \n(1)Department of Radiation Oncology, Ohio State University Comprehensive Cancer\nCenter & Arthur G James Cancer Hospital, Columbus, OH 43012, USA.\ndeliang.guo@osumc.edu.\n\nConstitutively activated oncogenic signaling via genetic mutations such as in the\nEGFR/PI3K/Akt and Ras/RAF/MEK pathways has been recognized as a major driver for \ntumorigenesis in most cancers. Recent insights into tumor metabolism have further\nrevealed that oncogenic signaling pathways directly promote metabolic\nreprogramming to upregulate biosynthesis of lipids, carbohydrates, protein, DNA\nand RNA, leading to enhanced growth of human tumors. Therefore, targeting cell\nmetabolism has become a novel direction for drug development in oncology. In\nmalignant gliomas, metabolism pathways of glucose, glutamine and lipid are\nsignificantly reprogrammed. Moreover, molecular mechanisms causing these\nmetabolic changes are just starting to be unraveled. In this review, we will\nsummarize recent studies revealing critical gene alterations that lead to\nmetabolic changes in malignant gliomas, and also discuss promising therapeutic\nstrategies via targeting the key players in metabolic regulation.\n\nPMCID: PMC3875949\nPMID: 24217114  [PubMed]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7877",
        "source" : "7743",
        "target" : "7670",
        "pmid" : "24217114",
        "selected" : false,
        "canonicalName" : "NF-kB:HIF1A (contains) RELA",
        "edgeType" : "contains",
        "SUID" : 7877,
        "name" : "NF-kB:HIF1A (contains) RELA",
        "interaction" : "contains",
        "shared_interaction" : "contains",
        "shared_name" : "NF-kB:HIF1A (contains) RELA",
        "edgeDetail" : "NF-kB:HIF1A (contains) RELA\n\n1. Cancers (Basel). 2013 Nov 8;5(4):1469-84. doi: 10.3390/cancers5041469.\n\nTumor metabolism of malignant gliomas.\n\nRu P(1), Williams TM, Chakravarti A, Guo D.\n\nAuthor information: \n(1)Department of Radiation Oncology, Ohio State University Comprehensive Cancer\nCenter & Arthur G James Cancer Hospital, Columbus, OH 43012, USA.\ndeliang.guo@osumc.edu.\n\nConstitutively activated oncogenic signaling via genetic mutations such as in the\nEGFR/PI3K/Akt and Ras/RAF/MEK pathways has been recognized as a major driver for \ntumorigenesis in most cancers. Recent insights into tumor metabolism have further\nrevealed that oncogenic signaling pathways directly promote metabolic\nreprogramming to upregulate biosynthesis of lipids, carbohydrates, protein, DNA\nand RNA, leading to enhanced growth of human tumors. Therefore, targeting cell\nmetabolism has become a novel direction for drug development in oncology. In\nmalignant gliomas, metabolism pathways of glucose, glutamine and lipid are\nsignificantly reprogrammed. Moreover, molecular mechanisms causing these\nmetabolic changes are just starting to be unraveled. In this review, we will\nsummarize recent studies revealing critical gene alterations that lead to\nmetabolic changes in malignant gliomas, and also discuss promising therapeutic\nstrategies via targeting the key players in metabolic regulation.\n\nPMCID: PMC3875949\nPMID: 24217114  [PubMed]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7876",
        "source" : "7743",
        "target" : "7733",
        "pmid" : "24217114",
        "selected" : false,
        "canonicalName" : "NF-kB:HIF1A (activates) PKM2",
        "edgeType" : "activates",
        "SUID" : 7876,
        "name" : "NF-kB:HIF1A (activates) PKM2",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "NF-kB:HIF1A (activates) PKM2",
        "edgeDetail" : "NF-kB:HIF1A (activates) PKM2\n\n1. Cancers (Basel). 2013 Nov 8;5(4):1469-84. doi: 10.3390/cancers5041469.\n\nTumor metabolism of malignant gliomas.\n\nRu P(1), Williams TM, Chakravarti A, Guo D.\n\nAuthor information: \n(1)Department of Radiation Oncology, Ohio State University Comprehensive Cancer\nCenter & Arthur G James Cancer Hospital, Columbus, OH 43012, USA.\ndeliang.guo@osumc.edu.\n\nConstitutively activated oncogenic signaling via genetic mutations such as in the\nEGFR/PI3K/Akt and Ras/RAF/MEK pathways has been recognized as a major driver for \ntumorigenesis in most cancers. Recent insights into tumor metabolism have further\nrevealed that oncogenic signaling pathways directly promote metabolic\nreprogramming to upregulate biosynthesis of lipids, carbohydrates, protein, DNA\nand RNA, leading to enhanced growth of human tumors. Therefore, targeting cell\nmetabolism has become a novel direction for drug development in oncology. In\nmalignant gliomas, metabolism pathways of glucose, glutamine and lipid are\nsignificantly reprogrammed. Moreover, molecular mechanisms causing these\nmetabolic changes are just starting to be unraveled. In this review, we will\nsummarize recent studies revealing critical gene alterations that lead to\nmetabolic changes in malignant gliomas, and also discuss promising therapeutic\nstrategies via targeting the key players in metabolic regulation.\n\nPMCID: PMC3875949\nPMID: 24217114  [PubMed]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7875",
        "source" : "7742",
        "target" : "7669",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "PDGFR family (family.includes) PDGFRB",
        "edgeType" : "family.includes",
        "SUID" : 7875,
        "name" : "PDGFR family (family.includes) PDGFRB",
        "interaction" : "family.includes",
        "shared_interaction" : "family.includes",
        "shared_name" : "PDGFR family (family.includes) PDGFRB",
        "edgeDetail" : "PDGFR family (family.includes) PDGFRB\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7874",
        "source" : "7742",
        "target" : "7668",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "PDGFR family (family.includes) PDGFRA",
        "edgeType" : "family.includes",
        "SUID" : 7874,
        "name" : "PDGFR family (family.includes) PDGFRA",
        "interaction" : "family.includes",
        "shared_interaction" : "family.includes",
        "shared_name" : "PDGFR family (family.includes) PDGFRA",
        "edgeDetail" : "PDGFR family (family.includes) PDGFRA\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7873",
        "source" : "7741",
        "target" : "7796",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "PDPK1 (activates) AKT family",
        "edgeType" : "activates",
        "SUID" : 7873,
        "name" : "PDPK1 (activates) AKT family",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "PDPK1 (activates) AKT family",
        "edgeDetail" : "PDPK1 (activates) AKT family\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7872",
        "source" : "7740",
        "target" : "7667",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "PI3K class1a (contains) PIK3R1",
        "edgeType" : "contains",
        "SUID" : 7872,
        "name" : "PI3K class1a (contains) PIK3R1",
        "interaction" : "contains",
        "shared_interaction" : "contains",
        "shared_name" : "PI3K class1a (contains) PIK3R1",
        "edgeDetail" : "PI3K class1a (contains) PIK3R1\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7871",
        "source" : "7740",
        "target" : "7666",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "PI3K class1a (contains) PIK3R2",
        "edgeType" : "contains",
        "SUID" : 7871,
        "name" : "PI3K class1a (contains) PIK3R2",
        "interaction" : "contains",
        "shared_interaction" : "contains",
        "shared_name" : "PI3K class1a (contains) PIK3R2",
        "edgeDetail" : "PI3K class1a (contains) PIK3R2\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7870",
        "source" : "7740",
        "target" : "7665",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "PI3K class1a (contains) PIK3CA",
        "edgeType" : "contains",
        "SUID" : 7870,
        "name" : "PI3K class1a (contains) PIK3CA",
        "interaction" : "contains",
        "shared_interaction" : "contains",
        "shared_name" : "PI3K class1a (contains) PIK3CA",
        "edgeDetail" : "PI3K class1a (contains) PIK3CA\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7869",
        "source" : "7740",
        "target" : "7664",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "PI3K class1a (contains) PIK3CB",
        "edgeType" : "contains",
        "SUID" : 7869,
        "name" : "PI3K class1a (contains) PIK3CB",
        "interaction" : "contains",
        "shared_interaction" : "contains",
        "shared_name" : "PI3K class1a (contains) PIK3CB",
        "edgeDetail" : "PI3K class1a (contains) PIK3CB\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7868",
        "source" : "7740",
        "target" : "7663",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "PI3K class1a (contains) PIK3CD",
        "edgeType" : "contains",
        "SUID" : 7868,
        "name" : "PI3K class1a (contains) PIK3CD",
        "interaction" : "contains",
        "shared_interaction" : "contains",
        "shared_name" : "PI3K class1a (contains) PIK3CD",
        "edgeDetail" : "PI3K class1a (contains) PIK3CD\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7867",
        "source" : "7740",
        "target" : "7735",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "PI3K class1a (activates) PIP3",
        "edgeType" : "activates",
        "SUID" : 7867,
        "name" : "PI3K class1a (activates) PIP3",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "PI3K class1a (activates) PIP3",
        "edgeDetail" : "PI3K class1a (activates) PIP3\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7866",
        "source" : "7739",
        "target" : "7662",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "PI3K class1b (contains) PIK3CG",
        "edgeType" : "contains",
        "SUID" : 7866,
        "name" : "PI3K class1b (contains) PIK3CG",
        "interaction" : "contains",
        "shared_interaction" : "contains",
        "shared_name" : "PI3K class1b (contains) PIK3CG",
        "edgeDetail" : "PI3K class1b (contains) PIK3CG\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7865",
        "source" : "7739",
        "target" : "7735",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "PI3K class1b (activates) PIP3",
        "edgeType" : "activates",
        "SUID" : 7865,
        "name" : "PI3K class1b (activates) PIP3",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "PI3K class1b (activates) PIP3",
        "edgeDetail" : "PI3K class1b (activates) PIP3\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7864",
        "source" : "7738",
        "target" : "7661",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "PI3K class2 (class.includes) PIK3C2A",
        "edgeType" : "class.includes",
        "SUID" : 7864,
        "name" : "PI3K class2 (class.includes) PIK3C2A",
        "interaction" : "class.includes",
        "shared_interaction" : "class.includes",
        "shared_name" : "PI3K class2 (class.includes) PIK3C2A",
        "edgeDetail" : "PI3K class2 (class.includes) PIK3C2A\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7863",
        "source" : "7738",
        "target" : "7736",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "PI3K class2 (class.includes) PIK3C2B",
        "edgeType" : "class.includes",
        "SUID" : 7863,
        "name" : "PI3K class2 (class.includes) PIK3C2B",
        "interaction" : "class.includes",
        "shared_interaction" : "class.includes",
        "shared_name" : "PI3K class2 (class.includes) PIK3C2B",
        "edgeDetail" : "PI3K class2 (class.includes) PIK3C2B\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7862",
        "source" : "7738",
        "target" : "7660",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "PI3K class2 (class.includes) PIK3C2G",
        "edgeType" : "class.includes",
        "SUID" : 7862,
        "name" : "PI3K class2 (class.includes) PIK3C2G",
        "interaction" : "class.includes",
        "shared_interaction" : "class.includes",
        "shared_name" : "PI3K class2 (class.includes) PIK3C2G",
        "edgeDetail" : "PI3K class2 (class.includes) PIK3C2G\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7861",
        "source" : "7737",
        "target" : "7705",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "PI-3-P (activates.process) CELL MIGRATION",
        "edgeType" : "activates.process",
        "SUID" : 7861,
        "name" : "PI-3-P (activates.process) CELL MIGRATION",
        "interaction" : "activates.process",
        "shared_interaction" : "activates.process",
        "shared_name" : "PI-3-P (activates.process) CELL MIGRATION",
        "edgeDetail" : "PI-3-P (activates.process) CELL MIGRATION\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7860",
        "source" : "7736",
        "target" : "7737",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "PIK3C2B (activates) PI-3-P",
        "edgeType" : "activates",
        "SUID" : 7860,
        "name" : "PIK3C2B (activates) PI-3-P",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "PIK3C2B (activates) PI-3-P",
        "edgeDetail" : "PIK3C2B (activates) PI-3-P\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7859",
        "source" : "7735",
        "target" : "7731",
        "pmid" : "20068170",
        "selected" : false,
        "canonicalName" : "PIP3 (inhibits) polycomb repressive complex",
        "edgeType" : "inhibits",
        "SUID" : 7859,
        "name" : "PIP3 (inhibits) polycomb repressive complex",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "PIP3 (inhibits) polycomb repressive complex",
        "edgeDetail" : "PIP3 (inhibits) polycomb repressive complex\n\n1. Cancer Res. 2010 Jan 15;70(2):453-62. doi: 10.1158/0008-5472.CAN-09-2189. Epub\n2010 Jan 12.\n\nReversing HOXA9 oncogene activation by PI3K inhibition: epigenetic mechanism and \nprognostic significance in human glioblastoma.\n\nCosta BM, Smith JS, Chen Y, Chen J, Phillips HS, Aldape KD, Zardo G, Nigro J,\nJames CD, Fridlyand J, Reis RM, Costello JF.\n\nThe Brain Tumor Research Center, Department of Neurological Surgery, University\nof California-San Francisco, San Francisco, California 94158-9001, USA.\n\nHOXA genes encode critical transcriptional regulators of embryonic development\nthat have been implicated in cancer. In this study, we documented functional\nrelevance and mechanism of activation of HOXA9 in glioblastoma (GBM), the most\ncommon malignant brain tumor. Expression of HOXA genes was investigated using\nreverse transcription-PCR in primary gliomas and glioblastoma cell lines and was \nvalidated in two sets of expression array data. In a subset of GBM, HOXA genes\nare aberrently activated within confined chromosomal domains. Transcriptional\nactivation of the HOXA cluster was reversible by a phosphoinostide 3-kinase\n(PI3K) inhibitor through an epigenetic mechanism involving histone H3K27\ntrimethylation. Functional studies of HOXA9 showed its capacity to decrease\napoptosis and increase cellular proliferation along with tumor necrosis\nfactor-related apoptosis-including ligand resistance. Notably, aberrant\nexpression of HOXA9 was independently predictive of shorter overall and\nprogression-free survival in two GBM patient sets and improved survival\nprediction by MGMT promoter methylation. Thus, HOXA9 activation is a novel,\nindependent, and negative prognostic marker in GBM that is reversible through a\nPI3K-associated epigenetic mechanism. Our findings suggest a transcriptional\npathway through which PI3K activates oncogenic HOXA expression with implications \nfor mTOR or PI3K targeted therapies.\n\nPMCID: PMC2849935\nPMID: 20068170  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7858",
        "source" : "7735",
        "target" : "7741",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "PIP3 (activates) PDPK1",
        "edgeType" : "activates",
        "SUID" : 7858,
        "name" : "PIP3 (activates) PDPK1",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "PIP3 (activates) PDPK1",
        "edgeDetail" : "PIP3 (activates) PDPK1\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7857",
        "source" : "7734",
        "target" : "7730",
        "pmid" : "24217114",
        "selected" : false,
        "canonicalName" : "PKC class (class.includes) PRKCE",
        "edgeType" : "class.includes",
        "SUID" : 7857,
        "name" : "PKC class (class.includes) PRKCE",
        "interaction" : "class.includes",
        "shared_interaction" : "class.includes",
        "shared_name" : "PKC class (class.includes) PRKCE",
        "edgeDetail" : "PKC class (class.includes) PRKCE\n\n1. Cancers (Basel). 2013 Nov 8;5(4):1469-84. doi: 10.3390/cancers5041469.\n\nTumor metabolism of malignant gliomas.\n\nRu P(1), Williams TM, Chakravarti A, Guo D.\n\nAuthor information: \n(1)Department of Radiation Oncology, Ohio State University Comprehensive Cancer\nCenter & Arthur G James Cancer Hospital, Columbus, OH 43012, USA.\ndeliang.guo@osumc.edu.\n\nConstitutively activated oncogenic signaling via genetic mutations such as in the\nEGFR/PI3K/Akt and Ras/RAF/MEK pathways has been recognized as a major driver for \ntumorigenesis in most cancers. Recent insights into tumor metabolism have further\nrevealed that oncogenic signaling pathways directly promote metabolic\nreprogramming to upregulate biosynthesis of lipids, carbohydrates, protein, DNA\nand RNA, leading to enhanced growth of human tumors. Therefore, targeting cell\nmetabolism has become a novel direction for drug development in oncology. In\nmalignant gliomas, metabolism pathways of glucose, glutamine and lipid are\nsignificantly reprogrammed. Moreover, molecular mechanisms causing these\nmetabolic changes are just starting to be unraveled. In this review, we will\nsummarize recent studies revealing critical gene alterations that lead to\nmetabolic changes in malignant gliomas, and also discuss promising therapeutic\nstrategies via targeting the key players in metabolic regulation.\n\nPMCID: PMC3875949\nPMID: 24217114  [PubMed]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7856",
        "source" : "7734",
        "target" : "7659",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "PKC class (class.includes) PRKCA",
        "edgeType" : "class.includes",
        "SUID" : 7856,
        "name" : "PKC class (class.includes) PRKCA",
        "interaction" : "class.includes",
        "shared_interaction" : "class.includes",
        "shared_name" : "PKC class (class.includes) PRKCA",
        "edgeDetail" : "PKC class (class.includes) PRKCA\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7855",
        "source" : "7734",
        "target" : "7658",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "PKC class (class.includes) PRKCB1",
        "edgeType" : "class.includes",
        "SUID" : 7855,
        "name" : "PKC class (class.includes) PRKCB1",
        "interaction" : "class.includes",
        "shared_interaction" : "class.includes",
        "shared_name" : "PKC class (class.includes) PRKCB1",
        "edgeDetail" : "PKC class (class.includes) PRKCB1\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7854",
        "source" : "7734",
        "target" : "7657",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "PKC class (class.includes) PRKCD",
        "edgeType" : "class.includes",
        "SUID" : 7854,
        "name" : "PKC class (class.includes) PRKCD",
        "interaction" : "class.includes",
        "shared_interaction" : "class.includes",
        "shared_name" : "PKC class (class.includes) PRKCD",
        "edgeDetail" : "PKC class (class.includes) PRKCD\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7853",
        "source" : "7734",
        "target" : "7656",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "PKC class (class.includes) PRKCG",
        "edgeType" : "class.includes",
        "SUID" : 7853,
        "name" : "PKC class (class.includes) PRKCG",
        "interaction" : "class.includes",
        "shared_interaction" : "class.includes",
        "shared_name" : "PKC class (class.includes) PRKCG",
        "edgeDetail" : "PKC class (class.includes) PRKCG\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7852",
        "source" : "7734",
        "target" : "7655",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "PKC class (class.includes) PRKCH",
        "edgeType" : "class.includes",
        "SUID" : 7852,
        "name" : "PKC class (class.includes) PRKCH",
        "interaction" : "class.includes",
        "shared_interaction" : "class.includes",
        "shared_name" : "PKC class (class.includes) PRKCH",
        "edgeDetail" : "PKC class (class.includes) PRKCH\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7851",
        "source" : "7734",
        "target" : "7654",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "PKC class (class.includes) PRKCI",
        "edgeType" : "class.includes",
        "SUID" : 7851,
        "name" : "PKC class (class.includes) PRKCI",
        "interaction" : "class.includes",
        "shared_interaction" : "class.includes",
        "shared_name" : "PKC class (class.includes) PRKCI",
        "edgeDetail" : "PKC class (class.includes) PRKCI\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7850",
        "source" : "7734",
        "target" : "7653",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "PKC class (class.includes) PRKCQ",
        "edgeType" : "class.includes",
        "SUID" : 7850,
        "name" : "PKC class (class.includes) PRKCQ",
        "interaction" : "class.includes",
        "shared_interaction" : "class.includes",
        "shared_name" : "PKC class (class.includes) PRKCQ",
        "edgeDetail" : "PKC class (class.includes) PRKCQ\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7849",
        "source" : "7734",
        "target" : "7652",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "PKC class (class.includes) PRKCZ",
        "edgeType" : "class.includes",
        "SUID" : 7849,
        "name" : "PKC class (class.includes) PRKCZ",
        "interaction" : "class.includes",
        "shared_interaction" : "class.includes",
        "shared_name" : "PKC class (class.includes) PRKCZ",
        "edgeDetail" : "PKC class (class.includes) PRKCZ\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7848",
        "source" : "7734",
        "target" : "7727",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "PKC class (activates) RAF family",
        "edgeType" : "activates",
        "SUID" : 7848,
        "name" : "PKC class (activates) RAF family",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "PKC class (activates) RAF family",
        "edgeDetail" : "PKC class (activates) RAF family\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7847",
        "source" : "7734",
        "target" : "7753",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "PKC class (activates) MEK1/2",
        "edgeType" : "activates",
        "SUID" : 7847,
        "name" : "PKC class (activates) MEK1/2",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "PKC class (activates) MEK1/2",
        "edgeDetail" : "PKC class (activates) MEK1/2\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7846",
        "source" : "7733",
        "target" : "7779",
        "pmid" : "24217114",
        "selected" : false,
        "canonicalName" : "PKM2 (activates) CTNNB1",
        "edgeType" : "activates",
        "SUID" : 7846,
        "name" : "PKM2 (activates) CTNNB1",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "PKM2 (activates) CTNNB1",
        "edgeDetail" : "PKM2 (activates) CTNNB1\n\n1. Ann Med Interne (Paris). 1974 Aug-Sep;125(8-9):641-50.\n\n[Obstructive myocardiopathy, Turner-Ullrich syndrome and von Recklinghausen's\ndisease].\n\n[Article in French]\n\nGerbaux A, Belfante M, Hiltgen M.\n\nPMID: 4217114  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7845",
        "source" : "7732",
        "target" : "7734",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "PLCG (activates) PKC class",
        "edgeType" : "activates",
        "SUID" : 7845,
        "name" : "PLCG (activates) PKC class",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "PLCG (activates) PKC class",
        "edgeDetail" : "PLCG (activates) PKC class\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7844",
        "source" : "7731",
        "target" : "7765",
        "pmid" : "20068170",
        "selected" : false,
        "canonicalName" : "polycomb repressive complex (inhibits) HOXA9",
        "edgeType" : "inhibits",
        "SUID" : 7844,
        "name" : "polycomb repressive complex (inhibits) HOXA9",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "polycomb repressive complex (inhibits) HOXA9",
        "edgeDetail" : "polycomb repressive complex (inhibits) HOXA9\n\n1. Cancer Res. 2010 Jan 15;70(2):453-62. doi: 10.1158/0008-5472.CAN-09-2189. Epub\n2010 Jan 12.\n\nReversing HOXA9 oncogene activation by PI3K inhibition: epigenetic mechanism and \nprognostic significance in human glioblastoma.\n\nCosta BM, Smith JS, Chen Y, Chen J, Phillips HS, Aldape KD, Zardo G, Nigro J,\nJames CD, Fridlyand J, Reis RM, Costello JF.\n\nThe Brain Tumor Research Center, Department of Neurological Surgery, University\nof California-San Francisco, San Francisco, California 94158-9001, USA.\n\nHOXA genes encode critical transcriptional regulators of embryonic development\nthat have been implicated in cancer. In this study, we documented functional\nrelevance and mechanism of activation of HOXA9 in glioblastoma (GBM), the most\ncommon malignant brain tumor. Expression of HOXA genes was investigated using\nreverse transcription-PCR in primary gliomas and glioblastoma cell lines and was \nvalidated in two sets of expression array data. In a subset of GBM, HOXA genes\nare aberrently activated within confined chromosomal domains. Transcriptional\nactivation of the HOXA cluster was reversible by a phosphoinostide 3-kinase\n(PI3K) inhibitor through an epigenetic mechanism involving histone H3K27\ntrimethylation. Functional studies of HOXA9 showed its capacity to decrease\napoptosis and increase cellular proliferation along with tumor necrosis\nfactor-related apoptosis-including ligand resistance. Notably, aberrant\nexpression of HOXA9 was independently predictive of shorter overall and\nprogression-free survival in two GBM patient sets and improved survival\nprediction by MGMT promoter methylation. Thus, HOXA9 activation is a novel,\nindependent, and negative prognostic marker in GBM that is reversible through a\nPI3K-associated epigenetic mechanism. Our findings suggest a transcriptional\npathway through which PI3K activates oncogenic HOXA expression with implications \nfor mTOR or PI3K targeted therapies.\n\nPMCID: PMC2849935\nPMID: 20068170  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7843",
        "source" : "7731",
        "target" : "7685",
        "pmid" : "17060944",
        "selected" : false,
        "canonicalName" : "polycomb repressive complex (activates.process) Histone methylation",
        "edgeType" : "activates.process",
        "SUID" : 7843,
        "name" : "polycomb repressive complex (activates.process) Histone methylation",
        "interaction" : "activates.process",
        "shared_interaction" : "activates.process",
        "shared_name" : "polycomb repressive complex (activates.process) Histone methylation",
        "edgeDetail" : "polycomb repressive complex (activates.process) Histone methylation\n\n1. Nat Rev Cancer. 2006 Nov;6(11):846-56.\n\nPolycomb silencers control cell fate, development and cancer.\n\nSparmann A(1), van Lohuizen M.\n\nAuthor information: \n(1)Division of Molecular Genetics and Centre for Biomedical Genetics, The\nNetherlands Cancer Institute, Plesmanlaan 121, 1066CX Amsterdam, The Netherlands.\n\nPolycomb group (PcG) proteins are epigenetic gene silencers that are implicated\nin neoplastic development. Their oncogenic function might be associated with\ntheir well-established role in the maintenance of embryonic and adult stem cells.\nIn this review, we discuss new insights into the possible mechanisms by which\nPcGs regulate cellular identity, and speculate how these functions might be\nrelevant during tumorigenesis.\n\nPMID: 17060944  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7842",
        "source" : "7730",
        "target" : "7743",
        "pmid" : "24217114",
        "selected" : false,
        "canonicalName" : "PRKCE (activates) NF-kB:HIF1A",
        "edgeType" : "activates",
        "SUID" : 7842,
        "name" : "PRKCE (activates) NF-kB:HIF1A",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "PRKCE (activates) NF-kB:HIF1A",
        "edgeDetail" : "PRKCE (activates) NF-kB:HIF1A\n\n1. Cancers (Basel). 2013 Nov 8;5(4):1469-84. doi: 10.3390/cancers5041469.\n\nTumor metabolism of malignant gliomas.\n\nRu P(1), Williams TM, Chakravarti A, Guo D.\n\nAuthor information: \n(1)Department of Radiation Oncology, Ohio State University Comprehensive Cancer\nCenter & Arthur G James Cancer Hospital, Columbus, OH 43012, USA.\ndeliang.guo@osumc.edu.\n\nConstitutively activated oncogenic signaling via genetic mutations such as in the\nEGFR/PI3K/Akt and Ras/RAF/MEK pathways has been recognized as a major driver for \ntumorigenesis in most cancers. Recent insights into tumor metabolism have further\nrevealed that oncogenic signaling pathways directly promote metabolic\nreprogramming to upregulate biosynthesis of lipids, carbohydrates, protein, DNA\nand RNA, leading to enhanced growth of human tumors. Therefore, targeting cell\nmetabolism has become a novel direction for drug development in oncology. In\nmalignant gliomas, metabolism pathways of glucose, glutamine and lipid are\nsignificantly reprogrammed. Moreover, molecular mechanisms causing these\nmetabolic changes are just starting to be unraveled. In this review, we will\nsummarize recent studies revealing critical gene alterations that lead to\nmetabolic changes in malignant gliomas, and also discuss promising therapeutic\nstrategies via targeting the key players in metabolic regulation.\n\nPMCID: PMC3875949\nPMID: 24217114  [PubMed]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7841",
        "source" : "7729",
        "target" : "7721",
        "pmid" : "23641065",
        "selected" : false,
        "canonicalName" : "pS6K (activates) SREBP",
        "edgeType" : "activates",
        "SUID" : 7841,
        "name" : "pS6K (activates) SREBP",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "pS6K (activates) SREBP",
        "edgeDetail" : "pS6K (activates) SREBP\n\n1. J Cell Sci. 2013 Apr 15;126(Pt 8):1713-9. doi: 10.1242/jcs.125773. Epub 2013 May \n2.\n\nRegulation of mTORC1 and its impact on gene expression at a glance.\n\nLaplante M, Sabatini DM.\n\nCentre de Recherche de l'Institut Universitaire de Cardiologie et de Pneumologie \nde Qubec (CRIUCPQ), Facult de Mdecine, Universit Laval, 2725 Chemin Ste-Foy, \nQubec, QC, G1V 4G5, Canada. mathieu.laplante@criucpq.ulaval.ca\n\nThe mechanistic (or mammalian) target of rapamycin (mTOR) is a kinase that\nregulates key cellular functions linked to the promotion of cell growth and\nmetabolism. This kinase, which is part of two protein complexes termed mTOR\ncomplex 1 (mTORC1) and 2 (mTORC2), has a fundamental role in coordinating\nanabolic and catabolic processes in response to growth factors and nutrients. Of \nthe two mTOR complexes, mTORC1 is by far the best characterized. When active,\nmTORC1 triggers cell growth and proliferation by promoting protein synthesis,\nlipid biogenesis, and metabolism, and by reducing autophagy. The fact that mTORC1\nderegulation is associated with several human diseases, such as type 2 diabetes, \ncancer, obesity and neurodegeneration, highlights its importance in the\nmaintenance of cellular homeostasis. Over the last years, several groups observed\nthat mTORC1 inhibition, in addition to reducing protein synthesis, deeply affects\ngene transcription. Here, we review the connections between mTORC1 and gene\ntranscription by focusing on its impact in regulating the activation of specific \ntranscription factors including including STAT3, SREBPs, PPAR, PPAR, HIF1,\nYY1PGC1 and TFEB. We also discuss the importance of these transcription factors\nin mediating the effects of mTORC1 on various cellular processes in physiological\nand pathological contexts.\n\nPMCID: PMC3678406 [Available on 2014/4/15]\nPMID: 23641065  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7840",
        "source" : "7728",
        "target" : "7735",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "PTEN (inhibits) PIP3",
        "edgeType" : "inhibits",
        "SUID" : 7840,
        "name" : "PTEN (inhibits) PIP3",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "PTEN (inhibits) PIP3",
        "edgeDetail" : "PTEN (inhibits) PIP3\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7839",
        "source" : "7727",
        "target" : "7651",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "RAF family (family.includes) RAF1",
        "edgeType" : "family.includes",
        "SUID" : 7839,
        "name" : "RAF family (family.includes) RAF1",
        "interaction" : "family.includes",
        "shared_interaction" : "family.includes",
        "shared_name" : "RAF family (family.includes) RAF1",
        "edgeDetail" : "RAF family (family.includes) RAF1\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7838",
        "source" : "7727",
        "target" : "7650",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "RAF family (family.includes) ARAF",
        "edgeType" : "family.includes",
        "SUID" : 7838,
        "name" : "RAF family (family.includes) ARAF",
        "interaction" : "family.includes",
        "shared_interaction" : "family.includes",
        "shared_name" : "RAF family (family.includes) ARAF",
        "edgeDetail" : "RAF family (family.includes) ARAF\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7837",
        "source" : "7727",
        "target" : "7649",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "RAF family (family.includes) BRAF",
        "edgeType" : "family.includes",
        "SUID" : 7837,
        "name" : "RAF family (family.includes) BRAF",
        "interaction" : "family.includes",
        "shared_interaction" : "family.includes",
        "shared_name" : "RAF family (family.includes) BRAF",
        "edgeDetail" : "RAF family (family.includes) BRAF\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7836",
        "source" : "7727",
        "target" : "7753",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "RAF family (activates) MEK1/2",
        "edgeType" : "activates",
        "SUID" : 7836,
        "name" : "RAF family (activates) MEK1/2",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "RAF family (activates) MEK1/2",
        "edgeDetail" : "RAF family (activates) MEK1/2\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7835",
        "source" : "7726",
        "target" : "7648",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "RAS family (family.includes) HRAS",
        "edgeType" : "family.includes",
        "SUID" : 7835,
        "name" : "RAS family (family.includes) HRAS",
        "interaction" : "family.includes",
        "shared_interaction" : "family.includes",
        "shared_name" : "RAS family (family.includes) HRAS",
        "edgeDetail" : "RAS family (family.includes) HRAS\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7834",
        "source" : "7726",
        "target" : "7647",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "RAS family (family.includes) KRAS",
        "edgeType" : "family.includes",
        "SUID" : 7834,
        "name" : "RAS family (family.includes) KRAS",
        "interaction" : "family.includes",
        "shared_interaction" : "family.includes",
        "shared_name" : "RAS family (family.includes) KRAS",
        "edgeDetail" : "RAS family (family.includes) KRAS\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7833",
        "source" : "7726",
        "target" : "7646",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "RAS family (family.includes) NRAS",
        "edgeType" : "family.includes",
        "SUID" : 7833,
        "name" : "RAS family (family.includes) NRAS",
        "interaction" : "family.includes",
        "shared_interaction" : "family.includes",
        "shared_name" : "RAS family (family.includes) NRAS",
        "edgeDetail" : "RAS family (family.includes) NRAS\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7832",
        "source" : "7726",
        "target" : "7727",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "RAS family (activates) RAF family",
        "edgeType" : "activates",
        "SUID" : 7832,
        "name" : "RAS family (activates) RAF family",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "RAS family (activates) RAF family",
        "edgeDetail" : "RAS family (activates) RAF family\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7831",
        "source" : "7726",
        "target" : "7739",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "RAS family (activates) PI3K class1b",
        "edgeType" : "activates",
        "SUID" : 7831,
        "name" : "RAS family (activates) PI3K class1b",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "RAS family (activates) PI3K class1b",
        "edgeDetail" : "RAS family (activates) PI3K class1b\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7830",
        "source" : "7725",
        "target" : "7778",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "RB1 (inhibits) E2F1",
        "edgeType" : "inhibits",
        "SUID" : 7830,
        "name" : "RB1 (inhibits) E2F1",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "RB1 (inhibits) E2F1",
        "edgeDetail" : "RB1 (inhibits) E2F1\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7829",
        "source" : "7724",
        "target" : "7738",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "RTK class (activates) PI3K class2",
        "edgeType" : "activates",
        "SUID" : 7829,
        "name" : "RTK class (activates) PI3K class2",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "RTK class (activates) PI3K class2",
        "edgeDetail" : "RTK class (activates) PI3K class2\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7828",
        "source" : "7724",
        "target" : "7773",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "RTK class (class.includes) FGFR family",
        "edgeType" : "class.includes",
        "SUID" : 7828,
        "name" : "RTK class (class.includes) FGFR family",
        "interaction" : "class.includes",
        "shared_interaction" : "class.includes",
        "shared_name" : "RTK class (class.includes) FGFR family",
        "edgeDetail" : "RTK class (class.includes) FGFR family\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7827",
        "source" : "7724",
        "target" : "7762",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "RTK class (class.includes) IGF1R",
        "edgeType" : "class.includes",
        "SUID" : 7827,
        "name" : "RTK class (class.includes) IGF1R",
        "interaction" : "class.includes",
        "shared_interaction" : "class.includes",
        "shared_name" : "RTK class (class.includes) IGF1R",
        "edgeDetail" : "RTK class (class.includes) IGF1R\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7826",
        "source" : "7724",
        "target" : "7732",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "RTK class (activates) PLCG",
        "edgeType" : "activates",
        "SUID" : 7826,
        "name" : "RTK class (activates) PLCG",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "RTK class (activates) PLCG",
        "edgeDetail" : "RTK class (activates) PLCG\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7825",
        "source" : "7724",
        "target" : "7722",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "RTK class (activates) SRC",
        "edgeType" : "activates",
        "SUID" : 7825,
        "name" : "RTK class (activates) SRC",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "RTK class (activates) SRC",
        "edgeDetail" : "RTK class (activates) SRC\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7824",
        "source" : "7724",
        "target" : "7742",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "RTK class (class.includes) PDGFR family",
        "edgeType" : "class.includes",
        "SUID" : 7824,
        "name" : "RTK class (class.includes) PDGFR family",
        "interaction" : "class.includes",
        "shared_interaction" : "class.includes",
        "shared_name" : "RTK class (class.includes) PDGFR family",
        "edgeDetail" : "RTK class (class.includes) PDGFR family\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7823",
        "source" : "7724",
        "target" : "7687",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "RTK class (class.includes) MET",
        "edgeType" : "class.includes",
        "SUID" : 7823,
        "name" : "RTK class (class.includes) MET",
        "interaction" : "class.includes",
        "shared_interaction" : "class.includes",
        "shared_name" : "RTK class (class.includes) MET",
        "edgeDetail" : "RTK class (class.includes) MET\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7822",
        "source" : "7724",
        "target" : "7777",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "RTK class (class.includes) EGFR family",
        "edgeType" : "class.includes",
        "SUID" : 7822,
        "name" : "RTK class (class.includes) EGFR family",
        "interaction" : "class.includes",
        "shared_interaction" : "class.includes",
        "shared_name" : "RTK class (class.includes) EGFR family",
        "edgeDetail" : "RTK class (class.includes) EGFR family\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7821",
        "source" : "7724",
        "target" : "7768",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "RTK class (activates) GRB2",
        "edgeType" : "activates",
        "SUID" : 7821,
        "name" : "RTK class (activates) GRB2",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "RTK class (activates) GRB2",
        "edgeDetail" : "RTK class (activates) GRB2\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7820",
        "source" : "7723",
        "target" : "7727",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "SPRY2 (inhibits) RAF family",
        "edgeType" : "inhibits",
        "SUID" : 7820,
        "name" : "SPRY2 (inhibits) RAF family",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "SPRY2 (inhibits) RAF family",
        "edgeDetail" : "SPRY2 (inhibits) RAF family\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7819",
        "source" : "7723",
        "target" : "7790",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "SPRY2 (inhibits) CBL",
        "edgeType" : "inhibits",
        "SUID" : 7819,
        "name" : "SPRY2 (inhibits) CBL",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "SPRY2 (inhibits) CBL",
        "edgeDetail" : "SPRY2 (inhibits) CBL\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7818",
        "source" : "7722",
        "target" : "7728",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "SRC (inhibits) PTEN",
        "edgeType" : "inhibits",
        "SUID" : 7818,
        "name" : "SRC (inhibits) PTEN",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "SRC (inhibits) PTEN",
        "edgeDetail" : "SRC (inhibits) PTEN\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7817",
        "source" : "7722",
        "target" : "7740",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "SRC (activates) PI3K class1a",
        "edgeType" : "activates",
        "SUID" : 7817,
        "name" : "SRC (activates) PI3K class1a",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "SRC (activates) PI3K class1a",
        "edgeDetail" : "SRC (activates) PI3K class1a\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7816",
        "source" : "7721",
        "target" : "7720",
        "pmid" : "21428927",
        "selected" : false,
        "canonicalName" : "SREBP (family.includes) SREBP1a",
        "edgeType" : "family.includes",
        "SUID" : 7816,
        "name" : "SREBP (family.includes) SREBP1a",
        "interaction" : "family.includes",
        "shared_interaction" : "family.includes",
        "shared_name" : "SREBP (family.includes) SREBP1a",
        "edgeDetail" : "SREBP (family.includes) SREBP1a\n\n1. Biochem Soc Trans. 2011 Apr;39(2):495-9. doi: 10.1042/BST0390495.\n\nRegulation of the SREBP transcription factors by mTORC1.\n\nLewis CA, Griffiths B, Santos CR, Pende M, Schulze A.\n\nGene Expression Analysis Laboratory, Cancer Research UK London Research\nInstitute, 44 Lincoln's Inn Fields, London WC2A 3LY, UK.\n\nIn recent years several reports have linked mTORC1 (mammalian target of rapamycin\ncomplex 1) to lipogenesis via the SREBPs (sterol-regulatory-element-binding\nproteins). SREBPs regulate the expression of genes encoding enzymes required for \nfatty acid and cholesterol biosynthesis. Lipid metabolism is perturbed in some\ndiseases and SREBP target genes, such as FASN (fatty acid synthase), have been\nshown to be up-regulated in some cancers. We have previously shown that mTORC1\nplays a role in SREBP activation and Akt/PKB (protein kinase B)-dependent de novo\nlipogenesis. Our findings suggest that mTORC1 plays a crucial role in the\nactivation of SREBP and that the activation of lipid biosynthesis through the\ninduction of SREBP could be part of a regulatory pathway that co-ordinates\nprotein and lipid biosynthesis during cell growth. In the present paper, we\ndiscuss the increasing amount of data supporting the potential mechanisms of\nmTORC1-dependent activation of SREBP as well as the implications of this\nsignalling pathway in cancer.\n\nPMID: 21428927  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7815",
        "source" : "7721",
        "target" : "7719",
        "pmid" : "21428927",
        "selected" : false,
        "canonicalName" : "SREBP (family.includes) SREBP2",
        "edgeType" : "family.includes",
        "SUID" : 7815,
        "name" : "SREBP (family.includes) SREBP2",
        "interaction" : "family.includes",
        "shared_interaction" : "family.includes",
        "shared_name" : "SREBP (family.includes) SREBP2",
        "edgeDetail" : "SREBP (family.includes) SREBP2\n\n1. Biochem Soc Trans. 2011 Apr;39(2):495-9. doi: 10.1042/BST0390495.\n\nRegulation of the SREBP transcription factors by mTORC1.\n\nLewis CA, Griffiths B, Santos CR, Pende M, Schulze A.\n\nGene Expression Analysis Laboratory, Cancer Research UK London Research\nInstitute, 44 Lincoln's Inn Fields, London WC2A 3LY, UK.\n\nIn recent years several reports have linked mTORC1 (mammalian target of rapamycin\ncomplex 1) to lipogenesis via the SREBPs (sterol-regulatory-element-binding\nproteins). SREBPs regulate the expression of genes encoding enzymes required for \nfatty acid and cholesterol biosynthesis. Lipid metabolism is perturbed in some\ndiseases and SREBP target genes, such as FASN (fatty acid synthase), have been\nshown to be up-regulated in some cancers. We have previously shown that mTORC1\nplays a role in SREBP activation and Akt/PKB (protein kinase B)-dependent de novo\nlipogenesis. Our findings suggest that mTORC1 plays a crucial role in the\nactivation of SREBP and that the activation of lipid biosynthesis through the\ninduction of SREBP could be part of a regulatory pathway that co-ordinates\nprotein and lipid biosynthesis during cell growth. In the present paper, we\ndiscuss the increasing amount of data supporting the potential mechanisms of\nmTORC1-dependent activation of SREBP as well as the implications of this\nsignalling pathway in cancer.\n\nPMID: 21428927  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7814",
        "source" : "7720",
        "target" : "7758",
        "pmid" : "24217114",
        "selected" : false,
        "canonicalName" : "SREBP1a (activates) LDLR",
        "edgeType" : "activates",
        "SUID" : 7814,
        "name" : "SREBP1a (activates) LDLR",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "SREBP1a (activates) LDLR",
        "edgeDetail" : "SREBP1a (activates) LDLR\n\n1. Cancers (Basel). 2013 Nov 8;5(4):1469-84. doi: 10.3390/cancers5041469.\n\nTumor metabolism of malignant gliomas.\n\nRu P(1), Williams TM, Chakravarti A, Guo D.\n\nAuthor information: \n(1)Department of Radiation Oncology, Ohio State University Comprehensive Cancer\nCenter & Arthur G James Cancer Hospital, Columbus, OH 43012, USA.\ndeliang.guo@osumc.edu.\n\nConstitutively activated oncogenic signaling via genetic mutations such as in the\nEGFR/PI3K/Akt and Ras/RAF/MEK pathways has been recognized as a major driver for \ntumorigenesis in most cancers. Recent insights into tumor metabolism have further\nrevealed that oncogenic signaling pathways directly promote metabolic\nreprogramming to upregulate biosynthesis of lipids, carbohydrates, protein, DNA\nand RNA, leading to enhanced growth of human tumors. Therefore, targeting cell\nmetabolism has become a novel direction for drug development in oncology. In\nmalignant gliomas, metabolism pathways of glucose, glutamine and lipid are\nsignificantly reprogrammed. Moreover, molecular mechanisms causing these\nmetabolic changes are just starting to be unraveled. In this review, we will\nsummarize recent studies revealing critical gene alterations that lead to\nmetabolic changes in malignant gliomas, and also discuss promising therapeutic\nstrategies via targeting the key players in metabolic regulation.\n\nPMCID: PMC3875949\nPMID: 24217114  [PubMed]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7813",
        "source" : "7720",
        "target" : "7785",
        "pmid" : "24077277",
        "selected" : false,
        "canonicalName" : "SREBP1a (activates) CDKN1A",
        "edgeType" : "activates",
        "SUID" : 7813,
        "name" : "SREBP1a (activates) CDKN1A",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "SREBP1a (activates) CDKN1A",
        "edgeDetail" : "SREBP1a (activates) CDKN1A\n\n1. Neurol Med Chir (Tokyo). 2013;53(10):645-54. Epub 2013 Sep 27.\n\nAn R132H mutation in isocitrate dehydrogenase 1 enhances p21 expression and\ninhibits phosphorylation of retinoblastoma protein in glioma cells.\n\nMiyata S(1), Urabe M, Gomi A, Nagai M, Yamaguchi T, Tsukahara T, Mizukami H, Kume\nA, Ozawa K, Watanabe E.\n\nAuthor information: \n(1)Department of Neurosurgery, Center for Molecular Medicine, Jichi Medical\nUniversity.\n\nCytosolic isocitrate dehydrogenase 1 (IDH1) with an R132H mutation in brain\ntumors loses its enzymatic activity for catalyzing isocitrate to -ketoglutarate \n(-KG) and acquires new activity whereby it converts -KG to 2-hydroxyglutarate. \nThe IDH1 mutation induces down-regulation of tricarboxylic acid cycle\nintermediates and up-regulation of lipid metabolism. Sterol regulatory\nelement-binding proteins (SREBPs) regulate not only the synthesis of cholesterol \nand fatty acids but also acyclin-dependent kinase inhibitor p21 that halts the\ncell cycle at G1. Here we show that SREBPs were up-regulated in U87 human\nglioblastoma cells transfected with an IDH1(R132H)-expression plasmid. Small\ninterfering ribonucleic acid (siRNA) for SREBP1 specifically decreased p21\nmessenger RNA (mRNA) levels independent of the p53 pathway. In\nIDH1(R132H)-expressing U87 cells, phosphorylation of Retinoblastoma (Rb) protein \nalso decreased. We propose that metabolic changes induced by the IDH1 mutation\nenhance p21 expression via SREBP1 and inhibit phosphorylation of Rb, which slows \nprogression of the cell cycle and may be associated with non-aggressive features \nof gliomas with an IDH1 mutation.\n\nPMID: 24077277  [PubMed - in process]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7812",
        "source" : "7719",
        "target" : "7785",
        "pmid" : "16199872",
        "selected" : false,
        "canonicalName" : "SREBP2 (activates) CDKN1A",
        "edgeType" : "activates",
        "SUID" : 7812,
        "name" : "SREBP2 (activates) CDKN1A",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "SREBP2 (activates) CDKN1A",
        "edgeDetail" : "SREBP2 (activates) CDKN1A\n\n1. Mol Cell Biol. 2005 Oct;25(20):8938-47.\n\nLipid synthetic transcription factor SREBP-1a activates p21WAF1/CIP1, a universal\ncyclin-dependent kinase inhibitor.\n\nInoue N, Shimano H, Nakakuki M, Matsuzaka T, Nakagawa Y, Yamamoto T, Sato R,\nTakahashi A, Sone H, Yahagi N, Suzuki H, Toyoshima H, Yamada N.\n\nDepartment of Internal Medicine, Institute of Clinical Medicine, University of\nTsukuba, Ibaraki, Japan.\n\nSterol regulatory element-binding proteins (SREBPs) are membrane-bound\ntranscription factors that regulate lipid synthetic genes. In contrast to\nSREBP-2, which regulates cellular cholesterol level in normal cells, SREBP-1a is \nhighly expressed in actively growing cells and activates entire programs of genes\ninvolved in lipid synthesis such as cholesterol, fatty acids, triglycerides, and \nphospholipids. Previously, the physiological relevance of this potent activity of\nSREBP-1a has been thought to regulate the supply of membrane lipids in response\nto cell growth. Here we show that nuclear SREBP-1a and SREBP-2 bind directly to a\nnovel SREBP binding site in the promoter of the p21(WAF1/CIP1) gene, the major\ncyclin-dependent kinase inhibitor, and strongly activate its promoter activity.\nOnly the SREBP-1a isoform consistently causes induction of p21 at both the mRNA\nand protein levels. Colony formation assays and polyploidy of livers from\ntransgenic mice suggest that activation of p21 by SREBP-1a could inhibit cell\ngrowth. Activation of endogenous SREBPs in lipid deprivation conditions was\nassociated with induction of p21 mRNA and protein. Expression of p21 was reduced \nin SREBP-1 null mice. These data suggest a physiological role of SREBP-1a in p21 \nregulation. Identification of p21 as a new SREBP target might implicate a new\nparadigm in the link between lipid synthesis and cell growth.\n\nPMCID: PMC1265776\nPMID: 16199872  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7811",
        "source" : "7718",
        "target" : "7705",
        "pmid" : "21969816",
        "selected" : false,
        "canonicalName" : "STAT3 (activates.process) CELL MIGRATION",
        "edgeType" : "activates.process",
        "SUID" : 7811,
        "name" : "STAT3 (activates.process) CELL MIGRATION",
        "interaction" : "activates.process",
        "shared_interaction" : "activates.process",
        "shared_name" : "STAT3 (activates.process) CELL MIGRATION",
        "edgeDetail" : "STAT3 (activates.process) CELL MIGRATION\n\n1. Neoplasia. 2011 Sep;13(9):831-40.\n\nGlioma cell migration on three-dimensional nanofiber scaffolds is regulated by\nsubstrate topography and abolished by inhibition of STAT3 signaling.\n\nAgudelo-Garcia PA(1), De Jesus JK, Williams SP, Nowicki MO, Chiocca EA,\nLiyanarachchi S, Li PK, Lannutti JJ, Johnson JK, Lawler SE, Viapiano MS.\n\nAuthor information: \n(1)Center for Molecular Neurobiology, The Ohio State University, Columbus, OH 43210,\nUSA.\n\nA hallmark of malignant gliomas is their ability to disperse through neural\ntissue, leading to long-term failure of all known therapies. Identifying new\nantimigratory targets could reduce glioma recurrence and improve therapeutic\nefficacy, but screens based on conventional migration assays are hampered by the \nlimited ability of these assays to reproduce native cell motility. Here, we have \nanalyzed the motility, gene expression, and sensitivity to migration inhibitors\nof glioma cells cultured on scaffolds formed by submicron-sized fibers\n(nanofibers) mimicking the neural topography. Glioma cells cultured on aligned\nnanofiber scaffolds reproduced the elongated morphology of cells migrating in\nwhite matter tissue and were highly sensitive to myosin II inhibition but only\nmoderately affected by stress fiber disruption. In contrast, the same cells\ndisplayed a flat morphology and opposite sensitivity to myosin II and actin\ninhibition when cultured on conventional tissue culture polystyrene. Gene\nexpression analysis indicated a correlation between migration on aligned\nnanofibers and increased STAT3 signaling, a known driver of glioma progression.\nAccordingly, cell migration out of glioblastoma-derived neurospheres and tumor\nexplants was reduced by STAT3 inhibitors at subtoxic concentrations. Remarkably, \nthese inhibitors were ineffective when tested at the same concentrations in a\nconventional two-dimensional migration assay. We conclude that migration of\nglioma cells is regulated by topographical cues that affect cell adhesion and\ngene expression. Cell migration analysis using nanofiber scaffolds could be used \nto reproduce native mechanisms of migration and to identify antimigratory\nstrategies not disclosed by other in vitro models.\n\nPMCID: PMC3182275\nPMID: 21969816  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7810",
        "source" : "7718",
        "target" : "7766",
        "pmid" : "23359207",
        "selected" : false,
        "canonicalName" : "STAT3 (activates) HGF",
        "edgeType" : "activates",
        "SUID" : 7810,
        "name" : "STAT3 (activates) HGF",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "STAT3 (activates) HGF",
        "edgeDetail" : "STAT3 (activates) HGF\n\n1. Neoplasia. 2013 Jan;15(1):73-84.\n\nRegulation of HGF expression by EGFR-mediated c-Met activation in glioblastoma\ncells.\n\nGarnett J(1), Chumbalkar V, Vaillant B, Gururaj AE, Hill KS, Latha K, Yao J,\nPriebe W, Colman H, Elferink LA, Bogler O.\n\nAuthor information: \n(1)Department of Neurosurgery, University of Texas MD Anderson Cancer Center,\nHouston, TX 77030, USA.\n\nThe hepatocyte growth factor receptor (c-Met) and a constitutively active mutant \nof the epidermal growth factor receptor (EGFR/EGFRvIII) are frequently\noverexpressed in glioblastoma (GBM) and promote tumorigenesis. The mechanisms\nunderlying elevated hepatocyte growth factor (HGF) production in GBM are not\nunderstood. We found higher, coordinated mRNA expression levels of HGF and c-Met \nin mesenchymal (Mes) GBMs, a subtype associated with poor treatment response and \nshorter overall survival. In an HGF/c-Met-dependent GBM cell line, HGF expression\ndeclined upon silencing of c-Met using RNAi or by inhibiting its activity with\nSU11274. Silencing c-Met decreased anchorage-independent colony formation and\nincreased the survival of mice bearing intracranial GBM xenografts. Consistent\nwith these findings, c-Met activation by EGFR also elevated HGF expression, and \nthe inhibition of EGFR with AG1478 reduced HGF levels. Interestingly, c-Met\nexpression was required for EGFR-mediated HGF production, anchorage-independent \ngrowth, and in vivo tumorigenicity, suggesting that these pathways are coupled.\nUsing an unbiased mass spectrometry-based screen, we show that signal transducer \nand activator of transcription 3 (STAT3) Y705 is a downstream target of c-Met\nsignaling. Suppression of STAT3 phosphorylation with WP1193 reduced HGF\nexpression in EGFR-expressing GBM cells, whereas constitutively active STAT3\npartially rescued HGF expression and colony formation in c-Met knockdown cells\nexpressing EGFR. These results suggest that the c-Met/HGF signaling axis is\nenhanced by EGFR through increased STAT3-dependent HGF expression and that\ntargeting c-Met in Mes GBMs may be an important strategy for therapy.\n\nPMCID: PMC3556940\nPMID: 23359207  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7809",
        "source" : "7718",
        "target" : "7687",
        "pmid" : "23359207",
        "selected" : false,
        "canonicalName" : "STAT3 (activates) MET",
        "edgeType" : "activates",
        "SUID" : 7809,
        "name" : "STAT3 (activates) MET",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "STAT3 (activates) MET",
        "edgeDetail" : "STAT3 (activates) MET\n\n1. Neoplasia. 2013 Jan;15(1):73-84.\n\nRegulation of HGF expression by EGFR-mediated c-Met activation in glioblastoma\ncells.\n\nGarnett J(1), Chumbalkar V, Vaillant B, Gururaj AE, Hill KS, Latha K, Yao J,\nPriebe W, Colman H, Elferink LA, Bogler O.\n\nAuthor information: \n(1)Department of Neurosurgery, University of Texas MD Anderson Cancer Center,\nHouston, TX 77030, USA.\n\nThe hepatocyte growth factor receptor (c-Met) and a constitutively active mutant \nof the epidermal growth factor receptor (EGFR/EGFRvIII) are frequently\noverexpressed in glioblastoma (GBM) and promote tumorigenesis. The mechanisms\nunderlying elevated hepatocyte growth factor (HGF) production in GBM are not\nunderstood. We found higher, coordinated mRNA expression levels of HGF and c-Met \nin mesenchymal (Mes) GBMs, a subtype associated with poor treatment response and \nshorter overall survival. In an HGF/c-Met-dependent GBM cell line, HGF expression\ndeclined upon silencing of c-Met using RNAi or by inhibiting its activity with\nSU11274. Silencing c-Met decreased anchorage-independent colony formation and\nincreased the survival of mice bearing intracranial GBM xenografts. Consistent\nwith these findings, c-Met activation by EGFR also elevated HGF expression, and \nthe inhibition of EGFR with AG1478 reduced HGF levels. Interestingly, c-Met\nexpression was required for EGFR-mediated HGF production, anchorage-independent \ngrowth, and in vivo tumorigenicity, suggesting that these pathways are coupled.\nUsing an unbiased mass spectrometry-based screen, we show that signal transducer \nand activator of transcription 3 (STAT3) Y705 is a downstream target of c-Met\nsignaling. Suppression of STAT3 phosphorylation with WP1193 reduced HGF\nexpression in EGFR-expressing GBM cells, whereas constitutively active STAT3\npartially rescued HGF expression and colony formation in c-Met knockdown cells\nexpressing EGFR. These results suggest that the c-Met/HGF signaling axis is\nenhanced by EGFR through increased STAT3-dependent HGF expression and that\ntargeting c-Met in Mes GBMs may be an important strategy for therapy.\n\nPMCID: PMC3556940\nPMID: 23359207  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7808",
        "source" : "7717",
        "target" : "7792",
        "pmid" : "21815198",
        "selected" : false,
        "canonicalName" : "STAT5 (activates) BCL2L1",
        "edgeType" : "activates",
        "SUID" : 7808,
        "name" : "STAT5 (activates) BCL2L1",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "STAT5 (activates) BCL2L1",
        "edgeDetail" : "STAT5 (activates) BCL2L1\n\nPubmed-entry ::= {\n  pmid 2279867,\n  medent {\n    em std {\n      year 1991,\n      month 3,\n      day 14\n    },\n    cit {\n      title {\n        name \"Modulatory effects of vasoactive intestinal peptide on immune\n functions of spleen and lymphoid cells.\"\n      },\n      authors {\n        names std {\n          {\n            name ml \"Nock IH\",\n            affil str \"School of Environmental Sciences, Jawaharlal Nehru\n University, New Delhi, India.\"\n          },\n          {\n            name ml \"Sharma JD\"\n          }\n        }\n      },\n      from journal {\n        title {\n          iso-jta \"Int. J. Neurosci.\",\n          ml-jta \"Int J Neurosci\",\n          issn \"0020-7454\"\n        },\n        imp {\n          date std {\n            year 1990,\n            month 4\n          },\n          volume \"51\",\n          issue \"3-4\",\n          pages \"205-206\",\n          language \"eng\",\n          pubstatus ppublish,\n          history {\n            {\n              pubstatus pubmed,\n              date std {\n                year 1990,\n                month 4,\n                day 1\n              }\n            },\n            {\n              pubstatus medline,\n              date std {\n                year 1990,\n                month 4,\n                day 1,\n                hour 0,\n                minute 1\n              }\n            },\n            {\n              pubstatus other,\n              date std {\n                year 1990,\n                month 4,\n                day 1,\n                hour 0,\n                minute 0\n              }\n            }\n          }\n        }\n      },\n      ids {\n        pubmed 2279867\n      }\n    },\n    mesh {\n      {\n        term \"Animals\"\n      },\n      {\n        term \"Cells, Cultured\"\n      },\n      {\n        term \"Concanavalin A\"\n      },\n      {\n        term \"Lymph Nodes\",\n        qual {\n          {\n            mp TRUE,\n            subh \"immunology\"\n          }\n        }\n      },\n      {\n        term \"Lymphocyte Activation\",\n        qual {\n          {\n            mp TRUE,\n            subh \"drug effects\"\n          }\n        }\n      },\n      {\n        term \"Lymphocytes\",\n        qual {\n          {\n            subh \"drug effects\"\n          },\n          {\n            mp TRUE,\n            subh \"immunology\"\n          }\n        }\n      },\n      {\n        term \"Mice\"\n      },\n      {\n        term \"Spleen\",\n        qual {\n          {\n            mp TRUE,\n            subh \"immunology\"\n          }\n        }\n      },\n      {\n        term \"Vasoactive Intestinal Peptide\",\n        qual {\n          {\n            mp TRUE,\n            subh \"pharmacology\"\n          }\n        }\n      }\n    },\n    substance {\n      {\n        type cas,\n        cit \"11028-71-0\",\n        name \"Concanavalin A\"\n      },\n      {\n        type cas,\n        cit \"37221-79-7\",\n        name \"Vasoactive Intestinal Peptide\"\n      }\n    },\n    pmid 2279867,\n    pub-type {\n      \"Journal Article\"\n    },\n    status medline\n  }\n}\n\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7807",
        "source" : "7716",
        "target" : "7645",
        "pmid" : "23863747",
        "selected" : false,
        "canonicalName" : "TET2 (inhibits.process) DNA methylation",
        "edgeType" : "inhibits.process",
        "SUID" : 7807,
        "name" : "TET2 (inhibits.process) DNA methylation",
        "interaction" : "inhibits.process",
        "shared_interaction" : "inhibits.process",
        "shared_name" : "TET2 (inhibits.process) DNA methylation",
        "edgeDetail" : "TET2 (inhibits.process) DNA methylation\n\n1. Nat Commun. 2013;4:2166. doi: 10.1038/ncomms3166.\n\nMeta-analysis of IDH-mutant cancers identifies EBF1 as an interaction partner for\nTET2.\n\nGuilhamon P(1), Eskandarpour M, Halai D, Wilson GA, Feber A, Teschendorff AE,\nGomez V, Hergovich A, Tirabosco R, Fernanda Amary M, Baumhoer D, Jundt G, Ross\nMT, Flanagan AM, Beck S.\n\nAuthor information: \n(1)Medical Genomics, UCL Cancer Institute, University College London, London, UK.\n\nIsocitrate dehydrogenase (IDH) genes 1 and 2 are frequently mutated in acute\nmyeloid leukaemia (AML), low-grade glioma, cholangiocarcinoma (CC) and\nchondrosarcoma (CS). For AML, low-grade glioma and CC, mutant IDH status is\nassociated with a DNA hypermethylation phenotype, implicating altered epigenome\ndynamics in the aetiology of these cancers. Here we show that the IDH variants in\nCS are also associated with a hypermethylation phenotype and display increased\nproduction of the oncometabolite 2-hydroxyglutarate, supporting the role of\nmutant IDH-produced 2-hydroxyglutarate as an inhibitor of TET-mediated DNA\ndemethylation. Meta-analysis of the acute myeloid leukaemia, low-grade glioma,\ncholangiocarcinoma and CS methylation data identifies cancer-specific effectors\nwithin the retinoic acid receptor activation pathway among the hypermethylated\ntargets. By analysing sequence motifs surrounding hypermethylated sites across\nthe four cancer types, and using chromatin immunoprecipitation and western\nblotting, we identify the transcription factor EBF1 (early B-cell factor 1) as an\ninteraction partner for TET2, suggesting a sequence-specific mechanism for\nregulating DNA methylation.\n\nPMCID: PMC3759038\nPMID: 23863747  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7806",
        "source" : "7715",
        "target" : "7644",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "TP53 (activates.process) G2/M ARREST",
        "edgeType" : "activates.process",
        "SUID" : 7806,
        "name" : "TP53 (activates.process) G2/M ARREST",
        "interaction" : "activates.process",
        "shared_interaction" : "activates.process",
        "shared_name" : "TP53 (activates.process) G2/M ARREST",
        "edgeDetail" : "TP53 (activates.process) G2/M ARREST\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7805",
        "source" : "7715",
        "target" : "7686",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "TP53 (activates.process) APOPTOSIS",
        "edgeType" : "activates.process",
        "SUID" : 7805,
        "name" : "TP53 (activates.process) APOPTOSIS",
        "interaction" : "activates.process",
        "shared_interaction" : "activates.process",
        "shared_name" : "TP53 (activates.process) APOPTOSIS",
        "edgeDetail" : "TP53 (activates.process) APOPTOSIS\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7804",
        "source" : "7715",
        "target" : "7755",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "TP53 (activates) MDM2",
        "edgeType" : "activates",
        "SUID" : 7804,
        "name" : "TP53 (activates) MDM2",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "TP53 (activates) MDM2",
        "edgeDetail" : "TP53 (activates) MDM2\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7803",
        "source" : "7715",
        "target" : "7785",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "TP53 (activates) CDKN1A",
        "edgeType" : "activates",
        "SUID" : 7803,
        "name" : "TP53 (activates) CDKN1A",
        "interaction" : "activates",
        "shared_interaction" : "activates",
        "shared_name" : "TP53 (activates) CDKN1A",
        "edgeDetail" : "TP53 (activates) CDKN1A\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7802",
        "source" : "7715",
        "target" : "7751",
        "pmid" : "23224642",
        "selected" : false,
        "canonicalName" : "TP53 (inhibits) MGMT",
        "edgeType" : "inhibits",
        "SUID" : 7802,
        "name" : "TP53 (inhibits) MGMT",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "TP53 (inhibits) MGMT",
        "edgeDetail" : "TP53 (inhibits) MGMT\n\n1. Neurol Sci. 2013 Aug;34(8):1421-8. doi: 10.1007/s10072-012-1257-9. Epub 2012 Dec \n8.\n\nMutant TP53 enhances the resistance of glioblastoma cells to temozolomide by\nup-regulating O(6)-methylguanine DNA-methyltransferase.\n\nWang X(1), Chen JX, Liu YH, You C, Mao Q.\n\nAuthor information: \n(1)Department of Neurosurgery, West China Hospital, Sichuan University, 37 Guo\nXuexiang, Chengdu, 610041, Sichuan, China.\n\nThe \"gain of function\" of mutant TP53 is an important determinant in human tumor \ndevelopment and progression. This study aimed to investigate the possible\nmechanism of mutant TP53 inducing temozolomide resistance in glioblastoma cells. \nThree established human glioma cell lines, T98G, U87, and U138, were\nchemoresistant cells. The mRNA of cells was sequenced to confirm the status of\nTP53. Synthetic small interfering RNA (siRNA) was used to knock down TP53 in\ncells. TP53 mRNA was detected \"silenced\" by reverse transcriptase-polymerase\nchain reaction (RT-PCR) in five consecutive days. Viable cell survival was\nmeasured when these cells were exposed to temozolomide or semustine in step-up\nconcentrations. The expression of O(6)-methylguanine DNA-methyltransferase (MGMT)\nat mRNA level was also determined. T98G, U87, and U138 cells were resistant to\ntemozolomide. T98G and U138 cells expressed mutant-type TP53 with positive MGMT, \nwhile U87 cell expressed wild-type TP53 with negative MGMT. TP53-siRNA knocked\ndown TP53 effectively (P = 0.021) in five consecutive days. Knockdown of mutant\nTP53 in T98G and U138 cells led to a fivefold increase in chemosensitivity to\ntemozolomide, but not semustine. Knockdown of wild TP53 in U87 cell did not\naffect the chemoresistance. In addition, mutant TP53 knockdown induced a dramatic\ndecrease of MGMT expression (P = 0.0000034). TP53 mutation decreases the\nchemosensitivity of malignant gliomas to temozolomide. This \"gain of function\" in\ndrug resistance may be obtained by increasing MGMT expression.\n\nPMID: 23224642  [PubMed - in process]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7801",
        "source" : "7714",
        "target" : "7643",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "TSC complex (contains) TSC1",
        "edgeType" : "contains",
        "SUID" : 7801,
        "name" : "TSC complex (contains) TSC1",
        "interaction" : "contains",
        "shared_interaction" : "contains",
        "shared_name" : "TSC complex (contains) TSC1",
        "edgeDetail" : "TSC complex (contains) TSC1\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7800",
        "source" : "7714",
        "target" : "7642",
        "pmid" : "18772890",
        "selected" : false,
        "canonicalName" : "TSC complex (contains) TSC2",
        "edgeType" : "contains",
        "SUID" : 7800,
        "name" : "TSC complex (contains) TSC2",
        "interaction" : "contains",
        "shared_interaction" : "contains",
        "shared_name" : "TSC complex (contains) TSC2",
        "edgeDetail" : "TSC complex (contains) TSC2\nID list is empty! Possibly it has no correct IDs.\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7799",
        "source" : "7714",
        "target" : "7747",
        "pmid" : "23641065",
        "selected" : false,
        "canonicalName" : "TSC complex (inhibits) mTORC1",
        "edgeType" : "inhibits",
        "SUID" : 7799,
        "name" : "TSC complex (inhibits) mTORC1",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "TSC complex (inhibits) mTORC1",
        "edgeDetail" : "TSC complex (inhibits) mTORC1\n\n1. J Cell Sci. 2013 Apr 15;126(Pt 8):1713-9. doi: 10.1242/jcs.125773. Epub 2013 May \n2.\n\nRegulation of mTORC1 and its impact on gene expression at a glance.\n\nLaplante M, Sabatini DM.\n\nCentre de Recherche de l'Institut Universitaire de Cardiologie et de Pneumologie \nde Qubec (CRIUCPQ), Facult de Mdecine, Universit Laval, 2725 Chemin Ste-Foy, \nQubec, QC, G1V 4G5, Canada. mathieu.laplante@criucpq.ulaval.ca\n\nThe mechanistic (or mammalian) target of rapamycin (mTOR) is a kinase that\nregulates key cellular functions linked to the promotion of cell growth and\nmetabolism. This kinase, which is part of two protein complexes termed mTOR\ncomplex 1 (mTORC1) and 2 (mTORC2), has a fundamental role in coordinating\nanabolic and catabolic processes in response to growth factors and nutrients. Of \nthe two mTOR complexes, mTORC1 is by far the best characterized. When active,\nmTORC1 triggers cell growth and proliferation by promoting protein synthesis,\nlipid biogenesis, and metabolism, and by reducing autophagy. The fact that mTORC1\nderegulation is associated with several human diseases, such as type 2 diabetes, \ncancer, obesity and neurodegeneration, highlights its importance in the\nmaintenance of cellular homeostasis. Over the last years, several groups observed\nthat mTORC1 inhibition, in addition to reducing protein synthesis, deeply affects\ngene transcription. Here, we review the connections between mTORC1 and gene\ntranscription by focusing on its impact in regulating the activation of specific \ntranscription factors including including STAT3, SREBPs, PPAR, PPAR, HIF1,\nYY1PGC1 and TFEB. We also discuss the importance of these transcription factors\nin mediating the effects of mTORC1 on various cellular processes in physiological\nand pathological contexts.\n\nPMCID: PMC3678406 [Available on 2014/4/15]\nPMID: 23641065  [PubMed - indexed for MEDLINE]\n\n"
      },
      "selected" : false
    }, {
      "data" : {
        "id" : "7798",
        "source" : "7713",
        "target" : "7750",
        "pmid" : "23818228",
        "selected" : false,
        "canonicalName" : "ZEB1 (inhibits) MIR200C",
        "edgeType" : "inhibits",
        "SUID" : 7798,
        "name" : "ZEB1 (inhibits) MIR200C",
        "interaction" : "inhibits",
        "shared_interaction" : "inhibits",
        "shared_name" : "ZEB1 (inhibits) MIR200C",
        "edgeDetail" : "ZEB1 (inhibits) MIR200C\n\n1. EMBO Mol Med. 2013 Aug;5(8):1196-212. doi: 10.1002/emmm.201302827. Epub 2013 Jul \n1.\n\nThe ZEB1 pathway links glioblastoma initiation, invasion and chemoresistance.\n\nSiebzehnrubl FA, Silver DJ, Tugertimur B, Deleyrolle LP, Siebzehnrubl D,\nSarkisian MR, Devers KG, Yachnis AT, Kupper MD, Neal D, Nabilsi NH, Kladde MP,\nSuslov O, Brabletz S, Brabletz T, Reynolds BA, Steindler DA.\n\nDepartment of Neurosurgery, University of Florida, Gainesville, FL, USA.\nfas@ufl.edu\n\nGlioblastoma remains one of the most lethal types of cancer, and is the most\ncommon brain tumour in adults. In particular, tumour recurrence after surgical\nresection and radiation invariably occurs regardless of aggressive chemotherapy. \nHere, we provide evidence that the transcription factor ZEB1 (zinc finger E-box\nbinding homeobox 1) exerts simultaneous influence over invasion, chemoresistance \nand tumourigenesis in glioblastoma. ZEB1 is preferentially expressed in invasive \nglioblastoma cells, where the ZEB1-miR-200 feedback loop interconnects these\nprocesses through the downstream effectors ROBO1, c-MYB and MGMT. Moreover, ZEB1 \nexpression in glioblastoma patients is predictive of shorter survival and poor\nTemozolomide response. Our findings indicate that this regulator of\nepithelial-mesenchymal transition orchestrates key features of cancer stem cells \nin malignant glioma and identify ROBO1, OLIG2, CD133 and MGMT as novel targets of\nthe ZEB1 pathway. Thus, ZEB1 is an important candidate molecule for glioblastoma \nrecurrence, a marker of invasive tumour cells and a potential therapeutic target,\nalong with its downstream effectors.\n\n 2013 The Authors. Published by John Wiley and Sons, Ltd on behalf of EMBO.\n\nPMID: 23818228  [PubMed - in process]\n\n"
      },
      "selected" : false
    } ]
  }
}
curatedGBMVizmap = [ {
  "format_version" : "1.0",
  "generated_by" : "cytoscape-3.1.0",
  "target_cytoscapejs_version" : "~2.1",
  "title" : "GBMcurated",
  "style" : [ {
    "selector" : "node",
    "css" : {
      "text-valign" : "center",
      "text-halign" : "center",
      "font-size" : 60,
      "background-color" : "rgb(255,255,255)",
      "color" : "rgb(0,0,0)",
      "shape" : "ellipse",
      "height" : 100.0,
      "border-width" : 1.0,
      "border-opacity" : 1.0,
      "width" : 100.0,
      "text-opacity" : 1.0,
      "font-family" : "SansSerif",
      "font-weight" : "bold",
      "border-color" : "rgb(128,128,128)",
      "background-opacity" : 1.0,
      "content" : "data(label)"
    }
  }, {
    "selector" : "node[nodeType = 'process']",
    "css" : {
      "shape" : "rectangle"
    }
  }, {
    "selector" : "node[nodeType = 'complex']",
    "css" : {
      "shape" : "hexagon"
    }
  }, {
    "selector" : "node[nodeType = 'mutation']",
    "css" : {
      "shape" : "star"
    }
  }, {
    "selector" : "node[nodeType = 'gene']",
    "css" : {
      "shape" : "ellipse"
    }
  }, {
    "selector" : "node[nodeType = 'family']",
    "css" : {
      "shape" : "roundrectangle"
    }
  }, {
    "selector" : "node[nodeType = 'smallMolecule']",
    "css" : {
      "shape" : "diamond"
    }


  }, {
    "selector" : "node[score = 20000]",
    "css" : {
      "width" : 250.0
    }

  }, {
    "selector" : "node[score > 100000]",
    "css" : {
      "width" : 400.0
    }

  }, {
    "selector" : "node[score > 100000]",
    "css" : {
      "height" : 400.0
    }

  }, {
    "selector" : "node[score > 20000.0][score < 100000.0]",
    "css" : {
      "width" : "mapData(score,20000.0,100000.0, 350.0, 600.0)"
    }

  }, {
    "selector" : "node[score > 0.0][score < 20000.0]",
    "css" : {
      "width" : "mapData(score,0.0,20000.0, 200.0, 350.0)"
    }


  }, {
    "selector" : "node[score = 20000]",
    "css" : {
      "heigth" : 250.0
    }

  }, {
    "selector" : "node[score > 20000.0][score < 100000.0]",
    "css" : {
      "height" : "mapData(score,20000.0,100000.0, 350.0, 600.0)"
    }

  }, {
    "selector" : "node[score > 0.0][score < 20000.0]",
    "css" : {
      "height" : "mapData(score,0.0,20000.0,200.0, 350.0)"
    }

  }, {
    "selector" : "node[score > 8000.0]",
    "css" : {
      "background-color" : "rgb(221,0,0)"
    }
  }, {
    "selector" : "node[score = 20000.0]",
    "css" : {
      "background-color" : "rgb(221,0,0)"
    }

  }, {
    "selector" : "node[score > 1.0][score < 20000.0]",
    "css" : {
      "background-color" : "mapData(score,1.0,20000.0,rgb(255,200,200),rgb(221,0,0))"
    }

  }, {
    "selector" : "node[score < 1.0]",
    "css" : {
      "background-color" : "rgb(255,255,255)"
    }

  }, {
    "selector" : "node[score = -80.0]",
    "css" : {
      "background-color" : "rgb(0,0,221)"
    }
  }, {
    "selector" : "node[score < -80.0]",
    "css" : {
      "background-color" : "rgb(0,0,221)"
    }
  }, {
    "selector" : "node[mut > 20.0]",
    "css" : {
      "border-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "node[cnv = 2]",
    "css" : {
      "border-width" : 40.0
    }
  }, {
    "selector" : "node[cnv = 1]",
    "css" : {
      "border-width" : 20.0
    }
  }, {
    "selector" : "node[cnv = 0]",
    "css" : {
      "border-width" : 1.0
    }
  }, {
    "selector" : "node[cnv = -2]",
    "css" : {
      "border-width" : 40.0
    }
  }, {
    "selector" : "node[cnv = -1]",
    "css" : {
      "border-width" : 20.0
    }
  }, {
    "selector" : "node[cnv = 2]",
    "css" : {
      "border-color" : "rgb(102,255,255)"
    }
  }, {
    "selector" : "node[cnv = 1]",
    "css" : {
      "border-color" : "rgb(102,255,255)"
    }
  }, {
    "selector" : "node[cnv = 0]",
    "css" : {
      "border-color" : "rgb(128, 128, 128)"
    }
  }, {
    "selector" : "node[cnv = -2]",
    "css" : {
      "border-color" : "rgb(102, 255, 51)"
    }
  }, {
    "selector" : "node[cnv = -1]",
    "css" : {
      "border-color" : "rgb(102, 255, 51)"
    }
  }, {
    "selector" : "node[mut = 20.0]",
    "css" : {
      "border-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "node[mut > 1.0][mut < 20.0]",
    "css" : {
      "border-color" : "mapData(mut,1.0,20.0,rgb(255,170,255),rgb(221,0,221))"
    }
  }, {
    "selector" : "node[mut > 0.0][mut < 1.0]",
    "css" : {
      "border-color" : "mapData(mut,0.0,1.0,rgb(255,255,255),rgb(255,170,255))"
    }
  }, {
    "selector" : "node[mut = 0.0]",
    "css" : {
      "border-color" : "rgb(255,255,255)"
    }
  }, {
    "selector" : "node[mut < 0.0]",
    "css" : {
      "border-color" : "rgb(255,255,255)"
    }
  }, {
    "selector" : "node:selected",
    "css" : {
      "background-color" : "rgb(255,255,0)"
    }
  }, {
    "selector" : "edge",
    "css" : {
      "line-color" : "rgb(255,255,255)",
      "text-opacity" : 1.0,
      "source-arrow-shape" : "triangle",
      "opacity" : 1.0,
      "width" : 6.0,
      "content" : "",
      "source-arrow-color" : "rgb(0,0,0)",
      "font-family" : "Dialog",
      "font-weight" : "normal",
      "font-size" : 10,
      "color" : "rgb(0,0,0)",
      "line-style" : "solid",
      "target-arrow-color" : "rgb(0,0,0)",
      "target-arrow-shape" : "triangle"
    }
  }, {
    "selector" : "edge[edgeType = 'expression']",
    "css" : {
      "line-color" : "rgb(0,0,170)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0403(colocalization)']",
    "css" : {
      "line-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'compound']",
    "css" : {
      "line-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'dissociation']",
    "css" : {
      "line-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'functional']",
    "css" : {
      "line-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'inhibition;phosphorylation']",
    "css" : {
      "line-color" : "rgb(221,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'contains']",
    "css" : {
      "line-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0407(direct interaction)']",
    "css" : {
      "line-color" : "rgb(0,170,170)"
    }
  }, {
    "selector" : "edge[edgeType = 'family.includes']",
    "css" : {
      "line-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'Mi:0933(negative genetic interaction)']",
    "css" : {
      "line-color" : "rgb(0,0,170)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0799(additive genetic interaction defined by inequality)']",
    "css" : {
      "line-color" : "rgb(170,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0915(physical association)']",
    "css" : {
      "line-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0217(phosphorylation reaction)']",
    "css" : {
      "line-color" : "rgb(0,170,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0935(positive genetic interaction)']",
    "css" : {
      "line-color" : "rgb(0,221,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0203(dephosphorylation reaction)']",
    "css" : {
      "line-color" : "rgb(0,170,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'inhibition']",
    "css" : {
      "line-color" : "rgb(221,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0914(association)']",
    "css" : {
      "line-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'leads_to']",
    "css" : {
      "line-color" : "rgb(0,0,170)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0414(enzymatic reaction)']",
    "css" : {
      "line-color" : "rgb(0,170,0)"
    }
  }, {
    "selector" : "edge[edgeType = '-']",
    "css" : {
      "line-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'activates.process']",
    "css" : {
      "line-color" : "rgb(0,170,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'unspecified']",
    "css" : {
      "line-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0794(synthetic genetic interaction defined by inequality)']",
    "css" : {
      "line-color" : "rgb(170,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0796(suppressive genetic interaction defined by inequality)']",
    "css" : {
      "line-color" : "rgb(170,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'complex.includes']",
    "css" : {
      "line-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0194(cleavage)']",
    "css" : {
      "line-color" : "rgb(0,170,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'includes']",
    "css" : {
      "line-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0220(ubiquitination reaction)']",
    "css" : {
      "line-color" : "rgb(0,170,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'ubiquination']",
    "css" : {
      "line-color" : "rgb(221,0,221)"
    }
  }, {
    "selector" : "edge[edgeType = 'binding/association']",
    "css" : {
      "line-color" : "rgb(0,170,170)"
    }
  }, {
    "selector" : "edge[edgeType = 'activation;phosphorylation']",
    "css" : {
      "line-color" : "rgb(0,170,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'indirect effect']",
    "css" : {
      "line-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'repression']",
    "css" : {
      "line-color" : "rgb(221,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'activation']",
    "css" : {
      "line-color" : "rgb(0,170,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0567(neddylation reaction)']",
    "css" : {
      "line-color" : "rgb(0,170,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0191(aggregation)']",
    "css" : {
      "line-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'activates']",
    "css" : {
      "line-color" : "rgb(0,170,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'dimerize']",
    "css" : {
      "line-color" : "rgb(0,170,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'missing interaction']",
    "css" : {
      "line-color" : "rgb(170,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0192(acetylation reaction)']",
    "css" : {
      "line-color" : "rgb(0,170,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'class.includes']",
    "css" : {
      "line-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0190']",
    "css" : {
      "line-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0217(phosphorylation)']",
    "css" : {
      "line-color" : "rgb(221,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0570(protein cleavage)']",
    "css" : {
      "line-color" : "rgb(0,170,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0194(cleavage reaction)']",
    "css" : {
      "line-color" : "rgb(0,170,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'ubiquitination']",
    "css" : {
      "line-color" : "rgb(221,0,221)"
    }
  }, {
    "selector" : "edge[edgeType = 'inhibits']",
    "css" : {
      "line-color" : "rgb(221,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'dephosphorylation']",
    "css" : {
      "line-color" : "rgb(0,170,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'inhibits.process']",
    "css" : {
      "line-color" : "rgb(221,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'phosphorylation']",
    "css" : {
      "line-color" : "rgb(0,170,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'expression']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0403(colocalization)']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'dissociation']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'compound']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'functional']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'inhibition;phosphorylation']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0407(direct interaction)']",
    "css" : {
      "source-arrow-shape" : "triangle"
    }
  }, {
    "selector" : "edge[edgeType = 'contains']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'family.includes']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'Mi:0933(negative genetic interaction)']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0799(additive genetic interaction defined by inequality)']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0915(physical association)']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0217(phosphorylation reaction)']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0935(positive genetic interaction)']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0203(dephosphorylation reaction)']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'inhibition']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0914(association)']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'leads_to']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0414(enzymatic reaction)']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = '-']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'activates.process']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'unspecified']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0796(suppressive genetic interaction defined by inequality)']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0794(synthetic genetic interaction defined by inequality)']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'complex.includes']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0194(cleavage)']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'includes']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0220(ubiquitination reaction)']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'binding/association']",
    "css" : {
      "source-arrow-shape" : "triangle"
    }
  }, {
    "selector" : "edge[edgeType = 'ubiquination']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'activation;phosphorylation']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'indirect effect']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'repression']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'activation']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0567(neddylation reaction)']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0191(aggregation)']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'activates']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'dimerize']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'missing interaction']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'class.includes']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0192(acetylation reaction)']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0217(phosphorylation)']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0190']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0570(protein cleavage)']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0194(cleavage reaction)']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'ubiquitination']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'inhibits']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'dephosphorylation']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'inhibits.process']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'phosphorylation']",
    "css" : {
      "source-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'expression']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0403(colocalization)']",
    "css" : {
      "width" : 4.0
    }
  }, {
    "selector" : "edge[edgeType = 'compound']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'dissociation']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'functional']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'inhibition;phosphorylation']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0407(direct interaction)']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'contains']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'family.includes']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'Mi:0933(negative genetic interaction)']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0799(additive genetic interaction defined by inequality)']",
    "css" : {
      "width" : 5.0
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0915(physical association)']",
    "css" : {
      "width" : 2.0
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0217(phosphorylation reaction)']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0935(positive genetic interaction)']",
    "css" : {
      "width" : 5.0
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0203(dephosphorylation reaction)']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0914(association)']",
    "css" : {
      "width" : 2.0
    }
  }, {
    "selector" : "edge[edgeType = 'leads_to']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'inhibition']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0414(enzymatic reaction)']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = '-']",
    "css" : {
      "width" : 1.0
    }
  }, {
    "selector" : "edge[edgeType = 'activates.process']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'unspecified']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0794(synthetic genetic interaction defined by inequality)']",
    "css" : {
      "width" : 5.0
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0796(suppressive genetic interaction defined by inequality)']",
    "css" : {
      "width" : 5.0
    }
  }, {
    "selector" : "edge[edgeType = 'complex.includes']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0194(cleavage)']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'includes']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0220(ubiquitination reaction)']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'binding/association']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'ubiquination']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'activation;phosphorylation']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'indirect effect']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'repression']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'activation']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0567(neddylation reaction)']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0191(aggregation)']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'activates']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'dimerize']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'missing interaction']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0192(acetylation reaction)']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'class.includes']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0190']",
    "css" : {
      "width" : 1.0
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0217(phosphorylation)']",
    "css" : {
      "width" : 7.0
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0570(protein cleavage)']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0194(cleavage reaction)']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'ubiquitination']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'inhibits']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'dephosphorylation']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'inhibits.process']",
    "css" : {
      "width" : 4.0
    }
  }, {
    "selector" : "edge[edgeType = 'phosphorylation']",
    "css" : {
      "width" : 6.0
    }
  }, {
    "selector" : "edge[edgeType = 'expression']",
    "css" : {
      "source-arrow-color" : "rgb(0,0,170)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0403(colocalization)']",
    "css" : {
      "source-arrow-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'compound']",
    "css" : {
      "source-arrow-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'dissociation']",
    "css" : {
      "source-arrow-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'functional']",
    "css" : {
      "source-arrow-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'inhibition;phosphorylation']",
    "css" : {
      "source-arrow-color" : "rgb(221,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'contains']",
    "css" : {
      "source-arrow-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0407(direct interaction)']",
    "css" : {
      "source-arrow-color" : "rgb(0,170,170)"
    }
  }, {
    "selector" : "edge[edgeType = 'family.includes']",
    "css" : {
      "source-arrow-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'Mi:0933(negative genetic interaction)']",
    "css" : {
      "source-arrow-color" : "rgb(0,0,170)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0799(additive genetic interaction defined by inequality)']",
    "css" : {
      "source-arrow-color" : "rgb(170,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0915(physical association)']",
    "css" : {
      "source-arrow-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0217(phosphorylation reaction)']",
    "css" : {
      "source-arrow-color" : "rgb(0,170,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0935(positive genetic interaction)']",
    "css" : {
      "source-arrow-color" : "rgb(0,221,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0203(dephosphorylation reaction)']",
    "css" : {
      "source-arrow-color" : "rgb(0,170,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0914(association)']",
    "css" : {
      "source-arrow-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'leads_to']",
    "css" : {
      "source-arrow-color" : "rgb(0,0,170)"
    }
  }, {
    "selector" : "edge[edgeType = 'inhibition']",
    "css" : {
      "source-arrow-color" : "rgb(221,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0414(enzymatic reaction)']",
    "css" : {
      "source-arrow-color" : "rgb(0,170,0)"
    }
  }, {
    "selector" : "edge[edgeType = '-']",
    "css" : {
      "source-arrow-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'activates.process']",
    "css" : {
      "source-arrow-color" : "rgb(0,170,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'unspecified']",
    "css" : {
      "source-arrow-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0794(synthetic genetic interaction defined by inequality)']",
    "css" : {
      "source-arrow-color" : "rgb(170,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0796(suppressive genetic interaction defined by inequality)']",
    "css" : {
      "source-arrow-color" : "rgb(170,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'complex.includes']",
    "css" : {
      "source-arrow-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0194(cleavage)']",
    "css" : {
      "source-arrow-color" : "rgb(0,170,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'includes']",
    "css" : {
      "source-arrow-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0220(ubiquitination reaction)']",
    "css" : {
      "source-arrow-color" : "rgb(0,170,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'binding/association']",
    "css" : {
      "source-arrow-color" : "rgb(0,170,170)"
    }
  }, {
    "selector" : "edge[edgeType = 'ubiquination']",
    "css" : {
      "source-arrow-color" : "rgb(221,0,221)"
    }
  }, {
    "selector" : "edge[edgeType = 'activation;phosphorylation']",
    "css" : {
      "source-arrow-color" : "rgb(0,170,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'indirect effect']",
    "css" : {
      "source-arrow-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'repression']",
    "css" : {
      "source-arrow-color" : "rgb(221,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'activation']",
    "css" : {
      "source-arrow-color" : "rgb(0,170,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0567(neddylation reaction)']",
    "css" : {
      "source-arrow-color" : "rgb(0,170,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0191(aggregation)']",
    "css" : {
      "source-arrow-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'activates']",
    "css" : {
      "source-arrow-color" : "rgb(0,170,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'dimerize']",
    "css" : {
      "source-arrow-color" : "rgb(0,170,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'missing interaction']",
    "css" : {
      "source-arrow-color" : "rgb(170,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'class.includes']",
    "css" : {
      "source-arrow-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0192(acetylation reaction)']",
    "css" : {
      "source-arrow-color" : "rgb(0,170,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0217(phosphorylation)']",
    "css" : {
      "source-arrow-color" : "rgb(221,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0190']",
    "css" : {
      "source-arrow-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0570(protein cleavage)']",
    "css" : {
      "source-arrow-color" : "rgb(0,170,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'ubiquitination']",
    "css" : {
      "source-arrow-color" : "rgb(221,0,221)"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0194(cleavage reaction)']",
    "css" : {
      "source-arrow-color" : "rgb(0,170,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'inhibits']",
    "css" : {
      "source-arrow-color" : "rgb(221,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'dephosphorylation']",
    "css" : {
      "source-arrow-color" : "rgb(0,170,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'inhibits.process']",
    "css" : {
      "source-arrow-color" : "rgb(221,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'phosphorylation']",
    "css" : {
      "source-arrow-color" : "rgb(0,170,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'expression']",
    "css" : {
      "line-style" : "solid"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0403(colocalization)']",
    "css" : {
      "line-style" : "solid"
    }
  }, {
    "selector" : "edge[edgeType = 'dissociation']",
    "css" : {
      "line-style" : "dotted"
    }
  }, {
    "selector" : "edge[edgeType = 'compound']",
    "css" : {
      "line-style" : "dotted"
    }
  }, {
    "selector" : "edge[edgeType = 'inhibition;phosphorylation']",
    "css" : {
      "line-style" : "dashed"
    }
  }, {
    "selector" : "edge[edgeType = 'functional']",
    "css" : {
      "line-style" : "dotted"
    }
  }, {
    "selector" : "edge[edgeType = 'contains']",
    "css" : {
      "line-style" : "dotted"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0407(direct interaction)']",
    "css" : {
      "line-style" : "solid"
    }
  }, {
    "selector" : "edge[edgeType = 'family.includes']",
    "css" : {
      "line-style" : "dotted"
    }
  }, {
    "selector" : "edge[edgeType = 'Mi:0933(negative genetic interaction)']",
    "css" : {
      "line-style" : "solid"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0799(additive genetic interaction defined by inequality)']",
    "css" : {
      "line-style" : "solid"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0915(physical association)']",
    "css" : {
      "line-style" : "dotted"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0217(phosphorylation reaction)']",
    "css" : {
      "line-style" : "dashed"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0935(positive genetic interaction)']",
    "css" : {
      "line-style" : "solid"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0203(dephosphorylation reaction)']",
    "css" : {
      "line-style" : "dotted"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0914(association)']",
    "css" : {
      "line-style" : "dotted"
    }
  }, {
    "selector" : "edge[edgeType = 'inhibition']",
    "css" : {
      "line-style" : "dotted"
    }
  }, {
    "selector" : "edge[edgeType = 'leads_to']",
    "css" : {
      "line-style" : "solid"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0414(enzymatic reaction)']",
    "css" : {
      "line-style" : "dashed"
    }
  }, {
    "selector" : "edge[edgeType = '-']",
    "css" : {
      "line-style" : "dashed"
    }
  }, {
    "selector" : "edge[edgeType = 'activates.process']",
    "css" : {
      "line-style" : "solid"
    }
  }, {
    "selector" : "edge[edgeType = 'unspecified']",
    "css" : {
      "line-style" : "dotted"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0794(synthetic genetic interaction defined by inequality)']",
    "css" : {
      "line-style" : "solid"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0796(suppressive genetic interaction defined by inequality)']",
    "css" : {
      "line-style" : "solid"
    }
  }, {
    "selector" : "edge[edgeType = 'complex.includes']",
    "css" : {
      "line-style" : "dotted"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0194(cleavage)']",
    "css" : {
      "line-style" : "solid"
    }
  }, {
    "selector" : "edge[edgeType = 'includes']",
    "css" : {
      "line-style" : "dotted"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0220(ubiquitination reaction)']",
    "css" : {
      "line-style" : "dashed"
    }
  }, {
    "selector" : "edge[edgeType = 'ubiquination']",
    "css" : {
      "line-style" : "solid"
    }
  }, {
    "selector" : "edge[edgeType = 'binding/association']",
    "css" : {
      "line-style" : "solid"
    }
  }, {
    "selector" : "edge[edgeType = 'activation;phosphorylation']",
    "css" : {
      "line-style" : "dashed"
    }
  }, {
    "selector" : "edge[edgeType = 'repression']",
    "css" : {
      "line-style" : "solid"
    }
  }, {
    "selector" : "edge[edgeType = 'indirect effect']",
    "css" : {
      "line-style" : "dashed"
    }
  }, {
    "selector" : "edge[edgeType = 'activation']",
    "css" : {
      "line-style" : "solid"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0567(neddylation reaction)']",
    "css" : {
      "line-style" : "dashed"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0191(aggregation)']",
    "css" : {
      "line-style" : "dotted"
    }
  }, {
    "selector" : "edge[edgeType = 'activates']",
    "css" : {
      "line-style" : "solid"
    }
  }, {
    "selector" : "edge[edgeType = 'dimerize']",
    "css" : {
      "line-style" : "dotted"
    }
  }, {
    "selector" : "edge[edgeType = 'missing interaction']",
    "css" : {
      "line-style" : "dotted"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0192(acetylation reaction)']",
    "css" : {
      "line-style" : "dashed"
    }
  }, {
    "selector" : "edge[edgeType = 'class.includes']",
    "css" : {
      "line-style" : "dotted"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0217(phosphorylation)']",
    "css" : {
      "line-style" : "solid"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0190']",
    "css" : {
      "line-style" : "dashed"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0570(protein cleavage)']",
    "css" : {
      "line-style" : "solid"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0194(cleavage reaction)']",
    "css" : {
      "line-style" : "solid"
    }
  }, {
    "selector" : "edge[edgeType = 'ubiquitination']",
    "css" : {
      "line-style" : "solid"
    }
  }, {
    "selector" : "edge[edgeType = 'inhibits']",
    "css" : {
      "line-style" : "solid"
    }
  }, {
    "selector" : "edge[edgeType = 'dephosphorylation']",
    "css" : {
      "line-style" : "dotted"
    }
  }, {
    "selector" : "edge[edgeType = 'inhibits.process']",
    "css" : {
      "line-style" : "solid"
    }
  }, {
    "selector" : "edge[edgeType = 'phosphorylation']",
    "css" : {
      "line-style" : "dashed"
    }
  }, {
    "selector" : "edge[edgeType = 'family.includes']",
    "css" : {
      "target-arrow-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'inhibits']",
    "css" : {
      "target-arrow-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'activates.process']",
    "css" : {
      "target-arrow-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'activates']",
    "css" : {
      "target-arrow-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'inhibits.process']",
    "css" : {
      "target-arrow-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'contains']",
    "css" : {
      "target-arrow-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'class.includes']",
    "css" : {
      "target-arrow-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'complex.includes']",
    "css" : {
      "target-arrow-color" : "rgb(0,0,0)"
    }
  }, {
    "selector" : "edge[edgeType = 'expression']",
    "css" : {
      "target-arrow-shape" : "triangle"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0403(colocalization)']",
    "css" : {
      "target-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'compound']",
    "css" : {
      "target-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'dissociation']",
    "css" : {
      "target-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'inhibition;phosphorylation']",
    "css" : {
      "target-arrow-shape" : "tee"
    }
  }, {
    "selector" : "edge[edgeType = 'functional']",
    "css" : {
      "target-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'contains']",
    "css" : {
      "target-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0407(direct interaction)']",
    "css" : {
      "target-arrow-shape" : "triangle"
    }
  }, {
    "selector" : "edge[edgeType = 'family.includes']",
    "css" : {
      "target-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'Mi:0933(negative genetic interaction)']",
    "css" : {
      "target-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0799(additive genetic interaction defined by inequality)']",
    "css" : {
      "target-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0915(physical association)']",
    "css" : {
      "target-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0217(phosphorylation reaction)']",
    "css" : {
      "target-arrow-shape" : "triangle"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0935(positive genetic interaction)']",
    "css" : {
      "target-arrow-shape" : "triangle"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0203(dephosphorylation reaction)']",
    "css" : {
      "target-arrow-shape" : "triangle"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0914(association)']",
    "css" : {
      "target-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'inhibition']",
    "css" : {
      "target-arrow-shape" : "tee"
    }
  }, {
    "selector" : "edge[edgeType = 'leads_to']",
    "css" : {
      "target-arrow-shape" : "triangle"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0414(enzymatic reaction)']",
    "css" : {
      "target-arrow-shape" : "triangle"
    }
  }, {
    "selector" : "edge[edgeType = '-']",
    "css" : {
      "target-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'activates.process']",
    "css" : {
      "target-arrow-shape" : "triangle"
    }
  }, {
    "selector" : "edge[edgeType = 'unspecified']",
    "css" : {
      "target-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0796(suppressive genetic interaction defined by inequality)']",
    "css" : {
      "target-arrow-shape" : "tee"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0794(synthetic genetic interaction defined by inequality)']",
    "css" : {
      "target-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'complex.includes']",
    "css" : {
      "target-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0194(cleavage)']",
    "css" : {
      "target-arrow-shape" : "triangle"
    }
  }, {
    "selector" : "edge[edgeType = 'includes']",
    "css" : {
      "target-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0220(ubiquitination reaction)']",
    "css" : {
      "target-arrow-shape" : "triangle"
    }
  }, {
    "selector" : "edge[edgeType = 'ubiquination']",
    "css" : {
      "target-arrow-shape" : "triangle"
    }
  }, {
    "selector" : "edge[edgeType = 'binding/association']",
    "css" : {
      "target-arrow-shape" : "triangle"
    }
  }, {
    "selector" : "edge[edgeType = 'activation;phosphorylation']",
    "css" : {
      "target-arrow-shape" : "triangle"
    }
  }, {
    "selector" : "edge[edgeType = 'indirect effect']",
    "css" : {
      "target-arrow-shape" : "triangle"
    }
  }, {
    "selector" : "edge[edgeType = 'repression']",
    "css" : {
      "target-arrow-shape" : "tee"
    }
  }, {
    "selector" : "edge[edgeType = 'activation']",
    "css" : {
      "target-arrow-shape" : "triangle"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0567(neddylation reaction)']",
    "css" : {
      "target-arrow-shape" : "triangle"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0191(aggregation)']",
    "css" : {
      "target-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'activates']",
    "css" : {
      "target-arrow-shape" : "triangle"
    }
  }, {
    "selector" : "edge[edgeType = 'dimerize']",
    "css" : {
      "target-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'missing interaction']",
    "css" : {
      "target-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'class.includes']",
    "css" : {
      "target-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0192(acetylation reaction)']",
    "css" : {
      "target-arrow-shape" : "triangle"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0190']",
    "css" : {
      "target-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0217(phosphorylation)']",
    "css" : {
      "target-arrow-shape" : "triangle"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0570(protein cleavage)']",
    "css" : {
      "target-arrow-shape" : "triangle"
    }
  }, {
    "selector" : "edge[edgeType = 'psi-mi:MI:0194(cleavage reaction)']",
    "css" : {
      "target-arrow-shape" : "triangle"
    }
  }, {
    "selector" : "edge[edgeType = 'ubiquitination']",
    "css" : {
      "target-arrow-shape" : "triangle"
    }
  }, {
    "selector" : "edge[edgeType = 'inhibits']",
    "css" : {
      "target-arrow-shape" : "tee"
    }
  }, {
    "selector" : "edge[edgeType = 'dephosphorylation']",
    "css" : {
      "target-arrow-shape" : "triangle"
    }
  }, {
    "selector" : "edge[edgeType = 'inhibits.process']",
    "css" : {
      "target-arrow-shape" : "tee"
    }
  }, {
    "selector" : "edge[edgeType = 'phosphorylation']",
    "css" : {
      "target-arrow-shape" : "triangle"
    }
  }, {
    "selector" : "edge:selected",
    "css" : {
      "line-color" : "rgb(255,0,0)"
    }
  } ]
}, {
  "format_version" : "1.0",
  "generated_by" : "cytoscape-3.1.0",
  "target_cytoscapejs_version" : "~2.1",
  "title" : "Nested Network Style",
  "style" : [ {
    "selector" : "node",
    "css" : {
      "text-valign" : "center",
      "text-halign" : "center",
      "font-size" : 13,
      "background-color" : "rgb(153,255,153)",
      "color" : "rgb(0,0,0)",
      "shape" : "ellipse",
      "height" : 40.0,
      "border-width" : 2.0,
      "border-opacity" : 0.7450980392156863,
      "width" : 40.0,
      "text-opacity" : 1.0,
      "font-family" : "SansSerif",
      "font-weight" : "normal",
      "border-color" : "rgb(0,153,0)",
      "background-opacity" : 0.5882352941176471,
      "content" : "data(name)"
    }
  }, {
    "selector" : "node[has_nested_network = 'true']",
    "css" : {
      "text-valign" : "bottom"
    }
  }, {
    "selector" : "node[has_nested_network = 'true']",
    "css" : {
      "border-color" : "rgb(0,102,204)"
    }
  }, {
    "selector" : "node[has_nested_network = 'true']",
    "css" : {
      "shape" : "rectangle"
    }
  }, {
    "selector" : "node[has_nested_network = 'true']",
    "css" : {
      "color" : "rgb(0,102,204)"
    }
  }, {
    "selector" : "node[has_nested_network = 'true']",
    "css" : {
      "background-color" : "rgb(255,255,255)"
    }
  }, {
    "selector" : "node[has_nested_network = 'true']",
    "css" : {
      "border-width" : 5.0
    }
  }, {
    "selector" : "node[has_nested_network = 'true']",
    "css" : {
      "width" : 60.0,
      "height" : 60.0
    }
  }, {
    "selector" : "node:selected",
    "css" : {
      "background-color" : "rgb(255,255,0)"
    }
  }, {
    "selector" : "edge",
    "css" : {
      "line-color" : "rgb(51,51,51)",
      "text-opacity" : 1.0,
      "source-arrow-shape" : "none",
      "opacity" : 1.0,
      "width" : 2.0,
      "content" : "",
      "source-arrow-color" : "rgb(0,0,0)",
      "font-family" : "Dialog",
      "font-weight" : "normal",
      "font-size" : 10,
      "color" : "rgb(0,0,0)",
      "line-style" : "solid",
      "target-arrow-color" : "rgb(0,0,0)",
      "target-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge:selected",
    "css" : {
      "line-color" : "rgb(255,0,0)"
    }
  } ]
}, {
  "format_version" : "1.0",
  "generated_by" : "cytoscape-3.1.0",
  "target_cytoscapejs_version" : "~2.1",
  "title" : "Minimal",
  "style" : [ {
    "selector" : "node",
    "css" : {
      "text-valign" : "center",
      "text-halign" : "center",
      "font-size" : 12,
      "background-color" : "rgb(255,255,255)",
      "color" : "rgb(0,0,0)",
      "shape" : "rectangle",
      "height" : 30.0,
      "border-width" : 0.0,
      "border-opacity" : 1.0,
      "width" : 30.0,
      "text-opacity" : 1.0,
      "font-family" : "Dialog",
      "font-weight" : "normal",
      "border-color" : "rgb(0,0,0)",
      "content" : "",
      "background-opacity" : 1.0
    }
  }, {
    "selector" : "node:selected",
    "css" : {
      "background-color" : "rgb(255,255,0)"
    }
  }, {
    "selector" : "edge",
    "css" : {
      "line-color" : "rgb(51,51,51)",
      "text-opacity" : 1.0,
      "source-arrow-shape" : "none",
      "opacity" : 1.0,
      "width" : 4.0,
      "content" : "",
      "source-arrow-color" : "rgb(0,0,0)",
      "font-family" : "Dialog",
      "font-weight" : "normal",
      "font-size" : 10,
      "color" : "rgb(0,0,0)",
      "line-style" : "solid",
      "target-arrow-color" : "rgb(0,0,0)",
      "target-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge:selected",
    "css" : {
      "line-color" : "rgb(255,0,0)"
    }
  } ]
}, {
  "format_version" : "1.0",
  "generated_by" : "cytoscape-3.1.0",
  "target_cytoscapejs_version" : "~2.1",
  "title" : "Solid",
  "style" : [ {
    "selector" : "node",
    "css" : {
      "text-valign" : "bottom",
      "text-halign" : "center",
      "font-size" : 18,
      "background-color" : "rgb(102,102,102)",
      "color" : "rgb(0,0,0)",
      "shape" : "ellipse",
      "height" : 40.0,
      "border-width" : 0.0,
      "border-opacity" : 1.0,
      "width" : 40.0,
      "text-opacity" : 0.6274509803921569,
      "font-family" : "Dialog",
      "font-weight" : "normal",
      "border-color" : "rgb(0,0,0)",
      "background-opacity" : 1.0,
      "content" : "data(name)"
    }
  }, {
    "selector" : "node:selected",
    "css" : {
      "background-color" : "rgb(255,255,0)"
    }
  }, {
    "selector" : "edge",
    "css" : {
      "line-color" : "rgb(204,204,204)",
      "text-opacity" : 0.7450980392156863,
      "source-arrow-shape" : "none",
      "opacity" : 1.0,
      "width" : 12.0,
      "source-arrow-color" : "rgb(0,0,0)",
      "font-family" : "Dialog",
      "font-weight" : "normal",
      "font-size" : 10,
      "color" : "rgb(102,102,102)",
      "line-style" : "solid",
      "target-arrow-color" : "rgb(0,0,0)",
      "target-arrow-shape" : "none",
      "content" : "data(interaction)"
    }
  }, {
    "selector" : "edge:selected",
    "css" : {
      "line-color" : "rgb(255,0,0)"
    }
  } ]
}, {
  "format_version" : "1.0",
  "generated_by" : "cytoscape-3.1.0",
  "target_cytoscapejs_version" : "~2.1",
  "title" : "Universe",
  "style" : [ {
    "selector" : "node",
    "css" : {
      "text-valign" : "center",
      "text-halign" : "center",
      "font-size" : 20,
      "background-color" : "rgb(0,0,0)",
      "color" : "rgb(255,255,204)",
      "shape" : "ellipse",
      "height" : 40.0,
      "border-width" : 0.0,
      "border-opacity" : 1.0,
      "width" : 40.0,
      "text-opacity" : 0.7058823529411765,
      "font-family" : "Monospaced",
      "font-weight" : "bold",
      "border-color" : "rgb(0,0,0)",
      "background-opacity" : 0.0,
      "content" : "data(name)"
    }
  }, {
    "selector" : "node:selected",
    "css" : {
      "background-color" : "rgb(255,255,0)"
    }
  }, {
    "selector" : "edge",
    "css" : {
      "line-color" : "rgb(255,255,255)",
      "text-opacity" : 1.0,
      "source-arrow-shape" : "none",
      "opacity" : 0.39215686274509803,
      "width" : 6.0,
      "content" : "",
      "source-arrow-color" : "rgb(0,0,0)",
      "font-family" : "Dialog",
      "font-weight" : "normal",
      "font-size" : 10,
      "color" : "rgb(0,0,0)",
      "line-style" : "dashed",
      "target-arrow-color" : "rgb(0,0,0)",
      "target-arrow-shape" : "none"
    }
  }, {
    "selector" : "edge:selected",
    "css" : {
      "line-color" : "rgb(255,0,0)"
    }
  } ]
}, {
  "format_version" : "1.0",
  "generated_by" : "cytoscape-3.1.0",
  "target_cytoscapejs_version" : "~2.1",
  "title" : "Sample1",
  "style" : [ {
    "selector" : "node",
    "css" : {
      "text-valign" : "center",
      "text-halign" : "center",
      "font-size" : 12,
      "background-color" : "rgb(204,204,255)",
      "color" : "rgb(0,0,0)",
      "shape" : "ellipse",
      "height" : 40.0,
      "border-width" : 0.0,
      "border-opacity" : 1.0,
      "width" : 40.0,
      "text-opacity" : 1.0,
      "font-family" : "Dialog",
      "font-weight" : "bold",
      "border-color" : "rgb(0,0,0)",
      "background-opacity" : 1.0,
      "content" : "data(name)"
    }
  }, {
    "selector" : "node:selected",
    "css" : {
      "background-color" : "rgb(255,255,0)"
    }
  }, {
    "selector" : "edge",
    "css" : {
      "line-color" : "rgb(0,0,0)",
      "text-opacity" : 1.0,
      "source-arrow-shape" : "none",
      "opacity" : 1.0,
      "width" : 1.0,
      "source-arrow-color" : "rgb(0,0,0)",
      "font-family" : "Dialog",
      "font-weight" : "normal",
      "font-size" : 10,
      "color" : "rgb(0,0,0)",
      "line-style" : "solid",
      "target-arrow-color" : "rgb(0,0,0)",
      "target-arrow-shape" : "none",
      "content" : "data(interaction)"
    }
  }, {
    "selector" : "edge[interaction = 'pd']",
    "css" : {
      "line-color" : "rgb(255,0,51)"
    }
  }, {
    "selector" : "edge[interaction = 'pp']",
    "css" : {
      "line-color" : "rgb(0,204,0)"
    }
  }, {
    "selector" : "edge[interaction = 'pd']",
    "css" : {
      "line-style" : "dashed"
    }
  }, {
    "selector" : "edge[interaction = 'pp']",
    "css" : {
      "line-style" : "solid"
    }
  }, {
    "selector" : "edge:selected",
    "css" : {
      "line-color" : "rgb(255,0,0)"
    }
  } ]
}, {
  "format_version" : "1.0",
  "generated_by" : "cytoscape-3.1.0",
  "target_cytoscapejs_version" : "~2.1",
  "title" : "default",
  "style" : [ {
    "selector" : "node",
    "css" : {
      "text-valign" : "center",
      "text-halign" : "center",
      "font-size" : 60,
      "background-color" : "rgb(255,255,255)",
      "color" : "rgb(0,0,0)",
      "shape" : "ellipse",
      "height" : 200.0,
      "border-width" : 40.0,
      "border-opacity" : 1.0,
      "width" : 200.0,
      "text-opacity" : 1.0,
      "font-family" : "SansSerif",
      "font-weight" : "bold",
      "border-color" : "rgb(0,0,0)",
      "content" : "",
      "background-opacity" : 1.0
    }
  }, {
    "selector" : "node:selected",
    "css" : {
      "background-color" : "rgb(255,255,0)"
    }
  }, {
    "selector" : "edge",
    "css" : {
      "line-color" : "rgb(255,255,255)",
      "text-opacity" : 1.0,
      "source-arrow-shape" : "triangle",
      "opacity" : 1.0,
      "width" : 1.0,
      "content" : "",
      "source-arrow-color" : "rgb(0,0,0)",
      "font-family" : "Dialog",
      "font-weight" : "normal",
      "font-size" : 10,
      "color" : "rgb(0,0,0)",
      "line-style" : "solid",
      "target-arrow-color" : "rgb(0,0,0)",
      "target-arrow-shape" : "triangle"
    }
  }, {
    "selector" : "edge:selected",
    "css" : {
      "line-color" : "rgb(255,0,0)"
    }
  } ]
} ]

</script>
<style>

div {
   display: block;
   }

#cyGbmPathways {
  background-color: #DDD;
  height: 400px;
  width: 600px;
  position: relative;
  margin-top: 0px;
  margin-left: 0px;
  border: 1px solid #aaa;
  padding: 0px;
  display:block;
  margin-bottom: 0px;
  }

#gbmPathwaysMouseOverReadoutDiv{
  border: 1px solid #aaa;
  width: 800px;
  height: 20px;
  }

#gbmPathwaysButtonDiv{
  margin-top: 1px;
  margin-bottom: 1px;
  }

.fubar{
   font-size:8pt !important;
}
</style>

<div id="gbmPathwaysDiv">
   <div id="gbmPathwaysMouseOverReadoutDiv"></div>
   <div id="gbmPathwaysButtonDiv">
      <label for="select">Select a sample:</label>
      <select id="sampleSelector"></select>
      <button id="cwGBMMovieButton" class="fubar">Movie</button>
      <!-- button id="clearSamplesButton">Clear Sample Selector</button -->
      <button id="cwGBMZoomSelectedButton" class="fubar">Zoom Selected</button>
      <button id="cwGBMViewAbstractsButton" class="fubar">Enable Abstracts</button>
      <input type="text" id="gbmPathwaysSearchBox"></input>
   </div>
   <div id="cyGbmPathways"></div>
</div>





<style>
.node circle {
  cursor: pointer;
  fill: #fff;
  stroke: steelblue;
  stroke-width: 1.5px;
}

.node text {
  font-size: 11px;
}

path.link {
  fill: none;
  stroke: #ccc;
  stroke-width: 1.5px;
}
</style>


  <div id="SavedSelectionDiv">
     <div id="SavedSelectionTreeDiv"></div>
  <div id="SelectionBanner">
    <div id="Acknowledgement">Created by Lisa McFerrin</div>
    <script language="Javascript">
		document.write("This page was last modified on: " + document.lastModified +"");
	</script>
  </div>

  </div>
  
 

<style>

.domain { 
  fill: none; 
  } 


.extent {
  fill-opacity: .1;
  stroke: #f00;
}


.axis path, .axis line {
    stroke: black;
    stroke-width: 1px;
    }

#pairedDistributionsDiv {
   margin: 0px;
   padding: 0px;
   }

#pairedDistributionsDisplay {
   border: 1px solid #aaa;
   width: 50px;
   height: 50px;
   background-color: #E8E8E8;
   margin-top: 1px;
   margin-left: 3px;
   margin-right: 0;
   margin-bottom: 1px;
   padding: 1px;
   padding-bottom: 0px;
   }

</style>

<div id="pairedDistributionsDiv">
   <button id="generateRandomPairedDistributionsDataButton">Generate Random Paired Distributions</button>
   <div id="pairedDistributionsDisplay"></div>
</div>

<div id="todoDiv">
<h3> Tasks and Schedules </h3>
   <button id=ToDoLink>Edit The Table</button>
   <iframe
      height="600px",
      width="95%",
      src="https://docs.google.com/spreadsheets/d/1Rqqpma1M8aF5bX4BM2cYYgDd5hazUysqzCxLhGUwldo/pubhtml;headers=false">  
   </iframe>
</div>



</div>


</body>
</html>
